<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>WorkflowFinder</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.17/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-buttons-1.10.20/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="site_libs/dt-ext-buttons-1.10.20/js/dataTables.buttons.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.flash.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.html5.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.colVis.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.print.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>

<!DOCTYPE html>
<link rel="shortcut icon" href="./images/Australian-Biocommons-Favicon.png" />

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-T8GWK081T9"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-T8GWK081T9');
</script>
<!--/html_preserve-->

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">About</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="2_tools.html">ToolFinder</a>
</li>
<li>
  <a href="2_1_workflows.html">WorkflowFinder</a>
</li>
<li>
  <a href="5_attributions.html">Attributions</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">WorkflowFinder</h1>

</div>


<style type="text/css">
.main-container {
  max-width: 1500px;
  margin-left: auto;
  margin-right: auto;
}
</style>
<p>The interactive table below allows you to search for workflows registered by Australian BioCommons partners on <a href="https://workflowhub.eu/programmes/8">workflowhub</a>.</p>
<hr />
<hr />
<p><font size="2"></p>
<div id="htmlwidget-4d430e408fbf00f4ef8c" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-4d430e408fbf00f4ef8c">{"x":{"filter":"none","class":"cell-border stripe","extensions":["Buttons"],"data":[["<a href='https://workflowhub.eu/workflows/142' target='_blank'  rel='noopener noreferrer'>16S_biodiversity_BIOM<\/a>","<a href='https://workflowhub.eu/workflows/233' target='_blank'  rel='noopener noreferrer'>16S_biodiversity_for_nonoverlap_paired_end<\/a>","<a href='https://workflowhub.eu/workflows/232' target='_blank'  rel='noopener noreferrer'>16S_biodiversity_for_overlap_paired_end<\/a>","<a href='https://workflowhub.eu/workflows/226' target='_blank'  rel='noopener noreferrer'>Assembly polishing<\/a>","<a href='https://workflowhub.eu/workflows/225' target='_blank'  rel='noopener noreferrer'>Assembly with Flye<\/a>","<a href='https://workflowhub.eu/workflows/229' target='_blank'  rel='noopener noreferrer'>Assess genome quality<\/a>","<a href='https://workflowhub.eu/workflows/153' target='_blank'  rel='noopener noreferrer'>Bootstrapping-for-BQSR @ NCI-Gadi<\/a>","<a href='https://workflowhub.eu/workflows/220' target='_blank'  rel='noopener noreferrer'>CCS.BAM to FASTQ + QC (HiFi genome assembly stage 1)<\/a>","<a href='https://workflowhub.eu/workflows/230' target='_blank'  rel='noopener noreferrer'>Combined workflows for large genome assembly<\/a>","<a href='https://workflowhub.eu/workflows/222' target='_blank'  rel='noopener noreferrer'>Data QC<\/a>","<a href='https://workflowhub.eu/workflows/146' target='_blank'  rel='noopener noreferrer'>Fastq-to-bam @ NCI-Gadi<\/a>","<a href='https://workflowhub.eu/workflows/150' target='_blank'  rel='noopener noreferrer'>Flashlite-Juicer<\/a>","<a href='https://workflowhub.eu/workflows/151' target='_blank'  rel='noopener noreferrer'>Flashlite-Supernova<\/a>","<a href='https://workflowhub.eu/workflows/149' target='_blank'  rel='noopener noreferrer'>Flashlite-Trinity<\/a>","<a href='https://workflowhub.eu/workflows/147' target='_blank'  rel='noopener noreferrer'>GATK4 Fastq to joint-called cohort VCF with Cromwell on local cluster (no job scheduler)<\/a>","<a href='https://workflowhub.eu/workflows/144' target='_blank'  rel='noopener noreferrer'>GATK4 Fastq to joint-called cohort VCF with Cromwell on SLURM<\/a>","<a href='https://workflowhub.eu/workflows/143' target='_blank'  rel='noopener noreferrer'>Germline-ShortV @ NCI-Gadi<\/a>","<a href='https://workflowhub.eu/workflows/236' target='_blank'  rel='noopener noreferrer'>Janis Germline Variant-Calling Workflow (GATK)<\/a>","<a href='https://workflowhub.eu/workflows/223' target='_blank'  rel='noopener noreferrer'>kmer counting - meryl<\/a>","<a href='https://workflowhub.eu/workflows/140' target='_blank'  rel='noopener noreferrer'>microPIPE: a pipeline for high-quality bacterial genome construction using ONT and Illumina sequencing<\/a>","<a href='https://workflowhub.eu/workflows/221' target='_blank'  rel='noopener noreferrer'>PacBio HiFi genome assembly using hifiasm (HiFi genome assembly stage 2)<\/a>","<a href='https://workflowhub.eu/workflows/237' target='_blank'  rel='noopener noreferrer'>Purge duplicates from hifiasm assembly v1.0 (HiFi genome assembly stage 3)<\/a>","<a href='https://workflowhub.eu/workflows/228' target='_blank'  rel='noopener noreferrer'>Racon polish with Illumina reads, x2<\/a>","<a href='https://workflowhub.eu/workflows/227' target='_blank'  rel='noopener noreferrer'>Racon polish with long reads, x4<\/a>","<a href='https://workflowhub.eu/workflows/152' target='_blank'  rel='noopener noreferrer'>RNASeq-DE @ NCI-Gadi<\/a>","<a href='https://workflowhub.eu/workflows/148' target='_blank'  rel='noopener noreferrer'>Somatic-ShortV @ NCI-Gadi<\/a>","<a href='https://workflowhub.eu/workflows/224' target='_blank'  rel='noopener noreferrer'>Trim and filter reads - fastp<\/a>","<a href='https://workflowhub.eu/workflows/145' target='_blank'  rel='noopener noreferrer'>Trinity @ NCI-Gadi<\/a>","<a href='https://workflowhub.eu/workflows/114' target='_blank'  rel='noopener noreferrer'>Trinity RNA Assembly<\/a>"],["MetaDEGalaxy","MetaDEGalaxy","MetaDEGalaxy",null,null,null,"Alignment, BQSR, Bootstrapping, DNA, GATK4, Genomics, INDELs, NCI, NCI-Gadi, PBS, SNPs, WGS, illumina, model, non-model, scalable, variant calling","BAM, Conversion, FASTQ, QC",null,null,"Alignment, BROAD, BWA-mem, DNA, Gadi, Genomics, NCI, PBS, WGS, genome, mapping, scalable","BWA, FASTQ, Flashlite, Hi-C, Juicer, PBS, TAD, map, scalable, topologically associating domains","10X, Flashlite, Supernova, TELLSeq","Assembly, PBS, Transcriptomics, container, de novo, global assemblies, illumina, rna, rna-seq, salmon, scalable, singularity, strandedness, transcriptome, trinity","Alignment, GATK4, Genomics, INDELs, SNPs, variant_calling, workflow","Alignment, GATK4, Genomics, INDELs, SNPs, variant_calling","BROAD, DNA, DNA-seq, GATK4, Genomics, Germline, HaplotyeCaller, INDELs, SNPs, WGS, genome, variant_calling",null,null,"Assembly, Nextflow, ONT, bacterial-genomics, workflow","FASTQ, HiFi, genome_assembly, hifiasm","Assembly, HiFi, purge_dups",null,null,"BBduk, DE, FASTQ, FastQC, Gadi, HTSeq, MultiQC, NCI, NCI-Gadi, PBS, RNASEQ, RSeQC, SAMTools, STAR, bash, counts, differential expression, differential_expression, expression, illumina, parallel, rna, rna-seq, scalable, workflow","GATK4, Gadi, INDELs, Mutect2, NCI, NCI Gadi, NCI-Gadi, SNPs, Somatic, VCF, cancer, scalable, tumour, variant_calling",null,"Assembly, Gadi, NCI, PBS, RNASEQ, Transcriptomics, rna, rna-seq, scalable, trinity","Assembly, Nextflow, RNASEQ, Transcriptomics"],["<p>This is a Galaxy workflow that uses to convert the16S BIOM file to table and figures. It is part of the metaDEGalaxy workflow MetaDEGalaxy: Galaxy workflow for differential abundance analysis of 16s metagenomic data. <\/p> ...","<p>A description is not available.<\/p> ...","<p>MetaDEGalaxy: Galaxy workflow for differential abundance analysis of 16s metagenomic data<\/p> ...","<p>Assembly polishing; can run alone or as part of a combined workflow for large genome assembly. <br /><br />* What it does: Polishes (corrects) an assembly, using long reads (with the tools Racon and Medaka) and short reads (with the tool Racon). (Note: medaka is only for nanopore reads, not PacBi ...","<p>Assembly with Flye; can run alone or as part of a combined workflow for large genome assembly. <br /><br />* What it does: Assembles long reads with the tool Flye<br />* Inputs: long reads (may be raw, or filtered, and/or corrected); fastq.gz format<br />* Outputs: Flye assembly fasta; Fasta stat ...","<p>Assess genome quality; can run alone or as part of a combined workflow for large genome assembly. <br /><br />* What it does: Assesses the quality of the genome assembly: generate some statistics and determine if expected genes are present; align contigs to a reference genome.<br />* Inputs: poli ...","<p>Bootstrapping-for-BQSR @ NCI-Gadi is a pipeline for bootstrapping a variant resource to enable GATK base quality score recalibration (BQSR) for non-model organisms that lack a publicly available variant resource. This implementation is optimised for the National Compute Infrastucture&#39;s Gadi H ...","<p>Conversion of files from BAM to FASTQ, including FASTQC &amp; CCS.BAM file quality control (QC) steps.<br /><br />Infrastructure_deployment_metadata: Galaxy Australia (Galaxy)<\/p> ...","<p>Combined workflow for large genome assembly<br /><br />The tutorial document for this workflow is here: <a href=\"https://doi.org/10.5281/zenodo.5655813\">https://doi.org/10.5281/zenodo.5655813<\/a><br /><br /><br />What it does:  A workflow for genome assembly, containing subworkflows:<br />* Data  ...","<p>Data QC step, can run alone or as part of a combined workflow for large genome assembly. <br /><br />* What it does: Reports statistics from sequencing reads.<br />* Inputs: long reads (fastq.gz format), short reads (R1 and R2) (fastq.gz format).<br />* Outputs: For long reads: a nanoplot report  ...","<p>Fastq-to-BAM @ NCI-Gadi is a genome alignment workflow that takes raw FASTQ files, aligns them to a reference genome and outputs analysis ready BAM files. This workflow is designed for the National Computational Infrastructure&#39;s (NCI) Gadi supercompter, leveraging multiple nodes on NCI Gadi t ...","<p>Flashlite-Juicer is a PBS implementation of <a href=\"https://github.com/aidenlab/juicer\">Juicer<\/a> for University of Queensland&#39;s Flashlite HPC.<br /><br />Infrastructure_deployment_metadata: FlashLite (QRISCloud)<\/p> ...","<p>The Flashlite-Supernova pipeline runs Supernova to generate phased whole-genome de novo assemblies from a Chromium prepared library on <a href=\"https://rcc.uq.edu.au/flashlite\">University of Queensland&#39;s HPC, Flashlite<\/a>. <br /><br />Infrastructure_deployment_metadata: FlashLite (QRISCloud) ...","<p>Flashlite-Trinity contains two workflows that run Trinity on the <a href=\"https://rcc.uq.edu.au/flashlite\">University of Queensland&#39;s HPC, Flashlite<\/a>.  Trinity performs de novo transcriptome assembly of RNA-seq data by combining three independent software modules Inchworm, Chrysalis and Bu ...","Local Cromwell implementation of GATK4 germline variant calling pipeline<br />See the <a href=\"https://gatk.broadinstitute.org/hc/en-us\">GATK<\/a> website for more information on this toolset <br />## Assumptions<br />- Using hg38 human reference genome build<br />- Running &#39;locally&#39; i.e. ...","SLURM HPC Cromwell implementation of GATK4 germline variant calling pipeline<br />See the <a href=\"https://gatk.broadinstitute.org/hc/en-us\">GATK<\/a> website for more information on this toolset <br />## Assumptions<br />- Using hg38 human reference genome build<br />- Running using HPC/SLURM sc ...","<p>Germline-ShortV @ NCI-Gadi is an implementation of the BROAD Institute&#39;s best practice workflow for germline short variant discovery. This implementation is optimised for the National Compute Infrastucture&#39;s Gadi HPC, utilising scatter-gather parallelism to enable use of multiple nodes wi ...","<p>This is a genomics pipeline to do a single germline sample variant-calling, adapted from GATK Best Practice Workflow.<br /><br />This workflow is a reference pipeline for using the Janis Python framework (pipelines assistant).<br />- Alignment: bwa-mem<br />- Variant-Calling: GATK HaplotypeCaller ...","<p>Kmer counting step, can run alone or as part of a combined workflow for large genome assembly. <br /><br />* What it does: Estimates genome size and heterozygosity based on counts of kmers<br />* Inputs: One set of short reads: e.g. R1.fq.gz<br />* Outputs: GenomeScope graphs<br />* Tools used: M ...","<p>microPIPE was developed to automate high-quality complete bacterial genome assembly using Oxford Nanopore Sequencing in combination with Illumina sequencing.<br /><br />To build microPIPE we evaluated the performance of several tools at each step of bacterial genome assembly, including basecallin ...","<p>Assembly, visualisation and quality control workflow for high fidelity reads built from circular consensus sequence (PacBio HiFi) data.<br /><br />Infrastructure_deployment_metadata: Galaxy Australia (Galaxy)<\/p> ...","<p>Optional workflow to purge duplicates from the contig assembly.<br /><br />Infrastructure_deployment_metadata: Galaxy Australia (Galaxy)<\/p> ...","<p>Assembly polishing subworkflow: Racon polishing with short reads<br /><br />Inputs: short reads and assembly (usually pre-polished with other tools first, e.g. Racon + long reads; Medaka)<br /><br />Workflow steps: <br />* minimap2: short reads (R1 only) are mapped to the assembly =&gt; overlaps. ...","<p>Assembly polishing subworkflow: Racon polishing with long reads<br /><br />Inputs: long reads and assembly contigs<br /><br />Workflow steps:<br />* minimap2 : long reads are mapped to assembly =&gt; overlaps.paf. <br />* overaps, long reads, assembly =&gt; Racon =&gt; polished assembly 1<br />*  ...","<p>RNASeq-DE @ NCI-Gadi processes RNA sequencing data (single, paired and/or multiplexed) for differential expression (raw FASTQ to counts). This pipeline consists of multiple stages and is designed for the National Computational Infrastructure&#39;s (NCI) Gadi supercompter, leveraging multiple node ...","<p>Somatic-ShortV @ NCI-Gadi is a variant calling pipeline that calls somatic short variants (SNPs and indels) from tumour and matched normal BAM files following <a href=\"https://gatk.broadinstitute.org/hc/en-us/articles/360035894731-Somatic-short-variant-discovery-SNVs-Indels-\">GATK&#39;s Best Prac ...","<p>Trim and filter reads; can run alone or as part of a combined workflow for large genome assembly. <br /><br />* What it does: Trims and filters raw sequence reads according to specified settings. <br />* Inputs: Long reads (format fastq); Short reads R1 and R2 (format fastq) <br />* Outputs: Trim ...","<p>Description: Trinity @ NCI-Gadi contains a staged <a href=\"https://github.com/trinityrnaseq/trinityrnaseq/wiki\">Trinity<\/a> workflow that can be run on the National Computational Infrastructure’s (NCI) Gadi supercomputer. Trinity performs de novo transcriptome assembly of RNA-seq data by combinin ...","<p>A porting of the Trinity RNA assembly pipeline, <a href=\"https://trinityrnaseq.github.io\">https://trinityrnaseq.github.io<\/a>, that uses Nextflow to handle the underlying sub-tasks.<br />This enables additional capabilities to better use HPC resources, such as packing of tasks to fill up nodes an ..."],["GPL-3.0","GPL-3.0","GPL-3.0","Apache-2.0","Apache-2.0","Apache-2.0","GPL-3.0","GPL-3.0","Apache-2.0","Apache-2.0","GPL-3.0","GPL-3.0","GPL-3.0","GPL-3.0","Apache-2.0","Apache-2.0","GPL-3.0","Apache-2.0","Apache-2.0","GPL-3.0","GPL-3.0","GPL-3.0","Apache-2.0","Apache-2.0","GPL-3.0","GPL-3.0","Apache-2.0","Apache-2.0","GPL-3.0"],["Galaxy","Galaxy","Galaxy","Galaxy","Galaxy","Galaxy","Shell Script","Galaxy","Galaxy","Galaxy","Shell Script","Shell Script","Shell Script","Shell Script","Unrecognized workflow type","Unrecognized workflow type","Shell Script","Janis","Galaxy","Nextflow","Galaxy","Galaxy","Galaxy","Galaxy","Shell Script","Shell Script","Galaxy","Shell Script","Nextflow"],["Galaxy Australia","QCIF Bioinformatics","QCIF Bioinformatics","Galaxy Australia<br />Australian BioCommons","Galaxy Australia<br />Australian BioCommons","Galaxy Australia<br />Australian BioCommons","Sydney Informatics Hub","Australian BioCommons<br />Galaxy Australia","Galaxy Australia<br />Australian BioCommons","Galaxy Australia<br />Australian BioCommons","Australian BioCommons<br />Sydney Informatics Hub","Sydney Informatics Hub","Australian BioCommons<br />Sydney Informatics Hub","Australian BioCommons<br />Sydney Informatics Hub","Australian BioCommons<br />Pawsey Supercomputing Centre","Australian BioCommons<br />Pawsey Supercomputing Centre","Australian BioCommons<br />Sydney Informatics Hub","Janis","Galaxy Australia<br />Australian BioCommons","QCIF Bioinformatics","Galaxy Australia<br />Australian BioCommons","Galaxy Australia<br />Australian BioCommons","Galaxy Australia<br />Australian BioCommons","Galaxy Australia<br />Australian BioCommons","Sydney Informatics Hub","Sydney Informatics Hub","Galaxy Australia<br />Australian BioCommons","Australian BioCommons<br />Sydney Informatics Hub","Australian BioCommons"],[null,null,null,"Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Gadi (NCI)","Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Gadi (NCI)","FlashLite (QRISCloud)","FlashLite (QRISCloud)","FlashLite (QRISCloud)",null,null,"Gadi (NCI)","Spartan (Unimelb)","Galaxy Australia (Galaxy)","Zeus (Pawsey)","Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Galaxy Australia (Galaxy)","Gadi (NCI)","Gadi (NCI)","Galaxy Australia (Galaxy)","Gadi (NCI)",null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th class=\"dt-center\" style=\"font-size:11pt\" colspan=\"6\">Workflow metadata<\/th>\n      <th class=\"dt-center\" style=\"font-size:11pt\" colspan=\"1\">Deployment<\/th>\n    <\/tr>\n    <tr>\n      <th>Title<\/th>\n      <th>Tag(s)<\/th>\n      <th>Description<\/th>\n      <th>License<\/th>\n      <th>Workflow Class<\/th>\n      <th>Contributing team(s)<\/th>\n      <th>Successful infrastructure deployment(s)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"headerCallback":"function( thead, data, start, end, display ) {\n  $(thead).closest('thead').find('th').eq(0).css('background-color', '#205a86').css('color', '#ffffff');\n  $(thead).closest('thead').find('th').eq(1).css('background-color', '#2cb77c').css('color', '#ffffff');\n}","pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","colvis"],"columnDefs":[{"className":"dt-center","targets":[0,3,4,5,6]},{"width":"150px","targets":1}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.headerCallback"],"jsHooks":[]}</script>
<p></font></p>
<p><strong>Please note:</strong></p>
<ul>
<li>This table was last updated on <code>2022-01-25</code>.</li>
</ul>
<hr />
<hr />


<!DOCTYPE html>

<div class="footer">This site uses Cookies. We use cookies to help the website run effectively. To find out more, read our <a href="https://www.biocommons.org.au/privacy">Privacy</a> and <a href="https://www.biocommons.org.au/cookies">Cookies</a> Statements. </div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
