<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>ToolFinder</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.17/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-buttons-1.10.20/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="site_libs/dt-ext-buttons-1.10.20/js/dataTables.buttons.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.flash.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.html5.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.colVis.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.10.20/js/buttons.print.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<!DOCTYPE html>
<link rel="shortcut icon" href="./images/Australian-Biocommons-Favicon.png" />

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-T8GWK081T9"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-T8GWK081T9');
</script>
<!--/html_preserve-->

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">About</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="2_tools.html">ToolFinder</a>
</li>
<li>
  <a href="2_1_workflows.html">WorkflowFinder</a>
</li>
<li>
  <a href="3_biocommons_activities.html">Optimised tools &amp; workflows</a>
</li>
<li>
  <a href="5_attributions.html">Attributions</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">ToolFinder</h1>

</div>


<style type="text/css">
.main-container {
  max-width: 1500px;
  margin-left: auto;
  margin-right: auto;
}
</style>
<p>This page lists bioinformatics tools installed across several of the <a href="http://support.biocommons.org.au/support/solutions/articles/6000251977-compute-systems-at-the-biocommons-partner-infrastructures">BioCommons infrastructure partner systems</a> and tool installation request mechanisms on these systems. Tool status on other partner infrastructures will be added over time.</p>
<hr />
<hr />
<p><a name="tool_information"></a></p>
<p><font size="2"></p>
<div id="htmlwidget-6104b0ea7151e0ec2c98" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-6104b0ea7151e0ec2c98">{"x":{"filter":"none","class":"cell-border stripe","extensions":["Buttons"],"data":[["<a href='https://github.com/aidenlab/3d-dna' target='_blank'  rel='noopener noreferrer'>3D-DNA<\/a>","<a href='https://github.com/tseemann/abricate' target='_blank'  rel='noopener noreferrer'>ABRicate<\/a>","<a href='https://github.com/bcgsc/abyss/' target='_blank'  rel='noopener noreferrer'>ABySS<\/a>","<a href='https://github.com/tanghaibao/allhic' target='_blank'  rel='noopener noreferrer'>allhic<\/a>","<a href='https://github.com/tanghaibao/jcvi/wiki/ALLMAPS' target='_blank'  rel='noopener noreferrer'>ALLMAPS<\/a>","<a href='http://software.broadinstitute.org/allpaths-lg/blog/' target='_blank'  rel='noopener noreferrer'>allpathslg<\/a>","<a href='http://amos.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>amos<\/a>","<a href='https://msberends.github.io/AMR/' target='_blank'  rel='noopener noreferrer'>AMR++<\/a>","<a href='http://bioinformatics.ua.pt/software/anaconda/' target='_blank'  rel='noopener noreferrer'>anaconda<\/a>","<a href='http://www.popgen.dk/angsd' target='_blank'  rel='noopener noreferrer'>ANGSD<\/a>","<a href='https://merenlab.org/software/anvio/' target='_blank'  rel='noopener noreferrer'>Anvio<\/a>","<a href='http://apollo.berkeleybop.org' target='_blank'  rel='noopener noreferrer'>Apollo<\/a>","<a href='http://www.sanger.ac.uk/science/tools/artemis' target='_blank'  rel='noopener noreferrer'>Artemis<\/a>","<a href='http://bioinf.uni-greifswald.de/augustus' target='_blank'  rel='noopener noreferrer'>Augustus<\/a>","<a href='https://github.com/hall-lab/bamkit' target='_blank'  rel='noopener noreferrer'>bamkit<\/a>","<a href='https://bammmotif.mpibpc.mpg.de' target='_blank'  rel='noopener noreferrer'>bamm<\/a>","<a href='https://github.com/pezmaster31/bamtools' target='_blank'  rel='noopener noreferrer'>bamtools<\/a>","<a href='https://jgi.doe.gov/data-and-tools/bbtools/' target='_blank'  rel='noopener noreferrer'>bbmap<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>BCFtools<\/a>","<a href='https://faculty.washington.edu/browning/beagle/beagle.html' target='_blank'  rel='noopener noreferrer'>Beagle<\/a>","<a href='https://github.com/beagle-dev/beagle-lib' target='_blank'  rel='noopener noreferrer'>beagle-lib<\/a>","<a href='http://beast.bio.ed.ac.uk/' target='_blank'  rel='noopener noreferrer'>BEAST<\/a>","<a href='http://www.beast2.org' target='_blank'  rel='noopener noreferrer'>BEAST2<\/a>","<a href='https://github.com/arq5x/bedtools2' target='_blank'  rel='noopener noreferrer'>BEDTools<\/a>","<a href='https://github.com/davebx/bellerophon' target='_blank'  rel='noopener noreferrer'>bellerophon<\/a>","<a href='https://github.com/edgraham/BinSanity' target='_blank'  rel='noopener noreferrer'>BinSanity<\/a>","<a href='https://www.yumpu.com/en/document/view/3851346/bio-roll-users-guide-rocks-clusters' target='_blank'  rel='noopener noreferrer'>bio-roll<\/a>","<a href='http://www.bioconductor.org' target='_blank'  rel='noopener noreferrer'>bioconductor<\/a>","<a href='https://github.com/Bio-Linux' target='_blank'  rel='noopener noreferrer'>biolinux<\/a>","<a href='https://bionanogenomics.com/support-page/bionano-access-software/' target='_blank'  rel='noopener noreferrer'>Bionano Access<\/a>","<a href='http://bioperl.org/' target='_blank'  rel='noopener noreferrer'>bioperl<\/a>","<a href='http://biopython.org/' target='_blank'  rel='noopener noreferrer'>biopython<\/a>","<a href='https://github.com/FelixKrueger/Bismark' target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST+<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST2<\/a>","<a href='https://www.kegg.jp/blastkoala/' target='_blank'  rel='noopener noreferrer'>BlastKOALA<\/a>","<a href='http://genome.ucsc.edu/cgi-bin/hgBlat?command=start' target='_blank'  rel='noopener noreferrer'>blat<\/a>","<a href='https://blobtoolkit.genomehubs.org/view' target='_blank'  rel='noopener noreferrer'>BlobToolKit<\/a>","<a href='https://github.com/DRL/blobtools' target='_blank'  rel='noopener noreferrer'>BlobTools1<\/a>","<a href='https://research.pasteur.fr/en/software/bmge-block-mapping-and-gathering-with-entropy/' target='_blank'  rel='noopener noreferrer'>BMGE<\/a>","<a href='https://www.boost.org/' target='_blank'  rel='noopener noreferrer'>Boost<\/a>","<a href='http://bowtie-bio.sourceforge.net/index.shtml' target='_blank'  rel='noopener noreferrer'>Bowtie<\/a>","<a href='http://bowtie-bio.sourceforge.net/bowtie2/index.shtml' target='_blank'  rel='noopener noreferrer'>Bowtie2<\/a>","<a href='http://bioinf.uni-greifswald.de/bioinf/braker/index.html' target='_blank'  rel='noopener noreferrer'>BRAKER<\/a>","<a href='http://bioinf.uni-greifswald.de/bioinf/braker/index.html' target='_blank'  rel='noopener noreferrer'>BRAKER2<\/a>","<a href='http://sourceforge.net/projects/breakdancer/' target='_blank'  rel='noopener noreferrer'>breakdancer<\/a>","<a href='https://code.google.com/archive/p/bsmap/' target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","<a href='http://pellegrini.mcdb.ucla.edu/BS_Seeker/BS_Seeker.html' target='_blank'  rel='noopener noreferrer'>bsseeker<\/a>","<a href='http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2/' target='_blank'  rel='noopener noreferrer'>bsseeker2<\/a>","<a href='http://busco.ezlab.org/' target='_blank'  rel='noopener noreferrer'>BUSCO<\/a>","<a href='http://bio-bwa.sourceforge.net' target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://github.com/bxlab/bx-python' target='_blank'  rel='noopener noreferrer'>bx-python<\/a>","<a href='https://github.com/ComparativeGenomicsToolkit/cactus' target='_blank'  rel='noopener noreferrer'>Cactus<\/a>","<a href='http://cgenome.net/wiki/index.php/Calypso' target='_blank'  rel='noopener noreferrer'>Calypso<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/CAMERA.html' target='_blank'  rel='noopener noreferrer'>CAMERA<\/a>","<a href='https://github.com/marbl/canu' target='_blank'  rel='noopener noreferrer'>Canu<\/a>","<a href='http://biosrv.cab.unina.it/webcap3/' target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/Cardinal.html' target='_blank'  rel='noopener noreferrer'>Cardinal<\/a>","<a href='https://github.com/dutilh/CAT' target='_blank'  rel='noopener noreferrer'>CAT<\/a>","<a href='https://github.com/weizhongli/cdhit' target='_blank'  rel='noopener noreferrer'>cd-hit<\/a>","<a href='https://github.com/yanailab/celseq2' target='_blank'  rel='noopener noreferrer'>CEL-Seq 2<\/a>","<a href='http://wgs-assembler.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>Celera / wgs-assembler<\/a>","<a href='https://cellprofiler.org/' target='_blank'  rel='noopener noreferrer'>CellProfiler<\/a>","<a href='https://ccb.jhu.edu/software/centrifuge/' target='_blank'  rel='noopener noreferrer'>Centrifuge<\/a>","<a href='https://ecogenomics.github.io/CheckM' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://ecogenomics.github.io/CheckM' target='_blank'  rel='noopener noreferrer'>checkm-genome<\/a>","<a href='http://bitbucket.org/berkeleylab/CheckV' target='_blank'  rel='noopener noreferrer'>CheckV<\/a>","<a href='http://microbiomeutil.sourceforge.net/#A_CS' target='_blank'  rel='noopener noreferrer'>ChimeraSlayer<\/a>","<a href='https://github.com/estebanpw/chromeister' target='_blank'  rel='noopener noreferrer'>chromeister<\/a>","<a href='https://genome.sph.umich.edu/wiki/ChunkChromosome' target='_blank'  rel='noopener noreferrer'>chunkchromosome<\/a>","<a href='https://github.com/YangLab/CIRCexplorer' target='_blank'  rel='noopener noreferrer'>circexplorer<\/a>","<a href='https://sanger-pathogens.github.io/circlator/' target='_blank'  rel='noopener noreferrer'>circlator<\/a>","<a href='http://www.clustal.org/download/1.X/ftp-igbmc.u-strasbg.fr/pub/' target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","<a href='https://github.com/bsc-performance-tools/clustering-suite' target='_blank'  rel='noopener noreferrer'>clusteringsuite<\/a>","<a href='https://github.com/etal/cnvkit' target='_blank'  rel='noopener noreferrer'>CNVkit<\/a>","<a href='http://cpc.cbi.pku.edu.cn/' target='_blank'  rel='noopener noreferrer'>Coding Potential Calculator<\/a>","<a href='http://cpc2.cbi.pku.edu.cn/' target='_blank'  rel='noopener noreferrer'>Coding Potential Calculator 2<\/a>","<a href='https://github.com/dparks1134/CompareM' target='_blank'  rel='noopener noreferrer'>CompareM<\/a>","<a href='https://github.com/wwood/CoverM' target='_blank'  rel='noopener noreferrer'>CoverM<\/a>","<a href='http://lilab.research.bcm.edu/cpat/' target='_blank'  rel='noopener noreferrer'>CPAT: Coding Potential Assessment Tool<\/a>","<a href='https://github.com/NordicESMhub/ctsm/tree/fates_emerald_api' target='_blank'  rel='noopener noreferrer'>ctsm_fates<\/a>","<a href='https://developer.nvidia.com/cuda-toolkit' target='_blank'  rel='noopener noreferrer'>cuda<\/a>","<a href='https://developer.nvidia.com/cudnn' target='_blank'  rel='noopener noreferrer'>cudnn<\/a>","<a href='http://cole-trapnell-lab.github.io/cufflinks' target='_blank'  rel='noopener noreferrer'>Cufflinks<\/a>","<a href='https://pypi.python.org/pypi/cutadapt' target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='http://www.cytoscape.org/' target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/dada2.html' target='_blank'  rel='noopener noreferrer'>DADA2<\/a>","<a href='http://csbl.bmb.uga.edu/dbCAN/annotate.php' target='_blank'  rel='noopener noreferrer'>dbCAN<\/a>","<a href='http://cys.bios.niu.edu/dbCAN2/' target='_blank'  rel='noopener noreferrer'>dbCAN2<\/a>","<a href='https://deeptools.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>DeepTools<\/a>","<a href='https://github.com/google/deepvariant' target='_blank'  rel='noopener noreferrer'>deepvariant<\/a>","<a href='https://dendropy.org/' target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://sourceforge.net/projects/diaumpire/' target='_blank'  rel='noopener noreferrer'>DIA-Umpire<\/a>","<a href='https://github.com/bbuchfink/diamond' target='_blank'  rel='noopener noreferrer'>diamond<\/a>","<a href='http://www.broadinstitute.org/software/discovar/blog/' target='_blank'  rel='noopener noreferrer'>DISCOVAR de novo<\/a>","<a href='https://github.com/jbloomlab/dms_tools2' target='_blank'  rel='noopener noreferrer'>dms_tools2<\/a>","<a href='http://swift.cmbi.ru.nl/gv/dssp/' target='_blank'  rel='noopener noreferrer'>dssp<\/a>","dyninst","<a href='https://github.com/easybuilders/easybuild' target='_blank'  rel='noopener noreferrer'>easybuild<\/a>","<a href='http://www.genomic.ch/edena' target='_blank'  rel='noopener noreferrer'>Edena<\/a>","<a href='https://www.ncbi.nlm.nih.gov/books/NBK179288/' target='_blank'  rel='noopener noreferrer'>edirect<\/a>","<a href='http://beta-eggnogdb.embl.de/#/app/emapper' target='_blank'  rel='noopener noreferrer'>eggNOG-mapper<\/a>","<a href='http://emboss.bioinformatics.nl/' target='_blank'  rel='noopener noreferrer'>EMBOSS (European Molecular Biology Open Software Suite)<\/a>","<a href='https://bitbucket.org/searleb/encyclopedia/' target='_blank'  rel='noopener noreferrer'>EncyclopeDIA<\/a>","<a href='https://github.com/geronimp/enrichM' target='_blank'  rel='noopener noreferrer'>enrichm<\/a>","<a href='http://www.ensembl.org/' target='_blank'  rel='noopener noreferrer'>Ensembl browser<\/a>","<a href='http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html' target='_blank'  rel='noopener noreferrer'>Ensembl Genebuild<\/a>","<a href='https://mblab.wustl.edu/software.html' target='_blank'  rel='noopener noreferrer'>eval<\/a>","<a href='https://github.com/EVidenceModeler/EVidenceModeler.github.io' target='_blank'  rel='noopener noreferrer'>EVidenceModeler (EVM)<\/a>","<a href='https://github.com/aberer/exabayes' target='_blank'  rel='noopener noreferrer'>ExaBayes<\/a>","<a href='http://www.ebi.ac.uk/%7Eguy/exonerate' target='_blank'  rel='noopener noreferrer'>Exonerate<\/a>","<a href='https://github.com/SegataLab/export2graphlan/' target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","<a href='https://github.com/pratas/falcon' target='_blank'  rel='noopener noreferrer'>Falcon<\/a>","<a href='https://github.com/pratas/falcon' target='_blank'  rel='noopener noreferrer'>Falcon (pb-assembly)<\/a>","<a href='http://fastml.tau.ac.il/' target='_blank'  rel='noopener noreferrer'>fastml<\/a>","<a href='https://github.com/OpenGene/fastp' target='_blank'  rel='noopener noreferrer'>fastp<\/a>","<a href='http://www.bioinformatics.babraham.ac.uk/projects/fastqc/' target='_blank'  rel='noopener noreferrer'>FASTQC<\/a>","<a href='http://rajanil.github.io/fastStructure/' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='http://rajanil.github.io/fastStructure/' target='_blank'  rel='noopener noreferrer'>faststructure (biocontainers)<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/' target='_blank'  rel='noopener noreferrer'>FASTX-Toolkit<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/' target='_blank'  rel='noopener noreferrer'>FASTX-Toolkit<\/a>","<a href='http://nhjy.hzau.edu.cn/kech/swxxx/jakj/dianzi/Bioinf6/GeneFinding/GeneFinding2.htm' target='_blank'  rel='noopener noreferrer'>FgenesH++<\/a>","<a href='http://www.aps.uoguelph.ca/~msargol/fimpute/' target='_blank'  rel='noopener noreferrer'>FImpute<\/a>","<a href='http://ccb.jhu.edu/software/FLASH/' target='_blank'  rel='noopener noreferrer'>FLASH<\/a>","<a href='https://github.com/smith-chem-wisc/FlashLFQ' target='_blank'  rel='noopener noreferrer'>flashLFQ<\/a>","<a href='https://github.com/fenderglass/Flye' target='_blank'  rel='noopener noreferrer'>Flye<\/a>","<a href='https://github.com/metageni/FOCUS' target='_blank'  rel='noopener noreferrer'>Focus<\/a>","<a href='https://sourceforge.net/projects/fraggenescan/' target='_blank'  rel='noopener noreferrer'>FragGeneScan<\/a>","<a href='https://github.com/ekg/freebayes' target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","<a href='https://cran.r-project.org/web/packages/freesurfer/index.html' target='_blank'  rel='noopener noreferrer'>freesurfer<\/a>","<a href='https://github.com/nextgenusfs/funannotate' target='_blank'  rel='noopener noreferrer'>funannotate<\/a>","<a href='https://code.google.com/archive/p/garli/' target='_blank'  rel='noopener noreferrer'>GARLI<\/a>","<a href='http://gatb.inria.fr/' target='_blank'  rel='noopener noreferrer'>GATB<\/a>","<a href='https://software.broadinstitute.org/gatk/' target='_blank'  rel='noopener noreferrer'>GATK<\/a>","gatk4","<a href='http://www.embl-heidelberg.de/~castresa/Gblocks/Gblocks.html' target='_blank'  rel='noopener noreferrer'>gblocks<\/a>","<a href='http://cnsgenomics.com/software/gcta/' target='_blank'  rel='noopener noreferrer'>gcta<\/a>","<a href='http://bitlab-es.com/gecko/' target='_blank'  rel='noopener noreferrer'>gecko<\/a>","<a href='http://genome.crg.es/software/geneid/index.html' target='_blank'  rel='noopener noreferrer'>geneid<\/a>","<a href='http://exon.gatech.edu/GeneMark/' target='_blank'  rel='noopener noreferrer'>Genemark<\/a>","<a href='https://github.com/tbenavi1/genomescope2.0' target='_blank'  rel='noopener noreferrer'>GenomeScope<\/a>","<a href='https://github.com/PacificBiosciences/GenomicConsensus' target='_blank'  rel='noopener noreferrer'>GenomicConsensus (Arrow)<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html' target='_blank'  rel='noopener noreferrer'>genomicfeatures<\/a>","<a href='https://www.kegg.jp/ghostkoala/' target='_blank'  rel='noopener noreferrer'>GhostKOALA<\/a>","<a href='http://www5.esu.edu/cpsc/bioinfo/software/GIST/' target='_blank'  rel='noopener noreferrer'>Gist<\/a>","<a href='https://ccb.jhu.edu/software/glimmer/' target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='https://ccb.jhu.edu/software/glimmerhmm/' target='_blank'  rel='noopener noreferrer'>glimmerhmm<\/a>","<a href='https://github.com/davek44/Glimmer-MG' target='_blank'  rel='noopener noreferrer'>GlimmerMG<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap (biocontainers)<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap_gsnap<\/a>","<a href='https://gnps.ucsd.edu/' target='_blank'  rel='noopener noreferrer'>GNPS<\/a>","<a href='https://www.gnu.org/' target='_blank'  rel='noopener noreferrer'>GNU<\/a>","<a href='https://www.gnu.org/software/parallel/' target='_blank'  rel='noopener noreferrer'>GNU parallel<\/a>","<a href='https://www.gnu.org/software/gsl/' target='_blank'  rel='noopener noreferrer'>GNU scientific library<\/a>","<a href='https://pypi.org/project/graftm/' target='_blank'  rel='noopener noreferrer'>GraftM<\/a>","<a href='https://github.com/biobakery/graphlan' target='_blank'  rel='noopener noreferrer'>GraPhlAn<\/a>","<a href='http://www.gromacs.org/' target='_blank'  rel='noopener noreferrer'>gromacs<\/a>","<a href='http://ecogenomics.github.io/GroopM' target='_blank'  rel='noopener noreferrer'>groopm<\/a>","<a href='https://github.com/ecogenomics/gtdbtk' target='_blank'  rel='noopener noreferrer'>GTDB-TK<\/a>","<a href='https://github.com/sanger-pathogens/gubbins' target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","guppy","<a href='https://hadoop.apache.org/' target='_blank'  rel='noopener noreferrer'>Hadoop<\/a>","<a href='https://github.com/vpc-ccg/haslr' target='_blank'  rel='noopener noreferrer'>HASLR<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php?title=Hawkeye' target='_blank'  rel='noopener noreferrer'>Hawkeye<\/a>","<a href='https://github.com/SegataLab/hclust2' target='_blank'  rel='noopener noreferrer'>Hclust2<\/a>","<a href='https://support.hdfgroup.org/products/hdf4_tools/' target='_blank'  rel='noopener noreferrer'>hdf4<\/a>","<a href='https://support.hdfgroup.org/products/hdf5_tools/' target='_blank'  rel='noopener noreferrer'>hdf5<\/a>","<a href='https://github.com/chhylp123/hifiasm' target='_blank'  rel='noopener noreferrer'>hifiasm<\/a>","<a href='https://github.com/higlass' target='_blank'  rel='noopener noreferrer'>HiGlass<\/a>","<a href='https://github.com/DovetailGenomics/HiRise_July2015_GR' target='_blank'  rel='noopener noreferrer'>HiRise<\/a>","<a href='https://ccb.jhu.edu/software/hisat2/index.shtml' target='_blank'  rel='noopener noreferrer'>HISAT2<\/a>","<a href='http://hmmer.org/' target='_blank'  rel='noopener noreferrer'>HMMER<\/a>","<a href='https://github.com/horovod/horovod' target='_blank'  rel='noopener noreferrer'>horovod<\/a>","hpctoolkit","hpcviewer","<a href='http://htseq.readthedocs.io/en/release_0.9.1/' target='_blank'  rel='noopener noreferrer'>HTSeq<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>HTSlib<\/a>","<a href='http://huttenhower.sph.harvard.edu/humann1' target='_blank'  rel='noopener noreferrer'>HUMAnN<\/a>","<a href='http://huttenhower.sph.harvard.edu/humann2' target='_blank'  rel='noopener noreferrer'>HUMAnN2<\/a>","<a href='http://huttenhower.sph.harvard.edu/humann3' target='_blank'  rel='noopener noreferrer'>HUMAnN3<\/a>","<a href='http://i.cs.hku.hk/%7Ealse/idba' target='_blank'  rel='noopener noreferrer'>idba-ud<\/a>","<a href='https://www.l3harrisgeospatial.com/Software-Technology/IDL' target='_blank'  rel='noopener noreferrer'>IDL<\/a>","<a href='https://imagemagick.org/script/index.php' target='_blank'  rel='noopener noreferrer'>ImageMagick<\/a>","<a href='http://img.jgi.doe.gov/cgi-bin/m/main.cgi' target='_blank'  rel='noopener noreferrer'>IMG/M<\/a>","<a href='https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler' target='_blank'  rel='noopener noreferrer'>Improved Phased Assembler (pbipa)<\/a>","<a href='http://eddylab.org/infernal/' target='_blank'  rel='noopener noreferrer'>Infernal<\/a>","<a href='http://software.broadinstitute.org/software/igv/home' target='_blank'  rel='noopener noreferrer'>Integrated Genomics Viewer<\/a>","intel-xed","<a href='https://interproscan-docs.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>InterProScan<\/a>","<a href='https://github.com/dereneaton/pyrad' target='_blank'  rel='noopener noreferrer'>iPyRAD<\/a>","<a href='http://www.cibiv.at/software/iqtree/' target='_blank'  rel='noopener noreferrer'>IQ-TREE<\/a>","iqtree","<a href='https://github.com/PacificBiosciences/IsoSeq' target='_blank'  rel='noopener noreferrer'>IsoSeq3<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/IWTomics.html' target='_blank'  rel='noopener noreferrer'>IWTomics<\/a>","<a href='https://mcmc-jags.sourceforge.io/' target='_blank'  rel='noopener noreferrer'>jags<\/a>","<a href='https://github.com/genomecuration/JAMg' target='_blank'  rel='noopener noreferrer'>JAMg<\/a>","<a href='http://gmod.org/wiki/JBrowse' target='_blank'  rel='noopener noreferrer'>JBrowse<\/a>","<a href='https://github.com/tanghaibao/jcvi' target='_blank'  rel='noopener noreferrer'>JCVI<\/a>","<a href='http://www.genome.umd.edu/jellyfish.html' target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","<a href='https://stedolan.github.io/jq/' target='_blank'  rel='noopener noreferrer'>jq<\/a>","<a href='https://aidenlab.org/juicebox/' target='_blank'  rel='noopener noreferrer'>juicebox<\/a>","<a href='https://github.com/aidenlab/juicer' target='_blank'  rel='noopener noreferrer'>Juicer<\/a>","<a href='http://www.genome.jp/kegg/kaas/' target='_blank'  rel='noopener noreferrer'>KAAS<\/a>","<a href='https://pachterlab.github.io/kallisto/about.html' target='_blank'  rel='noopener noreferrer'>Kallisto<\/a>","<a href='http://www.earlham.ac.uk/kat-tools' target='_blank'  rel='noopener noreferrer'>KAT<\/a>","<a href='http://khmer.readthedocs.org/en/v2.0/' target='_blank'  rel='noopener noreferrer'>khmer<\/a>","<a href='https://github.com/wwood/kingfisher-download' target='_blank'  rel='noopener noreferrer'>Kingfisher<\/a>","<a href='http://sun.aei.polsl.pl/REFRESH/index.php?page=projects&project=kmc&subpage=about' target='_blank'  rel='noopener noreferrer'>KMC<\/a>","<a href='https://www.genome.jp/tools/kofamkoala/' target='_blank'  rel='noopener noreferrer'>KofamKOALA<\/a>","<a href='https://ccb.jhu.edu/software/kraken/' target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://ccb.jhu.edu/software/kraken2/' target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","<a href='https://github.com/marbl/Krona/wiki' target='_blank'  rel='noopener noreferrer'>krona<\/a>","<a href='http://www.bx.psu.edu/~rsharris/lastz/' target='_blank'  rel='noopener noreferrer'>lastz<\/a>","<a href='https://bioinformatics.erc.monash.edu/apps/LFQ-Analyst' target='_blank'  rel='noopener noreferrer'>LFQ-Analyst<\/a>","libmonitor","<a href='https://github.com/bcgsc/LINKS' target='_blank'  rel='noopener noreferrer'>LINKS<\/a>","<a href='http://www.lipidmaps.org/tools/index.html' target='_blank'  rel='noopener noreferrer'>Lipid Maps<\/a>","<a href='https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger' target='_blank'  rel='noopener noreferrer'>Long Ranger<\/a>","lsdyna","<a href='http://csg.sph.umich.edu/abecasis/mach/' target='_blank'  rel='noopener noreferrer'>mach<\/a>","<a href='http://liulab.dfci.harvard.edu/MACS/' target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='http://liulab.dfci.harvard.edu/MACS/' target='_blank'  rel='noopener noreferrer'>macs2<\/a>","<a href='https://github.com/ranwez/MACSE_V2_PIPELINES' target='_blank'  rel='noopener noreferrer'>MACSE<\/a>","<a href='http://mafft.cbrc.jp/alignment/server/index.html' target='_blank'  rel='noopener noreferrer'>mafft<\/a>","<a href='https://github.com/WatsonLab/MAGpy' target='_blank'  rel='noopener noreferrer'>MAGpy<\/a>","<a href='http://www.yandell-lab.org/software/maker.html' target='_blank'  rel='noopener noreferrer'>MAKER<\/a>","<a href='http://www.yandell-lab.org/software/maker-p.html' target='_blank'  rel='noopener noreferrer'>MAKER-P<\/a>","<a href='https://cran.r-project.org/package=MALDIquant' target='_blank'  rel='noopener noreferrer'>MALDIquant<\/a>","<a href='https://github.com/marbl/mash' target='_blank'  rel='noopener noreferrer'>mash<\/a>","<a href='https://github.com/marbl/MashMap' target='_blank'  rel='noopener noreferrer'>mashmap<\/a>","<a href='https://github.com/kwongj/maskrc-svg' target='_blank'  rel='noopener noreferrer'>maskrc-svg<\/a>","<a href='http://www.genome.umd.edu/masurca.html' target='_blank'  rel='noopener noreferrer'>MaSuRCA<\/a>","<a href='http://darlinglab.org/mauve' target='_blank'  rel='noopener noreferrer'>Mauve<\/a>","<a href='http://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html' target='_blank'  rel='noopener noreferrer'>MaxBin2<\/a>","<a href='http://www.maxquant.org/' target='_blank'  rel='noopener noreferrer'>MaxQuant<\/a>","<a href='http://www.maxquant.org/' target='_blank'  rel='noopener noreferrer'>MaxQuant (biocontainers)<\/a>","<a href='http://micans.org/mcl/' target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://github.com/nanoporetech/medaka' target='_blank'  rel='noopener noreferrer'>Medaka<\/a>","<a href='https://github.com/voutcn/megahit' target='_blank'  rel='noopener noreferrer'>megahit<\/a>","<a href='http://ab.inf.uni-tuebingen.de/software/megan6/' target='_blank'  rel='noopener noreferrer'>MEGAN<\/a>","<a href='https://github.com/marbl/merqury' target='_blank'  rel='noopener noreferrer'>Merqury<\/a>","<a href='https://github.com/marbl/meryl' target='_blank'  rel='noopener noreferrer'>meryl<\/a>","<a href='http://i.cs.hku.hk/%7Ealse/idba' target='_blank'  rel='noopener noreferrer'>meta-IDBA<\/a>","<a href='https://bitbucket.org/berkeleylab/metabat' target='_blank'  rel='noopener noreferrer'>metabat<\/a>","<a href='http://www.metaboanalyst.ca' target='_blank'  rel='noopener noreferrer'>MetaboAnalyst<\/a>","<a href='http://i.cs.hku.hk/~alse/MetaCluster/index.html' target='_blank'  rel='noopener noreferrer'>MetaCluster<\/a>","<a href='https://sourceforge.net/projects/metacv/' target='_blank'  rel='noopener noreferrer'>metacv<\/a>","<a href='http://203.101.224.202/galaxy/' target='_blank'  rel='noopener noreferrer'>MetaDEGalaxy<\/a>","<a href='http://metagene.cb.k.u-tokyo.ac.jp/' target='_blank'  rel='noopener noreferrer'>MetaGeneAnnotator<\/a>","<a href='http://exon.gatech.edu/GeneMark/meta_gmhmmp.cgi' target='_blank'  rel='noopener noreferrer'>MetaGeneMark<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html' target='_blank'  rel='noopener noreferrer'>MetagenomeSeq<\/a>","<a href='https://github.com/DiltheyLab/MetaMaps' target='_blank'  rel='noopener noreferrer'>MetaMaps<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/metaMS.html' target='_blank'  rel='noopener noreferrer'>metaMS<\/a>","<a href='https://github.com/metaOmics/MetaPath' target='_blank'  rel='noopener noreferrer'>MetaPath<\/a>","<a href='http://huttenhower.sph.harvard.edu/metaphlan' target='_blank'  rel='noopener noreferrer'>MetaPhlAn<\/a>","<a href='http://metaphyler.cbcb.umd.edu/' target='_blank'  rel='noopener noreferrer'>MetaPhyler<\/a>","<a href='https://github.com/galaxyproteomics/metaquantome' target='_blank'  rel='noopener noreferrer'>metaQuantome<\/a>","<a href='http://cab.spbu.ru/software/spades/' target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","<a href='http://metavelvet.dna.bio.keio.ac.jp/' target='_blank'  rel='noopener noreferrer'>MetaVelvet<\/a>","<a href='https://github.com/bxlab/metaWRAP' target='_blank'  rel='noopener noreferrer'>MetaWRAP<\/a>","<a href='http://metagenomics.anl.gov' target='_blank'  rel='noopener noreferrer'>MG-RAST<\/a>","<a href='http://www.ebi.ac.uk/metagenomics' target='_blank'  rel='noopener noreferrer'>MGnify<\/a>","<a href='https://peterbeerli.com/migrate-html5/index.html' target='_blank'  rel='noopener noreferrer'>MIGRATE<\/a>","<a href='https://mimodd.readthedocs.io/en/latest/nacreousmap.html' target='_blank'  rel='noopener noreferrer'>MiModD<\/a>","<a href='http://minia.genouest.org/' target='_blank'  rel='noopener noreferrer'>minia<\/a>","<a href='https://genome.sph.umich.edu/wiki/Minimac4' target='_blank'  rel='noopener noreferrer'>Minimac4<\/a>","<a href='https://github.com/lh3/minimap2' target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","<a href='https://github.com/mgtools/MinPath' target='_blank'  rel='noopener noreferrer'>MinPath<\/a>","<a href='https://sourceforge.net/p/mira-assembler/wiki/Home/' target='_blank'  rel='noopener noreferrer'>mira<\/a>","<a href='https://github.com/ARTbio/tools-artbio/tree/master/tools/mircounts' target='_blank'  rel='noopener noreferrer'>mircounts<\/a>","<a href='https://www.mdc-berlin.de/8551903/en/' target='_blank'  rel='noopener noreferrer'>mirdeep<\/a>","<a href='https://github.com/rajewsky-lab/mirdeep2' target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","<a href='http://genes.mit.edu/burgelab/miso/docs/#releases' target='_blank'  rel='noopener noreferrer'>miso<\/a>","<a href='http://mixomics.org/' target='_blank'  rel='noopener noreferrer'>mixOmics<\/a>","<a href='http://cge.cbs.dtu.dk/services/MLST/' target='_blank'  rel='noopener noreferrer'>MLST<\/a>","<a href='http://vm-lux.embl.de/~kultima/MOCAT/' target='_blank'  rel='noopener noreferrer'>MOCAT<\/a>","<a href='https://github.com/compomics/moFF' target='_blank'  rel='noopener noreferrer'>moFF<\/a>","<a href='https://cwenger.github.io/Morpheus/' target='_blank'  rel='noopener noreferrer'>Morpheus<\/a>","<a href='https://www.mothur.org/' target='_blank'  rel='noopener noreferrer'>Mothur<\/a>","mpiBLAST","<a href='http://nbisweden.github.io/MrBayes/' target='_blank'  rel='noopener noreferrer'>MrBayes<\/a>","<a href='http://prime.psc.riken.jp/compms/msdial/main.html' target='_blank'  rel='noopener noreferrer'>MS-dial<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSGFplus.html' target='_blank'  rel='noopener noreferrer'>MS-GF+<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSnbase.html' target='_blank'  rel='noopener noreferrer'>MSnbase readMSData<\/a>","<a href='http://www.msstats.org/' target='_blank'  rel='noopener noreferrer'>MSstats<\/a>","<a href='http://multiqc.info/' target='_blank'  rel='noopener noreferrer'>MultiQC<\/a>","<a href='https://bitbucket.org/joseph07/multisnv/src/master/' target='_blank'  rel='noopener noreferrer'>multisnv<\/a>","<a href='http://mummer.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>MUMmer<\/a>","<a href='https://www.drive5.com/muscle/' target='_blank'  rel='noopener noreferrer'>MUSCLE<\/a>","<a href='https://www.nag.com/' target='_blank'  rel='noopener noreferrer'>nag<\/a>","<a href='https://github.com/jts/nanopolish' target='_blank'  rel='noopener noreferrer'>Nanopolish<\/a>","<a href='http://microbiomeutil.sourceforge.net/#A_NASTiEr' target='_blank'  rel='noopener noreferrer'>NAST-iEr<\/a>","<a href='https://gitlab.com/remikz/nccmp' target='_blank'  rel='noopener noreferrer'>nccmp<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml' target='_blank'  rel='noopener noreferrer'>NCL<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml' target='_blank'  rel='noopener noreferrer'>NCL<\/a>","<a href='http://nco.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>NCO<\/a>","<a href='https://cirrus.ucsd.edu/ncview/' target='_blank'  rel='noopener noreferrer'>Ncview<\/a>","<a href='https://github.com/xiaochuanle/NECAT' target='_blank'  rel='noopener noreferrer'>NECAT<\/a>","<a href='https://www.unidata.ucar.edu/software/netcdf/' target='_blank'  rel='noopener noreferrer'>NetCDF<\/a>","<a href='http://neuro.debian.net/' target='_blank'  rel='noopener noreferrer'>neurodebian<\/a>","<a href='https://github.com/Nextomics/NextDenovo' target='_blank'  rel='noopener noreferrer'>NextDenovo<\/a>","<a href='https://www.nextflow.io/' target='_blank'  rel='noopener noreferrer'>nextflow<\/a>","<a href='http://cibiv.github.io/NextGenMap/' target='_blank'  rel='noopener noreferrer'>nextgenmap<\/a>","<a href='https://github.com/Nextomics/NextPolish' target='_blank'  rel='noopener noreferrer'>NextPolish<\/a>","<a href='https://github.com/ncbi/ngs' target='_blank'  rel='noopener noreferrer'>NGS<\/a>","<a href='https://github.com/shenlab-sinai/ngsplot' target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","<a href='https://github.com/ndierckx/NOVOPlasty' target='_blank'  rel='noopener noreferrer'>NOVOPlasty<\/a>","<a href='https://nucleoatac.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>nucleoatac<\/a>","<a href='http://nvlabs.github.io/nvbio/index.html' target='_blank'  rel='noopener noreferrer'>nvbio<\/a>","<a href='https://github.com/dzerbino/oases' target='_blank'  rel='noopener noreferrer'>Oases<\/a>","<a href='https://git.metabarcoding.org/obitools/obitools/wikis/home' target='_blank'  rel='noopener noreferrer'>OBITools<\/a>","<a href='http://www.openms.de' target='_blank'  rel='noopener noreferrer'>OpenMS<\/a>","<a href='https://github.com/CSB5/OPERA-MS' target='_blank'  rel='noopener noreferrer'>OPERA-MS<\/a>","<a href='https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper' target='_blank'  rel='noopener noreferrer'>OPLS-DA_Contrasts (w4mcorcov)<\/a>","<a href='https://bitbucket.org/xvazquezc/otureporter/wiki/Home' target='_blank'  rel='noopener noreferrer'>OTUreporter<\/a>","<a href='https://github.com/kishwarshafin/pepper' target='_blank'  rel='noopener noreferrer'>P.E.P.P.E.R. (+ deep variant)<\/a>","<a href='https://github.com/TomSkelly/PacBioEDA' target='_blank'  rel='noopener noreferrer'>PacBioEDA<\/a>","<a href='https://pairtools.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>pairtools<\/a>","<a href='https://github.com/ColineRoyaux/PAMPA-Galaxy' target='_blank'  rel='noopener noreferrer'>PAMPA-Galaxy<\/a>","<a href='https://github.com/neufeld/pandaseq' target='_blank'  rel='noopener noreferrer'>PANDAseq<\/a>","<a href='https://github.com/cov-lineages/pangolin' target='_blank'  rel='noopener noreferrer'>Pangolin<\/a>","<a href='https://www.paraview.org/' target='_blank'  rel='noopener noreferrer'>ParaView<\/a>","<a href='https://github.com/marbl/parsnp' target='_blank'  rel='noopener noreferrer'>Parsnp<\/a>","<a href='https://sourceforge.net/projects/pasa/' target='_blank'  rel='noopener noreferrer'>PASA<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/pathview.html' target='_blank'  rel='noopener noreferrer'>Pathview<\/a>","<a href='http://sco.h-its.org/exelixis/web/software/pear/' target='_blank'  rel='noopener noreferrer'>PEAR<\/a>","<a href='http://compomics.github.io/projects/peptide-shaker.html' target='_blank'  rel='noopener noreferrer'>Peptide Shaker<\/a>","<a href='http://noble.gs.washington.edu/proj/percolator/' target='_blank'  rel='noopener noreferrer'>Percolator<\/a>","<a href='https://www.php.net/' target='_blank'  rel='noopener noreferrer'>PHP<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/phyloseq.html' target='_blank'  rel='noopener noreferrer'>Phyloseq<\/a>","<a href='https://phylosift.wordpress.com/about-2/' target='_blank'  rel='noopener noreferrer'>PhyloSift<\/a>","<a href='http://www.atgc-montpellier.fr/phyml/' target='_blank'  rel='noopener noreferrer'>PhyML<\/a>","<a href='https://ccb.jhu.edu/software/phymmbl/index.shtml' target='_blank'  rel='noopener noreferrer'>PhymmBL<\/a>","<a href='https://github.com/broadinstitute/picard' target='_blank'  rel='noopener noreferrer'>PICARD<\/a>","<a href='http://picrust.github.io/picrust/' target='_blank'  rel='noopener noreferrer'>PICRUSt<\/a>","<a href='https://github.com/picrust/picrust2' target='_blank'  rel='noopener noreferrer'>PICRUSt 2<\/a>","<a href='https://www.drive5.com/piler/' target='_blank'  rel='noopener noreferrer'>PILER-DF<\/a>","<a href='http://www.broadinstitute.org/software/pilon/' target='_blank'  rel='noopener noreferrer'>Pilon<\/a>","<a href='http://zzz.bwh.harvard.edu/plink/' target='_blank'  rel='noopener noreferrer'>plink<\/a>","<a href='https://atgu.mgh.harvard.edu/plinkseq/' target='_blank'  rel='noopener noreferrer'>plinkseq<\/a>","<a href='https://parallel-netcdf.github.io/' target='_blank'  rel='noopener noreferrer'>PnetCDF<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/clstutils.html' target='_blank'  rel='noopener noreferrer'>pplacer<\/a>","<a href='https://github.com/wtsi-hpag/PretextView' target='_blank'  rel='noopener noreferrer'>PretextView / Pretext Suite<\/a>","<a href='http://edwards.sdsu.edu/prinseq' target='_blank'  rel='noopener noreferrer'>PRINSEQ<\/a>","<a href='http://www.genabel.org/packages/ProbABEL' target='_blank'  rel='noopener noreferrer'>probabel<\/a>","<a href='http://labpib.fmrp.usp.br/methods/probmetab/' target='_blank'  rel='noopener noreferrer'>ProbMetab Tool<\/a>","<a href='https://code.google.com/p/gmv/' target='_blank'  rel='noopener noreferrer'>Prodigal<\/a>","<a href='https://github.com/tseemann/prokka' target='_blank'  rel='noopener noreferrer'>Prokka<\/a>","<a href='http://www.bioinfo.biozentrum.uni-wuerzburg.de/groups/genomics/research/development_of_bioinformatical_tools/' target='_blank'  rel='noopener noreferrer'>proovread<\/a>","<a href='http://www.psort.org/psortb/' target='_blank'  rel='noopener noreferrer'>PSORTb<\/a>","<a href='https://www.psort.org/psortm/' target='_blank'  rel='noopener noreferrer'>PSORTm<\/a>","<a href='https://github.com/dfguan/purge_dups' target='_blank'  rel='noopener noreferrer'>Purge_Dups<\/a>","<a href='https://bitbucket.org/mroachawri/purge_haplotigs' target='_blank'  rel='noopener noreferrer'>PurgeHaplotigs<\/a>","<a href='https://github.com/widdowquinn/pyani' target='_blank'  rel='noopener noreferrer'>pyani<\/a>","<a href='http://dereneaton.com/software/pyrad/' target='_blank'  rel='noopener noreferrer'>PyRAD<\/a>","<a href='https://github.com/pysam-developers/pysam' target='_blank'  rel='noopener noreferrer'>pysam<\/a>","<a href='https://pytorch.org/' target='_blank'  rel='noopener noreferrer'>PyTorch<\/a>","<a href='http://qiime.org/' target='_blank'  rel='noopener noreferrer'>QIIME<\/a>","<a href='http://qiime.org/' target='_blank'  rel='noopener noreferrer'>QIIME<\/a>","<a href='https://qiime2.org/' target='_blank'  rel='noopener noreferrer'>QIIME2.0<\/a>","<a href='http://quast.sourceforge.net/quast' target='_blank'  rel='noopener noreferrer'>QUAST<\/a>","<a href='https://github.com/mahulchak/quickmerge' target='_blank'  rel='noopener noreferrer'>Quickmerge<\/a>","<a href='http://www.r-project.org/' target='_blank'  rel='noopener noreferrer'>R<\/a>","<a href='https://github.com/isovic/racon' target='_blank'  rel='noopener noreferrer'>Racon<\/a>","<a href='https://github.com/bcgsc/RAILS' target='_blank'  rel='noopener noreferrer'>RAILS/Cobbler<\/a>","<a href='http://ratt.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>RATT<\/a>","<a href='https://github.com/comprna/RATTLE' target='_blank'  rel='noopener noreferrer'>RATTLE<\/a>","<a href='https://github.com/lbcb-sci/raven' target='_blank'  rel='noopener noreferrer'>raven<\/a>","<a href='https://github.com/kevinkovalchik/RawTools' target='_blank'  rel='noopener noreferrer'>Raw Tools<\/a>","<a href='http://sco.h-its.org/exelixis/web/software/raxml/' target='_blank'  rel='noopener noreferrer'>RAxML<\/a>","<a href='https://rclone.org/' target='_blank'  rel='noopener noreferrer'>rclone<\/a>","<a href='https://github.com/dparks1134/RefineM' target='_blank'  rel='noopener noreferrer'>RefineM<\/a>","<a href='https://github.com/Reedwarbler/REPdenovo' target='_blank'  rel='noopener noreferrer'>REPdenovo<\/a>","<a href='http://www.repeatmasker.org/' target='_blank'  rel='noopener noreferrer'>RepeatMasker<\/a>","<a href='http://www.repeatmasker.org/RepeatModeler/' target='_blank'  rel='noopener noreferrer'>RepeatModeler<\/a>","<a href='https://github.com/Dfam-consortium/RepeatModeler' target='_blank'  rel='noopener noreferrer'>RepeatModeler2<\/a>","<a href='http://www.yandell-lab.org/software/repeatrunner.html' target='_blank'  rel='noopener noreferrer'>RepeatRunner<\/a>","<a href='https://urgi.versailles.inra.fr/Tools/REPET' target='_blank'  rel='noopener noreferrer'>REPET<\/a>","<a href='https://cran.r-project.org/package=rjags' target='_blank'  rel='noopener noreferrer'>rjags<\/a>","rmpi","<a href='http://deweylab.github.io/RSEM/' target='_blank'  rel='noopener noreferrer'>RSEM<\/a>","<a href='http://rseqc.sourceforge.net' target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","<a href='https://rstudio.com/' target='_blank'  rel='noopener noreferrer'>Rstudio<\/a>","<a href='https://github.com/weizhouUMICH/SAIGE' target='_blank'  rel='noopener noreferrer'>SAIGE<\/a>","<a href='https://combine-lab.github.io/salmon/' target='_blank'  rel='noopener noreferrer'>Salmon<\/a>","<a href='https://github.com/machinegun/SALSA' target='_blank'  rel='noopener noreferrer'>salsa<\/a>","<a href='https://github.com/GregoryFaust/samblaster' target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>SAMTools<\/a>","<a href='https://www.sas.com/en_us/software/foundation.html' target='_blank'  rel='noopener noreferrer'>SAS<\/a>","<a href='http://compbio.cs.toronto.edu/hapsembler/scarpa.html' target='_blank'  rel='noopener noreferrer'>Scarpa<\/a>","<a href='http://scikit-learn.org' target='_blank'  rel='noopener noreferrer'>Scikit-learn<\/a>","<a href='https://pypi.org/project/screed/' target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='http://compomics.github.io/projects/searchgui.html' target='_blank'  rel='noopener noreferrer'>Search GUI<\/a>","<a href='https://github.com/shenwei356/seqkit' target='_blank'  rel='noopener noreferrer'>SeqKit<\/a>","<a href='https://github.com/drostlab/seqstats' target='_blank'  rel='noopener noreferrer'>seqstats<\/a>","<a href='https://github.com/trinityrnaseq/seqtk-trinity' target='_blank'  rel='noopener noreferrer'>seqtk-trinity<\/a>","<a href='https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html' target='_blank'  rel='noopener noreferrer'>shapeit<\/a>","<a href='https://github.com/chanzuckerberg/shasta' target='_blank'  rel='noopener noreferrer'>shasta<\/a>","<a href='https://github.com/najoshi/sickle' target='_blank'  rel='noopener noreferrer'>sickle<\/a>","<a href='https://github.com/mgramm1/simtext' target='_blank'  rel='noopener noreferrer'>SimText<\/a>","<a href='https://sourceforge.net/projects/skewer/' target='_blank'  rel='noopener noreferrer'>skewer<\/a>","<a href='http://www.pacb.com/products-and-services/analytical-software/smrt-analysis/' target='_blank'  rel='noopener noreferrer'>SMRT Link / Analysis<\/a>","<a href='https://snakemake.readthedocs.io/en/stable/index.html' target='_blank'  rel='noopener noreferrer'>Snakemake<\/a>","<a href='http://korflab.ucdavis.edu/software.html' target='_blank'  rel='noopener noreferrer'>SNAP<\/a>","<a href='https://github.com/PMMAraujo/snappy' target='_blank'  rel='noopener noreferrer'>SNAPPy<\/a>","<a href='https://github.com/fritzsedlazeck/Sniffles' target='_blank'  rel='noopener noreferrer'>sniffles<\/a>","<a href='http://lowelab.ucsc.edu/snoscan/' target='_blank'  rel='noopener noreferrer'>snoscan<\/a>","<a href='http://snpeff.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","<a href='http://soap.genomics.org.cn/' target='_blank'  rel='noopener noreferrer'>soap<\/a>","<a href='https://i.cs.hku.hk/2bwt-tools/soap3/index.html' target='_blank'  rel='noopener noreferrer'>soap3-gpu<\/a>","<a href='https://github.com/aquaskyline/SOAPdenovo2' target='_blank'  rel='noopener noreferrer'>SOAPdenovo2<\/a>","<a href='http://soap.genomics.org.cn/soapsnp.html' target='_blank'  rel='noopener noreferrer'>SOAPsnp<\/a>","<a href='http://bioinfo.lifl.fr/RNA/sortmerna/' target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='http://cab.spbu.ru/software/spades/' target='_blank'  rel='noopener noreferrer'>SPAdes<\/a>","<a href='https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi' target='_blank'  rel='noopener noreferrer'>Splign<\/a>","<a href='https://github.com/jtamames/SqueezeMeta' target='_blank'  rel='noopener noreferrer'>SqueezeMeta<\/a>","<a href='https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc' target='_blank'  rel='noopener noreferrer'>sra-tools<\/a>","<a href='http://catchenlab.life.illinois.edu/stacks/' target='_blank'  rel='noopener noreferrer'>Stacks<\/a>","<a href='http://code.google.com/p/rna-star/' target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='http://web.stanford.edu/group/pritchardlab/structure.html' target='_blank'  rel='noopener noreferrer'>structure<\/a>","<a href='http://edwards.sdsu.edu/superfocus/' target='_blank'  rel='noopener noreferrer'>SUPER-FOCUS<\/a>","<a href='https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome' target='_blank'  rel='noopener noreferrer'>Supernova<\/a>","<a href='https://github.com/llecompte/SVJedi.git' target='_blank'  rel='noopener noreferrer'>svjedi<\/a>","<a href='https://github.com/hall-lab/svtyper' target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/SwathXtend.html' target='_blank'  rel='noopener noreferrer'>SwathXtend<\/a>","<a href='http://tagcleaner.sourceforge.net/index.html' target='_blank'  rel='noopener noreferrer'>Tagcleaner<\/a>","<a href='http://genome.gsc.riken.jp/osc/english/software/' target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://github.com/carringtonlab/TargetFinder' target='_blank'  rel='noopener noreferrer'>TargetFinder<\/a>","<a href='http://www.maizegenetics.net/tassel' target='_blank'  rel='noopener noreferrer'>tassel<\/a>","<a href='https://github.com/bwemheu/Tax4Fun2' target='_blank'  rel='noopener noreferrer'>Tax4Fun2<\/a>","<a href='https://www.tensorflow.org/resources/tools' target='_blank'  rel='noopener noreferrer'>TensorFlow<\/a>","<a href='https://github.com/compomics/ThermoRawFileParser' target='_blank'  rel='noopener noreferrer'>ThermoRawFileParser<\/a>","<a href='https://github.com/bcgsc/tigmint' target='_blank'  rel='noopener noreferrer'>Tigmint<\/a>","<a href='http://ccb.jhu.edu/software/tophat/index.shtml' target='_blank'  rel='noopener noreferrer'>tophat<\/a>","<a href='https://totalview.io/' target='_blank'  rel='noopener noreferrer'>TotalView<\/a>","<a href='https://github.com/TransDecoder/TransDecoder/wiki' target='_blank'  rel='noopener noreferrer'>TransDecoder<\/a>","<a href='http://tandem.bu.edu/trf/trf.html' target='_blank'  rel='noopener noreferrer'>trf<\/a>","<a href='http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/' target='_blank'  rel='noopener noreferrer'>Trim Galore<\/a>","<a href='http://www.usadellab.org/cms/index.php?page=trimmomatic' target='_blank'  rel='noopener noreferrer'>Trimmomatic<\/a>","<a href='https://github.com/trinityrnaseq/trinityrnaseq/wiki' target='_blank'  rel='noopener noreferrer'>Trinity / trinityrnaseq<\/a>","<a href='https://github.com/Trinotate/Trinotate.github.io' target='_blank'  rel='noopener noreferrer'>Trinotate<\/a>","<a href='http://trna.ucsc.edu/tRNAscan-SE/' target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","<a href='https://github.com/rrwick/Trycycler' target='_blank'  rel='noopener noreferrer'>Trycycler toolkit<\/a>","<a href='http://hgdownload.cse.ucsc.edu/downloads.html' target='_blank'  rel='noopener noreferrer'>UCSC tools<\/a>","<a href='https://github.com/rrwick/Unicycler' target='_blank'  rel='noopener noreferrer'>Unicycler<\/a>","<a href='http://www.drive5.com/usearch/' target='_blank'  rel='noopener noreferrer'>USEARCH<\/a>","<a href='https://github.com/marbl/VALET' target='_blank'  rel='noopener noreferrer'>Valet<\/a>","<a href='https://vamps.mbl.edu/' target='_blank'  rel='noopener noreferrer'>VAMPS<\/a>","<a href='http://dkoboldt.github.io/varscan/' target='_blank'  rel='noopener noreferrer'>VarScan<\/a>","<a href='https://github.com/cc2qe/vawk' target='_blank'  rel='noopener noreferrer'>vawk<\/a>","<a href='https://vcftools.github.io/index.html' target='_blank'  rel='noopener noreferrer'>VCFTools<\/a>","<a href='https://github.com/dzerbino/velvet' target='_blank'  rel='noopener noreferrer'>Velvet<\/a>","<a href='https://www.tbi.univie.ac.at/RNA/' target='_blank'  rel='noopener noreferrer'>ViennaRNA<\/a>","<a href='https://wci.llnl.gov/simulation/computer-codes/visit' target='_blank'  rel='noopener noreferrer'>VisIt<\/a>","<a href='https://github.com/torognes/vsearch' target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","<a href='https://github.com/adigenova/wengan' target='_blank'  rel='noopener noreferrer'>Wengan<\/a>","<a href='http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html' target='_blank'  rel='noopener noreferrer'>WindowMasker<\/a>","<a href='http://www.ebi.ac.uk/Tools/Wise2/' target='_blank'  rel='noopener noreferrer'>wise<\/a>","<a href='https://workflow4metabolomics.org/' target='_blank'  rel='noopener noreferrer'>Workflow 4 Metabolomics<\/a>","<a href='https://github.com/ruanjue/wtdbg2' target='_blank'  rel='noopener noreferrer'>WTDBG2<\/a>","<a href='http://xarray.pydata.org/en/stable/' target='_blank'  rel='noopener noreferrer'>xarray<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/xcms.html' target='_blank'  rel='noopener noreferrer'>xcms<\/a>","<a href='https://github.com/AustralianBioCommons/xml4ena' target='_blank'  rel='noopener noreferrer'>xml4ena<\/a>","<a href='http://www.thegpm.org/TANDEM/' target='_blank'  rel='noopener noreferrer'>xtandem<\/a>"],["<a href='https://bio.tools/3d-dna' target='_blank'  rel='noopener noreferrer'>3d-dna<\/a>","<a href='https://bio.tools/ABRicate' target='_blank'  rel='noopener noreferrer'>ABRicate<\/a>","<a href='https://bio.tools/abyss' target='_blank'  rel='noopener noreferrer'>abyss<\/a>","","","","<a href='https://bio.tools/amos' target='_blank'  rel='noopener noreferrer'>amos<\/a>","<a href='https://bio.tools/AMR' target='_blank'  rel='noopener noreferrer'>AMR<\/a>","<a href='https://bio.tools/anaconda' target='_blank'  rel='noopener noreferrer'>anaconda<\/a>","<a href='https://bio.tools/angsd' target='_blank'  rel='noopener noreferrer'>angsd<\/a>","<a href='https://bio.tools/anvio' target='_blank'  rel='noopener noreferrer'>anvio<\/a>","<a href='https://bio.tools/apollo' target='_blank'  rel='noopener noreferrer'>apollo<\/a>","<a href='https://bio.tools/artemis' target='_blank'  rel='noopener noreferrer'>artemis<\/a>","<a href='https://bio.tools/augustus' target='_blank'  rel='noopener noreferrer'>augustus<\/a>","<a href='https://bio.tools/bamkit' target='_blank'  rel='noopener noreferrer'>bamkit<\/a>","<a href='https://bio.tools/bamm' target='_blank'  rel='noopener noreferrer'>bamm<\/a>","<a href='https://bio.tools/bamtools' target='_blank'  rel='noopener noreferrer'>bamtools<\/a>","<a href='https://bio.tools/bbmap' target='_blank'  rel='noopener noreferrer'>bbmap<\/a>","<a href='https://bio.tools/bcftools' target='_blank'  rel='noopener noreferrer'>bcftools<\/a>","<a href='https://bio.tools/beagle' target='_blank'  rel='noopener noreferrer'>beagle<\/a>","<a href='https://bio.tools/beagle-lib' target='_blank'  rel='noopener noreferrer'>beagle-lib<\/a>","<a href='https://bio.tools/beast' target='_blank'  rel='noopener noreferrer'>beast<\/a>","<a href='https://bio.tools/beast2' target='_blank'  rel='noopener noreferrer'>beast2<\/a>","<a href='https://bio.tools/bedtools' target='_blank'  rel='noopener noreferrer'>bedtools<\/a>","","<a href='https://bio.tools/binsanity' target='_blank'  rel='noopener noreferrer'>binsanity<\/a>","","<a href='https://bio.tools/bioconductor' target='_blank'  rel='noopener noreferrer'>bioconductor<\/a>","","","<a href='https://bio.tools/bioperl' target='_blank'  rel='noopener noreferrer'>bioperl<\/a>","<a href='https://bio.tools/Biopython' target='_blank'  rel='noopener noreferrer'>Biopython<\/a>","<a href='https://bio.tools/bismark' target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://bio.tools/blast' target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://bio.tools/blast' target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://bio.tools/blast' target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://bio.tools/blastkoala' target='_blank'  rel='noopener noreferrer'>blastkoala<\/a>","<a href='https://bio.tools/blat' target='_blank'  rel='noopener noreferrer'>blat<\/a>","<a href='https://bio.tools/BlobToolKit' target='_blank'  rel='noopener noreferrer'>BlobToolKit<\/a>","<a href='https://bio.tools/blobtools' target='_blank'  rel='noopener noreferrer'>blobtools<\/a>","<a href='https://bio.tools/bmge' target='_blank'  rel='noopener noreferrer'>bmge<\/a>","<a href='https://bio.tools/boost' target='_blank'  rel='noopener noreferrer'>boost<\/a>","<a href='https://bio.tools/bowtie' target='_blank'  rel='noopener noreferrer'>bowtie<\/a>","<a href='https://bio.tools/bowtie2' target='_blank'  rel='noopener noreferrer'>bowtie2<\/a>","<a href='https://bio.tools/braker1' target='_blank'  rel='noopener noreferrer'>braker1<\/a>","<a href='https://bio.tools/braker1' target='_blank'  rel='noopener noreferrer'>braker1<\/a>","<a href='https://bio.tools/breakdancer' target='_blank'  rel='noopener noreferrer'>breakdancer<\/a>","<a href='https://bio.tools/bsmap' target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","<a href='https://bio.tools/bs_seeker' target='_blank'  rel='noopener noreferrer'>bs_seeker<\/a>","<a href='https://bio.tools/bs-seeker2' target='_blank'  rel='noopener noreferrer'>bs-seeker2<\/a>","<a href='https://bio.tools/busco' target='_blank'  rel='noopener noreferrer'>busco<\/a>","<a href='https://bio.tools/bwa' target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://bio.tools/bx-python' target='_blank'  rel='noopener noreferrer'>bx-python<\/a>","<a href='https://bio.tools/cactus' target='_blank'  rel='noopener noreferrer'>cactus<\/a>","","<a href='https://bio.tools/camera' target='_blank'  rel='noopener noreferrer'>camera<\/a>","<a href='https://bio.tools/canu' target='_blank'  rel='noopener noreferrer'>canu<\/a>","<a href='https://bio.tools/cap3' target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='https://bio.tools/cardinal' target='_blank'  rel='noopener noreferrer'>cardinal<\/a>","<a href='https://bio.tools/cat_bins' target='_blank'  rel='noopener noreferrer'>cat_bins<\/a>","<a href='https://bio.tools/cd-hit' target='_blank'  rel='noopener noreferrer'>cd-hit<\/a>","<a href='https://bio.tools/celseq2' target='_blank'  rel='noopener noreferrer'>celseq2<\/a>","<a href='https://bio.tools/cabog' target='_blank'  rel='noopener noreferrer'>cabog<\/a>","<a href='https://bio.tools/CellProfiler' target='_blank'  rel='noopener noreferrer'>CellProfiler<\/a>","<a href='https://bio.tools/centrifuge' target='_blank'  rel='noopener noreferrer'>centrifuge<\/a>","<a href='https://bio.tools/checkm' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://bio.tools/checkm' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://bio.tools/checkv' target='_blank'  rel='noopener noreferrer'>checkv<\/a>","<a href='https://bio.tools/chimeraslayer' target='_blank'  rel='noopener noreferrer'>chimeraslayer<\/a>","","<a href='https://bio.tools/chunkchromosome' target='_blank'  rel='noopener noreferrer'>chunkchromosome<\/a>","","","<a href='https://bio.tools/clustalw' target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","<a href='https://bio.tools/clusteringsuite' target='_blank'  rel='noopener noreferrer'>clusteringsuite<\/a>","<a href='https://bio.tools/cnvkit' target='_blank'  rel='noopener noreferrer'>cnvkit<\/a>","<a href='https://bio.tools/cpc' target='_blank'  rel='noopener noreferrer'>cpc<\/a>","<a href='https://bio.tools/cpc2' target='_blank'  rel='noopener noreferrer'>cpc2<\/a>","<a href='https://bio.tools/comparem' target='_blank'  rel='noopener noreferrer'>comparem<\/a>","<a href='https://bio.tools/coverm' target='_blank'  rel='noopener noreferrer'>coverm<\/a>","<a href='https://bio.tools/cpat' target='_blank'  rel='noopener noreferrer'>cpat<\/a>","","","","<a href='https://bio.tools/cufflinks' target='_blank'  rel='noopener noreferrer'>cufflinks<\/a>","<a href='https://bio.tools/cutadapt' target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='https://bio.tools/cytoscape' target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","<a href='https://bio.tools/dada2' target='_blank'  rel='noopener noreferrer'>dada2<\/a>","<a href='https://bio.tools/dbcan' target='_blank'  rel='noopener noreferrer'>dbcan<\/a>","<a href='https://bio.tools/dbcan2' target='_blank'  rel='noopener noreferrer'>dbcan2<\/a>","<a href='https://bio.tools/deeptools' target='_blank'  rel='noopener noreferrer'>deeptools<\/a>","<a href='https://bio.tools/deepvariant' target='_blank'  rel='noopener noreferrer'>deepvariant<\/a>","<a href='https://bio.tools/dendropy' target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://bio.tools/DIA-Umpire' target='_blank'  rel='noopener noreferrer'>DIA-Umpire<\/a>","<a href='https://bio.tools/diamond' target='_blank'  rel='noopener noreferrer'>diamond<\/a>","<a href='https://bio.tools/discovar_de_novo' target='_blank'  rel='noopener noreferrer'>discovar_de_novo<\/a>","<a href='https://bio.tools/dms_tools2' target='_blank'  rel='noopener noreferrer'>dms_tools2<\/a>","<a href='https://bio.tools/dssp' target='_blank'  rel='noopener noreferrer'>dssp<\/a>","","<a href='https://bio.tools/easybuild' target='_blank'  rel='noopener noreferrer'>easybuild<\/a>","<a href='https://bio.tools/edena' target='_blank'  rel='noopener noreferrer'>edena<\/a>","","<a href='https://bio.tools/eggnog-mapper' target='_blank'  rel='noopener noreferrer'>eggnog-mapper<\/a>","<a href='https://bio.tools/emboss' target='_blank'  rel='noopener noreferrer'>emboss<\/a>","<a href='https://bio.tools/EncyclopeDIA' target='_blank'  rel='noopener noreferrer'>EncyclopeDIA<\/a>","","<a href='https://bio.tools/ensembl' target='_blank'  rel='noopener noreferrer'>ensembl<\/a>","","<a href='https://bio.tools/eval' target='_blank'  rel='noopener noreferrer'>eval<\/a>","<a href='https://bio.tools/EvidenceModeler' target='_blank'  rel='noopener noreferrer'>EvidenceModeler<\/a>","<a href='https://bio.tools/exabayes' target='_blank'  rel='noopener noreferrer'>exabayes<\/a>","<a href='https://bio.tools/exonerate' target='_blank'  rel='noopener noreferrer'>exonerate<\/a>","<a href='https://bio.tools/export2graphlan' target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","<a href='https://bio.tools/falcon' target='_blank'  rel='noopener noreferrer'>falcon<\/a>","<a href='https://bio.tools/falcon' target='_blank'  rel='noopener noreferrer'>falcon<\/a>","<a href='https://bio.tools/fastml' target='_blank'  rel='noopener noreferrer'>fastml<\/a>","<a href='https://bio.tools/fastp' target='_blank'  rel='noopener noreferrer'>fastp<\/a>","<a href='https://bio.tools/fastqc' target='_blank'  rel='noopener noreferrer'>fastqc<\/a>","<a href='https://bio.tools/faststructure' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='https://bio.tools/faststructure' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='https://bio.tools/fastx-toolkit' target='_blank'  rel='noopener noreferrer'>fastx-toolkit<\/a>","<a href='https://bio.tools/fastx-toolkit' target='_blank'  rel='noopener noreferrer'>fastx-toolkit<\/a>","<a href='https://bio.tools/fgenesh' target='_blank'  rel='noopener noreferrer'>fgenesh<\/a>","<a href='https://bio.tools/fimpute' target='_blank'  rel='noopener noreferrer'>fimpute<\/a>","<a href='https://bio.tools/flash' target='_blank'  rel='noopener noreferrer'>flash<\/a>","<a href='https://bio.tools/flashlfq' target='_blank'  rel='noopener noreferrer'>flashlfq<\/a>","<a href='https://bio.tools/Flye' target='_blank'  rel='noopener noreferrer'>Flye<\/a>","<a href='https://bio.tools/focus' target='_blank'  rel='noopener noreferrer'>focus<\/a>","<a href='https://bio.tools/fraggenescan' target='_blank'  rel='noopener noreferrer'>fraggenescan<\/a>","<a href='https://bio.tools/freebayes' target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","<a href='https://bio.tools/freesurfer' target='_blank'  rel='noopener noreferrer'>freesurfer<\/a>","<a href='https://bio.tools/funannotate' target='_blank'  rel='noopener noreferrer'>funannotate<\/a>","<a href='https://bio.tools/garli' target='_blank'  rel='noopener noreferrer'>garli<\/a>","<a href='https://bio.tools/gatb' target='_blank'  rel='noopener noreferrer'>gatb<\/a>","<a href='https://bio.tools/gatk' target='_blank'  rel='noopener noreferrer'>gatk<\/a>","","<a href='https://bio.tools/gblocks' target='_blank'  rel='noopener noreferrer'>gblocks<\/a>","<a href='https://bio.tools/gcta' target='_blank'  rel='noopener noreferrer'>gcta<\/a>","<a href='https://bio.tools/gecko' target='_blank'  rel='noopener noreferrer'>gecko<\/a>","<a href='https://bio.tools/geneid' target='_blank'  rel='noopener noreferrer'>geneid<\/a>","<a href='https://bio.tools/genemark' target='_blank'  rel='noopener noreferrer'>genemark<\/a>","<a href='https://bio.tools/GenomeScope_2.0' target='_blank'  rel='noopener noreferrer'>GenomeScope_2.0<\/a>","<a href='https://bio.tools/GenomicConsensus' target='_blank'  rel='noopener noreferrer'>GenomicConsensus<\/a>","<a href='https://bio.tools/genomicfeatures' target='_blank'  rel='noopener noreferrer'>genomicfeatures<\/a>","<a href='https://bio.tools/ghostkoala' target='_blank'  rel='noopener noreferrer'>ghostkoala<\/a>","<a href='https://bio.tools/gist' target='_blank'  rel='noopener noreferrer'>gist<\/a>","<a href='https://bio.tools/glimmer' target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='https://bio.tools/glimmer-hmm' target='_blank'  rel='noopener noreferrer'>glimmer-hmm<\/a>","<a href='https://bio.tools/glimmermg' target='_blank'  rel='noopener noreferrer'>glimmermg<\/a>","<a href='https://bio.tools/gmap' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='https://bio.tools/gmap' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='https://bio.tools/gmap' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='https://bio.tools/gnps' target='_blank'  rel='noopener noreferrer'>gnps<\/a>","","<a href='https://bio.tools/parallel' target='_blank'  rel='noopener noreferrer'>parallel<\/a>","<a href='https://bio.tools/gsl' target='_blank'  rel='noopener noreferrer'>gsl<\/a>","<a href='https://bio.tools/graftm' target='_blank'  rel='noopener noreferrer'>graftm<\/a>","<a href='https://bio.tools/graphlan' target='_blank'  rel='noopener noreferrer'>graphlan<\/a>","<a href='https://bio.tools/gromacs' target='_blank'  rel='noopener noreferrer'>gromacs<\/a>","<a href='https://bio.tools/groopm' target='_blank'  rel='noopener noreferrer'>groopm<\/a>","<a href='https://bio.tools/GTDB-Tk' target='_blank'  rel='noopener noreferrer'>GTDB-Tk<\/a>","<a href='https://bio.tools/gubbins' target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","","<a href='https://bio.tools/hadoop' target='_blank'  rel='noopener noreferrer'>hadoop<\/a>","","<a href='https://bio.tools/hawkeye' target='_blank'  rel='noopener noreferrer'>hawkeye<\/a>","<a href='https://bio.tools/hclust2' target='_blank'  rel='noopener noreferrer'>hclust2<\/a>","","","","","<a href='https://bio.tools/hirise' target='_blank'  rel='noopener noreferrer'>hirise<\/a>","<a href='https://bio.tools/hisat2' target='_blank'  rel='noopener noreferrer'>hisat2<\/a>","<a href='https://bio.tools/hmmer3' target='_blank'  rel='noopener noreferrer'>hmmer3<\/a>","<a href='https://bio.tools/horovod' target='_blank'  rel='noopener noreferrer'>horovod<\/a>","","","<a href='https://bio.tools/htseq' target='_blank'  rel='noopener noreferrer'>htseq<\/a>","<a href='https://bio.tools/htslib' target='_blank'  rel='noopener noreferrer'>htslib<\/a>","","<a href='https://bio.tools/humann2' target='_blank'  rel='noopener noreferrer'>humann2<\/a>","","<a href='https://bio.tools/idba' target='_blank'  rel='noopener noreferrer'>idba<\/a>","","","<a href='https://bio.tools/img_m' target='_blank'  rel='noopener noreferrer'>img_m<\/a>","","","","","<a href='https://bio.tools/interproscan4' target='_blank'  rel='noopener noreferrer'>interproscan4<\/a>","","<a href='https://bio.tools/iq-tree' target='_blank'  rel='noopener noreferrer'>iq-tree<\/a>","","<a href='https://bio.tools/isoseq3' target='_blank'  rel='noopener noreferrer'>isoseq3<\/a>","<a href='https://bio.tools/iwtomics' target='_blank'  rel='noopener noreferrer'>iwtomics<\/a>","<a href='https://bio.tools/jags' target='_blank'  rel='noopener noreferrer'>jags<\/a>","<a href='https://bio.tools/jamg' target='_blank'  rel='noopener noreferrer'>jamg<\/a>","<a href='https://bio.tools/jbrowse' target='_blank'  rel='noopener noreferrer'>jbrowse<\/a>","","<a href='https://bio.tools/jellyfish' target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","","<a href='https://bio.tools/juicebox' target='_blank'  rel='noopener noreferrer'>juicebox<\/a>","<a href='https://bio.tools/juicer' target='_blank'  rel='noopener noreferrer'>juicer<\/a>","<a href='https://bio.tools/kaas' target='_blank'  rel='noopener noreferrer'>kaas<\/a>","<a href='https://bio.tools/kallisto' target='_blank'  rel='noopener noreferrer'>kallisto<\/a>","<a href='https://bio.tools/kat' target='_blank'  rel='noopener noreferrer'>kat<\/a>","<a href='https://bio.tools/khmer' target='_blank'  rel='noopener noreferrer'>khmer<\/a>","<a href='https://bio.tools/kingfisher' target='_blank'  rel='noopener noreferrer'>kingfisher<\/a>","<a href='https://bio.tools/kmc' target='_blank'  rel='noopener noreferrer'>kmc<\/a>","<a href='https://bio.tools/KofamKOALA' target='_blank'  rel='noopener noreferrer'>KofamKOALA<\/a>","<a href='https://bio.tools/kraken' target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://bio.tools/kraken2' target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","<a href='https://bio.tools/krona' target='_blank'  rel='noopener noreferrer'>krona<\/a>","<a href='https://bio.tools/lastz' target='_blank'  rel='noopener noreferrer'>lastz<\/a>","<a href='https://bio.tools/LFQ-Analyst' target='_blank'  rel='noopener noreferrer'>LFQ-Analyst<\/a>","","<a href='https://bio.tools/links' target='_blank'  rel='noopener noreferrer'>links<\/a>","<a href='https://bio.tools/lipid_maps' target='_blank'  rel='noopener noreferrer'>lipid_maps<\/a>","<a href='https://bio.tools/longranger' target='_blank'  rel='noopener noreferrer'>longranger<\/a>","","<a href='https://bio.tools/mach' target='_blank'  rel='noopener noreferrer'>mach<\/a>","<a href='https://bio.tools/macs' target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='https://bio.tools/macs' target='_blank'  rel='noopener noreferrer'>macs<\/a>","","<a href='https://bio.tools/MAFFT' target='_blank'  rel='noopener noreferrer'>MAFFT<\/a>","<a href='https://bio.tools/magpy' target='_blank'  rel='noopener noreferrer'>magpy<\/a>","<a href='https://bio.tools/maker' target='_blank'  rel='noopener noreferrer'>maker<\/a>","<a href='https://bio.tools/maker-p' target='_blank'  rel='noopener noreferrer'>maker-p<\/a>","<a href='https://bio.tools/maldi_quant' target='_blank'  rel='noopener noreferrer'>maldi_quant<\/a>","<a href='https://bio.tools/mash' target='_blank'  rel='noopener noreferrer'>mash<\/a>","<a href='https://bio.tools/mashmap' target='_blank'  rel='noopener noreferrer'>mashmap<\/a>","<a href='https://bio.tools/maskrc-svg' target='_blank'  rel='noopener noreferrer'>maskrc-svg<\/a>","<a href='https://bio.tools/masurca' target='_blank'  rel='noopener noreferrer'>masurca<\/a>","<a href='https://bio.tools/mauve' target='_blank'  rel='noopener noreferrer'>mauve<\/a>","<a href='https://bio.tools/maxbin' target='_blank'  rel='noopener noreferrer'>maxbin<\/a>","<a href='https://bio.tools/maxquant' target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://bio.tools/maxquant' target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://bio.tools/mcl' target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://bio.tools/medaka' target='_blank'  rel='noopener noreferrer'>medaka<\/a>","<a href='https://bio.tools/megahit' target='_blank'  rel='noopener noreferrer'>megahit<\/a>","<a href='https://bio.tools/megan' target='_blank'  rel='noopener noreferrer'>megan<\/a>","<a href='https://bio.tools/merqury' target='_blank'  rel='noopener noreferrer'>merqury<\/a>","","<a href='https://bio.tools/idba' target='_blank'  rel='noopener noreferrer'>idba<\/a>","<a href='https://bio.tools/MetaBAT_2' target='_blank'  rel='noopener noreferrer'>MetaBAT_2<\/a>","<a href='https://bio.tools/metaboanalyst' target='_blank'  rel='noopener noreferrer'>metaboanalyst<\/a>","<a href='https://bio.tools/metacluster' target='_blank'  rel='noopener noreferrer'>metacluster<\/a>","<a href='https://bio.tools/metacv' target='_blank'  rel='noopener noreferrer'>metacv<\/a>","<a href='https://bio.tools/MetaDEGalaxy' target='_blank'  rel='noopener noreferrer'>MetaDEGalaxy<\/a>","<a href='https://bio.tools/metageneannotator' target='_blank'  rel='noopener noreferrer'>metageneannotator<\/a>","","<a href='https://bio.tools/metagenomeseq' target='_blank'  rel='noopener noreferrer'>metagenomeseq<\/a>","<a href='https://bio.tools/metamaps' target='_blank'  rel='noopener noreferrer'>metamaps<\/a>","<a href='https://bio.tools/metams' target='_blank'  rel='noopener noreferrer'>metams<\/a>","<a href='https://bio.tools/metapath' target='_blank'  rel='noopener noreferrer'>metapath<\/a>","<a href='https://bio.tools/metaphlan' target='_blank'  rel='noopener noreferrer'>metaphlan<\/a>","<a href='https://bio.tools/metaphyler' target='_blank'  rel='noopener noreferrer'>metaphyler<\/a>","<a href='https://bio.tools/metaQuantome' target='_blank'  rel='noopener noreferrer'>metaQuantome<\/a>","<a href='https://bio.tools/metaspades' target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","<a href='https://bio.tools/metavelvet' target='_blank'  rel='noopener noreferrer'>metavelvet<\/a>","<a href='https://bio.tools/MetaWRAP' target='_blank'  rel='noopener noreferrer'>MetaWRAP<\/a>","<a href='https://bio.tools/mg-rast' target='_blank'  rel='noopener noreferrer'>mg-rast<\/a>","<a href='https://bio.tools/MGnify' target='_blank'  rel='noopener noreferrer'>MGnify<\/a>","<a href='https://bio.tools/MIGRATE' target='_blank'  rel='noopener noreferrer'>MIGRATE<\/a>","","<a href='https://bio.tools/minia' target='_blank'  rel='noopener noreferrer'>minia<\/a>","<a href='https://bio.tools/minimac4' target='_blank'  rel='noopener noreferrer'>minimac4<\/a>","<a href='https://bio.tools/minimap2' target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","<a href='https://bio.tools/minpath' target='_blank'  rel='noopener noreferrer'>minpath<\/a>","<a href='https://bio.tools/mira' target='_blank'  rel='noopener noreferrer'>mira<\/a>","","<a href='https://bio.tools/mirdeep' target='_blank'  rel='noopener noreferrer'>mirdeep<\/a>","<a href='https://bio.tools/mirdeep2' target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","<a href='https://bio.tools/miso' target='_blank'  rel='noopener noreferrer'>miso<\/a>","<a href='https://bio.tools/mixomics' target='_blank'  rel='noopener noreferrer'>mixomics<\/a>","<a href='https://bio.tools/mlst' target='_blank'  rel='noopener noreferrer'>mlst<\/a>","<a href='https://bio.tools/mocat' target='_blank'  rel='noopener noreferrer'>mocat<\/a>","<a href='https://bio.tools/moff' target='_blank'  rel='noopener noreferrer'>moff<\/a>","<a href='https://bio.tools/morpheus' target='_blank'  rel='noopener noreferrer'>morpheus<\/a>","<a href='https://bio.tools/Mothur' target='_blank'  rel='noopener noreferrer'>Mothur<\/a>","","<a href='https://bio.tools/mrbayes' target='_blank'  rel='noopener noreferrer'>mrbayes<\/a>","<a href='https://bio.tools/ms_dial' target='_blank'  rel='noopener noreferrer'>ms_dial<\/a>","<a href='https://bio.tools/msgfplus' target='_blank'  rel='noopener noreferrer'>msgfplus<\/a>","<a href='https://bio.tools/msnbase' target='_blank'  rel='noopener noreferrer'>msnbase<\/a>","<a href='https://bio.tools/msstats' target='_blank'  rel='noopener noreferrer'>msstats<\/a>","<a href='https://bio.tools/multiqc' target='_blank'  rel='noopener noreferrer'>multiqc<\/a>","<a href='https://bio.tools/multisnv' target='_blank'  rel='noopener noreferrer'>multisnv<\/a>","<a href='https://bio.tools/mummer' target='_blank'  rel='noopener noreferrer'>mummer<\/a>","<a href='https://bio.tools/muscle' target='_blank'  rel='noopener noreferrer'>muscle<\/a>","","<a href='https://bio.tools/nanopolish' target='_blank'  rel='noopener noreferrer'>nanopolish<\/a>","<a href='https://bio.tools/nast-ier' target='_blank'  rel='noopener noreferrer'>nast-ier<\/a>","<a href='https://bio.tools/nccmp' target='_blank'  rel='noopener noreferrer'>nccmp<\/a>","","","<a href='https://bio.tools/nco' target='_blank'  rel='noopener noreferrer'>nco<\/a>","<a href='https://bio.tools/ncview' target='_blank'  rel='noopener noreferrer'>ncview<\/a>","<a href='https://bio.tools/necat' target='_blank'  rel='noopener noreferrer'>necat<\/a>","<a href='https://bio.tools/netcdf' target='_blank'  rel='noopener noreferrer'>netcdf<\/a>","<a href='https://bio.tools/neurodebian' target='_blank'  rel='noopener noreferrer'>neurodebian<\/a>","<a href='https://bio.tools/nextdenovo' target='_blank'  rel='noopener noreferrer'>nextdenovo<\/a>","","<a href='https://bio.tools/nextgenmap' target='_blank'  rel='noopener noreferrer'>nextgenmap<\/a>","<a href='https://bio.tools/NextPolish' target='_blank'  rel='noopener noreferrer'>NextPolish<\/a>","","<a href='https://bio.tools/ngsplot' target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","","","<a href='https://bio.tools/nvbio' target='_blank'  rel='noopener noreferrer'>nvbio<\/a>","<a href='https://bio.tools/oases' target='_blank'  rel='noopener noreferrer'>oases<\/a>","<a href='https://bio.tools/obitools' target='_blank'  rel='noopener noreferrer'>obitools<\/a>","<a href='https://bio.tools/openms' target='_blank'  rel='noopener noreferrer'>openms<\/a>","","","<a href='https://bio.tools/OTUreporter' target='_blank'  rel='noopener noreferrer'>OTUreporter<\/a>","<a href='https://bio.tools/pepper-deepvariant' target='_blank'  rel='noopener noreferrer'>pepper-deepvariant<\/a>","<a href='https://bio.tools/pacbioeda' target='_blank'  rel='noopener noreferrer'>pacbioeda<\/a>","","","<a href='https://bio.tools/pandaseq' target='_blank'  rel='noopener noreferrer'>pandaseq<\/a>","","<a href='https://bio.tools/paraview' target='_blank'  rel='noopener noreferrer'>paraview<\/a>","<a href='https://bio.tools/parsnp' target='_blank'  rel='noopener noreferrer'>parsnp<\/a>","<a href='https://bio.tools/PASA' target='_blank'  rel='noopener noreferrer'>PASA<\/a>","<a href='https://bio.tools/pathview' target='_blank'  rel='noopener noreferrer'>pathview<\/a>","<a href='https://bio.tools/pear' target='_blank'  rel='noopener noreferrer'>pear<\/a>","<a href='https://bio.tools/peptideshaker' target='_blank'  rel='noopener noreferrer'>peptideshaker<\/a>","<a href='https://bio.tools/percolator' target='_blank'  rel='noopener noreferrer'>percolator<\/a>","","<a href='https://bio.tools/phyloseq' target='_blank'  rel='noopener noreferrer'>phyloseq<\/a>","<a href='https://bio.tools/phylosift' target='_blank'  rel='noopener noreferrer'>phylosift<\/a>","<a href='https://bio.tools/phyml' target='_blank'  rel='noopener noreferrer'>phyml<\/a>","","<a href='https://bio.tools/picard_tools' target='_blank'  rel='noopener noreferrer'>picard_tools<\/a>","<a href='https://bio.tools/picrust' target='_blank'  rel='noopener noreferrer'>picrust<\/a>","<a href='https://bio.tools/picrust2' target='_blank'  rel='noopener noreferrer'>picrust2<\/a>","<a href='https://bio.tools/piler-df' target='_blank'  rel='noopener noreferrer'>piler-df<\/a>","<a href='https://bio.tools/pilon' target='_blank'  rel='noopener noreferrer'>pilon<\/a>","<a href='https://bio.tools/plink' target='_blank'  rel='noopener noreferrer'>plink<\/a>","<a href='https://bio.tools/plink-seq' target='_blank'  rel='noopener noreferrer'>plink-seq<\/a>","<a href='https://bio.tools/pnetcdf' target='_blank'  rel='noopener noreferrer'>pnetcdf<\/a>","<a href='https://bio.tools/clstutils' target='_blank'  rel='noopener noreferrer'>clstutils<\/a>","<a href='https://bio.tools/pretextview' target='_blank'  rel='noopener noreferrer'>pretextview<\/a>","<a href='https://bio.tools/prinseq' target='_blank'  rel='noopener noreferrer'>prinseq<\/a>","","<a href='https://bio.tools/probmetab' target='_blank'  rel='noopener noreferrer'>probmetab<\/a>","<a href='https://bio.tools/gmv' target='_blank'  rel='noopener noreferrer'>gmv<\/a>","<a href='https://bio.tools/prokka' target='_blank'  rel='noopener noreferrer'>prokka<\/a>","<a href='https://bio.tools/proovread' target='_blank'  rel='noopener noreferrer'>proovread<\/a>","<a href='https://bio.tools/psortb' target='_blank'  rel='noopener noreferrer'>psortb<\/a>","<a href='https://bio.tools/psortm' target='_blank'  rel='noopener noreferrer'>psortm<\/a>","<a href='https://bio.tools/purge_dups' target='_blank'  rel='noopener noreferrer'>purge_dups<\/a>","<a href='https://bio.tools/purgehaplotigs' target='_blank'  rel='noopener noreferrer'>purgehaplotigs<\/a>","<a href='https://bio.tools/pyani' target='_blank'  rel='noopener noreferrer'>pyani<\/a>","<a href='https://bio.tools/pyrad' target='_blank'  rel='noopener noreferrer'>pyrad<\/a>","<a href='https://bio.tools/pysam' target='_blank'  rel='noopener noreferrer'>pysam<\/a>","<a href='https://bio.tools/pytorch' target='_blank'  rel='noopener noreferrer'>pytorch<\/a>","<a href='https://bio.tools/qiime' target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://bio.tools/qiime' target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://bio.tools/qiime2' target='_blank'  rel='noopener noreferrer'>qiime2<\/a>","<a href='https://bio.tools/quast' target='_blank'  rel='noopener noreferrer'>quast<\/a>","<a href='https://bio.tools/quickmerge' target='_blank'  rel='noopener noreferrer'>quickmerge<\/a>","<a href='https://bio.tools/r' target='_blank'  rel='noopener noreferrer'>r<\/a>","<a href='https://bio.tools/Racon' target='_blank'  rel='noopener noreferrer'>Racon<\/a>","<a href='https://bio.tools/rails_cobbler' target='_blank'  rel='noopener noreferrer'>rails_cobbler<\/a>","<a href='https://bio.tools/ratt' target='_blank'  rel='noopener noreferrer'>ratt<\/a>","<a href='https://bio.tools/rattle' target='_blank'  rel='noopener noreferrer'>rattle<\/a>","<a href='https://bio.tools/raven' target='_blank'  rel='noopener noreferrer'>raven<\/a>","<a href='https://bio.tools/rawtools' target='_blank'  rel='noopener noreferrer'>rawtools<\/a>","<a href='https://bio.tools/raxml' target='_blank'  rel='noopener noreferrer'>raxml<\/a>","","<a href='https://bio.tools/refinem' target='_blank'  rel='noopener noreferrer'>refinem<\/a>","<a href='https://bio.tools/repdenovo' target='_blank'  rel='noopener noreferrer'>repdenovo<\/a>","<a href='https://bio.tools/repeatmasker' target='_blank'  rel='noopener noreferrer'>repeatmasker<\/a>","<a href='https://bio.tools/repeatmodeler' target='_blank'  rel='noopener noreferrer'>repeatmodeler<\/a>","<a href='https://bio.tools/repeatmodeler2' target='_blank'  rel='noopener noreferrer'>repeatmodeler2<\/a>","<a href='https://bio.tools/repeatrunner' target='_blank'  rel='noopener noreferrer'>repeatrunner<\/a>","<a href='https://bio.tools/repet' target='_blank'  rel='noopener noreferrer'>repet<\/a>","<a href='https://bio.tools/rjags' target='_blank'  rel='noopener noreferrer'>rjags<\/a>","","<a href='https://bio.tools/rsem' target='_blank'  rel='noopener noreferrer'>rsem<\/a>","<a href='https://bio.tools/rseqc' target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","","<a href='https://bio.tools/saige' target='_blank'  rel='noopener noreferrer'>saige<\/a>","","<a href='https://bio.tools/SALSA' target='_blank'  rel='noopener noreferrer'>SALSA<\/a>","<a href='https://bio.tools/samblaster' target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='https://bio.tools/samtools' target='_blank'  rel='noopener noreferrer'>samtools<\/a>","<a href='https://bio.tools/sas-access' target='_blank'  rel='noopener noreferrer'>sas-access<\/a>","<a href='https://bio.tools/scarpa' target='_blank'  rel='noopener noreferrer'>scarpa<\/a>","","<a href='https://bio.tools/screed' target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='https://bio.tools/searchgui' target='_blank'  rel='noopener noreferrer'>searchgui<\/a>","","<a href='https://bio.tools/seqstats' target='_blank'  rel='noopener noreferrer'>seqstats<\/a>","<a href='https://bio.tools/seqtk-trinity' target='_blank'  rel='noopener noreferrer'>seqtk-trinity<\/a>","<a href='https://bio.tools/shapeit' target='_blank'  rel='noopener noreferrer'>shapeit<\/a>","","<a href='https://bio.tools/sickle' target='_blank'  rel='noopener noreferrer'>sickle<\/a>","<a href='https://bio.tools/simtext' target='_blank'  rel='noopener noreferrer'>simtext<\/a>","<a href='https://bio.tools/skewer' target='_blank'  rel='noopener noreferrer'>skewer<\/a>","<a href='https://bio.tools/smrt_analysis' target='_blank'  rel='noopener noreferrer'>smrt_analysis<\/a>","<a href='https://bio.tools/snakemake' target='_blank'  rel='noopener noreferrer'>snakemake<\/a>","<a href='https://bio.tools/snap' target='_blank'  rel='noopener noreferrer'>snap<\/a>","<a href='https://bio.tools/SNAPPy' target='_blank'  rel='noopener noreferrer'>SNAPPy<\/a>","<a href='https://bio.tools/sniffles' target='_blank'  rel='noopener noreferrer'>sniffles<\/a>","<a href='https://bio.tools/snoscan' target='_blank'  rel='noopener noreferrer'>snoscan<\/a>","<a href='https://bio.tools/snpeff' target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","<a href='https://bio.tools/soap' target='_blank'  rel='noopener noreferrer'>soap<\/a>","<a href='https://bio.tools/soap3-gpu' target='_blank'  rel='noopener noreferrer'>soap3-gpu<\/a>","<a href='https://bio.tools/soapdenovo2' target='_blank'  rel='noopener noreferrer'>soapdenovo2<\/a>","<a href='https://bio.tools/soapsnp' target='_blank'  rel='noopener noreferrer'>soapsnp<\/a>","<a href='https://bio.tools/sortmerna' target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='https://bio.tools/spades' target='_blank'  rel='noopener noreferrer'>spades<\/a>","<a href='https://bio.tools/splign' target='_blank'  rel='noopener noreferrer'>splign<\/a>","<a href='https://bio.tools/squeezemeta' target='_blank'  rel='noopener noreferrer'>squeezemeta<\/a>","<a href='https://bio.tools/sra-tools' target='_blank'  rel='noopener noreferrer'>sra-tools<\/a>","<a href='https://bio.tools/stacks' target='_blank'  rel='noopener noreferrer'>stacks<\/a>","<a href='https://bio.tools/star' target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='https://bio.tools/structure' target='_blank'  rel='noopener noreferrer'>structure<\/a>","<a href='https://bio.tools/super-focus' target='_blank'  rel='noopener noreferrer'>super-focus<\/a>","<a href='https://bio.tools/supernova' target='_blank'  rel='noopener noreferrer'>supernova<\/a>","<a href='https://bio.tools/SVJedi' target='_blank'  rel='noopener noreferrer'>SVJedi<\/a>","<a href='https://bio.tools/svtyper' target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","<a href='https://bio.tools/swathxtend' target='_blank'  rel='noopener noreferrer'>swathxtend<\/a>","<a href='https://bio.tools/tagcleaner' target='_blank'  rel='noopener noreferrer'>tagcleaner<\/a>","<a href='https://bio.tools/tagdust' target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://bio.tools/targetfinder.org' target='_blank'  rel='noopener noreferrer'>targetfinder.org<\/a>","<a href='https://bio.tools/tassel' target='_blank'  rel='noopener noreferrer'>tassel<\/a>","","<a href='https://bio.tools/TensorFlow' target='_blank'  rel='noopener noreferrer'>TensorFlow<\/a>","<a href='https://bio.tools/ThermoRawFileParser' target='_blank'  rel='noopener noreferrer'>ThermoRawFileParser<\/a>","<a href='https://bio.tools/tigmint' target='_blank'  rel='noopener noreferrer'>tigmint<\/a>","<a href='https://bio.tools/tophat' target='_blank'  rel='noopener noreferrer'>tophat<\/a>","<a href='https://bio.tools/totalview' target='_blank'  rel='noopener noreferrer'>totalview<\/a>","<a href='https://bio.tools/TransDecoder' target='_blank'  rel='noopener noreferrer'>TransDecoder<\/a>","<a href='https://bio.tools/trf' target='_blank'  rel='noopener noreferrer'>trf<\/a>","<a href='https://bio.tools/trim_galore' target='_blank'  rel='noopener noreferrer'>trim_galore<\/a>","<a href='https://bio.tools/trimmomatic' target='_blank'  rel='noopener noreferrer'>trimmomatic<\/a>","<a href='https://bio.tools/trinity' target='_blank'  rel='noopener noreferrer'>trinity<\/a>","<a href='https://bio.tools/trinotate' target='_blank'  rel='noopener noreferrer'>trinotate<\/a>","<a href='https://bio.tools/trnascan-se' target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","","<a href='https://bio.tools/UCSC_Genome_Browser_Utilities' target='_blank'  rel='noopener noreferrer'>UCSC_Genome_Browser_Utilities<\/a>","<a href='https://bio.tools/unicycler' target='_blank'  rel='noopener noreferrer'>unicycler<\/a>","<a href='https://bio.tools/usearch' target='_blank'  rel='noopener noreferrer'>usearch<\/a>","<a href='https://bio.tools/valet' target='_blank'  rel='noopener noreferrer'>valet<\/a>","<a href='https://bio.tools/vamps' target='_blank'  rel='noopener noreferrer'>vamps<\/a>","<a href='https://bio.tools/varscan' target='_blank'  rel='noopener noreferrer'>varscan<\/a>","<a href='https://bio.tools/vawk' target='_blank'  rel='noopener noreferrer'>vawk<\/a>","<a href='https://bio.tools/vcftools' target='_blank'  rel='noopener noreferrer'>vcftools<\/a>","<a href='https://bio.tools/velvet' target='_blank'  rel='noopener noreferrer'>velvet<\/a>","<a href='https://bio.tools/viennarna' target='_blank'  rel='noopener noreferrer'>viennarna<\/a>","<a href='https://bio.tools/visit' target='_blank'  rel='noopener noreferrer'>visit<\/a>","<a href='https://bio.tools/vsearch' target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","<a href='https://bio.tools/WENGAN' target='_blank'  rel='noopener noreferrer'>WENGAN<\/a>","<a href='https://bio.tools/windowmasker' target='_blank'  rel='noopener noreferrer'>windowmasker<\/a>","<a href='https://bio.tools/wise' target='_blank'  rel='noopener noreferrer'>wise<\/a>","<a href='https://bio.tools/workflow4metabolomics' target='_blank'  rel='noopener noreferrer'>workflow4metabolomics<\/a>","<a href='https://bio.tools/wtdbg2' target='_blank'  rel='noopener noreferrer'>wtdbg2<\/a>","","<a href='https://bio.tools/xcms' target='_blank'  rel='noopener noreferrer'>xcms<\/a>","","<a href='https://bio.tools/xtandem' target='_blank'  rel='noopener noreferrer'>xtandem<\/a>"],["3D-DNA","abricate","abyss","allhic","allmaps","allpathslg","amos","amr","anaconda","angsd","anvio","apollo","artemis","augustus","bamkit","bamm","bamtools","bbmap","bcftools","beagle","beagle-lib","beast","beast2","bedtools","bellerophon","binsanity","bio-roll","bioconductor","biolinux","Bionano Access","bioperl","biopython","bismark","blast","blast+","blast2","blastkoala","blat","BlobToolKit","blobtools","bmge","boost","bowtie","bowtie2","braker1","braker2","breakdancer","bsmap","bs_seeker","bsseeker2","busco","bwa","bx-python","cactus","calypso","camera","canu","cap3","cardinal","cat_bins","cdhit","celseq2","celera","CellProfiler","centrifuge","checkm","checkm-genome","checkv","chimeraslayer","chromeister","chunkchromosome","circexplorer","circlator","clustalw","clusteringsuite","cnvkit","cpc","cpc2","comparem","coverm","cpat","ctsm_fates","cuda","cudnn","cufflinks","cutadapt","cytoscape","dada2","dbcan","dbcan2","deeptools","deepvariant","dendropy","DIA-Umpire","diamond","discovar_de_novo","dms_tools2","dssp","dyninst","easybuild","edena","edirect","eggnog-mapper","emboss","encyclopedia","enrichm","ensembl","genebuild","eval","EvidenceModeler","exabayes","exonerate","export2graphlan","falcon","pb-assembly","fastml","fastp","fastqc","faststructure","biocontainers-faststructure","fastx_toolkit","fastx","fgenesh","fimpute","flash","flashlfq","flye","focus","fraggenescan","freebayes","freesurfer","funannotate","garli","gatb","gatk","gatk4","gblock","gcta","gecko","geneid","genemark","genomescope","GenomicConsensus","genomicfeatures","ghostkoala","gist","glimmer","glimmerhmm","glimmermg","gmap","biocontainers-gmap","gmap_gsnap","gnps","gnu","parallel","gsl","graftm","graphlan","gromacs","groopm","gtdbtk","gubbins","guppy","hadoop","haslr","hawkeye","hclust2","hdf4","hdf5","hifiasm","higlass","hirise","hisat2","hmmer","horovod","hpctoolkit","hpcviewer","htseq","htslib","humann","humann2","humann3","idba-ud","idl","imagemagick","img_m","pbipa","infernal","igv","intel-xed","interproscan","ipyrad","iq-tree","iqtree","isoseq3","iwtomics","jags","jamg","jbrowse","jcvi","jellyfish","jq","juicebox","juicer","kaas","kallisto","kat","khmer","kingfisher","kmc","KofamKOALA","kraken","kraken2","krona","lastz","LFQ-Analyst","libmonitor","links","lipid_maps","longranger","lsdyna","mach","macs","macs2","macse","mafft","magpy","maker","maker-p","maldi_quant","mash","mashmap","maskrc-svg","masurca","mauve","maxbin","maxquant","biocontainers-maxquant","mcl","medaka","megahit","megan","merqury","meryl","meta_idba","metabat","metaboanalyst","metacluster","metacv","MetaDEGalaxy","metageneannotator","metagenemark","metagenomeseq","metamaps","metams","metapath","metaphlan","metaphyler","metaQuantome","metaspades","metavelvet","MetaWRAP","mg-rast","MGnify","MIGRATE","mimodd","minia","minimac4","minimap2","minpath","mira","mircounts","mirdeep","mirdeep2","miso","mixomics","mlst","mocat","moff","morpheus","Mothur","mpiblast","mrbayes","ms_dial","msgfplus","msnbase","msstats","multiqc","multisnv","mummer","muscle","nag","nanopolish","nast-ier","nccmp","ncl","ncl_ncarg","nco","ncview","necat","netcdf","neurodebian","nextdenovo","nextflow","nextgenmap","nextpolish","ngs","ngsplot","NOVOPlasty","nucleoatac","nvbio","oases","obitools","openms","opera-ms","w4mcorcov","OTUreporter","pepper+deepvariant","pacbioeda","pairtools","pampa","pandaseq","pangolin","paraview","parsnp","PASA","pathview","pear","peptideshaker","percolator","php","phyloseq","phylosift","phyml","phymmbl","picard","picrust","picrust2","piler-df","pilon","plink","plinkseq","pnetcdf","pplacer","pretextview","prinseq","probabel","probmetab","prodigal","prokka","proovread","psortb","psortm","purge_dups","purgehaplotigs","pyani","pyrad","pysam","pytorch","qiime","qiime-modules","qiime2","quast","quickmerge","r","Racon","rails_cobbler","ratt","rattle","raven","rawtools","raxml","rclone","refinem","repdenovo","repeatmasker","repeatmodeler","repeatmodeler2","repeatrunner","repet","rjags","rmpi","rsem","rseqc","rstudio","saige","salmon","salsa","samblaster","samtools","sas","scarpa","scikit","screed","searchgui","biocontainers-seqkit","seqstats","seqtk-trinity","shapeit","shasta","sickle","simtext","skewer","smrtlink","snakemake","snap","snappy","sniffles","snoscan","snpeff","soap","soap3-gpu","soapdenovo2","soapsnp","sortmerna","spades","splign","squeezemeta","sra-tools","stacks","star","structure","super-focus","supernova","svjedi","svtyper","swathxtend","tagcleaner","tagdust","targetfinder","tassel","tax4fun2","tensorflow","ThermoRawFileParser","tigmint","tophat","totalview","transdecoder","trf","trim_galore","trimmomatic","trinity","trinotate","trnascan-se","trycycler","ucsc","unicycler","usearch","valet","vamps","varscan","vawk","vcftools","velvet","viennarna","visit","vsearch","wengan","windowmasker","wise","workflow4metabolomics","wtdbg2","xarray","xcms","xml4ena","xtandem"],["3D de novo assembly (3D-DNA) is a pipeline for de novo assembly using HiC.","Mass screening of contigs for antimicrobial resistance or virulence genes.","De novo genome sequence assembler.","","","","AMOS is a Modular, Open-Source whole genome assembler.","An R Package for Working with Antimicrobial Resistance Data.","Software package specially developed for the study of genes’ primary structure. It uses gene sequences downloaded from public databases, as FASTA and GenBank, and it applies a set of statistical and visualization methods in different ways, to reveal information about codon context, codon usage, nucleotide repeats within open reading frames (ORFeome) and others.","Software for analyzing next generation sequencing data. The software can handle a number of different input types from mapped reads to imputed genotype probabilities. Most methods take genotype uncertainty into account instead of basing the analysis on called genotypes. This is especially useful for low and medium depth data.","Anvi’o is an open-source, community-driven analysis and visualization platform for microbial ‘omics. It brings together many aspects of today’s cutting-edge strategies including genomics, metagenomics, metatranscriptomics, pangenomics, metapangenomics, phylogenomics, and microbial population genetics in an integrated and easy-to-use fashion through extensive interactive visualization capabilities.","Apollo is a genome annotation viewer and editor. Apollo allows researchers to explore genomic annotations at many levels of detail, and to perform expert annotation curation, all in a graphical environment.","Free genome browser and annotation tool that allows visualisation of sequence features, next generation data and the results of analyses within the context of the sequence, and also its six-frame translation.","AUGUSTUS is a eukaryotic gene prediction tool. It can integrate evidence, e.g. from RNA-Seq, ESTs, proteomics, but can also predict genes ab initio. The PPX extension to AUGUSTUS can take a protein sequence multiple sequence alignment as input to find new members of the family in a genome. It can be run through a web interface (see https://bio.tools/webaugustus), or downloaded and run locally.","Bamkit is a collection of tools for common BAM file manipulations","de-novo motif discovery and regulatory sequence analysis.\nDiscovery of regulatory motifs with higher-order Bayesian Markov Models (BaMMs)","BamTools provides a fast, flexible C++ API & toolkit for reading, writing, and managing BAM files.","BBMap is a fast splice-aware aligner for RNA and DNA.  It is faster than almost all short-read aligners, yet retains unrivaled sensitivity and specificity, particularly for reads with many errors and indels.","BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.","Beagle is a software package that performs genotype calling, genotype phasing, imputation of ungenotyped markers, and identity-by-descent segment detection.","BEAGLE is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages.","The Bayesian Evolutionary Analysis Sampling Trees is a cross-platform program for Bayesian analysis of molecular sequences using MCMC (Markov chain Monte Carlo). It is entirely orientated towards rooted, time-measured phylogenies inferred using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology.","Bayesian phylogenetic analysis of molecular sequences. It estimates rooted, time-measured phylogenies using strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies but is also a framework for testing evolutionary hypotheses without conditioning on a single tree topology. It uses Markov chain Monte Carlo (MCMC) to average over tree space, so that each tree is weighted proportional to its posterior probability. It includes a graphical user-interface for setting up standard analyses and a suit of programs for analysing the results.","BEDTools is an extensive suite of utilities for comparing genomic features in BED format.","","Unsupervised Clustering of Environmental Microbial Assemblies Using Coverage and Affinity Propagation","","Open development software project, based on the R programming language, providing tools for the analysis of high-throughput genomic data. The project aims to enable interdisciplinary research, collaboration and rapid development of scientific software.","","","A collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications.  It provides software modules for many of the typical tasks of bioinformatics programming.","","Bismark is a tool to map bisulfite treated sequencing reads and perform methylation calling in a quick and easy-to-use fashion.","A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.","A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.","A tool that finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.","BlastKOALA (KEGG Orthology And Links Annotation) is a KEGG Web service for annotating genomes and understanding organisms.","Fast, accurate spliced alignment of DNA sequences.","BlobToolKit Viewer is a genome-scale dataset visualistion tool developed as part of the blobtoolkit project to allow browser-based identification and filtering of target and non-target data in genome assemblies.","Visualisation, quality control and taxonomic partitioning of genome datasets.","Block Mapping and Gathering using Entropy.","Boost is a set of libraries for the C++ programming language that provides support for tasks and structures such as linear algebra, pseudorandom number generation, multithreading, image processing, regular expressions, and unit testing.","Bowtie is an ultrafast, memory-efficient short read aligner.","Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.","Pipeline for unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS.","Pipeline for unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS.","Application for detecting structural rearrangements and indels in short read sequencing data.","short reads mapping software for bisulfite sequencing","Mapping tool for bisulfite treated reads.","A versatile aligning pipeline for bisulfite sequencing data.","Provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.","Fast, accurate, memory-efficient aligner for short and long sequencing reads","Tools for manipulating biological data, particularly multiple sequence alignments.","Cactus is a reference-free whole-genome multiple alignment program.","","Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, isotope validation, EIC correlation based tagging of unknown adducts and fragments.","De-novo assembly tool for long read chemistry like Nanopore data and PacBio data.","Web-based contig assembly.","Implements statistical and computational tools for analyzing mass spectrometry imaging datasets, including methods for efficient pre-processing, spatial segmentation, and classification.","Contig Annotation Tool (CAT) and Bin Annotation Tool (BAT) are pipelines for the taxonomic classification of long DNA sequences and metagenome assembled genomes (MAGs/bins) of both known and (highly) unknown microorganisms, as generated by contemporary metagenomics studies. The core algorithm of both programs involves gene calling, mapping of predicted ORFs against the nr protein database, and voting-based classification of the entire contig / MAG based on classification of the individual ORFs.","Cluster a nucleotide dataset into representative sequences.","celseq2 is a Python framework for generating the UMI count matrix from CEL-Seq2 sequencing data.","Celera Assembler is scientific software for DNA research.","Tool for quantifying data from biological images, particularly in high-throughput experiments.","A very rapid and memory-efficient system for the classification of DNA sequences from microbial samples. The system uses a novel indexing scheme based on the Burrows-Wheeler transform and the Ferragina-Manzini index, optimized specifically for the metagenomic classification problem. Together these advances enable timely and accurate analysis of large metagenomics data sets on conventional desktop computers.","CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.","CheckM provides a set of tools for assessing the quality of genomes recovered from isolates, single cells, or metagenomes.","CheckV is a fully automated command-line pipeline for assessing the quality of single-contig viral genomes, including identification of host contamination for integrated proviruses, estimating completeness for genome fragments, and identification of closed genomes.","A chimeric sequence detection utility, compatible with near-full length Sanger sequences and shorter 454-FLX sequences. It involves the different series of steps that operate to flag chimeric 16S rRNA sequences.To run it, you need NAST-formatted sequences generated by the included NAST-iEr utility.","","ChunkChromosome is a helper utility for minimac and MaCH. It can be used to facilitate analyses of very large datasets in overlapping slices.","","","Multiple sequence alignment software. Old deprecated versions. Even older versions were CLUSTAL and CLUSTAL V (ClustalV).","The ClusteringSuite is the software package that includes the cluster analysis tool.","CNVkit is a software toolkit to infer and visualize copy number from targeted DNA sequencing data.","Coding Potential Calculator (CPC) is a support vector machine-based classifier to assess the protein-coding potential of a transcript based on six biologically meaningful sequence features.","Coding Potential Calculator. Fast and accurate coding potential calculator based on sequence intrinsic features.","CompareM i a toolbox for comparative genomics.","Read coverage calculator for metagenomics","CPAT (Coding-Potential Assessment Tool) is a logistic regression model-based classifier that can accurately and quickly distinguish protein-coding and noncoding RNAs using pure linguistic features calculated from the RNA sequences. CPAT takes as input the nucleotides sequences or genomic coordinates of RNAs and outputs the probabilities p (0 ≤ p ≤ 1), which measure the likelihood of protein coding.","","","","Cufflinks assembles transcripts and estimates their abundances in RNA-Seq samples. It accepts aligned RNA-Seq reads and assembles the alignments into a parsimonious set of transcripts. Cufflinks then estimates the relative abundances of these transcripts based on how many reads support each one.","Find and remove adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.","Open source software platform for visualizing molecular interaction networks and biological pathways and integrating these networks with annotations, gene expression profiles and other state data.","This package infers exact sequence variants (SVs) from amplicon data, replacing the commonly used and coarser OTU clustering approach. This pipeline inputs demultiplexed fastq files, and outputs the sequence variants and their sample-wise abundances after removing substitution and chimera errors. Taxonomic classification is available via a native implementation of the RDP naive Bayesian classifier.","dbCAN provides automated carbohydrate-active enzyme signature domain-based annotation for a set of proteins. Each carbohydrate-active enzyme has an assigned signature domain.","Automated Carbohydrate-active enzyme ANnotation. Three state-of-the-art tools for CAZome (all CAZymes of a genome) annotation: (i) HMMER search against the dbCAN HMM (hidden Markov model) database; (ii) DIAMOND search against the CAZy pre-annotated CAZyme sequence database and (iii) Hotpep search against the conserved CAZyme short peptide database.","User-friendly tools for the normalization and visualization of deep-sequencing data.","DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file.","DendroPy is a Python library for phylogenetic computing. It provides classes and functions for the simulation, processing, and manipulation of phylogenetic trees and character matrices, and supports the reading and writing of phylogenetic data in a range of formats.","DIA-Umpire is an open source Java program for computational analysis of data independent acquisition (DIA) mass spectrometry-based proteomics data. It enables untargeted peptide and protein identification and quantitation using DIA data, and also incorporates targeted extraction to reduce the number of cases of missing quantitation.","Sequence aligner for protein and translated DNA searches and functions as a drop-in replacement for the NCBI BLAST software tools. It is suitable for protein-protein search as well as DNA-protein search on short reads and longer sequences including contigs and assemblies, providing a speedup of BLAST ranging up to x20,000.","Genome assembler designed for state-of-the-art data. It can generate de novo assemblies for both large and small genomes. It currently does not call variants.","dms_tools2 is a software package for analyzing deep mutational scanning data. It is tailored to analyze libraries created using comprehensive codon mutagenesis of protein-coding genes","DSSP secondary structure database . It produces secondary structure assignments (and much more) for all protein entries in the Protein Data Bank (PDB).","","EasyBuild is a software build and installation framework that allows you to manage (scientific) software on High Performance Computing (HPC) systems in an efficient way.","A program for assembling de novo bacterial genomes from short reads produced by Illumina sequencing platform.","","For fast functional annotation of novel sequences. It uses precomputed orthologous groups and phylogenies from the eggNOG database to transfer functional information from fine-grained orthologs only. Its common uses include the annotation of novel genomes, transcriptomes or even metagenomic gene catalogs.\n\nThe use of orthology predictions for functional annotation permits a higher precision than traditional homology searches, as it avoids transferring annotations from close paralogs.","Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.","EncyclopeDIA is library search engine comprised of several algorithms for DIA data analysis and can search for peptides using either DDA-based spectrum libraries or DIA-based chromatogram libraries.","","Genome databases for vertebrates genomes.","","Eval is a flexible tool for analyzing the performance of gene-structure prediction programs. It provides summaries and graphical distributions for many statistics describing any set of annotations, regardless of their source. It also compares sets of predictions to standard annotations and to one another.","The EVidenceModeler (aka EVM) software combines ab intio gene predictions and protein and transcript alignments into weighted consensus gene structures. EVM provides a flexible and intuitive framework for combining diverse evidence types into a single automated gene structure annotation system.","ExaBayes is a software package for Bayesian phylogenetic tree inference. It is particularly suitable for large-scale analyses on computer clusters.","A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.","export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn. In particular, the annotation file tries to highlight specific sub-trees deriving automatically from input file what nodes are important.","Experimental PacBio diploid assembler.","Experimental PacBio diploid assembler.","FastML is a webserver for computing and reconstructing ancestral sequences using maximum likelihood.","A tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.","This tool aims to provide a QC report which can spot problems or biases which originate either in the sequencer or in the starting library material. It can be run in one of two modes. It can either run as a stand alone interactive application for the immediate analysis of small numbers of FastQ files, or it can be run in a non-interactive mode where it would be suitable for integrating into a larger analysis pipeline for the systematic processing of large numbers of files.","fastStructure is an algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x.","fastStructure is an algorithm for inferring population structure from large SNP genotype data. It is based on a variational Bayesian framework for posterior inference and is written in Python2.x.","Collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.","Collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.","HMM-based gene structure prediction (multiple genes, both chains); Program for predicting multiple genes in genomic DNA sequences.","Developed for large scale genotype imputation in livestock where hundreds of thousands of individuals are genotypes with different panels. It uses an overlapping sliding window approach to exploit relationships or haplotype similarities between target and reference individuals. The process starts with long windows to capture haplotype similarity between close relatives.","Identifies paired-end reads which overlap in the middle, converting them to single long reads","FlashLFQ is an ultrafast label-free quantification algorithm for mass-spectrometry proteomics.","Flye is a de novo assembler for single molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. It is designed for a wide range of datasets, from small bacterial projects to large mammalian-scale assemblies. The package represents a complete pipeline: it takes raw PB / ONT reads as input and outputs polished contigs.","FOCUS (Find Organisms by Composition Usage) is an agile profiler for metagenomic data.","Application for finding (fragmented) genes in short reads","Bayesian genetic variant detector designed to find small polymorphisms, specifically SNPs, indels, multi-nucleotide polymorphisms, and complex events (composite insertion and substitution events) smaller than the length of a short-read sequencing alignment.","Wrapper functions that interface with 'Freesurfer', a powerful and commonly-used 'neuroimaging' software, using system commands. The goal is to be able to interface with 'Freesurfer' completely in R, where you pass R objects of class 'nifti', implemented by package 'oro.nifti', and the function executes an 'Freesurfer' command and returns an R object of class 'nifti' or necessary output.","funannotate is a pipeline for genome annotation (built specifically for fungi, but will also work with higher eukaryotes).","GARLI (Genetic Algorithm for Rapid Likelihood Inference) is a program for inferring phylogenetic trees. Using an approach similar to a classical genetic algorithm, it rapidly searches the space of evolutionary trees and model parameters to find the solution maximizing the likelihood score. It implements nucleotide, amino acid and codon-based models of sequence evolution, and runs on all platforms.","C++ library of high level functions designed to benefit from recent advances in NGS data structures for handling huge datasets of sequencing data.","The Genome Analysis Toolkit (GATK) is a set of bioinformatic tools for analyzing high-throughput sequencing (HTS) and variant call format (VCF) data. The toolkit is well established for germline short variant discovery from whole genome and exome sequencing data. GATK4 expands functionality into copy number and somatic analyses and offers pipeline scripts for workflows.  \n\nVersion 4 (GATK4) is open-source at https://github.com/broadinstitute/gatk.","","Cleaning aligned sequences.","Genome-wide Complex Trait Analysis. Estimate the proportion of phenotypic variance explained by genome- or chromosome-wide SNPs for complex traits (the GREML method), and has subsequently extended for many other analyses to better understand the genetic architecture of complex traits.","Software aimed at pairwise sequence comparison generating high quality results (equivalent to MUMmer) with controlled memory consumption and comparable or faster execution times particularly with long sequences.","Ab initio gene finding program used to predict genes along DNA sequences in a large set of organisms.","A family of gene prediction programs: Gene Prediction in Bacteria, Archaea, Metagenomes and Metatranscriptomes; Gene Prediction in Eukaryotes; Gene Prediction in Transcripts; Gene Prediction in Viruses, Phages and Plasmids.","Reference-free profiling of polyploid genomes | We have developed GenomeScope 2.0, which applies classical insights from combinatorial theory to establish a detailed mathematical model of how k-mer frequencies will be distributed in heterozygous and polyploid genomes | Average k-mer coverage for polyploid genome | Upload results from running Jellyfish or KMC","","A set of tools and methods for making and manipulating transcript centric annotations. With these tools the user can easily download the genomic locations of the transcripts, exons and CDS of a given organism, from either the UCSC Genome Browser or a BioMart database. This information is then stored in a local database that keeps track of the relationship between transcripts, exons, CDS and genes.","GhostKOALA (KEGG Orthology And Links Annotation) is another service for annotating metagenomes and understanding ecosystems.","Pplatform for scientific users to predict GIs (Genomic Islands ). This software package includes five commonly used tools, AlienHunter, IslandPath, Colombo SIGI-HMM, INDeGenIUS and Pai-Ida. It also includes an optimization program EGID that ensembles the result of existing tools for more accurate prediction.","System for finding genes in microbial DNA, especially the genomes of bacteria, archaea, and viruses. It uses interpolated Markov models (IMMs) to identify the coding regions and distinguish them from noncoding DNA.","A new gene finder based on a Generalized Hidden Markov Model. Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models . Currently, GlimmerHMM's GHMM structure includes introns of each phase, intergenic regions, and four types of exons.","Glimmer-MG is a system for finding genes in environmental shotgun DNA\nsequences. Glimmer-MG (Gene Locator and Interpolated Markov ModelER -\nMetaGenomics) uses interpolated Markov models (IMMs) to identify the\ncoding regions and distinguish them from noncoding DNA. Licensed under Artistic License.","Genomic Mapping and Alignment Program for mRNA and EST Sequences.","Genomic Mapping and Alignment Program for mRNA and EST Sequences.","Genomic Mapping and Alignment Program for mRNA and EST Sequences.","GNPS is a web-based mass spectrometry ecosystem that aims to be an open-access knowledge base for community-wide organization and sharing of raw, processed, or annotated fragmentation mass spectrometry data (MS/MS). GNPS aids in identification and discovery throughout the entire life cycle of data; from initial data acquisition/analysis to post publication.","","GNU parallel is a shell tool for executing jobs in parallel using one or more computers. A job can be a single command or a small script that has to be run for each of the lines in the input. The typical input is a list of files, a list of hosts, a list of users, a list of URLs, or a list of tables. A job can also be a command that reads from a pipe. GNU parallel can then split the input and pipe it into commands in parallel.","The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.","Scalable, phylogenetically informed classification of genes within metagenomes.","GraPhlAn is a software tool for producing high-quality circular representations of taxonomic and phylogenetic trees. GraPhlAn focuses on concise, integrative, informative, and publication-ready representations of phylogenetically- and taxonomically-driven investigation.","Versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since it is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers.","GroopM is an automated tool for the recovery of population genomes from related metagenomes.","a toolkit to classify genomes with the Genome Taxonomy Database.\n\nGTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.\n\nGTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).","Gubbins is a tool for rapid phylogenetic analysis of large samples of recombinant bacterial whole genome sequences.","","The Apache Hadoop software library is a framework that allows for the distributed processing of large data sets across clusters of computers using simple programming models. It is designed to scale up from single servers to thousands of machines, each offering local computation and storage.","","An interactive visual analytics tool for genome assemblies.","Hclust2 is a handy tool for plotting heat-maps with several useful options to produce high quality figures that can be used in publication.","","","","","HiRise is a software pipeline that can identify poor quality joins and produce accurate, long-range sequence scaffolds.","Alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).","This tool is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models. The new HMMER3 project, HMMER is now as fast as BLAST for protein search.","Horovod is a distributed deep learning training framework for TensorFlow, Keras, PyTorch, and Apache MXNet. The goal of Horovod is to make distributed deep learning fast and easy to use.","","","Python framework to process and analyse high-throughput sequencing (HTS) data","The main purpose of HTSlib is to provide access to genomic information files, both alignment data (SAM, BAM, and CRAM formats) and variant data (VCF and BCF formats). The library also provides interfaces to access and index genome reference data in FASTA format and tab-delimited files with genomic coordinates. It is utilized and incorporated into both SAMtools and BCFtools.","","HUMAnN 2.0 is a pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data (typically millions of short DNA/RNA reads). This process, referred to as functional profiling, aims to describe the metabolic potential of a microbial community and its members. More generally, functional profiling answers the question “What are the microbes in my community-of-interest doing (or capable of doing)?”","","A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.","","","The integrated microbial genomes and metagenomes (IMG/M) system provides support for comparative analysis of microbial community aggregate genomes (metagenomes) in a comprehensive integrated context.","","","","","","","Very efficient phylogenetic software for reconstructing maximum-likelihood trees and assessing branch supports with the ultrafast bootstrap approximation. It is based on the IQPNNI algorithm with 10-fold speedup together with substantially additional features.","","IsoSeq v3 contains the newest tools to identify transcripts in PacBio single-molecule sequencing data. Starting in SMRT Link v6.0.0, those tools power the IsoSeq GUI-based analysis application. A composable workflow of existing tools and algorithms, combined with a new clustering technique.","Implementation of the Interval-Wise Testing (IWT) for omics data. This inferential procedure tests for differences in \"Omics\" data between two groups of genomic regions (or between a group of genomic regions and a reference center of symmetry), and does not require fixing location and scale at the outset.","JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation.","Just Annotate My Genome (JAMg) is focused on delivering a process for feature annotation for all the species that are not model species","Slick, speedy genome browser with a responsive and dynamic AJAX interface for visualization of genome data. Being developed by the GMOD project as a successor to GBrowse.","","A command-line algorithm for counting k-mers in DNA sequence.","","Juicebox is visualization software for Hi-C data.","Juicer is a platform for analyzing kilobase resolution Hi-C data. In this distribution, we include the pipeline for generating Hi-C maps from fastq raw data files and command line tools for feature annotation on the Hi-C maps.","Tool for ortholog assignment and pathway mapping. It provides functional annotation of genes by BLAST comparisons against the manually curated KEGG GENES database. The result contains KO (KEGG Orthology) assignments and automatically generated KEGG pathways.","A program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads. It is based on the novel idea of pseudoalignment for rapidly determining the compatibility of reads with targets, without the need for alignment.","Suite of tools that generate, analyse and compare k-mer spectra produced from sequence files","khmer is a set of command-line tools for working with DNA shotgun sequencing data from genomes, transcriptomes, metagenomes, and single cells. khmer can make de novo assemblies faster, and sometimes better. khmer can also identify (and fix) problems with shotgun data.","Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.","KMC is a utility designed for counting k-mers (sequences of consecutive k symbols) in a set of reads from genome sequencing projects.","KEGG ortholog assignment based on profile HMM and adaptive score threshold.\n\nKofamKOALA - KEGG Orthology Search.\n\nK number assignment based on KO-dependent scoring criteria.\n\nKofamKOALA assigns K numbers to the user's sequence data by HMMER/HMMSEARCH against KOfam (a customized HMM database of KEGG Orthologs (KOs)). K number assignments with scores above the predefined thresholds for individual KOs are more reliable than other proposed assignments. Such high score assignments are highlighted with asterisks '*' in the output. The K number assignments facilitate the interpretation of the annotation results by linking the user's sequence data to the KEGG pathways and EC numbers.","System for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies. Previous attempts by other bioinformatics software to accomplish this task have often used sequence alignment or machine learning techniques that were quite slow, leading to the development of less sensitive but much faster abundance estimation programs. It aims to achieve high sensitivity and high speed by utilizing exact alignments of k-mers and a novel classification algorithm.","Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.","Krona creates interactive HTML5 charts of hierarchical data (such as taxonomic abundance in a metagenome).","A tool for (1) aligning two DNA sequences, and (2) inferring appropriate scoring parameters automatically.","An Easy-To-Use Interactive Web Platform To Analyze and Visualize Label-Free Proteomics Data Preprocessed with MaxQuant.\n\nA tool for analysing label-free quantitative proteomics dataset https://bioinformatics.erc.monash.edu/apps/LFQ-Analyst/.\n\nLFQ-Analyst: An easy-to-use interactive web-platform to analyze and visualize proteomics data preprocessed with MaxQuant.\n\nLFQ-Analyst is an easy-to-use, interactive web application developed to perform differential expression analysis with “one click” and to visualize label-free quantitative proteomic datasets preprocessed with MaxQuant. LFQ-Analyst provides a wealth of user-analytic features and offers numerous publication-quality result output graphics and tables to facilitate statistical and exploratory analysis of label-free quantitative datasets","","LINKS (Long Interval Nucleotide K-mer Scaffolder) is a genomics application for scaffolding genome assemblies with long reads, such as those produced by Oxford Nanopore Technologies Ltd. It can be used to scaffold high-quality draft genome assemblies with any long sequences (eg. ONT reads, PacBio reads, other draft genomes, etc). It is also used to scaffold contig pairs linked by ARCS/ARKS.","LIPID Metabolites And Pathways Strategy (LIPID MAPS) is a consortium that presents a collection of tools for performing tasks such as drawing lipid structures and predicting possible structures from mass spectrometry data.","Long Ranger is a set of five analysis pipelines that processes Chromium sequencing output to align reads and call and phase SNPs, indels, and structural variants.","","Markov Chain based haplotyper that can resolve long haplotypes or infer missing genotypes in samples of unrelated individuals.","Model-based Analysis of ChIP-seq data.","Model-based Analysis of ChIP-seq data.","","MAFFT (Multiple Alignment using Fast Fourier Transform) is a high speed multiple sequence alignment program.","MAGpy is a Snakemake pipeline for downstream analysis of metagenome-assembled genomes (MAGs).","Portable and easily configurable genome annotation pipeline. It’s purpose is to allow smaller eukaryotic and prokaryotic genome projects to independently annotate their genomes and to create genome databases.","Pipeline designed to annotate novel plant genomes and is build on the MAKER annotation tool. The MAKER-P pipeline generates species-specific repeat libraries, as well as structural annotations of protein coding genes, non-coding RNAs, and pseudogenes.","MALDIquant is a complete analysis pipeline for matrix-assisted laser desorption/ionization-time-of-flight (MALDI-TOF) and other two-dimensional mass spectrometry data. In addition to commonly used plotting and processing methods it includes distinctive features, namely baseline subtraction methods such as morphological filters (TopHat) or the statistics-sensitive non-linear iterative peak-clipping algorithm (SNIP), peak alignment using warping functions, handling of replicated measurements as well as allowing spectra with different resolutions.","Fast genome and metagenome distance estimation using MinHash.","MashMap implements a fast and approximate algorithm for computing local alignment boundaries between long DNA sequences. It can be useful for mapping genome assembly or long reads (PacBio/ONT) to reference genome(s). Given a minimum alignment length and an identity threshold for the desired local alignments, Mashmap computes alignment boundaries and identity estimates using k-mers","Masks recombinant regions in an alignment based on ClonalFrameML or Gubbins output","Whole genome assembly software. It combines the efficiency of the de Bruijn graph and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing only short reads from Illumina sequencing or a mixture of short reads and long reads (Sanger, 454).","Mauve Genome Alignment software, for comparing two or more draft or finished genomes","Software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm.","Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.","Quantitative proteomics software package designed for analyzing large mass-spectrometric data sets. It is specifically aimed at high-resolution MS data.","MCL is a clustering algorithm widely used in bioinformatics and gaining traction in other fields.","medaka is a tool to create consensus sequences and variant calls from nanopore sequencing data. This task is performed using neural networks applied a pileup of individual sequencing reads against a draft assembly.","Single node assembler for large and complex metagenomics NGS reads, such as soil. It makes use of succinct de Bruijn graph to achieve low memory usage, whereas its goal is not to make memory usage as low as possible.","Metagenome Analysis Software - MEGAN (MEtaGenome ANalyzer) is a new computer program that allows laptop analysis of large metagenomic datasets. In a preprocessing step, the set of DNA reads (or contigs) is compared against databases of known sequences using BLAST or another comparison tool. MEGAN can then be used to compute and interactively explore the taxonomical content of the dataset, employing the NCBI taxonomy to summarize and order the results.","Reference-free quality, completeness, and phasing assessment for genome assemblies.\n\nEvaluate genome assemblies with k-mers and more.\nOften, genome assembly projects have illumina whole genome sequencing reads available for the assembled individual.\n\nMerqury provides a set of tools for this purpose.","","A short read assembler based on iterative De Bruijn graph. It is developed under 64-bit Linux, but should be suitable for all unix-like system.","an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | MetaBAT2 clusters metagenomic contigs into different \"bins\", each of which should correspond to a putative genome | MetaBAT2 uses nucleotide composition information and source strain abundance (measured by depth-of-coverage by aligning the reads to the contigs) to perform binning","Statistical, functional and integrative analysis of metabolomics data , interpretation, and integration with other omics data.","Unsupervised binning method for metagenomic sequences. Existing binning methods based on sequence similarity and sequence composition markers rely heavily on the reference genomes of known microorganisms and phylogenetic markers.","MetaCV is a composition and phylogeny based algorithm to classify very short metagenomic reads (75-100 bp) into specific taxonomic and functional groups. MetaCV performs (for both sensitivity and specificity) as good as BlastX-based methods on simulated short reads, but runs 300 times faster thus provides effectively and efficiently analysis on huge amount of NGS data.","Galaxy workflow for differential abundance analysis of 16s metagenomic data.\n\nYou are over your disk quota. Tool execution is on hold until your disk usage drops below your allocated quota.\n\nThis history is empty. You can load your own data or get data from an external source","Prokaryotic gene finding program from environmental genome shotgun sequences or metagenomic sequences.","","Designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. It is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.","MetaMaps is tool specifically developed for the analysis of long-read (PacBio/Oxford Nanopore) metagenomic datasets.","MS-based metabolomics data processing and compound annotation pipeline.","MetaPath is a tool for meta analysis pathway enrichment.","Computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data.","Novel taxonomic classifier for metagenomic shotgun reads, which uses phylogenetic marker genes as a taxonomic reference. Our classifier, based on BLAST, uses different thresholds (automatically learned from the reference database) for each combination of taxonomic rank, reference gene, and sequence length.","metaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.","Genome assembler for metagenomics datasets.","MetaVelvet is an extension of Velvet assembler to de novo metagenome assembly from short sequence reads","","MG-RAST is a fully-automated service for annotating metagenome samples providing analysis tools for comparison","The microbiome analysis resource in 2020.\n\nMGnify (http: www.ebi.ac.uk metagenomics) provides a free to use platform for the assembly, analysis and archiving of microbiome data derived from sequencing microbial populations that are present in particular environments. Over the past 2 years, MGnify (formerly EBI Metagenomics) has more than doubled the number of publicly available analysed datasets held within the resource. Recently, an updated approach to data analysis has been unveiled (version 5.0), replacing the previous single pipeline with multiple analysis pipelines that are tailored according to the input data, and that are formally described using the Common Workflow Language, enabling greater provenance, reusability, and reproducibility.","Estimates effective population sizes,past migration rates between n population assuming a migration matrix model with asymmetric migration rates and different subpopulation sizes, and population divergences or admixture.","","Short-read assembler based on a de Bruijn graph, capable of assembling a human genome on a desktop computer in a day.","Minimac4 is a genotype imputation software.","Pairwise aligner for genomic and spliced nucleotide sequences","MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological pathways for a query dataset. Licensed under GNU GPL.","MIRA 3 - Whole Genome Shotgun and EST Sequence Assembler","","Discovering known and novel miRNAs from deep sequencing data.","miRDeep2 discovers active known or novel miRNAs from deep sequencing data.","An alternative to Cufflinks, MISO (Mixture-of-Isoforms) is a probabilistic framework that quantitates the expression level of alternatively spliced genes.","The tool offers a wide range of multivariate methods for the exploration and integration of biological datasets with a particular focus on variable selection.","Multi Locus Sequence Typing from an assembled genome or from a set of reads.","MOCAT is a modular and scalable software pipeline for analyzing shotgun metagenomics datasets generated with Illumina technology. Starting from raw fastQ files, it can quality-filter and remove contaminants from them, assemble metagenomic reads into contigs, predict prokaryotic genes on these, identify phylogenetic marker genes and generate taxonomic abundance profiles by mapping reads to these marker genes.","A modest Feature Finder to extract features in MS1 Data.","A proteomics search algorithm specifically designed for high-resolution tandem mass spectra.","","","Program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. It uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.","MS-DIAL was launched as a universal program for untargeted metabolomics that supports multiple instruments (GC/MS, GC/MS/MS, LC/MS, and LC/MS/MS) and MS vendors (Agilent, Bruker, LECO, Sciex, Shimadzu, Thermo, and Waters)","This package contains function to perform peptide identification using MS-GF+.","Basic plotting, data manipulation and processing of MS-based Proteomics data.","Statistical tool for quantitative mass spectrometry-based proteomics.","MultiQC aggregates results from multiple bioinformatics analyses across many samples into a single report. It searches a given directory for analysis logs and compiles a HTML report. It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.","multiSNV is a tool for calling somatic single-nucleotide variants (SNVs) using NGS data from a normal and multiple tumour samples of the same patient.","MUMmer is a modular system for the rapid whole genome alignment of finished or draft sequence. Basically it is a ultra-fast alignment of large-scale DNA and protein sequences","This tool performs multiple sequence alignments of nucleotide or amino acid sequences.","","A package for detecting cytosine methylations and genetic variations from nanopore MinION sequencing data.","The NAST-iEr alignment utility aligns a single raw nucleotide sequence against one or more NAST formatted sequences.","nccmp compares two NetCDF files bitwise, semantically or with a user defined tolerance (absolute or relative percentage).  Parallel comparisons are done in local memory without requiring temporary files.  Highly recommended for regression testing scientific models or datasets in a test-driven development environment.","","","The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5.","Ncview is a netCDF visual browser.","NECAT is an error correction and de-novo assembly tool for Nanopore long noisy reads.","NetCDF (Network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.","NeuroDebian is a popular turnkey platform for Neuroscience, where software is integrated, tested, and delivered conveniently and reliably so you could concentrate on your research and not on \"system maintenance\". It provides a large collection of popular neuroscience research software for the Debian operating system as well as Ubuntu and other derivatives.","NextDenovo is a string graph-based de novo assembler for long reads (CLR, HiFi and ONT). It uses a \"correct-then-assemble\" strategy similar to canu (no correction step for PacBio Hifi reads), but requires significantly less computing resources and storages.","","NextGenMap is a flexible highly sensitive short read mapping tool that handles much higher mismatch rates than comparable algorithms while still outperforming them in terms of runtime.","A fast and efficient genome polishing tool for long read assembly.\n\nFast and accurately polish the genome generated by noisy long reads.\n\nNextPolish is used to fix base errors (SNV/Indel) in the genome generated by noisy long reads, it can be used with short read data only or long read data only or a combination of both. It contains two core modules, and use a stepwise fashion to correct the error bases in reference genome. To correct the raw third-generation sequencing (TGS) long reads with approximately 15-10% sequencing errors, please use NextDenovo","","","","","NVBIO is a library of reusable components designed by NVIDIA Corporation to accelerate bioinformatics applications using CUDA.","De novo transcriptome assembler designed to produce transcripts from short read sequencing technologies. It uploads a preliminary assembly produced by Velvet, and clusters the contigs into small groups, called loci. It then exploits the paired-end read and long read information, when available, to construct transcript isoforms.","Set of python programs developed to simplify the manipulation of sequence files. They were mainly designed to help us for analyzing Next Generation Sequencer outputs (454 or Illumina) in the context of DNA Metabarcoding.","Open source library and a collection of tools and interfaces for the analysis of mass spectrometry data. Includes over 200 standalone (TOPP) tools that can be combined to a workflow with the integrated workflow editor TOPPAS. Raw and intermediate mass spectrometry data can be visualised with the included viewer TOPPView.","","","","PEPPER is a genome inference module based on recurrent neural networks that enables long-read variant calling and nanopore assembly polishing in the PEPPER-Margin-DeepVariant pipeline. This pipeline enables nanopore-based variant calling with DeepVariant.","Python scripts for Exploratory Data Analysis of Pacific Biosciences sequence data","","","Paired-end Assembler for DNA sequences (PANDAseq) is a program to align Illumina reads, optionally with PCR primers embedded in the sequence, and reconstruct an overlapping sequence.","","ParaView is an open-source, multi-platform data analysis and visualization application. ParaView users can quickly build visualizations to analyze their data using qualitative and quantitative techniques. The data exploration can be done interactively in 3D or programmatically using ParaView’s batch processing capabilities.","Parsnp is a command-line-tool for efficient microbial core genome alignment and SNP detection. Parsnp was designed to work in tandem with Gingr, a flexible platform for visualizing genome alignments and phylogenetic trees.","Program to Assemble Spliced Alignment. PASA, acronym for Program to Assemble Spliced Alignments, is a eukaryotic genome annotation tool that exploits spliced alignments of expressed transcript sequences to automatically model gene structures, and to maintain gene structure annotation consistent with the most recently available experimental sequence data. PASA also identifies and classifies all splicing variations supported by the transcript alignments.","Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.","Paired-end read merger. PEAR evaluates all possible paired-end read overlaps without requiring the target fragment size as input. In addition, it implements a statistical test for minimizing false-positive results.","PeptideShaker is a search engine independent platform for interpretation of proteomics identification results from multiple search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, OMSSA, MyriMatch, Comet, Tide, Mascot, Andromeda and mzIdentML. By combining the results from multiple search engines, while re-calculating PTM localization scores and redoing the protein inference, PeptideShaker attempts to give you the best possible understanding of your proteomics data","Semi-supervised learning for peptide identification from MS/MS data.","","Provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.","Suite of software tools to conduct phylogenetic analysis of genomes and metagenomes. Using a reference database of protein sequences, PhyloSift can scan new sequences against that data for homologs and identify phylogentic relationships of the new sequence compared to the database sequences.","Phylogenetic estimation software using Maximum Likelihood","","A set of command line tools for manipulating high-throughput sequencing (HTS) data in formats such as SAM/BAM/CRAM and VCF. Available as a standalone program or within the GATK4 program.","PICRUSt (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes.","PICRUSt2 (Phylogenetic Investigation of Communities by Reconstruction of Unobserved States) is a software for predicting functional abundances based only on marker gene sequences.","","Read alignment analysis to diagnose, report, and automatically improve de novo genome assemblies.","Free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.","Analytic tool development for variation data from large-scale resequencing and genotyping projects, particularly whole-exome and whole-genome studies. It is independent of (but designed to be complementary to) the existing PLINK package.","PnetCDF: A Parallel I/O Library for NetCDF File Access","Tools for performing taxonomic assignment based on phylogeny using pplacer and clst.","Pretext is an OpenGL-powered pretext contact map viewer.","PRINSEQ is a sequence processing tool that can be used to filter, reformat and trim genomic and metagenomic sequence data. It generates summary statistics of the input in graphical and tabular formats that can be used for quality control steps. PRINSEQ is available as both standalone and web-based versions.","","ProbMetab is an R package for Bayesian probabilistic annotation of LC–MS-based metabolomics","The pipeline runs PRODIGAL gene predictions on all genomes, runs pan-reciprocal BLAST, and identifies ortholog sets. For a set of orthologous genes, if the positions of the PRODIGAL selected starts coincide in a multiple sequence alignment, they are accepted. If they do not coincide, a consistent start position is sought where a majority of the highest-scoring PRODIGAL selected sites coincide. If such a position is found, it is accepted, and the predictions are changed for the outlying genes.","Software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.","Hybrid correction pipeline for SMRT reads, which can be flexibly adapted on existing hardware and infrastructure from a laptop to a high performance computing cluster.","It is a high-precision localization prediction method for bacterial proteins. It has remained the most precise bacterial protein subcellular localization (SCL) predictor.  PSORTb version improved recall, higher proteome-scale prediction coverage, and new refined localization subcategories. It is the first SCL predictor specifically geared for all prokaryotes, including archaea and bacteria with atypical membrane/cell wall topologies.","PSORTm is a modified version of PSORTb 3.0.2 for protein subcellular localization prediction (SCL), and PSORTdb-based cell envelope classification, that can predict protein SCL from metagenomic sequences.\n\nPSORT.org provides links to the PSORT family of programs for subcellular localization prediction as well as other datasets and resources relevant to localization prediction.","Identifying and removing haplotypic duplication in primary genome assemblies | haplotypic duplication identification tool | scripts/pd_config.py: script to generate a configuration file used by run_purge_dups.py | purge haplotigs and overlaps in an assembly based on read depth | Given a primary assembly pri_asm and an alternative assembly hap_asm (optional, if you have one), follow the steps shown below to build your own purge_dups pipeline, steps with same number can be run simultaneously. Among all the steps, although step 4 is optional, we highly recommend our users to do so, because assemblers may produce overrepresented seqeuences. In such a case, The final step 4 can be applied to remove those seqeuences","Purge Haplotigs is a pipeline to help with curating heterozygous diploid genome assemblies.","pyani is a Python package and standalone program for calculation of whole-genome similarity measures. It is designed to be used with draft or complete prokaryote (bacterial and archaeal) genomes.","Pipeline to assemble de novo RADseq loci with the aim of optimizing coverage across phylogenetic datasets.","A Python module for reading and manipulating SAM/BAM/VCF/BCF files.","PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.","Open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. The pipeline is designed to take users from raw sequencing data generated on the Illumina or other platforms through publication quality graphics and statistics. This includes demultiplexing and quality filtering, OTU picking, taxonomic assignment, and phylogenetic reconstruction, and diversity analyses and visualizations.","Open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data. The pipeline is designed to take users from raw sequencing data generated on the Illumina or other platforms through publication quality graphics and statistics. This includes demultiplexing and quality filtering, OTU picking, taxonomic assignment, and phylogenetic reconstruction, and diversity analyses and visualizations.","QIIME 2™ is a next-generation microbiome bioinformatics platform that is extensible, free, open source, and community developed.","QUAST stands for QUality ASsessment Tool.  \nIt evaluates a quality of genome assemblies by computing various metrics and providing nice reports.","Quickmerge is a program that uses complementary information from genomes assembled with long reads in order to improve contiguity, and works with assemblies derived from both Pacific Biosciences or Oxford Nanopore. Quickmerge will even work with hybrid assemblies made by combining long reads and Illumina short reads.","Free software environment for statistical computing and graphics.","The Possibility to Use Oxford Nanopore Technology | Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads. http://genome.cshlp.org/content/early/2017/01/18/gr.214270.116 Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: | Racon is intended as a standalone consensus module to correct raw contigs generated by rapid assembly methods which do not include a consensus step | Consensus module for raw de novo DNA assembly of long uncorrected reads","RAILS and Cobbler are genomics application for scaffolding and automated finishing of genome assemblies with long DNA sequences. They can be used to scaffold & finish high-quality draft genome assemblies with any long, preferably high-quality, sequences such as scaftigs/contigs from another genome draft.","Transfer annotation from a reference (annotated) genome to an unannotated query genome. It is able to transfer any entry present on a reference sequence, such as the systematic id or an annotator’s notes.","RATTLE is a tool for reference-free reconstruction and quantification of transcriptomes from long-read sequencing.","a de novo genome assembler for long reads.\n\nRaven is a de novo genome assembler for long uncorrected reads.","A standalone tool for extracting data directly from raw files generated by Thermo Orbitrap family instruments.","A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies.","","RefineM is a set of tools for improving population genomes. It provides methods designed to improve the completeness of a genome along with methods for identifying and removing contamination.","A new method which assembles repeat sequences directly from raw shotgun sequencing data.","A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns).","De-novo repeat family identification and modeling package. At the heart of RepeatModeler are two de-novo repeat finding programs ( RECON and RepeatScout ) which employ complementary computational methods for identifying repeat element boundaries and family relationships from sequence data. RepeatModeler assists in automating the runs of RECON and RepeatScout given a genomic database and uses the output to build, refine and classify consensus models of putative interspersed repeats.","","A CGL-based program that integrates RepeatMasker with BLASTX to provide a comprehensive means of identifying repetitive elements. Because RepeatMasker identifies repeats by means of similarity to a nucleotide library of known repeats, it often fails to identify highly divergent repeats and divergent portions of repeats, especially near repeat edges.","The REPET package integrates bioinformatics pipelines dedicated to detecte, annotate and analyse transposable elements (TEs) in genomic sequences. The main pipelines are (i) TEdenovo, which search for interspersed repeats, build consensus sequences and classify them according to TE features, and (ii)\n TEannot, which mines a genome with a library of TE sequences, for instance the one produced by the TEdenovo pipeline, to provide TE annotations exported into GFF3 files.","The rjags package provides an interface from R to the JAGS library for Bayesian data analysis. JAGS uses Markov Chain Monte Carlo (MCMC) to generate a sequence of dependent samples\nfrom the posterior distribution of the parameters.","","We present a generative statistical model and associated inference methods that handle read mapping uncertainty in a principled manner. Through simulations parameterized by real RNASeq data, we show that our method is more accurate than previous methods. Our improved accuracy is the result of handling read mapping uncertainty with a statistical model and the estimation of gene expression levels as the sum of isoform expression levels.","Provides a number of useful modules that can comprehensively evaluate high throughput sequence data especially RNA-seq data. Some basic modules quickly inspect sequence quality, nucleotide composition bias, PCR bias and GC bias, while RNA-seq specific modules evaluate sequencing saturation, mapped reads distribution, coverage uniformity, strand specificity, transcript level RNA integrity etc.","","SAIGE is an R package with Scalable and Accurate Implementation of Generalized mixed model. It accounts for sample relatedness and is feasible for genetic association tests in large cohorts and biobanks (N > 400,000).\n\nSAIGE performs single-variant association tests for binary traits and quantitative traits. For binary traits, SAIGE uses the saddlepoint approximation (SPA)(mhof, J. P. , 1961; Kuonen, D. 1999; Dey, R. et.al 2017) to account for case-control imbalance.\n\nSAIGE-GENE (implemented in the SAIGE R package) performs gene- or region-based association tests (Burde, SKAT, SKAT-O) for binary traits and quantitative traits. Note: SAIGE-GENE accounts for case-control imbalance in gene-based tests (>= 0.35.8.5)","","> VERY_LOW CONFIDENCE! | > CORRECT NAME OF TOOL COULD ALSO BE 'chromosome-scale', 'reference-quality', 'Hi-C', 'scaffolder' | Integrating Hi-C links with assembly graphs for chromosome-scale assembly | SALSA: A tool to scaffold long read assemblies with Hi-C data | SALSA: A tool to scaffold long read assemblies with Hi-C | This code is used to scaffold your assemblies using Hi-C data. This version implements some improvements in the original SALSA algorithm. If you want to use the old version, it can be found in the old_salsa branch","A tool to mark duplicates and extract discordant and split reads from SAM files.","SAMtools and BCFtools are widely used programs for processing and analysing high-throughput sequencing data. They include tools for file format conversion and manipulation, sorting, querying, statistics, variant calling, and effect analysis amongst other methods.","SAS is an integrated system of software that enables you to do everything from accessing data across multiple sources to performing sophisticated analyses and delivering information across your organization. SAS foundation products are available as individual software modules, and they serve as the foundation for SAS solutions.","Scarpa is a stand-alone scaffolding tool for NGS data. It can be used together with virtually any genome assembler and any NGS read mapper that supports SAM format. Other features include support for multiple libraries and an option to estimate insert size distributions from data.","","screed is a tool for short read sequence utils in Python.","Graphical user interface for proteomics identification search engines","","seqstats package provides a comprehensive framework for biological sequence statistics to allow researchers to design null hypotheses and random controls when performing exploratory genomics studies.","seqtk-trinity is a toolkit for processing sequences in FASTA/Q formats.","Fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.","","A  tool that uses sliding windows along with quality and length thresholds to determine when quality is sufficiently low to trim the 3'-end of reads and also determines when the quality is sufficiently high enough to trim the 5'-end of reads.","A text mining framework for interactive analysis and visualization of similarities among biomedical entities.\n\nFor each search query, PMIDs or abstracts from PubMed are saved.\n\n$ git clone https://github.com/dlal-group/simtext.\n\nFor all PMIDs in each row of a table the according abstracts are saved in additional columns.","Skewer implements a novel dynamic programming algorithm dedicated to the task of adapter trimming and it is specially designed for processing illumina paired-end sequences.","Abioinformatics software suite for analyzing single molecule, real-time DNA sequencing data from Pacific Biosciences.","Workflow engine and language. It aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern domain specific specification language (DSL) in python style.","The Semi-HMM-based Nucleic Acid Parser is a gene prediction tool.","a snakemake pipeline for scalable HIV-1 subtyping by phylogenetic pairing | SNAPPy is a Snakemake pipeline for HIV-1 subtyping by phylogenetic pairing | This is the repository for SNAPPy, a Snakemake pipeline for HIV-1 subtyping by phylogenetic pairing. SNAPPy allows high-throughput HIV-1 subtyping locally while being resource efficient and scalable. This pipeline was constructed using Snakemake , and it uses MAFFT and for multiple sequence alignment, BLAST for similarirty querys, IQ-TREE for phylogenetic inference, and several Biopython modules for data parsing an analysis. For in-depth information on how the tool works please visit the documentation page. SNAPPy was design for Linux based operative systems | Welcome to snappy’s documentation! — SNAPPy-HIV1-Subtyping 1.0.0 documentation | Free document hosting provided by Read the Docs","An algorithm for structural variation detection from third generation sequencing alignment.","Snoscan allows you to search for C/D box methylation guide snoRNA (small nucleolar RNA) genes in a genomic sequence","Variant annotation and effect prediction tool. It annotates and predicts the effects of variants on genes and proteins (such as amino acid changes).","Short Oligonucleotide Analysis Package. Efficient gapped and ungapped alignment of short oligonucleotides onto reference sequences and full solution to next generation sequencing data analysis.","SOAP3 is a GPU-based software for aligning short reads to a reference sequence.","SOAPdenovo2 is a next generation sequencing reads de novo assembler.","Accurate consensus sequence builder based on soap1 and SOAPaligner/soap2's alignment output. It calculates a quality score for each consensus base, which can be used for any latter process to call SNPs.","Sequence analysis tool for filtering, mapping and OTU-picking NGS reads.","St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies. SPAdes 3.9 works with Illumina or IonTorrent reads and is capable of providing hybrid assemblies using PacBio, Oxford Nanopore and Sanger reads. Additional contigs can be provided and can be used as long reads.","Splign is a utility for computing cDNA-to-Genomic, or spliced sequence alignments. At the heart of the program is a compartmentization algorithm which identifies possible gene duplications, and a refined alignment algorithm recognizing introns and splice signals.","SqueezeMeta is a fully automated metagenomics pipeline, from reads to bins.","The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.","Developed to work with restriction enzyme based sequence data, such as RADseq, for building genetic maps and conducting population genomics and phylogeography analysis.","Ultrafast universal RNA-seq data aligner","The program structureis a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed.","An agile homology-based approach using a reduced SEED database to report the subsystems present in metagenomic samples and profile their abundances.","Supernova is a software package for de novo assembly from Chromium Linked-Reads that are made from a single whole-genome library from an individual DNA source. A key feature of Supernova is that it creates diploid assemblies, thus separately representing maternal and paternal chromosomes over very long distances. Almost all other methods instead merge homologous chromosomes into single incorrect 'consensus' sequences. Supernova is the only practical method for creating diploid assemblies of large genomes.","Genotyping structural variations with long reads.\n\nSVJedi : Genotyping structural variations with long read data.\n\nSVJedi is a structural variation (SV) genotyper for long read data. Based on a representation of the different alleles, it estimates the genotype of each variant in a given individual sample based on allele-specific alignment counts. SVJedi takes as input a variant file (VCF), a reference genome (fasta) and a long read file (fasta/fastq) and outputs the initial variant file with an additional column containing genotyping information (VCF)","SVTyper performs breakpoint genotyping of structural variants (SVs) using whole genome sequencing data.","It contains utility functions for integrating spectral libraries for SWATH and statistical data analysis for SWATH generated data.","TagCleaner can be used to automatically detect and efficiently remove tag sequences (e.g. WTA or MID tags) from metagenomic datasets. TagCleaner is available as both standalone and web-based versions.","TagDust, a program identifying artifactual sequences in large sequencing runs. Given a user-defined cutoff for the false discovery rate (FDR), TagDust identifies all reads explainable by combinations and partial matches to known sequences used during library preparation.","Targetfinder.org provides a web based resource that allows users to find genes that have a similar expression to a query gene signature.","Software package to evaluate traits associations, evolutionary patterns, and linkage disequilibrium.","","","Open-source, crossplatform tool that converts Thermo RAW files into open file formats such as MGF and to the HUPO-PSI standard file format mzML","Tigmint is a tool for correcting misassemblies in genome assembly drafts using linked or long sequencing reads","Program that aligns RNA-Seq reads to a genome in order to identify exon-exon splice junctions. It is built on the ultrafast short read mapping program Bowtie. A stable SAMtools version is now packaged with the program.","TotalView is a debugger for High Performance Computing applications.","TransDecoder identifies candidate coding regions within transcript sequences, such as those generated by de novo RNA-Seq transcript assembly using Trinity, or constructed based on RNA-Seq alignments to the genome using Tophat and Cufflinks.","Tandem Repeats Finder. Find tandem repeats in DNA sequences without the need to specify either the pattern or pattern size. It uses the method of k-tuple matching to avoid the need for full scale alignment matrix computations. It requires no a priori knowledge of the pattern, pattern size or number of copies. There are no restrictions on the size of the repeats that can be detected. It determines a consensus pattern for the smallest repetitive unit in the tandem repeat.","A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.","A flexible read trimming tool for Illumina NGS data","Trinity is a transcriptome assembler which relies on three different tools, inchworm an assembler, chrysalis which pools contigs and butterfly which amongst others compacts a graph resulting from butterfly with reads.","Comprehensive annotation suite designed for automatic functional annotation of transcriptomes, particularly de novo assembled transcriptomes, from model or non-model organisms.","A program for improved detection of transfer RNA genes in genomic sequence.","","Utilities for handling sequences and assemblies from the UCSC Genome Browser project.","A tool for assembling bacterial genomes from a combination of short (2nd generation) and long (3rd generation) sequencing reads.","Algorithm designed to enable high-throughput, sensitive search of very large sequence databases.","VALET is a pipeline for detecting mis-assemblies in metagenomic assemblies.","A website for visualization and analysis of microbial population structures.","VarScan, an open source tool for variant detection that is compatible with several short read align-ers.","vawk is an awk-like VCF parser.","Provide easily accessible methods for working with complex genetic variation data in the form of VCF files.","A de novo genomic assembler specially designed for short read sequencing technologies, such as Solexa or 454 or SOLiD.","The ViennaRNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.","VisIt is an Open Source, interactive, scalable, visualization, animation and analysis tool.","High-throughput search and clustering sequence analysis tool. It supports de novo and reference based chimera detection, clustering, full-length and prefix dereplication, reverse complementation, masking, all-vs-all pairwise global alignment, exact and global alignment searching, shuffling, subsampling and sorting. It also supports FASTQ file analysis, filtering and conversion.","Wengan is a new genome assembler that unlike most of the current long-reads assemblers avoids entirely the all-vs-all read comparison.","windowmasker identifies and masks highly repetitive DNA sequences in a genome, using only the sequence of the genome itself.","Compare a protein sequences or protein HMMs to a genomic DNA sequence, allowing for introns and frameshifting errors.","First fully open-source and collaborative online platform for computational metabolomics. It includes preprocessing, normalization, quality control, statistical analysis of LC/MS, FIA-MS, GC/MS and NMR data.","Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). It assembles raw reads without error correction and then builds the consensus from intermediate assembly output. Wtdbg2 is able to assemble the human and even the 32Gb Axolotl genome at a speed tens of times faster than CANU and FALCON while producing contigs of comparable base accuracy.","","Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data.  The packages enables imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files and preprocesses data for high-throughput, untargeted analyte profiling.","","Matches tandem mass spectra with peptide sequences."],["Sequence assembly, Mapping, DNA","Genomics, Microbiology","Sequence assembly","Sequence assembly","Sequence assembly","Sequence assembly","Genomics, Sequence assembly","Microbiology, Public health and epidemiology, Data quality management, Infectious disease","DNA, Gene expression","Population genetics, Statistics and probability","Microbiology, Transcriptomics, Phylogenomics, Metagenomics, Omics","Data submission, annotation and curation, Gene transcripts, Genomics, Mapping, Molecular interactions, pathways and networks","Data visualisation","Gene transcripts, Gene and protein families","Sequence analysis","Sequence sites, features and motifs, ChIP-seq","Sequencing, Data management, Sequence analysis","Sequencing, RNA splicing, Whole genome sequencing, Phylogenetics, Metagenomics, RNA-Seq","Genetic variation, DNA polymorphism, GWAS study, Genotyping experiment","Population genetics","Phylogenetics, Statistics and probability, Computer science","Phylogenetics, Sequence analysis","Phylogenetics, Phylogenomics","Genomics","Sequence analysis","Metagenomics","Bioinformatics","Bioinformatics, Computational biology, Data management, Statistics and probability","","Sequence analysis","Genomics, Software engineering, Data management","Programming language / platform","Epigenomics, Genomics, Epigenetics","Sequence analysis, Genomics","Sequence analysis, Genomics","Sequence analysis, Genomics","Metagenomics, Molecular interactions, pathways and networks","Sequence analysis","Sequence assembly, Model organisms, Workflows, Phylogenomics, Zoology","Genomics, Sequence assembly, Data management","Phylogeny","No term available","Mapping, Genomics, Sequencing, Sequence assembly","Mapping, Genomics, Mapping","RNA-Seq, RNA, Workflows","RNA-Seq, RNA, Workflows","Genomics, DNA structural variation","Epigenetics","Epigenomics","Methylated DNA immunoprecipitation, Sequencing, Mapping, Nucleic acid structure analysis","Sequence assembly, Genomics, Transcriptomics, Sequence analysis","Mapping","Sequence analysis","Genomics, Sequence analysis, Phylogeny, Sequence assembly, Mapping, Phylogenetics","Taxonomy","Metabolomics","Genomics","Sequence assembly, Sequencing, Sequence analysis, Genetic variation","Proteomics, Data visualisation","Metagenomics, Metagenomic sequencing, Taxonomy, Sequence assembly","Sequencing","RNA-Seq, Transcriptomics","Sequence assembly","Imaging, Microarray experiment, Genotype and phenotype","Metagenomics, Microbiology","Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics","Genomics, Phylogenomics, Phylogenetics, Taxonomy, Metagenomics","Metagenomics, Sequence assembly, Metagenomic sequencing, Mapping","Sequences, DNA","Comparative genomics","No term available","Sequence analysis","Sequence assembly","Sequence analysis","Statistics and probability","DNA structural variation","Gene expression, Functional, regulatory and non-coding RNA, Gene transcripts, RNA, Sequence sites, features and motifs","Functional, regulatory and non-coding RNA, Gene expression, Gene transcripts, RNA","No term available","Bioinformatics","Functional, regulatory and non-coding RNA, Gene transcripts, RNA-Seq, Transcription factors and regulatory sites","Ecology","","Library","RNA splicing, Transcriptomics, RNA-Seq","Genomics, Probes and primers, Sequencing","Bioinformatics, Molecular interactions, pathways and networks, Data visualisation, Rare diseases","Sequencing, Genetic variation","Enzymes, Sequence sites, features and motifs, Protein folds and structural domains, Carbohydrates, Gene and protein families","Metagenomics, Carbohydrates, Sequence sites, features and motifs","Genomics, ChIP-seq","Whole genome sequencing, Exome sequencing, Genotype and phenotype, Genomics, DNA polymorphism","Phylogenetics, Phylogenomics","Proteomics, Proteomics experiment","Sequence analysis, Proteins","Sequence assembly","Genetic variation","Proteins, Protein secondary structure","","Computer science","Genomics, Microbiology, Sequence assembly","Knowledge discovery in databases","Metagenomics, Genomics","Molecular biology, Sequence analysis, Biology","Proteomics, Proteomics experiment","Metagenomics","Genomics, Rare diseases","Genomics / Genome annotation","Gene transcripts, Statistics and probability, Genomics","No term available","Phylogenetics","Sequence analysis, Sequence sites, features and motifs, Molecular interactions, pathways and networks","Taxonomy, Metabolomics, Biomarkers","Sequence assembly","Sequence assembly","Phylogeny, Statistics and probability, Sequencing, Sequence analysis, Experimental design and studies","No term available","Sequencing, Data quality management, Sequence analysis","Population genetics","Population genetics","Sequence analysis","Sequence analysis","Gene structure","Population genetics","Sequencing, Sequence assembly","Proteomics experiment, Proteomics","Sequence assembly, Metagenomics, Whole genome sequencing","Metagenomics, Metagenomic sequencing","Genetics, Sequence analysis","Genomics, Genetic variation, Rare diseases","MRI","Genomics","Phylogeny","DNA, Sequencing, Data management","Sequence analysis, Genetic variation, Sequencing, Workflows","","Sequence analysis","GWAS study","Sequence analysis","Nucleic acid sites, features and motifs, Sequence analysis, DNA, Gene structure, Molecular genetics","Gene structure","Sequence assembly, Mathematics, Mapping","Sequence assembly","ChIP-seq, RNA-seq, Data management","Molecular interactions, pathways and networks, Metagenomics, Metagenomic sequencing","Sequence analysis, DNA, DNA, Molecular genetics","Gene structure","Gene structure, Genomics","Metagenomics, Metagenomic sequencing","Mapping","Mapping","Mapping","Proteomics experiment, Metabolomics, Lipids, Biotechnology, Biochemistry","Programming language / platform","No term available","No term available","Genomics, Metagenomics, Transcriptomics","Metagenomics, Phylogenetics, Phylogenomics, Cladistics","Molecular dynamics","Metagenomics, Sequence assembly","Metagenomics, Taxonomy, Phylogenetics, Database management, Proteins","Phylogeny, Genotype and phenotype, Whole genome sequencing","Sequencing","Computer science","Sequence assembly","Sequence assembly","Microbiology, Metagenomics","","","Sequence assembly","Data visualisation","Mapping, Sequence assembly, Whole genome sequencing","RNA-seq","Sequence analysis, Sequence sites, features and motifs, Gene and protein families","Machine learning","","","Sequence analysis","Data management","Meta-Omics","No term available","Meta-Omics","Sequence assembly","Programming language / platform","Imaging","Metagenomics, Virology, Microbial ecology, Genomics, Comparative genomics","Sequence assembly","Genomics / Genome annotation","Data visualisation","","Protein sequence features","Sequence analysis","Phylogenetics, Statistics and probability, Phylogenomics","","Gene transcripts","Statistics and probability","Statistics and probability","Genomics, Bioinformatics","Genomics","Sequence assembly","Sequence analysis, Genomics","","Epigenetics, Sequencing, Mapping, Sequence assembly","Sequencing, Mapping, Genomics, Epigenetics","Molecular interactions, pathways and networks, Mapping, Data submission, annotation and curation","Transcriptomics, RNA-seq, Gene expression","Sequence analysis, Genomics","Sequence assembly","Nucleic acids","Whole genome sequencing, Genomics, Sequence assembly","Sequence sites, features and motifs, Molecular interactions, pathways and networks, Gene and protein families, Model organisms","Taxonomy, Metagenomics","Taxonomy, Metagenomics","Metagenomics","Genomics","Proteomics, Sequence analysis, Small molecules, Metabolomics, Proteomics experiment","","Sequence assembly, Mapping, Sequencing","Lipids, Endocrinology and metabolism, Small molecules, Mapping","Exome sequencing, Genomics, Whole genome sequencing, DNA polymorphism","","Genotype and phenotype, Genotyping experiment, Quantitative genetics","ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites","ChIP-seq, Molecular interactions, pathways and networks, Transcription factors and regulatory sites","Sequence alignment","Sequence analysis","Metagenomics, Sequence assembly, Phylogenetics, Taxonomy","Genomics, DNA, Sequence analysis","Sequencing, Data submission, annotation and curation","Proteomics experiment, Proteomics","Genomics, Metagenomics, Statistics and probability, Sequence analysis, DNA mutation","Sequence assembly, Mapping, Sequencing","DNA replication and recombination, Structure analysis","Sequence assembly, Whole genome sequencing, Sequence analysis","Genomics, Transcriptomics","Metagenomics, Sequence assembly, Microbiology","Proteomics experiment, Proteomics, Statistics and probability","Proteomics experiment, Proteomics, Statistics and probability","Molecular interactions, pathways and networks, Computer science","Sequence assembly, Machine learning","Metagenomics, Sequencing, Ecology, Sequence assembly","Sequence analysis","Sequence assembly, Whole genome sequencing, Plant biology","Sequence analysis","Sequence assembly","Metagenomics, Sequence assembly, Metagenomic sequencing","Metabolomics, Metagenomics, Small molecules","Microbiology, Sequence composition, complexity and repeats, Metagenomics, Phylogenetics","Metagenomics, Sequencing, Sequence composition, complexity and repeats","Metagenomics, Metagenomic sequencing, Workflows, Microbial ecology, RNA-Seq","Genomics, Model organisms, Data submission, annotation and curation","","Metagenomics, Sequencing","Metagenomics, Sequence composition, complexity and repeats","Metabolomics","Molecular interactions, pathways and networks, Gene expression","Metagenomics, Phylogenomics","Metagenomics, Taxonomy, Phylogenetics, Microbial ecology","Proteomics, Metatranscriptomics, Microbial ecology, Proteomics experiment, Metagenomics","Metagenomics, Sequence analysis","Assembly","","Metagenomics, Phylogenetics","Microbial ecology, Metagenomics, Cell biology, Workflows, Sequence assembly","Population genetics, Nucleic acid structure analysis, Zoology","Sequence alignment","Sequence assembly","Genotype and phenotype, Sequencing","Mapping","Molecular interactions, pathways and networks, Metagenomics, Metagenomic sequencing, Endocrinology and metabolism","Sequence assembly, RNA-Seq","","Functional, regulatory and non-coding RNA, Sequencing, RNA","Functional, regulatory and non-coding RNA, RNA immunoprecipitation","RNA-Seq, RNA splicing","Molecular interactions, pathways and networks, Statistics and probability","Immunoproteins, genes and antigens","Metagenomics","Proteomics","Proteomics","Metagenomics","Sequence alignment","Phylogenetics","Proteomics experiment, Lipids, Metabolomics","Proteomics experiment, Protein expression, Small molecules","Proteomics, Proteomics experiment, Data management, Data visualisation","Proteomics, Proteomics experiment, Statistics and probability, Workflows","Sequencing, Bioinformatics","Sequencing, Genetic variation, Oncology","Genomics, Mapping, Sequencing","Sequence analysis","Library","Sequence analysis, Sequencing, Genetic variation, Statistics and probability","Sequencing, Structure analysis","No term available","Data visualisation","Data visualisation","No term available","No term available","Sequence assembly","No term available","Neurobiology, Medical imaging","Sequence assembly","Workflows","DNA polymorphism, Mapping, Structure analysis","Sequence assembly, Plant biology, Mapping","Knowledge discovery in databases","Data visualisation","","Genomics / Genome annotation","Bioinformatics, Computer science","Transcriptomics, Whole genome sequencing, RNA-seq, Sequence assembly","Sequence analysis, DNA, Sequencing","Proteomics experiment, Proteomics, Metabolomics","Metagenomics","Metabolomics","","Sequence assembly, Mapping, Sequencing","Sequencing","Sequence analysis","Biodiversity","Sequence assembly, Probes and primers, Metagenomics","Virology","Data visualisation, Imaging","DNA polymorphism, Phylogenetics, Structure analysis","No term available","Molecular interactions, pathways and networks, Systems biology, Data visualisation","Sequence assembly","Proteomics, Proteomics experiment, Proteome","Proteomics, Proteomics experiment, Statistics and probability, Machine learning","Programming language / platform","Microbiology, Sequence analysis, Metagenomics","Metagenomics, Genomics","Phylogenetics, Bioinformatics, Phylogenetics","Metagenomics","Sequencing, Data management","Metagenomics, Microbial ecology, Functional, regulatory and non-coding RNA, Metagenomic sequencing","Metagenomics, Microbiology, Phylogenetics, Metagenomic sequencing","Genomics / Genome annotation","Assembly","GWAS study","Population genetics, Quantitative genetics, Whole genome sequencing","Computer science","Taxonomy","Mapping","Transcriptomics, Metagenomics, Genomics","Phylogenomics","Metabolomics, Molecular interactions, pathways and networks","DNA, Sequence analysis, Nucleic acid sites, features and motifs","Genomics, Model organisms, Virology","DNA, Sequencing, Data quality management","Sequences, Proteins, Plant","Metagenomics, Microbial ecology, Protein targeting and localisation, Membrane and lipoproteins, Metagenomic sequencing","Sequence assembly","Sequence assembly, Mapping, Genetic variation","Taxonomy, Public health and epidemiology, Genotype and phenotype","Phylogenetics, Sequence assembly, Sequence analysis","Sequencing, Mapping","Machine learning, Computer science","Biology","Biology","Microbial ecology, Phylogeny, Metatranscriptomics, Metagenomics","Sequence assembly","Structural variation, Sequence assembly, DNA polymorphism, Whole genome sequencing, Genotype and phenotype","Mathematics, Data architecture, analysis and design, Statistics and probability","Whole genome sequencing, Sequence assembly, Plant biology","Sequence assembly, DNA, Whole genome sequencing","Assembly, DNA","Transcriptomics, Sequencing","Sequence assembly, Whole genome sequencing, Genomics","Proteomics experiment, Proteomics","Phylogenetics, Sequence analysis","","No term available","Sequence assembly, Sequencing, Sequence composition, complexity and repeats","Sequence analysis, Sequence composition, complexity and repeats","Sequence composition, complexity and repeats, Sequence composition, complexity and repeats","Genomics / Genome annotation","Sequence composition, complexity and repeats, DNA","Sequence composition, complexity and repeats, DNA, Nucleic acid structure analysis","Statistics and probability","","RNA-Seq","Sequencing","Programming language / platform","No term available","Transcriptomics","Sequence assembly, DNA binding sites, Mapping","DNA, Sequencing, Mapping","Mapping, Sequence analysis, Sequencing, Rare diseases","Statistics and probability","Sequencing, Sequence assembly, DNA polymorphism","Machine learning","Sequencing","Proteomics, Proteomics experiment, Data visualisation","Sequence analysis","Statistics and probability, Sequence analysis, Whole genome sequencing","Sequencing","Genotype and phenotype, Genotyping experiment","Sequence assembly","Data quality management","Natural language processing, Workflows, Machine learning, Molecular interactions, pathways and networks","RNA-Seq, Sequence assembly, Sequencing","DNA, Sequencing, Data visualisation","Bioinformatics, Workflows, Software engineering","DNA, DNA polymorphism, Genetics","Phylogenetics, Workflows, Public health and epidemiology","DNA structural variation, Sequencing","Functional, regulatory and non-coding RNA, RNA, Methylated DNA immunoprecipitation, Epigenetics","DNA polymorphism, Genetic variation, Nucleic acid sites, features and motifs","Sequence assembly","Sequence analysis, Whole genome sequencing","Sequence assembly, Sequencing, Metagenomics","Sequence assembly, DNA polymorphism, Human biology","Transcriptomics","Sequence assembly","Gene transcripts, RNA splicing, Sequence analysis, Whole genome sequencing, Genetic variation","Metagenomics, Metatranscriptomics, RNA-Seq, Microbial ecology","DNA, Genomics, Sequencing","Mapping, Population genetics","RNA-Seq, Transcriptomics","Population genetics","DNA, Metagenomics, Sequence analysis","Sequencing, Whole genome sequencing, Sequence assembly","Genotype and phenotype, DNA structural variation, Sequencing, Mapping, Sequence assembly","Genetic variation, Genomics, Genotype and phenotype, Whole genome sequencing, Oncology","Proteomics experiment, Proteomics","Transcriptomics, Metagenomics","Data quality management","Gene expression, Transcription factors and regulatory sites, Gene transcripts, Small molecules, Gene regulation","GWAS study, Mapping","Metagenomics","","Proteomics, Proteomics experiment, Metabolomics","Sequence assembly, Structural variation","RNA-Seq, Mapping, Rare diseases","Computer science","No term available","Sequence analysis, Sequence composition, complexity and repeats","Sequence analysis","Sequencing, Sequence analysis, Data quality management","Transcriptomics, Gene expression, Gene transcripts","Gene expression, Transcriptomics","RNA, Genomics, Sequence analysis","Sequence assembly","Sequence analysis","Microbiology, Genomics, Sequencing, Sequence assembly","DNA, Sequence analysis, Sequencing","Metagenomics, Sequence assembly","Microbial ecology, Data visualisation, Population genetics","Sequencing, DNA polymorphism","Sequencing","Genetic variation","Sequence assembly","Structure prediction, Nucleic acid structure analysis, RNA","No term available","Metagenomics, Sequence analysis","Sequence assembly, Whole genome sequencing, Human biology","Sequencing","HMMs, DNA","Systems biology, Metabolomics, Computational biology","Sequence assembly, Computer science, Informatics","Mathematics","Biological imaging, Data visualisation, Metabolomics","Data submission, annotation and curation","Proteomics, Proteomics experiment"],["<a href='https://biocontainers.pro/tools/3d-dna' target='_blank'  rel='noopener noreferrer'>3d-dna<\/a>","<a href='https://biocontainers.pro/tools/ABRicate' target='_blank'  rel='noopener noreferrer'>ABRicate<\/a>","<a href='https://biocontainers.pro/tools/abyss' target='_blank'  rel='noopener noreferrer'>abyss<\/a>","","","","<a href='https://biocontainers.pro/tools/amos' target='_blank'  rel='noopener noreferrer'>amos<\/a>","<a href='https://biocontainers.pro/tools/AMR' target='_blank'  rel='noopener noreferrer'>AMR<\/a>","<a href='https://biocontainers.pro/tools/anaconda' target='_blank'  rel='noopener noreferrer'>anaconda<\/a>","<a href='https://biocontainers.pro/tools/angsd' target='_blank'  rel='noopener noreferrer'>angsd<\/a>","<a href='https://biocontainers.pro/tools/anvio' target='_blank'  rel='noopener noreferrer'>anvio<\/a>","<a href='https://biocontainers.pro/tools/apollo' target='_blank'  rel='noopener noreferrer'>apollo<\/a>","<a href='https://biocontainers.pro/tools/artemis' target='_blank'  rel='noopener noreferrer'>artemis<\/a>","<a href='https://biocontainers.pro/tools/augustus' target='_blank'  rel='noopener noreferrer'>augustus<\/a>","<a href='https://biocontainers.pro/tools/bamkit' target='_blank'  rel='noopener noreferrer'>bamkit<\/a>","<a href='https://biocontainers.pro/tools/bamm' target='_blank'  rel='noopener noreferrer'>bamm<\/a>","<a href='https://biocontainers.pro/tools/bamtools' target='_blank'  rel='noopener noreferrer'>bamtools<\/a>","<a href='https://biocontainers.pro/tools/bbmap' target='_blank'  rel='noopener noreferrer'>bbmap<\/a>","<a href='https://biocontainers.pro/tools/bcftools' target='_blank'  rel='noopener noreferrer'>bcftools<\/a>","<a href='https://biocontainers.pro/tools/beagle' target='_blank'  rel='noopener noreferrer'>beagle<\/a>","<a href='https://biocontainers.pro/tools/beagle-lib' target='_blank'  rel='noopener noreferrer'>beagle-lib<\/a>","<a href='https://biocontainers.pro/tools/beast' target='_blank'  rel='noopener noreferrer'>beast<\/a>","<a href='https://biocontainers.pro/tools/beast2' target='_blank'  rel='noopener noreferrer'>beast2<\/a>","<a href='https://biocontainers.pro/tools/bedtools' target='_blank'  rel='noopener noreferrer'>bedtools<\/a>","","<a href='https://biocontainers.pro/tools/binsanity' target='_blank'  rel='noopener noreferrer'>binsanity<\/a>","","<a href='https://biocontainers.pro/tools/bioconductor' target='_blank'  rel='noopener noreferrer'>bioconductor<\/a>","","","<a href='https://biocontainers.pro/tools/bioperl' target='_blank'  rel='noopener noreferrer'>bioperl<\/a>","<a href='https://biocontainers.pro/tools/Biopython' target='_blank'  rel='noopener noreferrer'>Biopython<\/a>","<a href='https://biocontainers.pro/tools/bismark' target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://biocontainers.pro/tools/blast' target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://biocontainers.pro/tools/blast' target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://biocontainers.pro/tools/blast' target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://biocontainers.pro/tools/blastkoala' target='_blank'  rel='noopener noreferrer'>blastkoala<\/a>","<a href='https://biocontainers.pro/tools/blat' target='_blank'  rel='noopener noreferrer'>blat<\/a>","<a href='https://biocontainers.pro/tools/BlobToolKit' target='_blank'  rel='noopener noreferrer'>BlobToolKit<\/a>","<a href='https://biocontainers.pro/tools/blobtools' target='_blank'  rel='noopener noreferrer'>blobtools<\/a>","<a href='https://biocontainers.pro/tools/bmge' target='_blank'  rel='noopener noreferrer'>bmge<\/a>","<a href='https://biocontainers.pro/tools/boost' target='_blank'  rel='noopener noreferrer'>boost<\/a>","<a href='https://biocontainers.pro/tools/bowtie' target='_blank'  rel='noopener noreferrer'>bowtie<\/a>","<a href='https://biocontainers.pro/tools/bowtie2' target='_blank'  rel='noopener noreferrer'>bowtie2<\/a>","<a href='https://biocontainers.pro/tools/braker1' target='_blank'  rel='noopener noreferrer'>braker1<\/a>","<a href='https://biocontainers.pro/tools/braker1' target='_blank'  rel='noopener noreferrer'>braker1<\/a>","<a href='https://biocontainers.pro/tools/breakdancer' target='_blank'  rel='noopener noreferrer'>breakdancer<\/a>","<a href='https://biocontainers.pro/tools/bsmap' target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","<a href='https://biocontainers.pro/tools/bs_seeker' target='_blank'  rel='noopener noreferrer'>bs_seeker<\/a>","<a href='https://biocontainers.pro/tools/bs-seeker2' target='_blank'  rel='noopener noreferrer'>bs-seeker2<\/a>","<a href='https://biocontainers.pro/tools/busco' target='_blank'  rel='noopener noreferrer'>busco<\/a>","<a href='https://biocontainers.pro/tools/bwa' target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://biocontainers.pro/tools/bx-python' target='_blank'  rel='noopener noreferrer'>bx-python<\/a>","<a href='https://biocontainers.pro/tools/cactus' target='_blank'  rel='noopener noreferrer'>cactus<\/a>","","<a href='https://biocontainers.pro/tools/camera' target='_blank'  rel='noopener noreferrer'>camera<\/a>","<a href='https://biocontainers.pro/tools/canu' target='_blank'  rel='noopener noreferrer'>canu<\/a>","<a href='https://biocontainers.pro/tools/cap3' target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='https://biocontainers.pro/tools/cardinal' target='_blank'  rel='noopener noreferrer'>cardinal<\/a>","<a href='https://biocontainers.pro/tools/cat_bins' target='_blank'  rel='noopener noreferrer'>cat_bins<\/a>","<a href='https://biocontainers.pro/tools/cd-hit' target='_blank'  rel='noopener noreferrer'>cd-hit<\/a>","<a href='https://biocontainers.pro/tools/celseq2' target='_blank'  rel='noopener noreferrer'>celseq2<\/a>","<a href='https://biocontainers.pro/tools/cabog' target='_blank'  rel='noopener noreferrer'>cabog<\/a>","<a href='https://biocontainers.pro/tools/CellProfiler' target='_blank'  rel='noopener noreferrer'>CellProfiler<\/a>","<a href='https://biocontainers.pro/tools/centrifuge' target='_blank'  rel='noopener noreferrer'>centrifuge<\/a>","<a href='https://biocontainers.pro/tools/checkm' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://biocontainers.pro/tools/checkm' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://biocontainers.pro/tools/checkv' target='_blank'  rel='noopener noreferrer'>checkv<\/a>","<a href='https://biocontainers.pro/tools/chimeraslayer' target='_blank'  rel='noopener noreferrer'>chimeraslayer<\/a>","","<a href='https://biocontainers.pro/tools/chunkchromosome' target='_blank'  rel='noopener noreferrer'>chunkchromosome<\/a>","","","<a href='https://biocontainers.pro/tools/clustalw' target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","<a href='https://biocontainers.pro/tools/clusteringsuite' target='_blank'  rel='noopener noreferrer'>clusteringsuite<\/a>","<a href='https://biocontainers.pro/tools/cnvkit' target='_blank'  rel='noopener noreferrer'>cnvkit<\/a>","<a href='https://biocontainers.pro/tools/cpc' target='_blank'  rel='noopener noreferrer'>cpc<\/a>","<a href='https://biocontainers.pro/tools/cpc2' target='_blank'  rel='noopener noreferrer'>cpc2<\/a>","<a href='https://biocontainers.pro/tools/comparem' target='_blank'  rel='noopener noreferrer'>comparem<\/a>","<a href='https://biocontainers.pro/tools/coverm' target='_blank'  rel='noopener noreferrer'>coverm<\/a>","<a href='https://biocontainers.pro/tools/cpat' target='_blank'  rel='noopener noreferrer'>cpat<\/a>","","","","<a href='https://biocontainers.pro/tools/cufflinks' target='_blank'  rel='noopener noreferrer'>cufflinks<\/a>","<a href='https://biocontainers.pro/tools/cutadapt' target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='https://biocontainers.pro/tools/cytoscape' target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","<a href='https://biocontainers.pro/tools/dada2' target='_blank'  rel='noopener noreferrer'>dada2<\/a>","<a href='https://biocontainers.pro/tools/dbcan' target='_blank'  rel='noopener noreferrer'>dbcan<\/a>","<a href='https://biocontainers.pro/tools/dbcan2' target='_blank'  rel='noopener noreferrer'>dbcan2<\/a>","<a href='https://biocontainers.pro/tools/deeptools' target='_blank'  rel='noopener noreferrer'>deeptools<\/a>","<a href='https://biocontainers.pro/tools/deepvariant' target='_blank'  rel='noopener noreferrer'>deepvariant<\/a>","<a href='https://biocontainers.pro/tools/dendropy' target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://biocontainers.pro/tools/DIA-Umpire' target='_blank'  rel='noopener noreferrer'>DIA-Umpire<\/a>","<a href='https://biocontainers.pro/tools/diamond' target='_blank'  rel='noopener noreferrer'>diamond<\/a>","<a href='https://biocontainers.pro/tools/discovar_de_novo' target='_blank'  rel='noopener noreferrer'>discovar_de_novo<\/a>","<a href='https://biocontainers.pro/tools/dms_tools2' target='_blank'  rel='noopener noreferrer'>dms_tools2<\/a>","<a href='https://biocontainers.pro/tools/dssp' target='_blank'  rel='noopener noreferrer'>dssp<\/a>","","<a href='https://biocontainers.pro/tools/easybuild' target='_blank'  rel='noopener noreferrer'>easybuild<\/a>","<a href='https://biocontainers.pro/tools/edena' target='_blank'  rel='noopener noreferrer'>edena<\/a>","","<a href='https://biocontainers.pro/tools/eggnog-mapper' target='_blank'  rel='noopener noreferrer'>eggnog-mapper<\/a>","<a href='https://biocontainers.pro/tools/emboss' target='_blank'  rel='noopener noreferrer'>emboss<\/a>","<a href='https://biocontainers.pro/tools/EncyclopeDIA' target='_blank'  rel='noopener noreferrer'>EncyclopeDIA<\/a>","","<a href='https://biocontainers.pro/tools/ensembl' target='_blank'  rel='noopener noreferrer'>ensembl<\/a>","","<a href='https://biocontainers.pro/tools/eval' target='_blank'  rel='noopener noreferrer'>eval<\/a>","<a href='https://biocontainers.pro/tools/EvidenceModeler' target='_blank'  rel='noopener noreferrer'>EvidenceModeler<\/a>","<a href='https://biocontainers.pro/tools/exabayes' target='_blank'  rel='noopener noreferrer'>exabayes<\/a>","<a href='https://biocontainers.pro/tools/exonerate' target='_blank'  rel='noopener noreferrer'>exonerate<\/a>","<a href='https://biocontainers.pro/tools/export2graphlan' target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","<a href='https://biocontainers.pro/tools/falcon' target='_blank'  rel='noopener noreferrer'>falcon<\/a>","<a href='https://biocontainers.pro/tools/falcon' target='_blank'  rel='noopener noreferrer'>falcon<\/a>","<a href='https://biocontainers.pro/tools/fastml' target='_blank'  rel='noopener noreferrer'>fastml<\/a>","<a href='https://biocontainers.pro/tools/fastp' target='_blank'  rel='noopener noreferrer'>fastp<\/a>","<a href='https://biocontainers.pro/tools/fastqc' target='_blank'  rel='noopener noreferrer'>fastqc<\/a>","<a href='https://biocontainers.pro/tools/faststructure' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='https://biocontainers.pro/tools/faststructure' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='https://biocontainers.pro/tools/fastx-toolkit' target='_blank'  rel='noopener noreferrer'>fastx-toolkit<\/a>","<a href='https://biocontainers.pro/tools/fastx-toolkit' target='_blank'  rel='noopener noreferrer'>fastx-toolkit<\/a>","<a href='https://biocontainers.pro/tools/fgenesh' target='_blank'  rel='noopener noreferrer'>fgenesh<\/a>","<a href='https://biocontainers.pro/tools/fimpute' target='_blank'  rel='noopener noreferrer'>fimpute<\/a>","<a href='https://biocontainers.pro/tools/flash' target='_blank'  rel='noopener noreferrer'>flash<\/a>","<a href='https://biocontainers.pro/tools/flashlfq' target='_blank'  rel='noopener noreferrer'>flashlfq<\/a>","<a href='https://biocontainers.pro/tools/Flye' target='_blank'  rel='noopener noreferrer'>Flye<\/a>","<a href='https://biocontainers.pro/tools/focus' target='_blank'  rel='noopener noreferrer'>focus<\/a>","<a href='https://biocontainers.pro/tools/fraggenescan' target='_blank'  rel='noopener noreferrer'>fraggenescan<\/a>","<a href='https://biocontainers.pro/tools/freebayes' target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","<a href='https://biocontainers.pro/tools/freesurfer' target='_blank'  rel='noopener noreferrer'>freesurfer<\/a>","<a href='https://biocontainers.pro/tools/funannotate' target='_blank'  rel='noopener noreferrer'>funannotate<\/a>","<a href='https://biocontainers.pro/tools/garli' target='_blank'  rel='noopener noreferrer'>garli<\/a>","<a href='https://biocontainers.pro/tools/gatb' target='_blank'  rel='noopener noreferrer'>gatb<\/a>","<a href='https://biocontainers.pro/tools/gatk' target='_blank'  rel='noopener noreferrer'>gatk<\/a>","","<a href='https://biocontainers.pro/tools/gblocks' target='_blank'  rel='noopener noreferrer'>gblocks<\/a>","<a href='https://biocontainers.pro/tools/gcta' target='_blank'  rel='noopener noreferrer'>gcta<\/a>","<a href='https://biocontainers.pro/tools/gecko' target='_blank'  rel='noopener noreferrer'>gecko<\/a>","<a href='https://biocontainers.pro/tools/geneid' target='_blank'  rel='noopener noreferrer'>geneid<\/a>","<a href='https://biocontainers.pro/tools/genemark' target='_blank'  rel='noopener noreferrer'>genemark<\/a>","<a href='https://biocontainers.pro/tools/GenomeScope_2.0' target='_blank'  rel='noopener noreferrer'>GenomeScope_2.0<\/a>","<a href='https://biocontainers.pro/tools/GenomicConsensus' target='_blank'  rel='noopener noreferrer'>GenomicConsensus<\/a>","<a href='https://biocontainers.pro/tools/genomicfeatures' target='_blank'  rel='noopener noreferrer'>genomicfeatures<\/a>","<a href='https://biocontainers.pro/tools/ghostkoala' target='_blank'  rel='noopener noreferrer'>ghostkoala<\/a>","<a href='https://biocontainers.pro/tools/gist' target='_blank'  rel='noopener noreferrer'>gist<\/a>","<a href='https://biocontainers.pro/tools/glimmer' target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='https://biocontainers.pro/tools/glimmer-hmm' target='_blank'  rel='noopener noreferrer'>glimmer-hmm<\/a>","<a href='https://biocontainers.pro/tools/glimmermg' target='_blank'  rel='noopener noreferrer'>glimmermg<\/a>","<a href='https://biocontainers.pro/tools/gmap' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='https://biocontainers.pro/tools/gmap' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='https://biocontainers.pro/tools/gmap' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='https://biocontainers.pro/tools/gnps' target='_blank'  rel='noopener noreferrer'>gnps<\/a>","","<a href='https://biocontainers.pro/tools/parallel' target='_blank'  rel='noopener noreferrer'>parallel<\/a>","<a href='https://biocontainers.pro/tools/gsl' target='_blank'  rel='noopener noreferrer'>gsl<\/a>","<a href='https://biocontainers.pro/tools/graftm' target='_blank'  rel='noopener noreferrer'>graftm<\/a>","<a href='https://biocontainers.pro/tools/graphlan' target='_blank'  rel='noopener noreferrer'>graphlan<\/a>","<a href='https://biocontainers.pro/tools/gromacs' target='_blank'  rel='noopener noreferrer'>gromacs<\/a>","<a href='https://biocontainers.pro/tools/groopm' target='_blank'  rel='noopener noreferrer'>groopm<\/a>","<a href='https://biocontainers.pro/tools/GTDB-Tk' target='_blank'  rel='noopener noreferrer'>GTDB-Tk<\/a>","<a href='https://biocontainers.pro/tools/gubbins' target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","","<a href='https://biocontainers.pro/tools/hadoop' target='_blank'  rel='noopener noreferrer'>hadoop<\/a>","","<a href='https://biocontainers.pro/tools/hawkeye' target='_blank'  rel='noopener noreferrer'>hawkeye<\/a>","<a href='https://biocontainers.pro/tools/hclust2' target='_blank'  rel='noopener noreferrer'>hclust2<\/a>","","","","","<a href='https://biocontainers.pro/tools/hirise' target='_blank'  rel='noopener noreferrer'>hirise<\/a>","<a href='https://biocontainers.pro/tools/hisat2' target='_blank'  rel='noopener noreferrer'>hisat2<\/a>","<a href='https://biocontainers.pro/tools/hmmer3' target='_blank'  rel='noopener noreferrer'>hmmer3<\/a>","<a href='https://biocontainers.pro/tools/horovod' target='_blank'  rel='noopener noreferrer'>horovod<\/a>","","","<a href='https://biocontainers.pro/tools/htseq' target='_blank'  rel='noopener noreferrer'>htseq<\/a>","<a href='https://biocontainers.pro/tools/htslib' target='_blank'  rel='noopener noreferrer'>htslib<\/a>","","<a href='https://biocontainers.pro/tools/humann2' target='_blank'  rel='noopener noreferrer'>humann2<\/a>","","<a href='https://biocontainers.pro/tools/idba' target='_blank'  rel='noopener noreferrer'>idba<\/a>","","","<a href='https://biocontainers.pro/tools/img_m' target='_blank'  rel='noopener noreferrer'>img_m<\/a>","","","","","<a href='https://biocontainers.pro/tools/interproscan4' target='_blank'  rel='noopener noreferrer'>interproscan4<\/a>","","<a href='https://biocontainers.pro/tools/iq-tree' target='_blank'  rel='noopener noreferrer'>iq-tree<\/a>","","<a href='https://biocontainers.pro/tools/isoseq3' target='_blank'  rel='noopener noreferrer'>isoseq3<\/a>","<a href='https://biocontainers.pro/tools/iwtomics' target='_blank'  rel='noopener noreferrer'>iwtomics<\/a>","<a href='https://biocontainers.pro/tools/jags' target='_blank'  rel='noopener noreferrer'>jags<\/a>","<a href='https://biocontainers.pro/tools/jamg' target='_blank'  rel='noopener noreferrer'>jamg<\/a>","<a href='https://biocontainers.pro/tools/jbrowse' target='_blank'  rel='noopener noreferrer'>jbrowse<\/a>","","<a href='https://biocontainers.pro/tools/jellyfish' target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","","<a href='https://biocontainers.pro/tools/juicebox' target='_blank'  rel='noopener noreferrer'>juicebox<\/a>","<a href='https://biocontainers.pro/tools/juicer' target='_blank'  rel='noopener noreferrer'>juicer<\/a>","<a href='https://biocontainers.pro/tools/kaas' target='_blank'  rel='noopener noreferrer'>kaas<\/a>","<a href='https://biocontainers.pro/tools/kallisto' target='_blank'  rel='noopener noreferrer'>kallisto<\/a>","<a href='https://biocontainers.pro/tools/kat' target='_blank'  rel='noopener noreferrer'>kat<\/a>","<a href='https://biocontainers.pro/tools/khmer' target='_blank'  rel='noopener noreferrer'>khmer<\/a>","<a href='https://biocontainers.pro/tools/kingfisher' target='_blank'  rel='noopener noreferrer'>kingfisher<\/a>","<a href='https://biocontainers.pro/tools/kmc' target='_blank'  rel='noopener noreferrer'>kmc<\/a>","<a href='https://biocontainers.pro/tools/KofamKOALA' target='_blank'  rel='noopener noreferrer'>KofamKOALA<\/a>","<a href='https://biocontainers.pro/tools/kraken' target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://biocontainers.pro/tools/kraken2' target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","<a href='https://biocontainers.pro/tools/krona' target='_blank'  rel='noopener noreferrer'>krona<\/a>","<a href='https://biocontainers.pro/tools/lastz' target='_blank'  rel='noopener noreferrer'>lastz<\/a>","<a href='https://biocontainers.pro/tools/LFQ-Analyst' target='_blank'  rel='noopener noreferrer'>LFQ-Analyst<\/a>","","<a href='https://biocontainers.pro/tools/links' target='_blank'  rel='noopener noreferrer'>links<\/a>","<a href='https://biocontainers.pro/tools/lipid_maps' target='_blank'  rel='noopener noreferrer'>lipid_maps<\/a>","<a href='https://biocontainers.pro/tools/longranger' target='_blank'  rel='noopener noreferrer'>longranger<\/a>","","<a href='https://biocontainers.pro/tools/mach' target='_blank'  rel='noopener noreferrer'>mach<\/a>","<a href='https://biocontainers.pro/tools/macs' target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='https://biocontainers.pro/tools/macs' target='_blank'  rel='noopener noreferrer'>macs<\/a>","","<a href='https://biocontainers.pro/tools/MAFFT' target='_blank'  rel='noopener noreferrer'>MAFFT<\/a>","<a href='https://biocontainers.pro/tools/magpy' target='_blank'  rel='noopener noreferrer'>magpy<\/a>","<a href='https://biocontainers.pro/tools/maker' target='_blank'  rel='noopener noreferrer'>maker<\/a>","<a href='https://biocontainers.pro/tools/maker-p' target='_blank'  rel='noopener noreferrer'>maker-p<\/a>","<a href='https://biocontainers.pro/tools/maldi_quant' target='_blank'  rel='noopener noreferrer'>maldi_quant<\/a>","<a href='https://biocontainers.pro/tools/mash' target='_blank'  rel='noopener noreferrer'>mash<\/a>","<a href='https://biocontainers.pro/tools/mashmap' target='_blank'  rel='noopener noreferrer'>mashmap<\/a>","<a href='https://biocontainers.pro/tools/maskrc-svg' target='_blank'  rel='noopener noreferrer'>maskrc-svg<\/a>","<a href='https://biocontainers.pro/tools/masurca' target='_blank'  rel='noopener noreferrer'>masurca<\/a>","<a href='https://biocontainers.pro/tools/mauve' target='_blank'  rel='noopener noreferrer'>mauve<\/a>","<a href='https://biocontainers.pro/tools/maxbin' target='_blank'  rel='noopener noreferrer'>maxbin<\/a>","<a href='https://biocontainers.pro/tools/maxquant' target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://biocontainers.pro/tools/maxquant' target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://biocontainers.pro/tools/mcl' target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://biocontainers.pro/tools/medaka' target='_blank'  rel='noopener noreferrer'>medaka<\/a>","<a href='https://biocontainers.pro/tools/megahit' target='_blank'  rel='noopener noreferrer'>megahit<\/a>","<a href='https://biocontainers.pro/tools/megan' target='_blank'  rel='noopener noreferrer'>megan<\/a>","<a href='https://biocontainers.pro/tools/merqury' target='_blank'  rel='noopener noreferrer'>merqury<\/a>","","<a href='https://biocontainers.pro/tools/idba' target='_blank'  rel='noopener noreferrer'>idba<\/a>","<a href='https://biocontainers.pro/tools/MetaBAT_2' target='_blank'  rel='noopener noreferrer'>MetaBAT_2<\/a>","<a href='https://biocontainers.pro/tools/metaboanalyst' target='_blank'  rel='noopener noreferrer'>metaboanalyst<\/a>","<a href='https://biocontainers.pro/tools/metacluster' target='_blank'  rel='noopener noreferrer'>metacluster<\/a>","<a href='https://biocontainers.pro/tools/metacv' target='_blank'  rel='noopener noreferrer'>metacv<\/a>","<a href='https://biocontainers.pro/tools/MetaDEGalaxy' target='_blank'  rel='noopener noreferrer'>MetaDEGalaxy<\/a>","<a href='https://biocontainers.pro/tools/metageneannotator' target='_blank'  rel='noopener noreferrer'>metageneannotator<\/a>","","<a href='https://biocontainers.pro/tools/metagenomeseq' target='_blank'  rel='noopener noreferrer'>metagenomeseq<\/a>","<a href='https://biocontainers.pro/tools/metamaps' target='_blank'  rel='noopener noreferrer'>metamaps<\/a>","<a href='https://biocontainers.pro/tools/metams' target='_blank'  rel='noopener noreferrer'>metams<\/a>","<a href='https://biocontainers.pro/tools/metapath' target='_blank'  rel='noopener noreferrer'>metapath<\/a>","<a href='https://biocontainers.pro/tools/metaphlan' target='_blank'  rel='noopener noreferrer'>metaphlan<\/a>","<a href='https://biocontainers.pro/tools/metaphyler' target='_blank'  rel='noopener noreferrer'>metaphyler<\/a>","<a href='https://biocontainers.pro/tools/metaQuantome' target='_blank'  rel='noopener noreferrer'>metaQuantome<\/a>","<a href='https://biocontainers.pro/tools/metaspades' target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","<a href='https://biocontainers.pro/tools/metavelvet' target='_blank'  rel='noopener noreferrer'>metavelvet<\/a>","<a href='https://biocontainers.pro/tools/MetaWRAP' target='_blank'  rel='noopener noreferrer'>MetaWRAP<\/a>","<a href='https://biocontainers.pro/tools/mg-rast' target='_blank'  rel='noopener noreferrer'>mg-rast<\/a>","<a href='https://biocontainers.pro/tools/MGnify' target='_blank'  rel='noopener noreferrer'>MGnify<\/a>","<a href='https://biocontainers.pro/tools/MIGRATE' target='_blank'  rel='noopener noreferrer'>MIGRATE<\/a>","","<a href='https://biocontainers.pro/tools/minia' target='_blank'  rel='noopener noreferrer'>minia<\/a>","<a href='https://biocontainers.pro/tools/minimac4' target='_blank'  rel='noopener noreferrer'>minimac4<\/a>","<a href='https://biocontainers.pro/tools/minimap2' target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","<a href='https://biocontainers.pro/tools/minpath' target='_blank'  rel='noopener noreferrer'>minpath<\/a>","<a href='https://biocontainers.pro/tools/mira' target='_blank'  rel='noopener noreferrer'>mira<\/a>","","<a href='https://biocontainers.pro/tools/mirdeep' target='_blank'  rel='noopener noreferrer'>mirdeep<\/a>","<a href='https://biocontainers.pro/tools/mirdeep2' target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","<a href='https://biocontainers.pro/tools/miso' target='_blank'  rel='noopener noreferrer'>miso<\/a>","<a href='https://biocontainers.pro/tools/mixomics' target='_blank'  rel='noopener noreferrer'>mixomics<\/a>","<a href='https://biocontainers.pro/tools/mlst' target='_blank'  rel='noopener noreferrer'>mlst<\/a>","<a href='https://biocontainers.pro/tools/mocat' target='_blank'  rel='noopener noreferrer'>mocat<\/a>","<a href='https://biocontainers.pro/tools/moff' target='_blank'  rel='noopener noreferrer'>moff<\/a>","<a href='https://biocontainers.pro/tools/morpheus' target='_blank'  rel='noopener noreferrer'>morpheus<\/a>","<a href='https://biocontainers.pro/tools/Mothur' target='_blank'  rel='noopener noreferrer'>Mothur<\/a>","","<a href='https://biocontainers.pro/tools/mrbayes' target='_blank'  rel='noopener noreferrer'>mrbayes<\/a>","<a href='https://biocontainers.pro/tools/ms_dial' target='_blank'  rel='noopener noreferrer'>ms_dial<\/a>","<a href='https://biocontainers.pro/tools/msgfplus' target='_blank'  rel='noopener noreferrer'>msgfplus<\/a>","<a href='https://biocontainers.pro/tools/msnbase' target='_blank'  rel='noopener noreferrer'>msnbase<\/a>","<a href='https://biocontainers.pro/tools/msstats' target='_blank'  rel='noopener noreferrer'>msstats<\/a>","<a href='https://biocontainers.pro/tools/multiqc' target='_blank'  rel='noopener noreferrer'>multiqc<\/a>","<a href='https://biocontainers.pro/tools/multisnv' target='_blank'  rel='noopener noreferrer'>multisnv<\/a>","<a href='https://biocontainers.pro/tools/mummer' target='_blank'  rel='noopener noreferrer'>mummer<\/a>","<a href='https://biocontainers.pro/tools/muscle' target='_blank'  rel='noopener noreferrer'>muscle<\/a>","","<a href='https://biocontainers.pro/tools/nanopolish' target='_blank'  rel='noopener noreferrer'>nanopolish<\/a>","<a href='https://biocontainers.pro/tools/nast-ier' target='_blank'  rel='noopener noreferrer'>nast-ier<\/a>","<a href='https://biocontainers.pro/tools/nccmp' target='_blank'  rel='noopener noreferrer'>nccmp<\/a>","","","<a href='https://biocontainers.pro/tools/nco' target='_blank'  rel='noopener noreferrer'>nco<\/a>","<a href='https://biocontainers.pro/tools/ncview' target='_blank'  rel='noopener noreferrer'>ncview<\/a>","<a href='https://biocontainers.pro/tools/necat' target='_blank'  rel='noopener noreferrer'>necat<\/a>","<a href='https://biocontainers.pro/tools/netcdf' target='_blank'  rel='noopener noreferrer'>netcdf<\/a>","<a href='https://biocontainers.pro/tools/neurodebian' target='_blank'  rel='noopener noreferrer'>neurodebian<\/a>","<a href='https://biocontainers.pro/tools/nextdenovo' target='_blank'  rel='noopener noreferrer'>nextdenovo<\/a>","","<a href='https://biocontainers.pro/tools/nextgenmap' target='_blank'  rel='noopener noreferrer'>nextgenmap<\/a>","<a href='https://biocontainers.pro/tools/NextPolish' target='_blank'  rel='noopener noreferrer'>NextPolish<\/a>","","<a href='https://biocontainers.pro/tools/ngsplot' target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","","","<a href='https://biocontainers.pro/tools/nvbio' target='_blank'  rel='noopener noreferrer'>nvbio<\/a>","<a href='https://biocontainers.pro/tools/oases' target='_blank'  rel='noopener noreferrer'>oases<\/a>","<a href='https://biocontainers.pro/tools/obitools' target='_blank'  rel='noopener noreferrer'>obitools<\/a>","<a href='https://biocontainers.pro/tools/openms' target='_blank'  rel='noopener noreferrer'>openms<\/a>","","","<a href='https://biocontainers.pro/tools/OTUreporter' target='_blank'  rel='noopener noreferrer'>OTUreporter<\/a>","<a href='https://biocontainers.pro/tools/pepper-deepvariant' target='_blank'  rel='noopener noreferrer'>pepper-deepvariant<\/a>","<a href='https://biocontainers.pro/tools/pacbioeda' target='_blank'  rel='noopener noreferrer'>pacbioeda<\/a>","","","<a href='https://biocontainers.pro/tools/pandaseq' target='_blank'  rel='noopener noreferrer'>pandaseq<\/a>","","<a href='https://biocontainers.pro/tools/paraview' target='_blank'  rel='noopener noreferrer'>paraview<\/a>","<a href='https://biocontainers.pro/tools/parsnp' target='_blank'  rel='noopener noreferrer'>parsnp<\/a>","<a href='https://biocontainers.pro/tools/PASA' target='_blank'  rel='noopener noreferrer'>PASA<\/a>","<a href='https://biocontainers.pro/tools/pathview' target='_blank'  rel='noopener noreferrer'>pathview<\/a>","<a href='https://biocontainers.pro/tools/pear' target='_blank'  rel='noopener noreferrer'>pear<\/a>","<a href='https://biocontainers.pro/tools/peptideshaker' target='_blank'  rel='noopener noreferrer'>peptideshaker<\/a>","<a href='https://biocontainers.pro/tools/percolator' target='_blank'  rel='noopener noreferrer'>percolator<\/a>","","<a href='https://biocontainers.pro/tools/phyloseq' target='_blank'  rel='noopener noreferrer'>phyloseq<\/a>","<a href='https://biocontainers.pro/tools/phylosift' target='_blank'  rel='noopener noreferrer'>phylosift<\/a>","<a href='https://biocontainers.pro/tools/phyml' target='_blank'  rel='noopener noreferrer'>phyml<\/a>","","<a href='https://biocontainers.pro/tools/picard_tools' target='_blank'  rel='noopener noreferrer'>picard_tools<\/a>","<a href='https://biocontainers.pro/tools/picrust' target='_blank'  rel='noopener noreferrer'>picrust<\/a>","<a href='https://biocontainers.pro/tools/picrust2' target='_blank'  rel='noopener noreferrer'>picrust2<\/a>","<a href='https://biocontainers.pro/tools/piler-df' target='_blank'  rel='noopener noreferrer'>piler-df<\/a>","<a href='https://biocontainers.pro/tools/pilon' target='_blank'  rel='noopener noreferrer'>pilon<\/a>","<a href='https://biocontainers.pro/tools/plink' target='_blank'  rel='noopener noreferrer'>plink<\/a>","<a href='https://biocontainers.pro/tools/plink-seq' target='_blank'  rel='noopener noreferrer'>plink-seq<\/a>","<a href='https://biocontainers.pro/tools/pnetcdf' target='_blank'  rel='noopener noreferrer'>pnetcdf<\/a>","<a href='https://biocontainers.pro/tools/clstutils' target='_blank'  rel='noopener noreferrer'>clstutils<\/a>","<a href='https://biocontainers.pro/tools/pretextview' target='_blank'  rel='noopener noreferrer'>pretextview<\/a>","<a href='https://biocontainers.pro/tools/prinseq' target='_blank'  rel='noopener noreferrer'>prinseq<\/a>","","<a href='https://biocontainers.pro/tools/probmetab' target='_blank'  rel='noopener noreferrer'>probmetab<\/a>","<a href='https://biocontainers.pro/tools/gmv' target='_blank'  rel='noopener noreferrer'>gmv<\/a>","<a href='https://biocontainers.pro/tools/prokka' target='_blank'  rel='noopener noreferrer'>prokka<\/a>","<a href='https://biocontainers.pro/tools/proovread' target='_blank'  rel='noopener noreferrer'>proovread<\/a>","<a href='https://biocontainers.pro/tools/psortb' target='_blank'  rel='noopener noreferrer'>psortb<\/a>","<a href='https://biocontainers.pro/tools/psortm' target='_blank'  rel='noopener noreferrer'>psortm<\/a>","<a href='https://biocontainers.pro/tools/purge_dups' target='_blank'  rel='noopener noreferrer'>purge_dups<\/a>","<a href='https://biocontainers.pro/tools/purgehaplotigs' target='_blank'  rel='noopener noreferrer'>purgehaplotigs<\/a>","<a href='https://biocontainers.pro/tools/pyani' target='_blank'  rel='noopener noreferrer'>pyani<\/a>","<a href='https://biocontainers.pro/tools/pyrad' target='_blank'  rel='noopener noreferrer'>pyrad<\/a>","<a href='https://biocontainers.pro/tools/pysam' target='_blank'  rel='noopener noreferrer'>pysam<\/a>","<a href='https://biocontainers.pro/tools/pytorch' target='_blank'  rel='noopener noreferrer'>pytorch<\/a>","<a href='https://biocontainers.pro/tools/qiime' target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://biocontainers.pro/tools/qiime' target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://biocontainers.pro/tools/qiime2' target='_blank'  rel='noopener noreferrer'>qiime2<\/a>","<a href='https://biocontainers.pro/tools/quast' target='_blank'  rel='noopener noreferrer'>quast<\/a>","<a href='https://biocontainers.pro/tools/quickmerge' target='_blank'  rel='noopener noreferrer'>quickmerge<\/a>","<a href='https://biocontainers.pro/tools/r' target='_blank'  rel='noopener noreferrer'>r<\/a>","<a href='https://biocontainers.pro/tools/Racon' target='_blank'  rel='noopener noreferrer'>Racon<\/a>","<a href='https://biocontainers.pro/tools/rails_cobbler' target='_blank'  rel='noopener noreferrer'>rails_cobbler<\/a>","<a href='https://biocontainers.pro/tools/ratt' target='_blank'  rel='noopener noreferrer'>ratt<\/a>","<a href='https://biocontainers.pro/tools/rattle' target='_blank'  rel='noopener noreferrer'>rattle<\/a>","<a href='https://biocontainers.pro/tools/raven' target='_blank'  rel='noopener noreferrer'>raven<\/a>","<a href='https://biocontainers.pro/tools/rawtools' target='_blank'  rel='noopener noreferrer'>rawtools<\/a>","<a href='https://biocontainers.pro/tools/raxml' target='_blank'  rel='noopener noreferrer'>raxml<\/a>","","<a href='https://biocontainers.pro/tools/refinem' target='_blank'  rel='noopener noreferrer'>refinem<\/a>","<a href='https://biocontainers.pro/tools/repdenovo' target='_blank'  rel='noopener noreferrer'>repdenovo<\/a>","<a href='https://biocontainers.pro/tools/repeatmasker' target='_blank'  rel='noopener noreferrer'>repeatmasker<\/a>","<a href='https://biocontainers.pro/tools/repeatmodeler' target='_blank'  rel='noopener noreferrer'>repeatmodeler<\/a>","<a href='https://biocontainers.pro/tools/repeatmodeler2' target='_blank'  rel='noopener noreferrer'>repeatmodeler2<\/a>","<a href='https://biocontainers.pro/tools/repeatrunner' target='_blank'  rel='noopener noreferrer'>repeatrunner<\/a>","<a href='https://biocontainers.pro/tools/repet' target='_blank'  rel='noopener noreferrer'>repet<\/a>","<a href='https://biocontainers.pro/tools/rjags' target='_blank'  rel='noopener noreferrer'>rjags<\/a>","","<a href='https://biocontainers.pro/tools/rsem' target='_blank'  rel='noopener noreferrer'>rsem<\/a>","<a href='https://biocontainers.pro/tools/rseqc' target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","","<a href='https://biocontainers.pro/tools/saige' target='_blank'  rel='noopener noreferrer'>saige<\/a>","","<a href='https://biocontainers.pro/tools/SALSA' target='_blank'  rel='noopener noreferrer'>SALSA<\/a>","<a href='https://biocontainers.pro/tools/samblaster' target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='https://biocontainers.pro/tools/samtools' target='_blank'  rel='noopener noreferrer'>samtools<\/a>","<a href='https://biocontainers.pro/tools/sas-access' target='_blank'  rel='noopener noreferrer'>sas-access<\/a>","<a href='https://biocontainers.pro/tools/scarpa' target='_blank'  rel='noopener noreferrer'>scarpa<\/a>","","<a href='https://biocontainers.pro/tools/screed' target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='https://biocontainers.pro/tools/searchgui' target='_blank'  rel='noopener noreferrer'>searchgui<\/a>","","<a href='https://biocontainers.pro/tools/seqstats' target='_blank'  rel='noopener noreferrer'>seqstats<\/a>","<a href='https://biocontainers.pro/tools/seqtk-trinity' target='_blank'  rel='noopener noreferrer'>seqtk-trinity<\/a>","<a href='https://biocontainers.pro/tools/shapeit' target='_blank'  rel='noopener noreferrer'>shapeit<\/a>","","<a href='https://biocontainers.pro/tools/sickle' target='_blank'  rel='noopener noreferrer'>sickle<\/a>","<a href='https://biocontainers.pro/tools/simtext' target='_blank'  rel='noopener noreferrer'>simtext<\/a>","<a href='https://biocontainers.pro/tools/skewer' target='_blank'  rel='noopener noreferrer'>skewer<\/a>","<a href='https://biocontainers.pro/tools/smrt_analysis' target='_blank'  rel='noopener noreferrer'>smrt_analysis<\/a>","<a href='https://biocontainers.pro/tools/snakemake' target='_blank'  rel='noopener noreferrer'>snakemake<\/a>","<a href='https://biocontainers.pro/tools/snap' target='_blank'  rel='noopener noreferrer'>snap<\/a>","<a href='https://biocontainers.pro/tools/SNAPPy' target='_blank'  rel='noopener noreferrer'>SNAPPy<\/a>","<a href='https://biocontainers.pro/tools/sniffles' target='_blank'  rel='noopener noreferrer'>sniffles<\/a>","<a href='https://biocontainers.pro/tools/snoscan' target='_blank'  rel='noopener noreferrer'>snoscan<\/a>","<a href='https://biocontainers.pro/tools/snpeff' target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","<a href='https://biocontainers.pro/tools/soap' target='_blank'  rel='noopener noreferrer'>soap<\/a>","<a href='https://biocontainers.pro/tools/soap3-gpu' target='_blank'  rel='noopener noreferrer'>soap3-gpu<\/a>","<a href='https://biocontainers.pro/tools/soapdenovo2' target='_blank'  rel='noopener noreferrer'>soapdenovo2<\/a>","<a href='https://biocontainers.pro/tools/soapsnp' target='_blank'  rel='noopener noreferrer'>soapsnp<\/a>","<a href='https://biocontainers.pro/tools/sortmerna' target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='https://biocontainers.pro/tools/spades' target='_blank'  rel='noopener noreferrer'>spades<\/a>","<a href='https://biocontainers.pro/tools/splign' target='_blank'  rel='noopener noreferrer'>splign<\/a>","<a href='https://biocontainers.pro/tools/squeezemeta' target='_blank'  rel='noopener noreferrer'>squeezemeta<\/a>","<a href='https://biocontainers.pro/tools/sra-tools' target='_blank'  rel='noopener noreferrer'>sra-tools<\/a>","<a href='https://biocontainers.pro/tools/stacks' target='_blank'  rel='noopener noreferrer'>stacks<\/a>","<a href='https://biocontainers.pro/tools/star' target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='https://biocontainers.pro/tools/structure' target='_blank'  rel='noopener noreferrer'>structure<\/a>","<a href='https://biocontainers.pro/tools/super-focus' target='_blank'  rel='noopener noreferrer'>super-focus<\/a>","<a href='https://biocontainers.pro/tools/supernova' target='_blank'  rel='noopener noreferrer'>supernova<\/a>","<a href='https://biocontainers.pro/tools/SVJedi' target='_blank'  rel='noopener noreferrer'>SVJedi<\/a>","<a href='https://biocontainers.pro/tools/svtyper' target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","<a href='https://biocontainers.pro/tools/swathxtend' target='_blank'  rel='noopener noreferrer'>swathxtend<\/a>","<a href='https://biocontainers.pro/tools/tagcleaner' target='_blank'  rel='noopener noreferrer'>tagcleaner<\/a>","<a href='https://biocontainers.pro/tools/tagdust' target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://biocontainers.pro/tools/targetfinder.org' target='_blank'  rel='noopener noreferrer'>targetfinder.org<\/a>","<a href='https://biocontainers.pro/tools/tassel' target='_blank'  rel='noopener noreferrer'>tassel<\/a>","","<a href='https://biocontainers.pro/tools/TensorFlow' target='_blank'  rel='noopener noreferrer'>TensorFlow<\/a>","<a href='https://biocontainers.pro/tools/ThermoRawFileParser' target='_blank'  rel='noopener noreferrer'>ThermoRawFileParser<\/a>","<a href='https://biocontainers.pro/tools/tigmint' target='_blank'  rel='noopener noreferrer'>tigmint<\/a>","<a href='https://biocontainers.pro/tools/tophat' target='_blank'  rel='noopener noreferrer'>tophat<\/a>","<a href='https://biocontainers.pro/tools/totalview' target='_blank'  rel='noopener noreferrer'>totalview<\/a>","<a href='https://biocontainers.pro/tools/TransDecoder' target='_blank'  rel='noopener noreferrer'>TransDecoder<\/a>","<a href='https://biocontainers.pro/tools/trf' target='_blank'  rel='noopener noreferrer'>trf<\/a>","<a href='https://biocontainers.pro/tools/trim_galore' target='_blank'  rel='noopener noreferrer'>trim_galore<\/a>","<a href='https://biocontainers.pro/tools/trimmomatic' target='_blank'  rel='noopener noreferrer'>trimmomatic<\/a>","<a href='https://biocontainers.pro/tools/trinity' target='_blank'  rel='noopener noreferrer'>trinity<\/a>","<a href='https://biocontainers.pro/tools/trinotate' target='_blank'  rel='noopener noreferrer'>trinotate<\/a>","<a href='https://biocontainers.pro/tools/trnascan-se' target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","","<a href='https://biocontainers.pro/tools/UCSC_Genome_Browser_Utilities' target='_blank'  rel='noopener noreferrer'>UCSC_Genome_Browser_Utilities<\/a>","<a href='https://biocontainers.pro/tools/unicycler' target='_blank'  rel='noopener noreferrer'>unicycler<\/a>","<a href='https://biocontainers.pro/tools/usearch' target='_blank'  rel='noopener noreferrer'>usearch<\/a>","<a href='https://biocontainers.pro/tools/valet' target='_blank'  rel='noopener noreferrer'>valet<\/a>","<a href='https://biocontainers.pro/tools/vamps' target='_blank'  rel='noopener noreferrer'>vamps<\/a>","<a href='https://biocontainers.pro/tools/varscan' target='_blank'  rel='noopener noreferrer'>varscan<\/a>","<a href='https://biocontainers.pro/tools/vawk' target='_blank'  rel='noopener noreferrer'>vawk<\/a>","<a href='https://biocontainers.pro/tools/vcftools' target='_blank'  rel='noopener noreferrer'>vcftools<\/a>","<a href='https://biocontainers.pro/tools/velvet' target='_blank'  rel='noopener noreferrer'>velvet<\/a>","<a href='https://biocontainers.pro/tools/viennarna' target='_blank'  rel='noopener noreferrer'>viennarna<\/a>","<a href='https://biocontainers.pro/tools/visit' target='_blank'  rel='noopener noreferrer'>visit<\/a>","<a href='https://biocontainers.pro/tools/vsearch' target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","<a href='https://biocontainers.pro/tools/WENGAN' target='_blank'  rel='noopener noreferrer'>WENGAN<\/a>","<a href='https://biocontainers.pro/tools/windowmasker' target='_blank'  rel='noopener noreferrer'>windowmasker<\/a>","<a href='https://biocontainers.pro/tools/wise' target='_blank'  rel='noopener noreferrer'>wise<\/a>","<a href='https://biocontainers.pro/tools/workflow4metabolomics' target='_blank'  rel='noopener noreferrer'>workflow4metabolomics<\/a>","<a href='https://biocontainers.pro/tools/wtdbg2' target='_blank'  rel='noopener noreferrer'>wtdbg2<\/a>","","<a href='https://biocontainers.pro/tools/xcms' target='_blank'  rel='noopener noreferrer'>xcms<\/a>","","<a href='https://biocontainers.pro/tools/xtandem' target='_blank'  rel='noopener noreferrer'>xtandem<\/a>"],["MIT","GPL-2.0",null,null,null,null,null,"GPL-2.0",null,"GPL-2.0","GPL-3.0",null,null,"Artistic-1.0","MIT","AGPL-3.0","MIT","BSD-3-Clause","MIT","GPL-3.0","GPL-3.0",null,null,"GPL-2.0",null,"GPL-3.0",null,null,null,null,null,null,"GPL-3.0",null,null,null,null,null,"MIT","GPL-3.0","GPL-2.0","Other",null,"GPL-3.0",null,null,"GPL-3.0","GPL-3.0",null,"MIT",null,"MIT","MIT",null,null,"GPL-2.0",null,null,"Artistic-2.0","MIT",null,"BSD-2-Clause",null,"BSD-3-Clause","GPL-3.0","GPL-3.0","GPL-3.0",null,null,null,null,null,null,null,null,"BSD-3-Clause","GPL-2.0",null,"GPL-3.0","GPL-3.0","GPL-3.0",null,null,null,"BSL-1.0","MIT",null,"GPL-3.0",null,null,"GPL-3.0","BSD-3-Clause","BSD-3-Clause","Apache-2.0","AGPL-3.0",null,"GPL-3.0","BSD-2-Clause",null,"GPL-2.0","GPL-3.0",null,"GPL-3.0",null,"Apache-2.0",null,null,null,null,null,"GPL-3.0","GPL-3.0","MIT",null,null,null,null,"GPL-3.0","MIT","MIT","AGPL-3.0","AGPL-3.0",null,"Not licensed",null,"LGPL-3.0",null,"GPL-3.0","GPL-3.0","MIT","GPL-3.0","BSD-2-Clause","GPL-3.0",null,null,null,null,"MIT","GPL-3.0","GPL-3.0",null,"Apache-2.0",null,"Artistic-2.0",null,null,null,"Artistic-1.0","Other",null,null,null,"Other",null,"GPL-3.0","GPL-3.0","GPL-3.0","MIT","LGPL-2.1","GPL-3.0","GPL-3.0","GPL-2.0",null,"Apache-2.0",null,"Artistic-2.0","MIT",null,null,null,null,"Not licensed","GPL-3.0","Other","Apache-2.0",null,null,"GPL-3.0","MIT",null,null,null,"Unlicense",null,null,null,null,null,null,null,null,null,null,null,"BSD-3-Clause-Clear","GPL-2.0","GPL-2.0","MPL-2.0",null,null,"GPL-3.0",null,"MIT","MIT",null,"BSD-2-Clause","GPL-3.0","BSD-3-Clause","GPL-3.0",null,null,"GFDL-1.3","MIT","Proprietary",null,"GPL-3.0",null,"GPL-3.0",null,"Proprietary",null,"Not licensed","Artistic-2.0","Artistic-2.0",null,"BSD-Source-Code",null,"Artistic-2.0","Artistic-2.0","GPL-3.0","CC-BY-4.0","Freeware","GPL-3.0",null,"GPL-3.0",null,null,null,null,"MPL-2.0",null,null,null,null,"Unlicense",null,"GPL-3.0",null,null,null,null,null,"Artistic-2.0","Freeware","GPL-2.0","Apache-2.0","MIT",null,null,null,null,null,null,null,"MIT",null,"CECILL-2.0","GPL-3.0","MIT",null,"GPL-3.0",null,null,"GPL-3.0",null,"GPL-3.0","Other","GPL-3.0",null,"MIT",null,null,"GPL-3.0",null,"GPL-2.0","Artistic-2.0",null,"GPL-3.0","GPL-3.0","Artistic-2.0","Other",null,"MIT","BSD-3-Clause","GPL-2.0",null,null,"BSD-3-Clause","GPL-1.0",null,null,null,null,null,"MIT",null,null,null,null,null,"BSD-3-Clause","GPL-3.0",null,"BSD-3-Clause",null,null,null,"MIT","GPL-3.0",null,null,"GPL-3.0",null,"BSD-3-Clause","BSD-3-Clause",null,"GPL-3.0","CC-BY-NC-1.0","Apache-2.0",null,null,"GPL-3.0","GPL-3.0","GPL-2.0",null,"MIT",null,"GPL-3.0",null,null,"GPL-2.0","Not licensed","Freeware","GPL-3.0","MIT","GPL-3.0",null,"GPL-2.0",null,null,null,null,"GPL-3.0","MIT","MIT","MIT",null,"MIT","BSD-3-Clause",null,null,"BSD-3-Clause","GPL-2.0","GPL-3.0",null,"MIT","GPL-3.0",null,"GPL-3.0","MIT","Apache-2.0",null,null,"GPL-3.0",null,"OSL-2.1","OSL-2.1",null,null,"CECILL-2.0","GPL-2.0",null,null,null,null,null,null,"MIT",null,"MIT","Proprietary","GPL-3.0",null,"BSD-3-Clause",null,null,"GPL-3.0","MIT","Other",null,"MIT",null,null,null,null,null,"MIT","MIT",null,null,"GPL-3.0","GPL-2.0","GPL-3.0",null,null,"GPL-2.0","Freeware","GPL-3.0",null,"GPL-3.0","GPL-3.0","Not licensed",null,"Proprietary","AGPL-3.0","MIT","GPL-2.0","GPL-3.0","GPL-3.0",null,"LGPL-2.1",null,null,"Apache-2.0","GPL-3.0",null,"Proprietary",null,"Other",null,null,null,null,null,null,"Other","GPL-3.0",null,"MIT",null,null,"MIT","GPL-3.0","GPL-3.0","Freeware",null,"GPL-3.0","AGPL-3.0",null,null,null,"GPL-3.0",null,"GPL-2.0",null,null],["","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abricate'  target='_blank'  rel='noopener noreferrer'>abricate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abyss'  target='_blank'  rel='noopener noreferrer'>abyss<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=amrplusplus'  target='_blank'  rel='noopener noreferrer'>amrplusplus<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=apollo'  target='_blank'  rel='noopener noreferrer'>apollo<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=augustus'  target='_blank'  rel='noopener noreferrer'>augustus<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bcftools'  target='_blank'  rel='noopener noreferrer'>bcftools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beagle'  target='_blank'  rel='noopener noreferrer'>beagle<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beast'  target='_blank'  rel='noopener noreferrer'>beast<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bedtools'  target='_blank'  rel='noopener noreferrer'>bedtools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bellerophon'  target='_blank'  rel='noopener noreferrer'>bellerophon<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bismark'  target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'  target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'  target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'  target='_blank'  rel='noopener noreferrer'>blast<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blat'  target='_blank'  rel='noopener noreferrer'>blat<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=boost'  target='_blank'  rel='noopener noreferrer'>boost<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie_wrappers'  target='_blank'  rel='noopener noreferrer'>bowtie_wrappers<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie2'  target='_blank'  rel='noopener noreferrer'>bowtie2<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=breakdancer_max'  target='_blank'  rel='noopener noreferrer'>breakdancer_max<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bsmap'  target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bs_seeker2'  target='_blank'  rel='noopener noreferrer'>bs_seeker2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=busco'  target='_blank'  rel='noopener noreferrer'>busco<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bwa'  target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bx_python'  target='_blank'  rel='noopener noreferrer'>bx_python<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=camera'  target='_blank'  rel='noopener noreferrer'>camera<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=canu'  target='_blank'  rel='noopener noreferrer'>canu<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cap3'  target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cardinal'  target='_blank'  rel='noopener noreferrer'>cardinal<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cat_bins'  target='_blank'  rel='noopener noreferrer'>cat_bins<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cdhit'  target='_blank'  rel='noopener noreferrer'>cdhit<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=CellProfiler'  target='_blank'  rel='noopener noreferrer'>CellProfiler<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=centrifuge'  target='_blank'  rel='noopener noreferrer'>centrifuge<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=chimera_slayer'  target='_blank'  rel='noopener noreferrer'>chimera_slayer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=chromeister'  target='_blank'  rel='noopener noreferrer'>chromeister<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=circexplorer'  target='_blank'  rel='noopener noreferrer'>circexplorer<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=clustalw'  target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ctsm_fates'  target='_blank'  rel='noopener noreferrer'>ctsm_fates<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cufflinks'  target='_blank'  rel='noopener noreferrer'>cufflinks<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cutadapt'  target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cytoscape'  target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dada2'  target='_blank'  rel='noopener noreferrer'>dada2<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=deeptools'  target='_blank'  rel='noopener noreferrer'>deeptools<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dendropy'  target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dia_umpire'  target='_blank'  rel='noopener noreferrer'>dia_umpire<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=diamond'  target='_blank'  rel='noopener noreferrer'>diamond<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=edena'  target='_blank'  rel='noopener noreferrer'>edena<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=eggnog-mapper'  target='_blank'  rel='noopener noreferrer'>eggnog-mapper<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=emboss'  target='_blank'  rel='noopener noreferrer'>emboss<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=encyclopedia'  target='_blank'  rel='noopener noreferrer'>encyclopedia<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ensembl'  target='_blank'  rel='noopener noreferrer'>ensembl<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=exonerate'  target='_blank'  rel='noopener noreferrer'>exonerate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=export2graphlan'  target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastp'  target='_blank'  rel='noopener noreferrer'>fastp<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastqc'  target='_blank'  rel='noopener noreferrer'>fastqc<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx'  target='_blank'  rel='noopener noreferrer'>fastx<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx'  target='_blank'  rel='noopener noreferrer'>fastx<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flash'  target='_blank'  rel='noopener noreferrer'>flash<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flashlfq'  target='_blank'  rel='noopener noreferrer'>flashlfq<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flye'  target='_blank'  rel='noopener noreferrer'>flye<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fraggenescan'  target='_blank'  rel='noopener noreferrer'>fraggenescan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=freebayes'  target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=garli'  target='_blank'  rel='noopener noreferrer'>garli<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gatk'  target='_blank'  rel='noopener noreferrer'>gatk<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gecko'  target='_blank'  rel='noopener noreferrer'>gecko<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=genomescope'  target='_blank'  rel='noopener noreferrer'>genomescope<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer'  target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer_hmm'  target='_blank'  rel='noopener noreferrer'>glimmer_hmm<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gsl'  target='_blank'  rel='noopener noreferrer'>gsl<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=graphlan'  target='_blank'  rel='noopener noreferrer'>graphlan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gmx'  target='_blank'  rel='noopener noreferrer'>gmx<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gubbins'  target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=guppy'  target='_blank'  rel='noopener noreferrer'>guppy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hadoop'  target='_blank'  rel='noopener noreferrer'>hadoop<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hifiasm'  target='_blank'  rel='noopener noreferrer'>hifiasm<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hisat2'  target='_blank'  rel='noopener noreferrer'>hisat2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hmmer'  target='_blank'  rel='noopener noreferrer'>hmmer<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=htseq'  target='_blank'  rel='noopener noreferrer'>htseq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=humann'  target='_blank'  rel='noopener noreferrer'>humann<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=humann'  target='_blank'  rel='noopener noreferrer'>humann<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=humann'  target='_blank'  rel='noopener noreferrer'>humann<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=idba'  target='_blank'  rel='noopener noreferrer'>idba<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=infernal'  target='_blank'  rel='noopener noreferrer'>infernal<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=interproscan'  target='_blank'  rel='noopener noreferrer'>interproscan<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=iqtree'  target='_blank'  rel='noopener noreferrer'>iqtree<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=iwtomics'  target='_blank'  rel='noopener noreferrer'>iwtomics<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jbrowse'  target='_blank'  rel='noopener noreferrer'>jbrowse<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jcvi'  target='_blank'  rel='noopener noreferrer'>jcvi<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jellyfish'  target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jq'  target='_blank'  rel='noopener noreferrer'>jq<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kallisto'  target='_blank'  rel='noopener noreferrer'>kallisto<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kat'  target='_blank'  rel='noopener noreferrer'>kat<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=khmer'  target='_blank'  rel='noopener noreferrer'>khmer<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken'  target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken2'  target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=krona'  target='_blank'  rel='noopener noreferrer'>krona<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lastz'  target='_blank'  rel='noopener noreferrer'>lastz<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lipidmaps'  target='_blank'  rel='noopener noreferrer'>lipidmaps<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs'  target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs2'  target='_blank'  rel='noopener noreferrer'>macs2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mafft'  target='_blank'  rel='noopener noreferrer'>mafft<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maker'  target='_blank'  rel='noopener noreferrer'>maker<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maldi_quant'  target='_blank'  rel='noopener noreferrer'>maldi_quant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mash'  target='_blank'  rel='noopener noreferrer'>mash<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mauve'  target='_blank'  rel='noopener noreferrer'>mauve<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxbin2'  target='_blank'  rel='noopener noreferrer'>maxbin2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant'  target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant'  target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mcl'  target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=medaka'  target='_blank'  rel='noopener noreferrer'>medaka<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=megahit'  target='_blank'  rel='noopener noreferrer'>megahit<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=merqury'  target='_blank'  rel='noopener noreferrer'>merqury<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=meryl'  target='_blank'  rel='noopener noreferrer'>meryl<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=idba'  target='_blank'  rel='noopener noreferrer'>idba<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=MetaDEGalaxy'  target='_blank'  rel='noopener noreferrer'>MetaDEGalaxy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metageneannotator'  target='_blank'  rel='noopener noreferrer'>metageneannotator<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metagenomeseq'  target='_blank'  rel='noopener noreferrer'>metagenomeseq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metams'  target='_blank'  rel='noopener noreferrer'>metams<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaphlan'  target='_blank'  rel='noopener noreferrer'>metaphlan<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaQuantome'  target='_blank'  rel='noopener noreferrer'>metaQuantome<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaspades'  target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metavelvet'  target='_blank'  rel='noopener noreferrer'>metavelvet<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mimodd'  target='_blank'  rel='noopener noreferrer'>mimodd<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minia'  target='_blank'  rel='noopener noreferrer'>minia<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minimap2'  target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mira'  target='_blank'  rel='noopener noreferrer'>mira<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mircounts'  target='_blank'  rel='noopener noreferrer'>mircounts<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mirdeep2'  target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mixomics'  target='_blank'  rel='noopener noreferrer'>mixomics<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mlst'  target='_blank'  rel='noopener noreferrer'>mlst<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=proteomics_moff'  target='_blank'  rel='noopener noreferrer'>proteomics_moff<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=morpheus'  target='_blank'  rel='noopener noreferrer'>morpheus<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mothur'  target='_blank'  rel='noopener noreferrer'>mothur<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mrbayes'  target='_blank'  rel='noopener noreferrer'>mrbayes<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msgfplus'  target='_blank'  rel='noopener noreferrer'>msgfplus<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msnbase'  target='_blank'  rel='noopener noreferrer'>msnbase<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msstats'  target='_blank'  rel='noopener noreferrer'>msstats<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=multiqc'  target='_blank'  rel='noopener noreferrer'>multiqc<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mummer'  target='_blank'  rel='noopener noreferrer'>mummer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=muscle'  target='_blank'  rel='noopener noreferrer'>muscle<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=nanopolish'  target='_blank'  rel='noopener noreferrer'>nanopolish<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=netcdf'  target='_blank'  rel='noopener noreferrer'>netcdf<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ngsplot'  target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=NOVOPlasty'  target='_blank'  rel='noopener noreferrer'>NOVOPlasty<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=oases'  target='_blank'  rel='noopener noreferrer'>oases<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=obitools'  target='_blank'  rel='noopener noreferrer'>obitools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=openms'  target='_blank'  rel='noopener noreferrer'>openms<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4mcorcov'  target='_blank'  rel='noopener noreferrer'>w4mcorcov<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pampa'  target='_blank'  rel='noopener noreferrer'>pampa<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pangolin'  target='_blank'  rel='noopener noreferrer'>pangolin<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=parsnp'  target='_blank'  rel='noopener noreferrer'>parsnp<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pathview'  target='_blank'  rel='noopener noreferrer'>pathview<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pear'  target='_blank'  rel='noopener noreferrer'>pear<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=peptideshaker'  target='_blank'  rel='noopener noreferrer'>peptideshaker<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=percolator'  target='_blank'  rel='noopener noreferrer'>percolator<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=phyloseq'  target='_blank'  rel='noopener noreferrer'>phyloseq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=phyml'  target='_blank'  rel='noopener noreferrer'>phyml<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picard'  target='_blank'  rel='noopener noreferrer'>picard<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picrust'  target='_blank'  rel='noopener noreferrer'>picrust<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pilon'  target='_blank'  rel='noopener noreferrer'>pilon<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=plink'  target='_blank'  rel='noopener noreferrer'>plink<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prinseq'  target='_blank'  rel='noopener noreferrer'>prinseq<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prodigal'  target='_blank'  rel='noopener noreferrer'>prodigal<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prokka'  target='_blank'  rel='noopener noreferrer'>prokka<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=psort'  target='_blank'  rel='noopener noreferrer'>psort<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=psort'  target='_blank'  rel='noopener noreferrer'>psort<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=purge_dups'  target='_blank'  rel='noopener noreferrer'>purge_dups<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pysam'  target='_blank'  rel='noopener noreferrer'>pysam<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime'  target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime'  target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime2'  target='_blank'  rel='noopener noreferrer'>qiime2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=quast'  target='_blank'  rel='noopener noreferrer'>quast<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=racon'  target='_blank'  rel='noopener noreferrer'>racon<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=raven'  target='_blank'  rel='noopener noreferrer'>raven<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rawtools'  target='_blank'  rel='noopener noreferrer'>rawtools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=raxml'  target='_blank'  rel='noopener noreferrer'>raxml<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repeat_masker'  target='_blank'  rel='noopener noreferrer'>repeat_masker<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repet'  target='_blank'  rel='noopener noreferrer'>repet<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rsem'  target='_blank'  rel='noopener noreferrer'>rsem<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rseqc'  target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=salmon'  target='_blank'  rel='noopener noreferrer'>salmon<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=salsa'  target='_blank'  rel='noopener noreferrer'>salsa<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samblaster'  target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samtools'  target='_blank'  rel='noopener noreferrer'>samtools<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=scikit'  target='_blank'  rel='noopener noreferrer'>scikit<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=screed'  target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=searchgui'  target='_blank'  rel='noopener noreferrer'>searchgui<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=seqkit'  target='_blank'  rel='noopener noreferrer'>seqkit<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=shasta'  target='_blank'  rel='noopener noreferrer'>shasta<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sickle'  target='_blank'  rel='noopener noreferrer'>sickle<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=simtext'  target='_blank'  rel='noopener noreferrer'>simtext<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snap'  target='_blank'  rel='noopener noreferrer'>snap<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sniffles'  target='_blank'  rel='noopener noreferrer'>sniffles<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snpeff'  target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sortmerna'  target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=spades'  target='_blank'  rel='noopener noreferrer'>spades<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=stacks'  target='_blank'  rel='noopener noreferrer'>stacks<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=star'  target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=structure'  target='_blank'  rel='noopener noreferrer'>structure<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=svtyper'  target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tagdust'  target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=targetfinder'  target='_blank'  rel='noopener noreferrer'>targetfinder<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tassel'  target='_blank'  rel='noopener noreferrer'>tassel<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=thermo_raw_file_converter'  target='_blank'  rel='noopener noreferrer'>thermo_raw_file_converter<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tophat'  target='_blank'  rel='noopener noreferrer'>tophat<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=transdecoder'  target='_blank'  rel='noopener noreferrer'>transdecoder<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trim_galore'  target='_blank'  rel='noopener noreferrer'>trim_galore<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trimmomatic'  target='_blank'  rel='noopener noreferrer'>trimmomatic<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinity'  target='_blank'  rel='noopener noreferrer'>trinity<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinotate'  target='_blank'  rel='noopener noreferrer'>trinotate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trnascan-se'  target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trycycler'  target='_blank'  rel='noopener noreferrer'>trycycler<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ucsc'  target='_blank'  rel='noopener noreferrer'>ucsc<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=unicycler'  target='_blank'  rel='noopener noreferrer'>unicycler<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=usearch'  target='_blank'  rel='noopener noreferrer'>usearch<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=valet'  target='_blank'  rel='noopener noreferrer'>valet<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=varscan'  target='_blank'  rel='noopener noreferrer'>varscan<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vcftools'  target='_blank'  rel='noopener noreferrer'>vcftools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=velvet'  target='_blank'  rel='noopener noreferrer'>velvet<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vienna'  target='_blank'  rel='noopener noreferrer'>vienna<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vsearch'  target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=windowmasker'  target='_blank'  rel='noopener noreferrer'>windowmasker<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4m'  target='_blank'  rel='noopener noreferrer'>w4m<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xarray'  target='_blank'  rel='noopener noreferrer'>xarray<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xcms'  target='_blank'  rel='noopener noreferrer'>xcms<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xtandem'  target='_blank'  rel='noopener noreferrer'>xtandem<\/a>"],[null,"Yes","Yes",null,null,null,null,null,null,null,null,null,null,"Yes",null,null,null,null,"Yes",null,null,null,null,"Yes","Yes",null,null,null,null,null,null,null,null,null,"Yes",null,null,null,null,null,null,null,null,"Yes",null,null,null,null,null,null,"Yes","Yes",null,null,null,"Yes","Yes","Yes","Yes","Yes","Yes",null,null,"Yes",null,null,null,null,"Yes","Yes",null,"Yes",null,"Yes",null,null,null,null,null,null,null,"Yes",null,null,"Yes","Yes",null,"Yes",null,null,"Yes",null,null,"Yes","Yes",null,null,null,null,null,null,null,null,"Yes","Yes",null,null,null,null,null,null,"Yes",null,null,null,null,"Yes","Yes",null,null,"Yes","Yes",null,null,null,"Yes","Yes",null,"Yes","Yes",null,null,null,null,null,null,null,null,"Yes",null,null,"Yes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Yes","Yes",null,null,"Yes",null,null,null,null,null,null,null,"Yes",null,null,"Yes","Yes",null,null,null,"Yes",null,null,null,"Yes",null,null,null,null,null,"Yes",null,null,null,null,"Yes",null,null,"Yes",null,null,"Yes","Yes","Yes","Yes",null,null,null,"Yes",null,"Yes",null,null,null,"Yes","Yes","Yes","Yes",null,null,null,"Yes",null,null,null,"Yes","Yes",null,"Yes",null,"Yes",null,"Yes","Yes",null,null,null,null,"Yes","Yes","Yes",null,"Yes","Yes",null,"Yes","Yes",null,null,null,null,null,"Yes",null,null,null,null,null,null,"Yes",null,"Yes","Yes",null,null,null,null,null,"Yes",null,null,"Yes",null,null,"Yes",null,"Yes",null,"Yes","Yes",null,"Yes","Yes","Yes",null,null,null,"Yes","Yes","Yes","Yes",null,"Yes",null,null,"Yes",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Yes",null,null,null,"Yes","Yes",null,"Yes",null,null,null,null,"Yes",null,"Yes",null,null,null,"Yes","Yes","Yes","Yes",null,"Yes",null,"Yes",null,"Yes","Yes",null,null,"Yes",null,null,null,null,null,null,null,null,null,"Yes",null,null,null,"Yes",null,null,null,null,null,null,null,null,"Yes",null,null,"Yes",null,null,null,"Yes","Yes","Yes",null,null,null,null,null,null,null,null,null,null,null,"Yes",null,null,"Yes","Yes",null,"Yes",null,null,"Yes",null,"Yes",null,null,null,null,"Yes",null,"Yes",null,null,null,"Yes",null,"Yes",null,"Yes",null,null,null,null,"Yes","Yes",null,null,null,"Yes","Yes",null,null,null,null,null,null,null,null,"Yes",null,null,null,"Yes",null,"Yes",null,"Yes",null,"Yes","Yes","Yes","Yes","Yes","Yes","Yes","Yes",null,null,null,"Yes",null,"Yes","Yes",null,null,"Yes",null,null,null,"Yes",null,"Yes","Yes",null,"Yes"],["","","2.2.3","","","","","","","","","","","","","","","","1.9<br />1.12","","3.1.2","1.10.4","","2.28.0","","","","","","","","","","2.10.1<br />2.11.0","","","","","","","","1.71.0<br />1.72.0","","2.3.5.1","","","","","","","","0.7.17","","","","","2.0<br />2.0t<br />1.9<br />2.1.1","","","","","","","","","","","","","","","","","","","","","","","","","","10.1<br />11.0.3<br />11.2.2<br />11.4.1","7.6.5-cuda10.1<br />8.1.1-cuda11<br />8.2.2-cuda11.4","","","","","","","","","","","","","","","11.0.1","","","","","","","","","","","","","","","","","","0.20.0","0.11.7","","","","","","","","","","","","","","","","","4.1.4.0<br />4.2.1.0<br />4.1.8.1","","","","","","","","","","","","","","","","","","","","20191022","2.6","","","2019.3<br />2019.3-plumed<br />2019.3-gpuvolta<br />2018.3<br />2018.3-plumed<br />2020.1<br />2020.1-plumed<br />2020.1-gpuvolta<br />2020.3<br />2020.3-gpuvolta<br />2021-gpuvolta<br />2021<br />2021.2<br />2021.2-gpuvolta","","","","","","","","","4.2.14<br />4.2.15","1.8.21<br />1.8.21p<br />1.10.5<br />1.10.5p<br />1.10.7<br />1.10.7p<br />1.12.1<br />1.12.1p","","","","","","0.19.0<br />0.22.1","2021.05.15","2021.05.15","","1.9<br />1.12","","","","","8.6<br />8.8","7.0.11","","","","","12.0.1","","","","","","","","","","","2.3.0","","","","","","","","","","","","","","","","2020.10.15","","","","13.0.0","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","2.17<br />2.22","","","","","","","","","","","","","","","","","","","","","","","nll6i27dbl<br />nll6i27dbl-i8<br />nll6i27dbl-mkl<br />fll6i26dcl<br />fll6i26dcl-mkl","","","1.8.5.0","6.6.2","","4.7.7<br />4.9.2","2.1.7","","4.7.1<br />4.7.1p<br />4.6.3<br />4.6.3p<br />4.7.3<br />4.7.3p<br />4.7.4<br />4.7.4p<br />4.6.3-i8r8<br />4.8.0<br />4.8.0p","","","","","","","","","","","","","","","","","","","","","","","5.8.0<br />5.8.0-mesa<br />5.8.0-gpu","","","","","","","5.6.40","","","","","","","","","","","","1.11.2","","","","","","","","","","","","","","","","1.4.0a0<br />1.5.1<br />1.9.0","","","","","","3.6.1<br />4.0.0<br />4.1.0","","","","","","","","","","","","","","","","","","","","","","1.1.0","","","1.9<br />1.10<br />1.12","","","","","","","","","","","","","","","","","","","","","","","2.41","","","","","","","","","","","","","","","","","","","","2.0.0<br />2.1.0<br />2.3.0<br />2.4.1<br />2.6.0","","","2.1.1","2019.3.14<br />2020.1.13","","","","","2.9.1<br />2.12.0","","","","","","","","","","","","1.2.10","","3.1.2","","","","","","","","","",""],["","","","","","","3.1.0","","","","","","","","0.1","","2.4.1","","1.6","","2.1.2","1.8.4<br />2.4.7","","2.26.0","","","","","","","1.7.1","1.70","0.19.0","2.2.26","2.10.1<br />2.7.1","","","3.6","","","","","1.2.1.1","2.3.3.1<br />2.3.4.1","","","1.1.2_2013_03_08","2.90","","2.1.3","","0.7.17","","","","","1.6","20130608","","","4.6.8","","8.3rc2","","","","","","","","2014-05-27","","","2.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","6.6.0","","","","","2.2.8","","","2.2.0","","","","3.1","","0.11.4","1.0","","0.0.14","","","","","","","","","0.9.21","","","","","","","0.91b","1.26.0","","1.4.4","","","","","","","3.02b","3.0.4","","2017-11-15","","","","","","","","","","","","","","","","","","","","","","","","3.0<br />3.1b2","","","","","1.2.1<br />1.6","","","","","","","","","","2.4.14","","5.27-66.0","","","","","","","","","","2.2.6","","","","","","","2.1.2","","","","","","","1.04.00","","","","","","","1.0.18","","","","7.313","","","","","","","","","","","","","14-137","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","3.2.6","","","","","","","3.23","3.8.1551","","","","","","","","","","","","","","","","1.3.0","","","","","0.2.08","","","","","","","","","","","","","","","","","","","","","","","","","","","","","1.07","0.10","","","","","0.5.0","","","","","","","","","","3.0.66","0.13","","","","","","","","","","","","","","8.2.11","","","","","","","","","","","1.3.0<br />1.3.3","","","","","","0.1.24","0.1.19<br />1.6","","0.241","","1.0","","","","0.0.2","r904","","1.33","","0.1.127","","","0.15","","","","4.3t","2.21","r146","r240","","","3.11.1","","","2.8.2-5","1.48<br />2.1","2.5.3a","2.3.4","","","","0.1.4","","","","","4.3","","","","","","","5.0.2","4.09","","0.36","2.5.1","","2.0.0","","357","","","","","","0.0.1","0.1.15","1.2.10","","","2.6.0","","","2.4.1","","","","","",""],["","","","","","","","","","","","","","","","","","","","","2.1.2","","","","","","","","","","","1.70","","","","","","","","","","","","","","","","","","2.1.3","","0.7.17","","","","","","","","","","","","","","","","","","","","","","2.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","1.4.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","2.1.2","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","3.2.6","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","0.13","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","0.1.19<br />1.6","","","","1.0","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],["","1.0.1","2.0.2","20120224","","","","","2019.07<br />2020.02<br />4.2.0<br />4.3.1<br />5.2.0","0.931<br />0.933","","","","3.3.3<br />3.4.0","","201710<br />201811","2.4.0","201710","1.12<br />1.3","2.1<br />4.1<br />5.1","","1.8.0<br />1.8.1<br />1.8.2","","2.25.0","","","bio-roll","3.6-rc2<br />3.8-rc2","8","","1.007002<br />1.007002-sw","1.66","0.15.0<br />0.22.3","2.3.0","","2.10.1<br />2.9.0","","35","","","","1.65.1","1.1.2","2.2.7<br />2.3.4.3","","2.1.5","","","","","1.22<br />3.0.2","0.7.13","0.8.1","2019.03.01","","","1.3<br />1.8<br />1.8<br />1.9<br />2.1","","","","","0.5.3.3","8.3rc2","","","1.0.11","1.0.13","","","","","","1.5.5","","2.6.8","0.9.4a0","","","","","","","","","2.2.1","1.18","3.7.1","","","","3.3.1","1.0.0","4.0.3","","0.7.12<br />0.9.22<br />0.9.23<br />0.9.30","","2.5.0","2.2.1","","2.3.0<br />2.3.1<br />2.6.0<br />3.6.1<br />3.8.0<br />3.9.4","3.131028","13.9","","6.5.7","","0.6.3","","","","","","2.4.0","0.22","201710","0.0.6<br />0.0.8","","0.20.1","0.11.4","","20210326","","0.0.14","","","","","2.4.2<br />2.6","","","","6.0<br />6.1dev","","","","3.5<br />4.0.4.0","4.2.1.0","","1.26.0","","","","","","1.42.0","","","","","","2021.05.27","20210527","20151231","","7.2.0","20170122<br />20180322","1.16<br />2.1","0.9.4","1.1.3","2018<br />5.1.4-intel<br />5.1.4-intelMPI","0.3.5<br />201811","1.4.1","2.4.1","3.3.0","2.5.0-cdh5.3.1-native","","","1.0.0","2.10","1.8.14","","","","2.1","3.1b2<br />3.3","","","","0.6.1p1<br />htseq-0.6.1","","","","","1.1.1","","","","","","","","","","","2.1.4_beta","3.2.2","","4.2.0","","","","2.2.10","","","","","0.46.2","","","","","","1.1.1","2.0.8","2.7.1<br />201712","","","","","","2.2.2","","","2-2.1.1","2.2.7.1","","v7.407","","2.31.10<br />3.01.03","","","","","0.5","3.2.4","20200714","","1.6.10.43<br />1.6.17.0<br />1.6.17.0-AOT","1.6.10.43","","","201710","","","","","201710","","","2.3.0","","","","","0.1","","","3.0.2","","","","","","","","","","","1.0.2","","","4.9.6","","","0.0.7","0.5.3","","2.19.0","","","","","","3.2.3<br />3.2.6","","","","","","201801","","","","","","","","6.3.0","","2.1.7","","3.6.2<br />4.3.2<br />4.4.1-gnu<br />4.4.1-intel","0.37.2<br />0.37.6<br />0.39.0<br />0.39.0+eddy<br />0.41.0","v2.2-beta.0","19.07","","1.1.0","","2.61","","0.3.1","","","","","0.9.0","","","0.1.0","20140812","0.3.0","","","","5.4.1","1.2<br />1.5.3","","","0.9.6","","","","","","","","2.18.2<br />2.25.7<br />2.3.0","1.1.3","2.2.0_b","","","1.9","","","v1.1.alpha17","","","","","201804","1.12<br />1.13<br />1.14.5","2.14.0","","","","","","","0.8.4","","","1.9.1","2018.6.0<br />2019.4<br />2021.2<br />2021.4","5.0.2<br />5.0.2cli","","3.4.3<br />3.5.0<br />3.5.0-gnu<br />3.5.0-intel<br />3.6.0-container<br />3.6.0+tidyverse+rstan+rtdists<br />3.6.1-container<br />3.6.1+CrocSim<br />3.6.1+exSTRa<br />3.6.1+Monocle3<br />3.6.1+Monocle3+Giotto<br />3.6.1+Seurat<br />3.6.3+Shawan<br />4.0.0<br />4.0.0+Spatial<br />4.0.2+Spatial<br />4.0.2+tradeSeq+Seurat<br />4.0.5+ubms<br />4.1.0+nsga2R<br />4.1.0+sf<br />4.1.0+SLiM","","","","","","","8.2.12","1.50.2-2","0.0.24","0.0.1","4.0.9_p2","2.0.1","","","","4.5.1<br />4.6.2--R-3.4.4","0.6_9","1.3.3","2.6.4","1.0.143<br />1.1.456-3.4.4<br />1.1.456-3.5.1","0.43","","","","0.1.17<br />1.10<br />1.2<br />1.3<br />1.6<br />1.9","9.4","","","","","0.13.2","","","","","","","","smrtlink_5.0.1<br />smrtlink_5.1<br />smrtlink_6.0<br />smrtlink_7.0<br />smrtlink_9.0","5.6","2006-07-28","1.1.2-GCC-4.9.2","","","","","","240","1.03","3.0.3<br />4.0.0","3.11.1<br />3.15.3<br />3.9.0","","","2.11.0","1.37","2.5.2b<br />2.5.2b-static","2.3.4","","2.1.1","1.1.4","","","","2.33","","","","","","","2.1.1","","5.5.0","","","0.35<br />0.39","2.1.1<br />2.9.1","3.2.1","","","","","","","","","","0.1.14","1.2.10","2.2.4","","","","","","","","","","",""]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th class=\"dt-center\" style=\"font-size:11pt\" colspan=\"8\">Tool metadata<\/th>\n      <th class=\"dt-center\" style=\"font-size:11pt\" colspan=\"5\">Availability on Australian compute infrastructures<\/th>\n    <\/tr>\n    <tr>\n      <th>Tool / workflow name<\/th>\n      <th>bio.tools documentation<\/th>\n      <th>Tool identifier (module name / bio.tools ID / placeholder)<\/th>\n      <th>Description<\/th>\n      <th>EDAM topics<\/th>\n      <th>Containers available? (BioContainers)<\/th>\n      <th>License<\/th>\n      <th>Available in Galaxy toolshed<\/th>\n      <th>Galaxy Australia<\/th>\n      <th>NCI (Gadi)<\/th>\n      <th>Pawsey (Zeus)<\/th>\n      <th>Pawsey (Magnus)<\/th>\n      <th>QRIScloud / UQ-RCC (Flashlite, Awoonga, Tinaroo)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"headerCallback":"function( thead, data, start, end, display ) {\n  $(thead).closest('thead').find('th').eq(0).css('background-color', '#205a86').css('color', '#ffffff');\n  $(thead).closest('thead').find('th').eq(1).css('background-color', '#2cb77c').css('color', '#ffffff');\n}","pageLength":20,"dom":"Bfrtip","buttons":["copy","csv","colvis"],"columnDefs":[{"visible":false,"targets":[2,6]},{"className":"dt-center","targets":[0,1,2,5,6,7,8,9,10,11,12]}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]}},"evals":["options.headerCallback"],"jsHooks":[]}</script>
<p></font></p>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-09-30</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>The <em>Tool identifier</em> column contains an identifier for the tool / workflow: typically either the module name (used for matching to HPC lists) or a placeholder value.</li>
<li>The <em>EDAM topics</em> column categorises the tools by purpose, using an <a href="https://github.com/edamontology/edamontology">EDAM</a> <em>term</em> where possible.</li>
<li>For tools which are not currently installed on <a href="https://usegalaxy.org.au/">Galaxy Australia</a>, but which are available <a href="https://toolshed.g2.bx.psu.edu/">in the Galaxy app store (aka toolshed)</a>, the <em>Available in Galaxy toolshed</em> column launches a toolshed search using the link label as the search term.</li>
<li>More information about a tool can be found by following the link in the <em><a href="https://bio.tools/">bio.tools</a> documentation</em> column.</li>
<li>When a tool has been containerised to allow for easier installation on any compute infrastructure, a link to the containerised software that can be downloaded from <a href="https://biocontainers.pro/#/">BioContainers</a> is shown in the <em>Containers available? (BioContainers)</em> column.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<hr />
<hr />
<p><a name="tool_request"></a></p>
<div id="requesting-tool-installations" class="section level1">
<h1>Requesting tool installations</h1>
<p>Tool installs can be requested for the infrastructures listed in the table:</p>
<font size="2">
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Infrastructure
</th>
<th style="text-align:left;">
Process
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="https://usegalaxy.org.au/">Galaxy Australia</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://request.usegalaxy.org.au/">tool install request</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://nci.org.au/">NCI</a>
</td>
<td style="text-align:left;">
<a href="https://opus.nci.org.au/display/Help/5.+Software+Applications">Contact NCI</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://pawsey.org.au/">Pawsey</a>
</td>
<td style="text-align:left;">
Visit the <a href="https://support.pawsey.org.au/portal/servicedesk/customer/portal">Helpdesk Portal</a> or email: <a href="help@pawsey.org.au">help@pawsey.org.au</a> with your request
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://www.qriscloud.org.au/">QRIScloud</a> / <a href="https://rcc.uq.edu.au/">UQ-RCC</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://docs.google.com/forms/d/e/1FAIpQLSefKCBkZbTQ-dUyU-pd4ZypkA-TA5GiMgpsJ2slWmD-B6elEg/viewform">tool install request</a>
</td>
</tr>
</tbody>
</table>
<p></font></p>
<div id="faq-how-can-i-get-a-hpc-allocation" class="section level3">
<h3>FAQ: How can I get a HPC allocation?</h3>
<p>Allocation of computational resources to individual researchers on partner HPC and cloud infrastructures is through the usual channels offered via each partner (see instructions for each by following these links: <a href="https://nci.org.au/users/how-access-nci">NCI</a>, <a href="https://support.pawsey.org.au/documentation/display/US/Getting+Access">Pawsey</a>, <a href="https://www.qcif.edu.au/services/computing/">QCIF</a>, <a href="http://cloud.nectar.org.au/start-now/">ARDC’s Nectar Research Cloud</a>). The <a href="https://www.biocommons.org.au/biocloud">BioCommons BioCloud</a> activity is working with these partners to examine ways to make it easier for life science researchers to receive HPC and HTC allocations.</p>
<p>Note that access to computational resources via the <a href="https://usegalaxy.org.au/">Galaxy Australia</a> hosted web platform is unlimited for Australian researchers thanks to our computational partners (ARDC’s Nectar Research Cloud, University of Queensland, QCIF, University of Melbourne and Pawsey) who underpin the Galaxy Australia service.</p>
<hr />
<hr />
</div>
</div>
<div id="how-were-the-tools-and-workflows-selected" class="section level1">
<h1>How were the tools and workflows selected?</h1>
<p>The tool list here includes those that have been identified through <a href="https://www.biocommons.org.au/get-involved">BioCommons Community consultations</a> and other engagements, as well as through matching to the tool sets installed across the various BioCommons partner compute infrastructures. The list is not intended to be exhaustive: i.e. it does not contain <em>ALL</em> bioinformatics tools.</p>
<hr />
<hr />
</div>
<div id="how-was-the-list-generated" class="section level1">
<h1>How was the list generated?</h1>
<p>RStudio (see session info at the bottom of this page) was used for processing.</p>
<p>The broad steps are described here:</p>
<p><font size="2"></p>
<ol style="list-style-type: decimal">
<li>The manually curated tool list and separate tool lists provided by the BioCommons partner infrastructures were parsed (Galaxy Australia, NCI, Pawsey and QRISCloud - RCC).</li>
<li>bio.tools metadata was imported for those tools where a biotoolsID was available.</li>
<li>The complete set of tool lists from the BioCommons partner infrastructures were joined using the <code>toolID</code> as a key, and</li>
<li>Links were embedded, where available, to the original documentation URLs (homepage or GitHub repository, for example) as well as the registry entries on bio.tools and BioContainers.</li>
<li>Galaxy wrapping often entails that tool suites are separated into their individual tool components. As such, if a Galaxy tool was matched to the manually curated tool list, a tool shed search link was embedded in the <code>Available in Galaxy toolshed</code> column. This allows a user to search for the tool using a relevant search term (e.g. vcftools) to identify all the component tools available in the toolshed.</li>
</ol>
<p></font></p>
<hr />
<hr />
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>## R version 4.0.3 (2020-10-10)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## Running under: Windows 10 x64 (build 19043)
## 
## Matrix products: default
## 
## locale:
## [1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
## [3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
## [5] LC_TIME=English_Australia.1252    
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] uuid_0.1-4        yaml_2.2.1        lubridate_1.7.9.2 DT_0.17          
##  [5] httr_1.4.2        tidyjson_0.3.1    knitr_1.31        kableExtra_1.3.1 
##  [9] forcats_0.5.1     stringr_1.4.0     dplyr_1.0.5       purrr_0.3.4      
## [13] readr_1.4.0       tidyr_1.1.2       tibble_3.1.0      ggplot2_3.3.3    
## [17] tidyverse_1.3.0  
## 
## loaded via a namespace (and not attached):
##  [1] tidyselect_1.1.0  xfun_0.20         haven_2.3.1       colorspace_2.0-0 
##  [5] vctrs_0.3.6       generics_0.1.0    viridisLite_0.3.0 htmltools_0.5.1.1
##  [9] utf8_1.1.4        rlang_0.4.10      pillar_1.5.1      glue_1.4.2       
## [13] withr_2.4.1       DBI_1.1.1         dbplyr_2.1.0      modelr_0.1.8     
## [17] readxl_1.3.1      lifecycle_1.0.0   munsell_0.5.0     gtable_0.3.0     
## [21] cellranger_1.1.0  rvest_0.3.6       htmlwidgets_1.5.3 evaluate_0.14    
## [25] crosstalk_1.1.1   curl_4.3          fansi_0.4.2       broom_0.7.3      
## [29] Rcpp_1.0.6        scales_1.1.1      backports_1.2.0   webshot_0.5.2    
## [33] jsonlite_1.7.2    fs_1.5.0          hms_1.0.0         digest_0.6.27    
## [37] stringi_1.5.3     grid_4.0.3        cli_2.3.1         tools_4.0.3      
## [41] magrittr_2.0.1    crayon_1.4.1      pkgconfig_2.0.3   ellipsis_0.3.1   
## [45] xml2_1.3.2        reprex_1.0.0      assertthat_0.2.1  rmarkdown_2.6    
## [49] rstudioapi_0.13   R6_2.5.0          compiler_4.0.3</code></pre>
<hr />
<hr />
</div>
</div>


<!DOCTYPE html>

<div class="footer">This site uses Cookies. We use cookies to help the website run effectively. To find out more, read our <a href="https://www.biocommons.org.au/privacy">Privacy</a> and <a href="https://www.biocommons.org.au/cookies">Cookies</a> Statements. </div>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
