<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Bioinformatics tools &amp; workflows on BioCommons partner infrastructures</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.17/datatables.js"></script>
<link href="site_libs/dt-core-bootstrap-1.10.20/css/dataTables.bootstrap.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-bootstrap-1.10.20/css/dataTables.bootstrap.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-bootstrap-1.10.20/js/jquery.dataTables.min.js"></script>
<script src="site_libs/dt-core-bootstrap-1.10.20/js/dataTables.bootstrap.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>

<!DOCTYPE html>
<link rel="shortcut icon" href="./images/Australian-Biocommons-Favicon.png" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="1_compute_systems.html">Compute systems</a>
</li>
<li>
  <a href="2_tools.html">Tools &amp; workflows</a>
</li>
<li>
  <a href="3_biocommons_activities.html">Featured tools &amp; workflows</a>
</li>
<li>
  <a href="4_help.html">Help!</a>
</li>
<li>
  <a href="5_attributions.html">Attributions</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Bioinformatics tools &amp; workflows on BioCommons partner infrastructures</h1>

</div>


<style type="text/css">
.main-container {
  max-width: 1500px;
  margin-left: auto;
  margin-right: auto;
}
</style>
<p>This page details tool request mechanisms and a list of tools and workflows installed across several of the <a href="./1_compute_systems.html">BioCommons infrastructure partner systems</a>. Tool status on other partner infrastructures will be added over time.</p>
<hr />
<hr />
<p><a name="tool_request"></a></p>
<div id="requesting-tool-installations" class="section level1">
<h1>Requesting tool installations</h1>
<p>Tool installs can be requested for the infrastructures listed in the table:</p>
<font size="-1.5">
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Infrastructure
</th>
<th style="text-align:left;">
Process
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="https://usegalaxy.org.au/">Galaxy Australia</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://request.usegalaxy.org.au/">tool install request</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://nci.org.au/">NCI</a>
</td>
<td style="text-align:left;">
<a href="https://opus.nci.org.au/display/Help/5.+Software+Applications">Contact NCI</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://pawsey.org.au/">Pawsey</a>
</td>
<td style="text-align:left;">
Visit the <a href="https://support.pawsey.org.au/portal/servicedesk/customer/portal">Helpdesk Portal</a> or email: <a href="help@pawsey.org.au">help@pawsey.org.au</a> with your request
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://www.qriscloud.org.au/">QRIScloud</a> / <a href="https://rcc.uq.edu.au/">UQ-RCC</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://docs.google.com/forms/d/e/1FAIpQLSefKCBkZbTQ-dUyU-pd4ZypkA-TA5GiMgpsJ2slWmD-B6elEg/viewform">tool install request</a>
</td>
</tr>
</tbody>
</table>
<p></font></p>
<div id="faq-can-i-get-a-hpc-allocation-through-the-biocommons" class="section level3">
<h3>FAQ: can I get a HPC allocation through the BioCommons?</h3>
<p>Allocation of computational resources to individual researchers on partner HPC and cloud infrastructures is through the usual channels offered via each partner (see instructions for each by following these links: <a href="https://nci.org.au/users/how-access-nci">NCI</a>, <a href="https://support.pawsey.org.au/documentation/display/US/Getting+Access">Pawsey</a>, <a href="https://www.qcif.edu.au/services/computing/">QCIF</a>, <a href="http://cloud.nectar.org.au/start-now/">ARDC-Nectar</a>). The <a href="https://www.biocommons.org.au/biocloud">BioCommons BioCloud</a> activity is working with these partners to examine ways to make it easier for life science researchers to receive HPC and HTC allocations.</p>
<p>Note that access to computational resources via the <a href="https://usegalaxy.org.au/">Galaxy Australia</a> hosted web platform is unlimited for Australian researchers thanks to our computational partners (ARDC-Nectar Cloud, University of Queensland, QCIF, University of Melbourne and Pawsey) who underpin the Galaxy Australia service.</p>
<hr />
<hr />
<p><a name="tool_information"></a></p>
</div>
</div>
<div id="tool-information" class="section level1 tabset tabset-pills">
<h1>Tool information</h1>
<div id="tool-install-status-and-version-availability" class="section level2">
<h2>Tool install status and version availability</h2>
<p><font size="-1.5"></p>
<div id="htmlwidget-2f0b0ff2510596dd6bc8" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2f0b0ff2510596dd6bc8">{"x":{"style":"bootstrap","filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432"],["<a href='https://github.com/aidenlab/3d-dna'>3D-DNA<\/a>","<a href='https://github.com/tseemann/abricate'>ABRicate<\/a>","<a href='https://github.com/bcgsc/abyss'>ABySS<\/a>","<a href='https://github.com/tanghaibao/jcvi/wiki/ALLMAPS'>ALLMAPS<\/a>","<a href='http://software.broadinstitute.org/allpaths-lg/blog/'>allpathslg<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php/AMOS'>amos<\/a>","<a href='http://bioinformatics.ua.pt/software/anaconda/'>anaconda<\/a>","<a href='http://www.popgen.dk/angsd/index.php/ANGSD'>ANGSD<\/a>","<a href='https://ant.apache.org/'>ant<\/a>","<a href='https://github.com/GMOD/Apollo'>Apollo<\/a>","<a href='https://arimagenomics.com/assembly'>arima mapping<\/a>","<a href='https://www.arm.com/products/development-tools/server-and-hpc/forge'>arm-forge<\/a>","arm-reports","<a href='https://github.com/sanger-pathogens/Artemis'>Artemis<\/a>","<a href='https://www.aspera.com/en/'>Aspera<\/a>","<a href='https://github.com/Gaius-Augustus/Augustus'>Augustus<\/a>","<a href='https://www.gnu.org/software/autoconf/'>Autoconf<\/a>","<a href='https://www.gnu.org/software/automake/'>Automake<\/a>","<a href='https://github.com/hall-lab/bamkit'>bamkit<\/a>","<a href='http://bamm-project.org/'>bamm<\/a>","<a href='https://github.com/pezmaster31/bamtools'>bamtools<\/a>","<a href='https://github.com/workflow4metabolomics/batch_correction'>batchcorrection<\/a>","<a href='https://bazel.build/'>bazel<\/a>","<a href='https://github.com/slaclab/bbcp'>bbcp<\/a>","<a href='https://jgi.doe.gov/data-and-tools/bbtools/'>bbmap<\/a>","<a href='http://www.htslib.org/'>BCFtools<\/a>","<a href='https://faculty.washington.edu/browning/beagle/beagle.html'>Beagle<\/a>","<a href='https://github.com/beagle-dev/beagle-lib'>beagle-lib<\/a>","<a href='https://github.com/beast-dev/beast-mcmc'>BEAST<\/a>","<a href='https://github.com/CompEvol/beast2'>BEAST2<\/a>","<a href='https://github.com/arq5x/bedtools2'>BEDTools<\/a>","<a href='https://www.gnu.org/software/binutils/'>binutils<\/a>","<a href='https://www.bioconductor.org/'>bioconductor<\/a>","<a href='https://github.com/Bio-Linux'>biolinux<\/a>","<a href='https://bionanogenomics.com/support-page/bionano-access-software/'>Bionano Access<\/a>","<a href='http://bioperl.org/'>bioperl<\/a>","<a href='http://biopython.org/'>biopython<\/a>","<a href='https://github.com/FelixKrueger/Bismark'>bismark<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST+<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST2<\/a>","<a href='https://genome.ucsc.edu/cgi-bin/hgBlat'>blat<\/a>","<a href='https://github.com/blobtoolkit/blobtools2'>BlobToolKit<\/a>","<a href='https://github.com/DRL/blobtools'>BlobTools1<\/a>","<a href='https://www.boost.org/'>Boost<\/a>","<a href='http://bowtie-bio.sourceforge.net/index.shtml'>Bowtie<\/a>","<a href='http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'>Bowtie2<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER'>BRAKER<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER'>BRAKER2<\/a>","<a href='https://github.com/genome/breakdancer'>breakdancer<\/a>","<a href='https://github.com/pace/bricks'>BRICKS<\/a>","<a href='https://code.google.com/archive/p/bsmap/'>bsmap<\/a>","<a href='http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker/BS_Seeker.html'>bsseeker<\/a>","<a href='https://github.com/BSSeeker/BSseeker2#bs-seeker2'>bsseeker2<\/a>","<a href='https://busco.ezlab.org/'>BUSCO<\/a>","<a href='https://github.com/lh3/bwa'>bwa<\/a>","<a href='https://github.com/bxlab/bx-python'>bx-python<\/a>","<a href='http://www.bzip.org/'>bzip2<\/a>","<a href='https://github.com/ComparativeGenomicsToolkit/cactus'>Cactus<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/CAMERA.html'>CAMERA<\/a>","<a href='https://github.com/marbl/canu'>Canu<\/a>","<a href='http://seq.cs.iastate.edu/cap3.html'>cap3<\/a>","<a href='https://cardinalmsi.org/'>Cardinal<\/a>","<a href='https://github.com/dutilh/CAT'>CAT<\/a>","<a href='https://github.com/weizhongli/cdhit'>cd-hit<\/a>","<a href='https://github.com/yanailab/celseq2'>CEL-Seq 2<\/a>","<a href='http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page'>Celera / wgs-assembler<\/a>","<a href='https://hub.docker.com/r/centos/tools/'>centos<\/a>","<a href='https://hub.docker.com/r/centos/tools/'>centos6<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki'>checkm<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki'>checkm-genome<\/a>","<a href='https://genome.sph.umich.edu/wiki/ChunkChromosome'>chunkchromosome<\/a>","<a href='http://www.clustal.org/clustal2/'>clustalw<\/a>","<a href='https://tools.bsc.es/doc/clusteringsuite_manual_0.pdf'>clusteringsuite<\/a>","<a href='https://cmake.org/'>cmake<\/a>","<a href='https://github.com/etal/cnvkit'>CNVkit<\/a>","<a href='https://github.com/biocoder/cpc'>Coding Potential Calculator<\/a>","<a href='https://github.com/biocoder/CPC2'>Coding Potential Calculator 2<\/a>","<a href='https://www.gnu.org/software/coreutils/'>coreutils<\/a>","<a href='https://sourceforge.net/projects/rna-cpat/files/'>CPAT: Coding Potential Assessment Tool<\/a>","<a href='https://sourceforge.net/projects/cppunit/'>cppunit<\/a>","<a href='https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cube-score-p-scalasca'>cube<\/a>","<a href='https://developer.nvidia.com/cuda-toolkit'>cuda<\/a>","<a href='https://developer.nvidia.com/cudnn'>cudnn<\/a>","<a href='http://cole-trapnell-lab.github.io/cufflinks/'>Cufflinks<\/a>","<a href='https://curl.se/'>curl<\/a>","<a href='https://github.com/marcelm/cutadapt'>cutadapt<\/a>","<a href='https://cytoscape.org/'>cytoscape<\/a>","<a href='https://www.mcs.anl.gov/research/projects/darshan/'>darshan<\/a>","<a href='https://deeptools.readthedocs.io/en/develop/'>DeepTools<\/a>","<a href='https://github.com/google/deepvariant'>deepvariant<\/a>","<a href='https://dendropy.org/'>dendropy<\/a>","<a href='https://diaumpire.nesvilab.org/'>DIA-Umpire<\/a>","<a href='https://github.com/bbuchfink/diamond'>diamond<\/a>","<a href='https://github.com/broadinstitute/discovar_de_novo'>DISCOVAR de novo<\/a>","<a href='https://jbloomlab.github.io/dms_tools2/'>dms_tools2<\/a>","<a href='https://github.com/dmtcp/dmtcp'>dmtcp<\/a>","<a href='https://github.com/cmbi/dssp'>dssp<\/a>","<a href='https://github.com/easybuilders/easybuild'>easybuild<\/a>","<a href='http://www.genomic.ch/edena.php'>Edena<\/a>","<a href='https://github.com/UQ-RCC/nimrod-embedded'>embedded-nimrod<\/a>","<a href='http://emboss.sourceforge.net/'>EMBOSS (European Molecular Biology Open Software Suite)<\/a>","<a href='https://asia.ensembl.org/info/website/index.html'>Ensembl browser<\/a>","<a href='http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html'>Ensembl Genebuild<\/a>","<a href='http://erlang.org/faq/introduction.html'>Erlang<\/a>","<a href='http://mblab.wustl.edu/software.html'>eval<\/a>","<a href='https://github.com/EVidenceModeler/EVidenceModeler/releases'>EVidenceModeler (EVM)<\/a>","<a href='https://github.com/aberer/exabayes'>ExaBayes<\/a>","<a href='http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate'>Exonerate<\/a>","<a href='https://github.com/SegataLab/export2graphlan/'>export2graphlan<\/a>","<a href='https://tools.bsc.es/extrae'>Extrae<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly'>Falcon<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly'>Falcon (pb-assembly)<\/a>","<a href='http://fastml.tau.ac.il/'>fastml<\/a>","<a href='https://www.bioinformatics.babraham.ac.uk/projects/fastqc/'>FASTQC<\/a>","<a href='https://rajanil.github.io/fastStructure/'>faststructure<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/'>FASTX-Toolkit<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/'>FASTX-Toolkit<\/a>","<a href='http://www.softberry.com/berry.phtml?topic=fgenesh_plus_plus&group=help&subgroup=pipelines'>FgenesH++<\/a>","<a href='http://animalbiosciences.uoguelph.ca/~msargol/fimpute/'>FImpute<\/a>","<a href='https://github.com/smith-chem-wisc/FlashLFQ'>flashLFQ<\/a>","<a href='https://www.fltk.org/index.php'>fltk<\/a>","<a href='https://github.com/fenderglass/Flye'>Flye<\/a>","<a href='https://sourceforge.net/projects/fpart/'>fpart<\/a>","<a href='https://fpm.readthedocs.io/en/latest/'>fpm<\/a>","<a href='https://sourceforge.net/projects/fraggenescan/'>FragGeneScan<\/a>","<a href='https://github.com/ekg/freebayes'>freebayes<\/a>","<a href='https://surfer.nmr.mgh.harvard.edu/'>freesurfer<\/a>","<a href='https://funannotate.readthedocs.io/en/latest/install.html'>funannotate<\/a>","<a href='https://code.google.com/archive/p/garli/'>GARLI<\/a>","<a href='https://github.com/broadinstitute/gatk'>GATK<\/a>","<a href='http://molevol.cmima.csic.es/castresana/Gblocks_server.html'>gblocks<\/a>","<a href='https://gcc.gnu.org/'>GCC<\/a>","<a href='http://cnsgenomics.com/software/gcta/'>gcta<\/a>","<a href='https://github.com/gnudatalanguage/gdl'>GDL<\/a>","<a href='http://genome.crg.es/software/geneid/'>geneid<\/a>","<a href='http://exon.gatech.edu/GeneMark/'>Genemark<\/a>","<a href='https://github.com/schatzlab/genomescope'>GenomeScope<\/a>","<a href='https://github.com/PacificBiosciences/GenomicConsensus'>GenomicConsensus (Arrow)<\/a>","<a href='https://git-scm.com/about'>Git<\/a>","<a href='http://glew.sourceforge.net/'>glew<\/a>","<a href='https://github.com/glfw/glfw'>glfw<\/a>","<a href='https://sourceware.org/git/?p=glibc.git'>glibc<\/a>","<a href='http://ccb.jhu.edu/software/glimmer/'>glimmer<\/a>","<a href='http://ccb.jhu.edu/software/glimmerhmm/'>glimmerhmm<\/a>","<a href='https://github.com/GlobalArrays/ga'>Global Arrays (GA) toolkit<\/a>","<a href='http://research-pub.gene.com/gmap/'>gmap<\/a>","<a href='http://research-pub.gene.com/gmap/'>gmap_gsnap<\/a>","<a href='https://github.com/epam/GMP'>gmp<\/a>","<a href='https://www.gnu.org/'>GNU<\/a>","<a href='https://www.gnu.org/software/libtool/manual/libtool.html'>GNU libtool<\/a>","<a href='https://www.gnu.org/software/parallel/'>GNU parallel<\/a>","<a href='https://www.gnu.org/software/gsl/'>GNU scientific library<\/a>","<a href='https://sourceforge.net/projects/cbadvanced/'>GNU Tools<\/a>","<a href='https://geronimp.github.io/graftM/'>GraftM<\/a>","<a href='https://github.com/biobakery/graphlan'>GraPhlAn<\/a>","<a href='http://www.gromacs.org/'>gromacs<\/a>","<a href='http://ecogenomics.github.io/GroopM/'>groopm<\/a>","<a href='https://github.com/sanger-pathogens/gubbins'>gubbins<\/a>","guppy","<a href='https://hadoop.apache.org/'>Hadoop<\/a>","<a href='https://github.com/vpc-ccg/haslr'>HASLR<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php?title=Hawkeye'>Hawkeye<\/a>","<a href='https://github.com/SegataLab/hclust2'>Hclust2<\/a>","<a href='https://support.hdfgroup.org/products/hdf4_tools/'>hdf4<\/a>","<a href='https://support.hdfgroup.org/products/hdf5_tools/'>hdf5<\/a>","<a href='https://github.com/higlass'>HiGlass<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772016/'>HiRise<\/a>","<a href='https://daehwankimlab.github.io/hisat2/'>HISAT2<\/a>","<a href='http://hmmer.org/'>HMMER<\/a>","<a href='https://github.com/horovod/horovod'>horovod<\/a>","<a href='https://github.com/htseq/htseq'>HTSeq<\/a>","<a href='http://www.htslib.org/'>HTSlib<\/a>","<a href='https://github.com/loneknightpy/idba'>idba-ud<\/a>","<a href='https://www.l3harrisgeospatial.com/Software-Technology/IDL'>IDL<\/a>","<a href='https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler'>Improved Phased Assembler (pbipa)<\/a>","<a href='http://eddylab.org/infernal/'>Infernal<\/a>","<a href='http://software.broadinstitute.org/software/igv/home'>Integrated Genomics Viewer<\/a>","<a href='https://interproscan-docs.readthedocs.io/en/latest/'>InterProScan<\/a>","<a href='https://github.com/nerscadmin/IPM'>ipm<\/a>","<a href='https://github.com/dereneaton/pyrad'>iPyRAD<\/a>","<a href='http://www.iqtree.org/'>IQ-TREE<\/a>","<a href='https://github.com/PacificBiosciences/IsoSeq'>IsoSeq3<\/a>","<a href='http://mcmc-jags.sourceforge.net/'>jags<\/a>","<a href='https://github.com/genomecuration/JAMg'>JAMg<\/a>","<a href='https://www.oracle.com/java/'>Java<\/a>","<a href='https://github.com/GMOD/jbrowse'>JBrowse<\/a>","<a href='http://www.genome.umd.edu/jellyfish.html'>jellyfish<\/a>","<a href='https://github.com/aidenlab/Juicebox'>juicebox<\/a>","<a href='https://github.com/aidenlab/juicer'>Juicer<\/a>","<a href='https://julialang.org/'>julia<\/a>","<a href='https://pachterlab.github.io/kallisto/about.html'>Kallisto<\/a>","<a href='https://kat.readthedocs.io/en/latest/'>KAT<\/a>","<a href='https://github.com/dib-lab/khmer'>khmer<\/a>","<a href='https://github.com/refresh-bio/KMC'>KMC<\/a>","<a href='https://ccb.jhu.edu/software/kraken/'>kraken<\/a>","<a href='https://github.com/DerrickWood/kraken2/wiki'>kraken2<\/a>","<a href='https://github.com/marbl/Krona/wiki'>krona<\/a>","<a href='https://kubernetes.io/'>Kubernetes<\/a>","<a href='http://www.bx.psu.edu/~rsharris/lastz/'>lastz<\/a>","<a href='https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst'>LFQ-Analyst<\/a>","<a href='https://ofiwg.github.io/libfabric/'>libfabric<\/a>","<a href='https://github.com/agordon/libgtextutils'>libgtextutils<\/a>","<a href='https://www.nongnu.org/libunwind/'>libunwind<\/a>","<a href='https://github.com/bcgsc/LINKS'>LINKS<\/a>","<a href='https://www.lipidmaps.org/'>Lipid Maps<\/a>","<a href='https://llvm.org/'>llvm<\/a>","<a href='https://lmod.readthedocs.io/en/latest/'>lmod<\/a>","<a href='https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger'>Long Ranger<\/a>","<a href='http://csg.sph.umich.edu/abecasis/mach/index.html'>mach<\/a>","<a href='https://github.com/macs3-project/MACS'>macs<\/a>","<a href='https://github.com/macs3-project/MACS'>macs2<\/a>","<a href='https://github.com/ranwez/MACSE_V2_PIPELINES'>MACSE<\/a>","<a href='http://mafft.cbrc.jp/alignment/software/'>mafft<\/a>","<a href='https://github.com/WatsonLab/MAGpy'>MAGpy<\/a>","<a href='http://www.yandell-lab.org/software/maker.html'>MAKER<\/a>","<a href='http://www.yandell-lab.org/software/maker-p.html'>MAKER-P<\/a>","<a href='https://cran.r-project.org/web/packages/MALDIquant/index.html'>MALDIquant<\/a>","<a href='https://github.com/marbl/MashMap'>mashmap<\/a>","<a href='https://github.com/kwongj/maskrc-svg'>maskrc-svg<\/a>","<a href='https://github.com/alekseyzimin/masurca'>MaSuRCA<\/a>","<a href='https://www.mathworks.com/products/matlab.html'>MATLAB<\/a>","<a href='https://github.com/dtreichler/matserver'>matserver<\/a>","<a href='http://darlinglab.org/mauve/download.html'>Mauve<\/a>","<a href='https://maven.apache.org/'>maven<\/a>","<a href='https://sourceforge.net/projects/maxbin2/'>MaxBin2<\/a>","<a href='https://www.maxquant.org/'>MaxQuant<\/a>","<a href='https://www.maxquant.org/'>MaxQuant (biocontainers)<\/a>","<a href='https://micans.org/mcl/'>mcl<\/a>","<a href='https://github.com/nanoporetech/medaka'>Medaka<\/a>","<a href='https://github.com/voutcn/megahit'>megahit<\/a>","<a href='https://github.com/marbl/merqury'>Merqury<\/a>","<a href='https://bitbucket.org/berkeleylab/metabat/src/master/'>metabat<\/a>","<a href='https://sourceforge.net/projects/metacv/'>metacv<\/a>","<a href='https://github.com/DiltheyLab/MetaMaps'>MetaMaps<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/metaMS.html'>metaMS<\/a>","<a href='https://huttenhower.sph.harvard.edu/metaphlan'>MetaPhlAn<\/a>","<a href='http://cab.spbu.ru/software/spades/'>metaspades<\/a>","<a href='https://peterbeerli.com/migrate-html5/index.html'>MIGRATE<\/a>","<a href='https://gatb.inria.fr/software/minia/'>minia<\/a>","<a href='https://genome.sph.umich.edu/wiki/Minimac4'>Minimac4<\/a>","<a href='https://github.com/lh3/minimap2'>minimap2<\/a>","<a href='https://sourceforge.net/p/mira-assembler/wiki/Home/'>mira<\/a>","<a href='https://www.nature.com/articles/nbt1394#Sec11'>mirdeep<\/a>","<a href='https://github.com/rajewsky-lab/mirdeep2'>mirdeep2<\/a>","<a href='https://github.com/yarden/MISO'>miso<\/a>","<a href='http://mixomics.org/'>mixOmics<\/a>","<a href='https://github.com/tseemann/mlst'>MLST<\/a>","<a href='https://github.com/compomics/moFF'>moFF<\/a>","<a href='https://www.mono-project.com/'>Mono<\/a>","<a href='https://cwenger.github.io/Morpheus/'>Morpheus<\/a>","<a href='https://www.mothur.org/'>Mothur<\/a>","<a href='https://www.mpfr.org/'>mpfr<\/a>","mpiBLAST","<a href='https://github.com/LLNL/mpiP'>mpiP (mpi profiling library)<\/a>","<a href='https://github.com/NBISweden/MrBayes'>MrBayes<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSGFplus.html'>MS-GF+<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSnbase.html'>MSnbase readMSData<\/a>","<a href='https://bitbucket.org/joseph07/multisnv/wiki/Home'>multisnv<\/a>","<a href='http://mummer.sourceforge.net/'>MUMmer<\/a>","<a href='http://www.drive5.com/muscle/'>MUSCLE<\/a>","<a href='https://mvapich.cse.ohio-state.edu/'>MVAPICH<\/a>","<a href='https://software.opensuse.org/package/mxml'>mxml<\/a>","<a href='https://www.nag.com/'>nag<\/a>","<a href='https://github.com/jts/nanopolish'>Nanopolish<\/a>","<a href='https://www.nasm.us/'>NASM<\/a>","<a href='https://github.com/NVIDIA/nccl'>NCCL<\/a>","<a href='https://gitlab.com/remikz/nccmp'>nccmp<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml'>NCL<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml'>NCL<\/a>","<a href='http://nco.sourceforge.net/'>NCO<\/a>","<a href='http://meteora.ucsd.edu/~pierce/ncview_home_page.html'>Ncview<\/a>","<a href='https://github.com/xiaochuanle/NECAT'>NECAT<\/a>","<a href='https://www.unidata.ucar.edu/software/netcdf/'>NetCDF<\/a>","<a href='https://github.com/neurodebian/neurodebian'>neurodebian<\/a>","<a href='https://github.com/Nextomics/NextDenovo'>NextDenovo<\/a>","<a href='https://www.nextflow.io/'>nextflow<\/a>","<a href='http://cibiv.github.io/NextGenMap/'>nextgenmap<\/a>","<a href='https://github.com/Nextomics/NextPolish'>NextPolish<\/a>","<a href='https://github.com/ncbi/ngs'>NGS<\/a>","<a href='https://github.com/shenlab-sinai/ngsplot'>ngsplot<\/a>","<a href='https://nucleoatac.readthedocs.io/en/latest/'>nucleoatac<\/a>","<a href='https://github.com/NVlabs/nvbio'>nvbio<\/a>","<a href='https://developer.nvidia.com/hpc-sdk'>NVIDIA HPC SDK<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/oases/'>Oases<\/a>","<a href='https://www.vi-hps.org/tools/opari2.html'>OPARI2<\/a>","<a href='https://www.open-mpi.org/'>Open MPI<\/a>","<a href='https://www.open-mpi.org/'>Open MPI (ib)<\/a>","<a href='https://www.open-mpi.org/'>Open MPI 2 (ib)<\/a>","<a href='https://www.openms.de/'>OpenMS<\/a>","<a href='https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper'>OPLS-DA_Contrasts (w4mcorcov)<\/a>","<a href='https://github.com/kishwarshafin/pepper'>P.E.P.P.E.R. (+ deep variant)<\/a>","<a href='https://github.com/TomSkelly/PacBioEDA'>PacBioEDA<\/a>","padb","<a href='http://icl.utk.edu/papi/software/'>PAPI (Performance Application Programming Interface)<\/a>","<a href='https://www.vi-hps.org/tools/paraver.html'>PARAVER<\/a>","<a href='https://www.paraview.org/'>ParaView<\/a>","<a href='https://github.com/marbl/parsnp'>Parsnp<\/a>","<a href='https://github.com/PASApipeline/PASApipeline/wiki'>PASA<\/a>","<a href='https://github.com/NixOS/patchelf'>PatchELF<\/a>","<a href='https://downloadcenter.intel.com/download/19792'>PDT (Processor Diagnostic Tool)<\/a>","<a href='http://compomics.github.io/projects/peptide-shaker.html'>Peptide Shaker<\/a>","<a href='http://percolator.ms/'>Percolator<\/a>","<a href='https://www.php.net/'>PHP<\/a>","<a href='https://broadinstitute.github.io/picard/'>PICARD<\/a>","<a href='http://picrust.github.io/picrust/'>PICRUSt<\/a>","<a href='https://github.com/picrust/picrust2/wiki'>PICRUSt 2<\/a>","<a href='https://www.drive5.com/piler/'>PILER-DF<\/a>","<a href='https://github.com/broadinstitute/pilon/wiki'>Pilon<\/a>","<a href='http://zzz.bwh.harvard.edu/plink/'>plink<\/a>","<a href='http://atgu.mgh.harvard.edu/plinkseq/index.shtml'>plinkseq<\/a>","<a href='https://parallel-netcdf.github.io/'>PnetCDF<\/a>","<a href='https://github.com/matsen/pplacer'>pplacer<\/a>","<a href='https://github.com/wtsi-hpag/PretextView'>PretextView / Pretext Suite<\/a>","<a href='http://prinseq.sourceforge.net/'>PRINSEQ<\/a>","<a href='http://www.genabel.org/packages/ProbABEL'>probabel<\/a>","<a href='https://github.com/rsilvabioinfo/ProbMetab'>ProbMetab Tool<\/a>","<a href='https://github.com/hyattpd/Prodigal'>Prodigal<\/a>","<a href='https://github.com/tseemann/prokka'>Prokka<\/a>","<a href='https://github.com/dfguan/purge_dups'>Purge_Dups<\/a>","<a href='https://bitbucket.org/mroachawri/purge_haplotigs/src/master/'>PurgeHaplotigs<\/a>","<a href='https://pylib.readthedocs.io/en/1.4.3/index.html'>pylibs<\/a>","<a href='https://ipyrad.readthedocs.io/en/latest/'>PyRAD<\/a>","<a href='https://github.com/pysam-developers/pysam'>pysam<\/a>","<a href='https://www.python.org/'>python<\/a>","<a href='https://www.python.org/'>python2<\/a>","<a href='https://www.python.org/'>python2-as-python<\/a>","<a href='https://www.python.org/'>python3<\/a>","<a href='https://www.python.org/'>python3-as-python<\/a>","<a href='https://pytorch.org/'>PyTorch<\/a>","<a href='http://qiime.org/'>QIIME<\/a>","<a href='http://qiime.org/'>QIIME<\/a>","<a href='https://qiime2.org/'>QIIME2.0<\/a>","<a href='http://bioinf.spbau.ru/quast'>QUAST<\/a>","<a href='https://github.com/mahulchak/quickmerge'>Quickmerge<\/a>","<a href='https://www.r-project.org/'>R<\/a>","<a href='https://www.rabbitmq.com/'>RabbitMQ server<\/a>","<a href='https://github.com/isovic/racon'>Racon<\/a>","<a href='https://github.com/bcgsc/RAILS'>RAILS/Cobbler<\/a>","<a href='http://ratt.sourceforge.net/'>RATT<\/a>","<a href='https://github.com/comprna/RATTLE'>RATTLE<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/galaxyp/rawtools/51e4aa55ea04'>Raw Tools<\/a>","<a href='https://sco.h-its.org/exelixis/web/software/raxml/index.html'>RAxML<\/a>","<a href='https://github.com/dparks1134/RefineM'>RefineM<\/a>","<a href='https://github.com/simoncchu/REPdenovo'>REPdenovo<\/a>","<a href='http://www.repeatmasker.org/'>RepeatMasker<\/a>","<a href='http://www.repeatmasker.org/RepeatModeler/'>RepeatModeler<\/a>","<a href='https://github.com/Dfam-consortium/RepeatModeler'>RepeatModeler2<\/a>","<a href='https://www.yandell-lab.org/software/repeatrunner.html'>RepeatRunner<\/a>","<a href='https://urgi.versailles.inra.fr/Tools/REPET'>REPET<\/a>","<a href='https://cran.r-project.org/web/packages/rjags/index.html'>rjags<\/a>","<a href='https://deweylab.github.io/RSEM/'>RSEM<\/a>","<a href='http://rseqc.sourceforge.net/'>rseqc<\/a>","<a href='https://rstudio.com/'>Rstudio<\/a>","<a href='https://www.ruby-lang.org/en/downloads/'>Ruby<\/a>","<a href='https://combine-lab.github.io/salmon/'>Salmon<\/a>","<a href='https://github.com/GregoryFaust/samblaster'>samblaster<\/a>","<a href='http://www.htslib.org/'>SAMTools<\/a>","<a href='https://www.sas.com/en_us/software/foundation.html'>SAS<\/a>","<a href='http://compbio.cs.toronto.edu/hapsembler/scarpa.html'>Scarpa<\/a>","<a href='https://www.scipy.org/'>SciPy<\/a>","<a href='https://scons.org/'>SCons<\/a>","<a href='https://pypi.python.org/pypi/screed'>screed<\/a>","<a href='http://compomics.github.io/projects/searchgui.html'>Search GUI<\/a>","<a href='https://github.com/shenwei356/seqkit'>SeqKit<\/a>","<a href='https://github.com/drostlab/seqstats'>seqstats<\/a>","<a href='https://github.com/trinityrnaseq/seqtk-trinity'>seqtk-trinity<\/a>","<a href='https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'>shapeit<\/a>","<a href='https://github.com/najoshi/sickle'>sickle<\/a>","<a href='https://www.bryntum.com/products/siesta/'>Siesta<\/a>","<a href='https://singularity.lbl.gov/'>singularity<\/a>","<a href='https://github.com/relipmoc/skewer'>skewer<\/a>","<a href='https://www.pacb.com/support/software-downloads/'>SMRT Link / Analysis<\/a>","<a href='https://snakemake.readthedocs.io/en/stable/'>Snakemake<\/a>","<a href='https://github.com/KorfLab/SNAP'>SNAP<\/a>","<a href='https://github.com/PMMAraujo/snappy'>SNAPPy<\/a>","<a href='http://lowelab.ucsc.edu/snoscan/'>snoscan<\/a>","<a href='http://snpeff.sourceforge.net/'>snpeff<\/a>","soap","<a href='http://i.cs.hku.hk/2bwt-tools/soap3/index.html'>soap3-gpu<\/a>","<a href='https://github.com/aquaskyline/SOAPdenovo2'>SOAPdenovo2<\/a>","<a href='https://sourceforge.net/projects/soapsnp/'>SOAPsnp<\/a>","<a href='https://github.com/biocore/sortmerna'>sortmerna<\/a>","<a href='https://github.com/ablab/spades'>SPAdes<\/a>","<a href='https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi'>Splign<\/a>","<a href='https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc'>sra-tools<\/a>","<a href='http://catchenlab.life.illinois.edu/stacks/'>Stacks<\/a>","<a href='https://github.com/alexdobin/STAR'>star<\/a>","<a href='http://web.stanford.edu/group/pritchardlab/structure.html'>structure<\/a>","<a href='https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome'>Supernova<\/a>","<a href='https://github.com/hall-lab/svtyper'>svtyper<\/a>","<a href='https://github.com/TimoLassmann/tagdust'>tagdust<\/a>","<a href='https://bitbucket.org/tasseladmin/tassel-5-standalone'>tassel<\/a>","<a href='http://www.cs.uoregon.edu/research/tau/home.php'>TAU<\/a>","<a href='https://www.tensorflow.org/resources/tools'>TensorFlow<\/a>","<a href='https://github.com/compomics/ThermoRawFileParser'>ThermoRawFileParser<\/a>","<a href='https://github.com/bcgsc/tigmint'>Tigmint<\/a>","<a href='https://ccb.jhu.edu/software/tophat/index.shtml'>tophat<\/a>","<a href='https://totalview.io/'>TotalView<\/a>","<a href='https://github.com/TransDecoder/TransDecoder/wiki'>TransDecoder<\/a>","<a href='https://tandem.bu.edu/trf/trf.html'>trf<\/a>","<a href='http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'>Trim Galore<\/a>","<a href='http://www.usadellab.org/cms/?page=trimmomatic'>Trimmomatic<\/a>","<a href='https://github.com/trinityrnaseq/trinityrnaseq/wiki'>Trinity / trinityrnaseq<\/a>","<a href='https://github.com/Trinotate/Trinotate.github.io/wiki'>Trinotate<\/a>","<a href='http://eddylab.org/software.html'>trnascan-se<\/a>","<a href='http://hgdownload.soe.ucsc.edu'>UCSC tools<\/a>","<a href='https://www.openucx.org/'>UCX (Unified Communication X)<\/a>","<a href='https://www.unidata.ucar.edu/software/udunits/'>UDUNITS<\/a>","<a href='https://github.com/rrwick/Unicycler'>Unicycler<\/a>","<a href='http://www.unixodbc.org/utilities.html'>unixODBC<\/a>","<a href='https://github.com/marbl/VALET'>Valet<\/a>","<a href='https://github.com/cc2qe/vawk'>vawk<\/a>","<a href='https://vcftools.github.io/index.html'>VCFTools<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/velvet/'>Velvet<\/a>","<a href='https://www.tbi.univie.ac.at/RNA/documentation.html'>ViennaRNA<\/a>","<a href='https://wci.llnl.gov/simulation/computer-codes/visit/'>VisIt<\/a>","<a href='https://github.com/torognes/vsearch'>vsearch<\/a>","<a href='https://github.com/adigenova/wengan'>Wengan<\/a>","<a href='http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html'>WindowMasker<\/a>","<a href='https://www.winehq.org/'>Wine<\/a>","<a href='http://www.ebi.ac.uk/~birney/wise2/'>wise<\/a>","<a href='https://workflow4metabolomics.org/'>Workflow 4 Metabolomics<\/a>","<a href='https://github.com/ruanjue/wtdbg2'>WTDBG2<\/a>","<a href='https://www.wxwidgets.org/'>wxWidgets<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/xcms.html'>xcms<\/a>","<a href='https://xerces.apache.org/xerces-c/'>Xerces-C++ XML parser<\/a>","<a href='https://github.com/AustralianBioCommons/xml4ena'>xml4ena<\/a>","<a href='https://github.com/freedesktop/xorg-macros'>xorg-macros<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem/29e2df3edf11'>xtandem<\/a>","<a href='https://zlib.net/'>zlib<\/a>","zstd"],["3D-DNA","abricate","abyss","allmaps","allpathslg","amos","anaconda","angsd","ant","apollo","arima mapping","arm-forge","arm-reports","artemis","aspera","augustus","autoconf","automake","bamkit","bamm","bamtools","batchcorrection","bazel","bbcp","bbmap","bcftools","beagle","beagle-lib","beast","beast2","bedtools","binutils","bioconductor","biolinux","Bionano Access","bioperl","biopython","bismark","blast","blast+","blast2","blat","BlobToolKit","blobtools","boost","bowtie","bowtie2","braker1","braker2","breakdancer","bricks","bsmap","bs_seeker","bsseeker2","busco","bwa","bx-python","bzip2","cactus","camera","canu","cap3","cardinal","cat_bins","cdhit","celseq2","celera","centos","centos6","checkm","checkm-genome","chunkchromosome","clustalw","clusteringsuite","cmake","cnvkit","cpc","cpc2","coreutils","cpat","cppunit","cube","cuda","cudnn","cufflinks","curl","cutadapt","cytoscape","darshan","deeptools","deepvariant","dendropy","DIA-Umpire","diamond","discovar_de_novo","dms_tools2","dmtcp","dssp","easybuild","edena","embedded-nimrod","emboss","ensembl","genebuild","erlang","eval","EvidenceModeler","exabayes","exonerate","export2graphlan","extrae","falcon","pb-assembly","fastml","fastqc","faststructure","fastx_toolkit","fastx","fgenesh","fimpute","flashlfq","fltk","flye","fpart","fpm","fraggenescan","freebayes","freesurfer","funannotate","garli","gatk","gblock","gcc","gcta","gdl","geneid","genemark","genomescope","GenomicConsensus","git","glew","glfw","glibc","glimmer","glimmerhmm","ga","gmap","gmap_gsnap","gmp","gnu","libtool","parallel","gsl","gnutools","graftm","graphlan","gromacs","groopm","gubbins","guppy","hadoop","haslr","hawkeye","hclust2","hdf4","hdf5","higlass","hirise","hisat2","hmmer","horovod","htseq","htslib","idba-ud","idl","pbipa","infernal","igv","interproscan","ipm","ipyrad","iq-tree","isoseq3","jags","jamg","java","jbrowse","jellyfish","juicebox","juicer","julia","kallisto","kat","khmer","kmc","kraken","kraken2","krona","k8","lastz","LFQ-Analyst","libfabric","libgtextutils","libunwind","links","lipid_maps","llvm","lmod","longranger","mach","macs","macs2","macse","mafft","magpy","maker","maker-p","maldi_quant","mashmap","maskrc-svg","masurca","matlab","matserver","mauve","maven","maxbin","maxquant","biocontainers-maxquant","mcl","medaka","megahit","merqury","metabat","metacv","metamaps","metams","metaphlan","metaspades","MIGRATE","minia","minimac4","minimap2","mira","mirdeep","mirdeep2","miso","mixomics","mlst","moff","mono","morpheus","Mothur","mpfr","mpiblast","mpip","mrbayes","msgfplus","msnbase","multisnv","mummer","muscle","mvapich2_ib","mxml","nag","nanopolish","nasm","nccl","nccmp","ncl","ncl_ncarg","nco","ncview","necat","netcdf","neurodebian","nextdenovo","nextflow","nextgenmap","nextpolish","ngs","ngsplot","nucleoatac","nvbio","nvidia-hpc-sdk","oases","opari2","openmpi","openmpi_ib","openmpi2_ib","openms","w4mcorcov","pepper+deepvariant","pacbioeda","padb","papi","paraver","paraview","parsnp","PASA","patchelf","pdt","peptideshaker","percolator","php","picard","picrust","picrust2","piler-df","pilon","plink","plinkseq","pnetcdf","pplacer","pretextview","prinseq","probabel","probmetab","prodigal","prokka","purge_dups","purgehaplotigs","pylibs","pyrad","pysam","python","python2","python2-as-python","python3","python3-as-python","pytorch","qiime","qiime-modules","qiime2","quast","quickmerge","r","rabbitmq-server","Racon","rails_cobbler","ratt","rattle","rawtools","raxml","refinem","repdenovo","repeatmasker","repeatmodeler","repeatmodeler2","repeatrunner","repet","rjags","rsem","rseqc","rstudio","ruby","salmon","samblaster","samtools","sas","scarpa","scipy","scons","screed","searchgui","biocontainers-seqkit","seqstats","seqtk-trinity","shapeit","sickle","siesta","singularity","skewer","smrtlink","snakemake","snap","snappy","snoscan","snpeff","soap","soap3-gpu","soapdenovo2","soapsnp","sortmerna","spades","splign","sra-tools","stacks","star","structure","supernova","svtyper","tagdust","tassel","tau","tensorflow","ThermoRawFileParser","tigmint","tophat","totalview","transdecoder","trf","trim_galore","trimmomatic","trinity","trinotate","trnascan-se","ucsc","ucx","udunits","unicycler","unixodbc","valet","vawk","vcftools","velvet","viennarna","visit","vsearch","wengan","windowmasker","wine","wise","workflow4metabolomics","wtdbg2","wxwidgets","xcms","xerces-c","xml4ena","xorg-macros","xtandem","zlib","zstd"],["Sequence assembly","Sequence analysis","Sequence assembly","Sequence assembly","Sequence assembly","Sequence assembly","Gene structure","Population genomics","","Genomics / Genome annotation","Chromosome conformation capture","","","Genomics / Genome annotation","","Genomics / Genome annotation","","","Sequence analysis","Phylogenomics","Sequence analysis","Metabolomics","","","Sequence analysis","Sequence analysis","Population genetics","Population genomics","Phylogenetics","Phylogenetics","Sequence analysis","","","","Sequence analysis","Programming language / platform","Programming language / platform","Epigenomics","Sequence alignment","Sequence alignment","Sequence alignment","Sequence alignment","Quality assurance","Quality assurance","Library","Sequence alignment","Sequence alignment","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","","Epigenomics","Epigenomics","Epigenomics","Quality assurance","Sequence alignment","Sequence analysis","","Sequence alignment","Metabolomics","Sequence assembly","Sequence assembly","Bioimaging","Metagenomics","Sequence analysis","Transcriptomics","Sequence assembly","","","Quality assurance","Quality assurance","Sequence analysis","Sequence alignment","Statistics and probability","","DNA structural variation","Genomics / Genome annotation","Genomics / Genome annotation","Library","Genomics / Genome annotation","","","","Library","Sequence assembly","","Sequence analysis","Multiomics","","Sequence analysis","Variant Calling","Phylogenomics","Proteomics / Metaproteomics","Sequence alignment","Sequence assembly","Genetic variation","","Protein folding, stability and design","","Sequence assembly","","Molecular biology","Data visualisation","Genomics / Genome annotation","Programming language / platform","Quality assurance","Genomics / Genome annotation","Phylogenomics","Sequence analysis","Phylogenomics","","","Sequence assembly","Phylogenomics","Quality assurance","Phylogenomics","Sequence analysis","Sequence analysis","Genomics / Genome annotation","Population genomics","Proteomics / Metaproteomics","","Sequence assembly","","","Sequence analysis","Phylogenomics","Bioimaging","Genomics / Genome annotation","Phylogenetics","Sequence analysis","Phylogenetics","Library","Phylogenomics","Library","Genomics / Genome annotation","Genomics / Genome annotation","Sequence analysis","Sequence assembly","","Library","Library","Library","Genomics / Genome annotation","Genomics / Genome annotation","","Sequence alignment","Sequence alignment","","Programming language / platform","Library","","Library","","Meta-Omics","Phylogenomics","Molecular dynamics","Metagenomics","Phylogenomics","Sequencing","Library","Sequence assembly","Quality assurance","Data visualisation","","","Data visualisation","Sequence assembly","Sequence alignment","Sequence analysis","","Sequence analysis","Library","Sequence assembly","Programming language / platform","Sequence assembly","Genomics / Genome annotation","Data visualisation","Protein sequence features","","Sequence analysis","Phylogenomics","Transcriptomics","Phylogenomics","Genomics / Genome annotation","Programming language / platform","Data visualisation","Sequence analysis","Sequence assembly","Sequence assembly","Programming language / platform","Transcriptomics","Sequence analysis","Sequence analysis","Sequence analysis","Taxonomy","Taxonomy","Data visualisation","Programming language / platform","Sequence alignment","Proteomics / Metaproteomics","Library","Library","","Sequence assembly","Metabolomics","","","Sequence alignment","Phylogenomics","ChIP-seq","ChIP-seq","Sequence alignment","Sequence alignment","Metagenomics","Genomics / Genome annotation","Genomics / Genome annotation","Proteomics","Sequence alignment","","Sequence assembly","","","Sequence alignment","","Metagenomics","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Statistics and probability","Sequence assembly","Metagenomics","Quality assurance","Metagenomics","Metagenomics","Metagenomics","Metabolomics","Metagenomics","Metagenomics","Population genomics","Sequence assembly","Phylogenomics","Sequence alignment","Sequence assembly","Transcriptomics","Transcriptomics","Transcriptomics","Multiomics","Sequence analysis","Proteomics / Metaproteomics","Programming language / platform","Proteomics / Metaproteomics","Metagenomics","","Sequence alignment","Library","Phylogenomics","Proteomics / Metaproteomics","Metabolomics","Variant Calling","Sequence alignment","Genomics / Genome annotation","","Library","Library","Sequence analysis","","","","Data visualisation","Data visualisation","","","Sequence assembly","Library","Neuroscience","Sequence assembly","Workflows","Sequence alignment","Sequence assembly","Knowledge discovery in databases","Data visualisation","Genomics / Genome annotation","Library","Library","Sequence assembly","","","","","Proteomics / Metaproteomics","Metabolomics","Sequence assembly","Sequence analysis","","","Data visualisation","Data visualisation","Phylogenomics","Genomics / Genome annotation","","","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Programming language / platform","Sequence analysis","Phylogenomics","Metagenomics","Genomics / Genome annotation","Sequence assembly","Phylogenomics","Variant Calling","Library","Phylogenomics","Data visualisation","Sequence analysis","Phylogenomics","Metabolomics","Genomics / Genome annotation","Genomics / Genome annotation","Sequence assembly","Sequence assembly","Library","Sequence analysis","Sequence analysis","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","","Metagenomics","Metagenomics","Metagenomics","Quality assurance","Sequence assembly","Programming language / platform","","Sequence assembly","Sequence assembly","Genomics / Genome annotation","Sequence assembly","Proteomics / Metaproteomics","Phylogenomics","Population genomics","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Phylogenomics","Transcriptomics","Quality assurance","Programming language / platform","Programming language / platform","Transcriptomics","Phylogenomics","Sequence analysis","Statistics and probability","Sequence assembly","Programming language / platform","","Sequence analysis","Proteomics / Metaproteomics","Sequence analysis","Sequence analysis","Sequence analysis","Phylogenomics","Sequence analysis","","Programming language / platform","Sequence analysis","Sequence analysis","Workflows","Genomics / Genome annotation","Phylogenomics","Genomics / Genome annotation","Variant Calling","Quality assurance","Sequence alignment","Sequence assembly","Sequence assembly","Metagenomics","Sequence assembly","Genomics / Genome annotation","Knowledge discovery in databases","Sequence assembly","Sequence alignment","Population genomics","Sequence assembly","Phylogenomics","Sequence analysis","Phylogenomics","","","Proteomics / Metaproteomics","Sequence assembly","Sequence alignment","","Transcriptomics","Genomics / Genome annotation","Sequence analysis","Sequence analysis","Sequence assembly","Transcriptomics","Transcriptomics","Sequence analysis","","","Sequence assembly","","Metagenomics","Quality assurance","Genetic variation","Sequence assembly","Transcriptomics","","Sequence analysis","Sequence assembly","Genomics / Genome annotation","","Sequence alignment","Metabolomics","Sequence assembly","","Metabolomics","","Data submission, annotation and curation","","Proteomics / Metaproteomics","",""],["","Yes","","","","","","","","","","","","","","Yes","","","","","","Yes","","","","Yes","","","","","Yes","","","","","","","","","Yes","","","","","","","Yes","","","","","","","","Yes","Yes","","","","Yes","Yes","","Yes","Yes","Yes","","","","","","","","Yes","","","","","","","","","","","","Yes","","Yes","","","Yes","","","Yes","Yes","","","","","","","","Yes","","","","","","","","","","","","","Yes","","Yes","Yes","","","Yes","","Yes","","","Yes","Yes","","","","","","","","","","","","","","","","","","","","","","","","","","","","","Yes","Yes","","Yes","","","","","","","","","","Yes","Yes","","Yes","","","","","Yes","","","","","Yes","","","","","Yes","","","","","Yes","","","","Yes","Yes","","","Yes","","","","","","Yes","","","","","Yes","Yes","","Yes","","","","Yes","","","","","","","","Yes","Yes","Yes","","","Yes","","","","","","","Yes","","","","Yes","","","Yes","","Yes","Yes","Yes","","Yes","Yes","","","","","Yes","Yes","","","","","","","Yes","","","","","","","","","","","","","","","","","","","","","","","","","Yes","Yes","","","","","","","","","","","Yes","Yes","","Yes","Yes","","","Yes","","","","","","","","","","Yes","","","","","","","","","","","","","","","Yes","","","","Yes","","","","Yes","Yes","","","","","","","","","","Yes","","","Yes","","Yes","","","","","","Yes","","","","","","","","","","","Yes","","","Yes","","","","","Yes","Yes","","","Yes","Yes","","","","","","","","Yes","","Yes","","Yes","","Yes","Yes","Yes","Yes","Yes","Yes","","","Yes","","","","Yes","Yes","","","Yes","","","","","Yes","","","Yes","","","","Yes","",""],["","","2.2.3","","","","","","1.10.7","","","20.0.2","20.0.2","","3.9.6","","","","","","","","0.26.0<br />0.27.1<br />3.1.0","15.02.03.00.1","","1.9","","3.1.2","1.10.4","","2.28.0","","","","","","","","2.10.1","","","","","","1.71.0<br />1.72.0","","2.3.5.1","","","","","","","","","0.7.17","","","","","1.9<br />2<br />2.0t","","","","","","","","","","","","","","3.16.2<br />3.18.2","","","","","","1.14.0","4.4<br />4.5","10.1<br />11.0.3","7.6.5-cuda10.1","","","","","3.2.1","","","","","","","","","","","","","","","","22.1","","","","","","","","","","","","","","","","","1.3.5","","","","","","","","","4.1.4.0","","system","","0.9.9","","","","","2.24.1","2.1.0","3.3.2","","","","5.7<br />5.7.2","","","","","","20191022","2.6","","","","2018.3<br />2018.3-plumed<br />2019.3<br />2019.3-gpuvolta<br />2019.3-plumed<br />2020.1<br />2020.1-gpuvolta<br />2020.1-plumed<br />2020.3<br />2020.3-gpuvolta","","","","","","","","4.2.14<br />4.2.15","1.10.5<br />1.10.5p<br />1.8.21<br />1.8.21p","","","","","0.19.0","","1.9","","8.6<br />8.7.2<br />8.8","","","","","2.0.6","","","","","","jdk-13.33<br />jdk-8.40","","2.3.0","","","1.5.3","","","","","","","","0.2.5","","","1.8.0a1","","1.2.1","","","10.0.0<br />11.0.0","","","","","","","","","","","","","","","R2019b","2018<br />2020","","","","","","","","","","","","","","","","","","","2.17","","","","","","","","","","","","","3.4.1","","","","","","","","","fll6i26dcl<br />fll6i26dcl-mkl<br />nll6i27dbl<br />nll6i27dbl-i8<br />nll6i27dbl-mkl","","","2.5.6-1+cuda10.1<br />2.6.4-1+cuda10.1","1.8.5.0","6.6.2","","4.7.7<br />4.9.2","2.1.7","","4.6.3<br />4.6.3-i8r8<br />4.6.3p<br />4.7.1<br />4.7.1p<br />4.7.3<br />4.7.3p<br />4.7.4<br />4.7.4p","","","","","","","","","","20.7","","2.0.3<br />2.0.5","2.1.6<br />2.1.6-debug<br />2.1.6-mt<br />2.1.6-mt-debug<br />3.0.4<br />3.0.4-debug<br />3.1.4<br />3.1.4-debug<br />4.0.1<br />4.0.1-debug<br />4.0.2<br />4.0.2-debug<br />4.0.3<br />4.0.3-debug","","","","","","","3.2","5.7.0","","5.8.0<br />5.8.0-gpu<br />5.8.0-mesa","","","0.1","","","","5.6.40","","","","","","","","1.11.2","","","","","","","","","","","","","","2.7.16<br />2.7.17","","3.7.4<br />3.8.5","","1.4.0a0<br />1.5.1","","","","","","3.6.1<br />4.0.0","3.8.0","","","","","","","","","","","","","","","","","","","1.1.0","","1.1<br />1.9","","","","3.1.1","","","","","","","","4.1-b4<br />4.1.5","","","","","","","","","","","2.41","","","","","","","","","","","","","","2.0.0<br />2.1.0<br />2.3.0","","","2.1.1","2019.3.14<br />2020.1.13","","","","","2.9.1","","","","1.6.1<br />1.8.0<br />1.8.1<br />1.9.0","1.12.11<br />2.2.26","","2.3.7","","","","1.2.10","","3.1.2","","","","","","","","3.0.4","","3.2.2","","","","","1.4.3"],["","","","","","3.1.0","","","","","","","","","","","","","0.1","","2.4.1","","","","","1.6","","2.1.2","1.8.4<br />2.4.7","","2.26.0","","","","","1.7.1","1.70","0.19.0","2.2.26","2.10.1<br />2.7.1","","3.6","","","","1.2.1.1","2.3.3.1<br />2.3.4.1","","","1.1.2_2013_03_08","","2.90","","2.1.3","","0.7.17","","","","","1.6","20130608","","","4.6.8","","8.3rc2","","","","","2014-05-27","2.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","6.6.0","","","","2.2.8","","","2.2.0","","","","","3.1","0.11.4","1.0","0.0.14","","","","","","","","","","0.9.21","","","","","0.91b","","1.26.0","","1.4.4","","","","","","","","3.02b","3.0.4","","2017-11-15","","","","","","","","","","","","","","","","","","","","","","","3.0<br />3.1b2","","","1.2.1<br />1.6","","","","","2.4.14","5.27-66.0","","","","","","","","","2.2.6","","","","","","2.1.2","","","","","","1.04.00","","","0.7","","","","","","","1.0.18","","","","7.313","","","","","","","","","","","","","","","14-137","","","","","","","","","","","","","","","","","","","","","","","","","","","3.2.6","","","","3.23","3.8.1551","","","","","","","","","","","","","","","","","","","1.3.0","","","","","0.2.08","","","","","","","","","","","","","","","","","","","","","","","","","1.07","0.10","","","","","0.5.0","","","","","","","3.0.66","0.13","","","","","","","","","","","","","","","","","","","8.2.11","","","","","","","","","1.3.0<br />1.3.3","","","","","0.1.24","0.1.19<br />1.6","","0.241","","","1.0","","","","0.0.2","r904","1.33","","","0.1.127","","","0.15","","","4.3t","2.21","r146","r240","","","3.11.1","","2.8.2-5","1.48<br />2.1","2.5.3a","2.3.4","","0.1.4","","4.3","","","","","","","5.0.2","4.09","","0.36","2.5.1","","2.0.0","357","","","","","","0.0.1","0.1.15","1.2.10","","","2.6.0","","","","2.4.1","","","","","","","","","",""],["","","","","","","","","","","","","","","","","","","","","","","","","","","","2.1.2","","","","","","","","","1.70","","","","","","","","","","","","","","","","","2.1.3","","0.7.17","","","","","","","","","","","","","","","","","2.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","1.4.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","2.1.2","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","3.2.6","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","0.13","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","0.1.19<br />1.6","","","","","1.0","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],["","1.0.1","2.0.2","","","","2019.07<br />2020.02<br />4.2.0<br />4.3.1<br />5.2.0","0.931<br />0.933","","","","","","","","3.3.3","2.69-GCC-4.9.2","1.15-gcc-4.8.2<br />1.15-GCC-4.9.2<br />1.5-gcc-4.8.2","","201710<br />201811","2.4.0","","","","201710","1.3","2.1<br />4.1","","1.8.0<br />1.8.1<br />1.8.2","","2.25.0","2.25<br />2.25-GCC-4.9.2","3.6-rc2<br />3.8-rc2","8","","1.007002<br />1.007002-sw","1.66","0.15.0<br />0.22.3","2.3.0","","2.10.1<br />2.9.0","35","","","1.65.1","1.1.2","2.2.7<br />2.3.4.3","","2.1.5","","20180416","","","","1.22<br />3.0.2","0.7.13","0.8.1","1.0.6-intel-2015.3.187","2019.03.01","","1.3<br />1.8<br />1.8<br />1.9<br />2.1","","","","","0.5.3.3","8.3rc2","6.9-tinaroo2018<br />rocks6_roll_modules<br />rocks7_roll_modules","9","1.0.11","1.0.13","","","2.6.8","3.1.3-GCC-4.9.2<br />3.15.0-gnu-7.2.0<br />3.2.3-intel-2015.3.187<br />3.9.1","0.9.4a0","","","8.22","","","","","","2.2.1","7.43.0-intel-2015.3.187","1.18","3.7.1","","3.3.1","1.0.0","4.0.3","","0.7.12<br />0.9.22<br />0.9.23<br />0.9.30","","2.5.0","2.5.0","2.2.1","2.3.0<br />2.3.1<br />2.6.0<br />3.6.1<br />3.8.0<br />3.9.4","3.131028","0.8.6<br />1.1.0<br />1.1.4<br />1.10.3<br />1.10.5<br />1.11.0<br />1.3.0<br />1.4.0<br />1.8.0<br />1.8.1<br />1.9.0","6.5.7","","","","","","","2.4.0","0.22","3.7.1-impi_2018.2.046<br />3.7.1-openmpi_2.0.2<br />3.7.1-openmpi_ib_1.8.4-intel","201710","0.0.6<br />0.0.8","","0.11.4","","","0.0.14","","","","","2.4.2<br />2.6","1.0.0","1.3.3-Ruby-2.1.6","","","6.0<br />6.1dev","","","3.5<br />4.0.4.0","","4.9.2","1.26.0","","","","","","","","","2.17<br />2.19<br />2.23","","","","","20151231","6.0.0a","7.2.0","2.4.2-GCC-4.9.2<br />2.4.5-GCC-4.9.2","20170122<br />20180322","1.16<br />2.1","2.69","0.9.4","1.1.3","2018<br />5.1.4-intel<br />5.1.4-intelMPI","0.3.5<br />201811","2.4.1","3.3.0","2.5.0-cdh5.3.1-native","","","1.0.0","2.10","1.8.14","","","2.1","3.1b2<br />3.3","","0.6.1p1<br />htseq-0.6.1","","1.1.1","","","","","","2.0.3","","","3.2.2","4.2.0","","1.7.0_79<br />1.8.0_45","","2.2.10","","","1.5.0","","","","","1.1.1","2.0.8","2.7.1<br />201712","","","","","","","","","4.0.1","7.4.17","2.2.2","","2-2.1.1","2.2.7.1","","v7.407","","2.31.10<br />3.01.03","","","","0.5","3.2.4","2015a<br />2017a<br />2018a<br />2018a<br />2019b<br />qcif-R2018a<br />qcif-R2018a","","20200714","3.3.3","","1.6.10.43<br />1.6.17.0","1.6.10.43","","","201710","","201710","2.3.0","0.1","","3.0.2","","","","1.0.2","","4.9.6","","0.0.7","0.5.3","","2.19.0","","6.10.0.104-0","","","3.1.2","","","3.2.3<br />3.2.6","","","201801","","","2.1","2.9","","","2.11.08-intel-2015.3.187","","","","6.3.0","","2.1.7","","3.6.2<br />4.3.2<br />4.4.1-gnu<br />4.4.1-intel","0.37.2<br />0.37.6<br />0.39.0<br />0.39.0+eddy","v2.2-beta.0","19.07","","1.1.0","","2.61","0.3.1","","","","","1.8.4<br />2.0.2","1.8.4","2.0.2<br />2.1.6","","","0.1.0","20140812","","5.4.1","4.8.1","5.4.1","1.2<br />1.5.3","","","3.20","","","","2.18.2<br />2.3.0","1.1.3","2.2.0_b","","","1.9","","","v1.1.alpha17","","","","","201804","1.12<br />1.13<br />1.14.5","","","1.0","","0.8.4","2.7.14","","","3.4.9-centos<br />3.6.2-sdsc","","","","1.9.1","2018.6.0<br />2019.4","5.0.2","","3.4.3<br />3.5.0<br />3.5.0-gnu<br />3.5.0-intel<br />3.6.0-container<br />3.6.0+tidyverse+rstan+rtdists<br />3.6.1-container<br />3.6.1+CrocSim<br />3.6.1+exSTRa<br />3.6.1+Monocle3<br />3.6.1+Monocle3+Giotto<br />3.6.1+Seurat<br />3.6.3+Shawan<br />4.0.0<br />4.0.0+Spatial<br />4.0.2+Spatial<br />4.0.2+tradeSeq+Seurat","","","","","","","8.2.12","0.0.24","0.0.1","4.0.9_p2","2.0.1","","","","4.5.1<br />4.6.2--R-3.4.4","1.3.3","2.6.4","1.0.143<br />1.1.456-3.4.4<br />1.1.456-3.5.1","2.1.6","","","0.1.17<br />1.10<br />1.2<br />1.3<br />1.6<br />1.9","9.4","","2.7","","","","0.13.2","","","","","","2.2.1<br />2.3.1<br />2.4.0<br />2.4.2<br />2.5.1<br />2.5.2<br />3.5.0","","smrtlink_5.0.1<br />smrtlink_5.1<br />smrtlink_6.0<br />smrtlink_7.0<br />smrtlink_9.0","5.6","2006-07-28","1.1.2-GCC-4.9.2","","","","","240","1.03","3.0.3<br />4.0.0","3.11.1<br />3.9.0","","","1.37","2.5.2b<br />2.5.2b-static","2.3.4","2.1.1","","2.33","","2.23","","","","2.1.1","","5.5.0","","","0.35<br />0.39","2.1.1<br />2.9.1","3.2.1","","","","","","","","","0.1.14","1.2.10","2.2.4","","","","","5.0.2","","","","","","","","1.17-GCC-4.7.2","","1.2.11-gnu-7.1.0",""]],"container":"<table class=\"table table-bordered table-striped\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Tool / workflow name<\/th>\n      <th>Tool identifier (module name / bio.tools ID / placeholder)<\/th>\n      <th>Primary purpose (EDAM, if available)<\/th>\n      <th>Galaxy Australia<\/th>\n      <th>NCI (Gadi)<\/th>\n      <th>Pawsey (Zeus)<\/th>\n      <th>Pawsey (Magnus)<\/th>\n      <th>QRIScloud / UQ-RCC (Flashlite, Awoonga, Tinaroo)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":20,"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}],"lengthMenu":[10,20,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<p></font></p>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-02-23</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>The <em>Tool identifier</em> column contains an identifier for the tool / workflow: typically either the module name (used for matching to HPC lists), a bio.tools ID, or a placeholder value.</li>
<li>For tools which are not currently installed on <a href="https://usegalaxy.org.au/">Galaxy Australia</a>, but which are available <a href="https://toolshed.g2.bx.psu.edu/">in the Galaxy app store (aka toolshed)</a>, the <em>Available in Galaxy toolshed</em> column launches a toolshed search using the link label as the search term.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<hr />
<hr />
</div>
<div id="tool-information-1" class="section level2">
<h2>Tool information</h2>
<p><font size="-1.5"></p>
<div id="htmlwidget-9e6693aaa57679b7f2d4" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9e6693aaa57679b7f2d4">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432"],["<a href='https://github.com/aidenlab/3d-dna'>3D-DNA<\/a>","<a href='https://github.com/tseemann/abricate'>ABRicate<\/a>","<a href='https://github.com/bcgsc/abyss'>ABySS<\/a>","<a href='https://github.com/tanghaibao/jcvi/wiki/ALLMAPS'>ALLMAPS<\/a>","<a href='http://software.broadinstitute.org/allpaths-lg/blog/'>allpathslg<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php/AMOS'>amos<\/a>","<a href='http://bioinformatics.ua.pt/software/anaconda/'>anaconda<\/a>","<a href='http://www.popgen.dk/angsd/index.php/ANGSD'>ANGSD<\/a>","<a href='https://ant.apache.org/'>ant<\/a>","<a href='https://github.com/GMOD/Apollo'>Apollo<\/a>","<a href='https://arimagenomics.com/assembly'>arima mapping<\/a>","<a href='https://www.arm.com/products/development-tools/server-and-hpc/forge'>arm-forge<\/a>","arm-reports","<a href='https://github.com/sanger-pathogens/Artemis'>Artemis<\/a>","<a href='https://www.aspera.com/en/'>Aspera<\/a>","<a href='https://github.com/Gaius-Augustus/Augustus'>Augustus<\/a>","<a href='https://www.gnu.org/software/autoconf/'>Autoconf<\/a>","<a href='https://www.gnu.org/software/automake/'>Automake<\/a>","<a href='https://github.com/hall-lab/bamkit'>bamkit<\/a>","<a href='http://bamm-project.org/'>bamm<\/a>","<a href='https://github.com/pezmaster31/bamtools'>bamtools<\/a>","<a href='https://github.com/workflow4metabolomics/batch_correction'>batchcorrection<\/a>","<a href='https://bazel.build/'>bazel<\/a>","<a href='https://github.com/slaclab/bbcp'>bbcp<\/a>","<a href='https://jgi.doe.gov/data-and-tools/bbtools/'>bbmap<\/a>","<a href='http://www.htslib.org/'>BCFtools<\/a>","<a href='https://faculty.washington.edu/browning/beagle/beagle.html'>Beagle<\/a>","<a href='https://github.com/beagle-dev/beagle-lib'>beagle-lib<\/a>","<a href='https://github.com/beast-dev/beast-mcmc'>BEAST<\/a>","<a href='https://github.com/CompEvol/beast2'>BEAST2<\/a>","<a href='https://github.com/arq5x/bedtools2'>BEDTools<\/a>","<a href='https://www.gnu.org/software/binutils/'>binutils<\/a>","<a href='https://www.bioconductor.org/'>bioconductor<\/a>","<a href='https://github.com/Bio-Linux'>biolinux<\/a>","<a href='https://bionanogenomics.com/support-page/bionano-access-software/'>Bionano Access<\/a>","<a href='http://bioperl.org/'>bioperl<\/a>","<a href='http://biopython.org/'>biopython<\/a>","<a href='https://github.com/FelixKrueger/Bismark'>bismark<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST+<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi'>BLAST2<\/a>","<a href='https://genome.ucsc.edu/cgi-bin/hgBlat'>blat<\/a>","<a href='https://github.com/blobtoolkit/blobtools2'>BlobToolKit<\/a>","<a href='https://github.com/DRL/blobtools'>BlobTools1<\/a>","<a href='https://www.boost.org/'>Boost<\/a>","<a href='http://bowtie-bio.sourceforge.net/index.shtml'>Bowtie<\/a>","<a href='http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'>Bowtie2<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER'>BRAKER<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER'>BRAKER2<\/a>","<a href='https://github.com/genome/breakdancer'>breakdancer<\/a>","<a href='https://github.com/pace/bricks'>BRICKS<\/a>","<a href='https://code.google.com/archive/p/bsmap/'>bsmap<\/a>","<a href='http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker/BS_Seeker.html'>bsseeker<\/a>","<a href='https://github.com/BSSeeker/BSseeker2#bs-seeker2'>bsseeker2<\/a>","<a href='https://busco.ezlab.org/'>BUSCO<\/a>","<a href='https://github.com/lh3/bwa'>bwa<\/a>","<a href='https://github.com/bxlab/bx-python'>bx-python<\/a>","<a href='http://www.bzip.org/'>bzip2<\/a>","<a href='https://github.com/ComparativeGenomicsToolkit/cactus'>Cactus<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/CAMERA.html'>CAMERA<\/a>","<a href='https://github.com/marbl/canu'>Canu<\/a>","<a href='http://seq.cs.iastate.edu/cap3.html'>cap3<\/a>","<a href='https://cardinalmsi.org/'>Cardinal<\/a>","<a href='https://github.com/dutilh/CAT'>CAT<\/a>","<a href='https://github.com/weizhongli/cdhit'>cd-hit<\/a>","<a href='https://github.com/yanailab/celseq2'>CEL-Seq 2<\/a>","<a href='http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page'>Celera / wgs-assembler<\/a>","<a href='https://hub.docker.com/r/centos/tools/'>centos<\/a>","<a href='https://hub.docker.com/r/centos/tools/'>centos6<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki'>checkm<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki'>checkm-genome<\/a>","<a href='https://genome.sph.umich.edu/wiki/ChunkChromosome'>chunkchromosome<\/a>","<a href='http://www.clustal.org/clustal2/'>clustalw<\/a>","<a href='https://tools.bsc.es/doc/clusteringsuite_manual_0.pdf'>clusteringsuite<\/a>","<a href='https://cmake.org/'>cmake<\/a>","<a href='https://github.com/etal/cnvkit'>CNVkit<\/a>","<a href='https://github.com/biocoder/cpc'>Coding Potential Calculator<\/a>","<a href='https://github.com/biocoder/CPC2'>Coding Potential Calculator 2<\/a>","<a href='https://www.gnu.org/software/coreutils/'>coreutils<\/a>","<a href='https://sourceforge.net/projects/rna-cpat/files/'>CPAT: Coding Potential Assessment Tool<\/a>","<a href='https://sourceforge.net/projects/cppunit/'>cppunit<\/a>","<a href='https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cube-score-p-scalasca'>cube<\/a>","<a href='https://developer.nvidia.com/cuda-toolkit'>cuda<\/a>","<a href='https://developer.nvidia.com/cudnn'>cudnn<\/a>","<a href='http://cole-trapnell-lab.github.io/cufflinks/'>Cufflinks<\/a>","<a href='https://curl.se/'>curl<\/a>","<a href='https://github.com/marcelm/cutadapt'>cutadapt<\/a>","<a href='https://cytoscape.org/'>cytoscape<\/a>","<a href='https://www.mcs.anl.gov/research/projects/darshan/'>darshan<\/a>","<a href='https://deeptools.readthedocs.io/en/develop/'>DeepTools<\/a>","<a href='https://github.com/google/deepvariant'>deepvariant<\/a>","<a href='https://dendropy.org/'>dendropy<\/a>","<a href='https://diaumpire.nesvilab.org/'>DIA-Umpire<\/a>","<a href='https://github.com/bbuchfink/diamond'>diamond<\/a>","<a href='https://github.com/broadinstitute/discovar_de_novo'>DISCOVAR de novo<\/a>","<a href='https://jbloomlab.github.io/dms_tools2/'>dms_tools2<\/a>","<a href='https://github.com/dmtcp/dmtcp'>dmtcp<\/a>","<a href='https://github.com/cmbi/dssp'>dssp<\/a>","<a href='https://github.com/easybuilders/easybuild'>easybuild<\/a>","<a href='http://www.genomic.ch/edena.php'>Edena<\/a>","<a href='https://github.com/UQ-RCC/nimrod-embedded'>embedded-nimrod<\/a>","<a href='http://emboss.sourceforge.net/'>EMBOSS (European Molecular Biology Open Software Suite)<\/a>","<a href='https://asia.ensembl.org/info/website/index.html'>Ensembl browser<\/a>","<a href='http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html'>Ensembl Genebuild<\/a>","<a href='http://erlang.org/faq/introduction.html'>Erlang<\/a>","<a href='http://mblab.wustl.edu/software.html'>eval<\/a>","<a href='https://github.com/EVidenceModeler/EVidenceModeler/releases'>EVidenceModeler (EVM)<\/a>","<a href='https://github.com/aberer/exabayes'>ExaBayes<\/a>","<a href='http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate'>Exonerate<\/a>","<a href='https://github.com/SegataLab/export2graphlan/'>export2graphlan<\/a>","<a href='https://tools.bsc.es/extrae'>Extrae<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly'>Falcon<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly'>Falcon (pb-assembly)<\/a>","<a href='http://fastml.tau.ac.il/'>fastml<\/a>","<a href='https://www.bioinformatics.babraham.ac.uk/projects/fastqc/'>FASTQC<\/a>","<a href='https://rajanil.github.io/fastStructure/'>faststructure<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/'>FASTX-Toolkit<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/'>FASTX-Toolkit<\/a>","<a href='http://www.softberry.com/berry.phtml?topic=fgenesh_plus_plus&group=help&subgroup=pipelines'>FgenesH++<\/a>","<a href='http://animalbiosciences.uoguelph.ca/~msargol/fimpute/'>FImpute<\/a>","<a href='https://github.com/smith-chem-wisc/FlashLFQ'>flashLFQ<\/a>","<a href='https://www.fltk.org/index.php'>fltk<\/a>","<a href='https://github.com/fenderglass/Flye'>Flye<\/a>","<a href='https://sourceforge.net/projects/fpart/'>fpart<\/a>","<a href='https://fpm.readthedocs.io/en/latest/'>fpm<\/a>","<a href='https://sourceforge.net/projects/fraggenescan/'>FragGeneScan<\/a>","<a href='https://github.com/ekg/freebayes'>freebayes<\/a>","<a href='https://surfer.nmr.mgh.harvard.edu/'>freesurfer<\/a>","<a href='https://funannotate.readthedocs.io/en/latest/install.html'>funannotate<\/a>","<a href='https://code.google.com/archive/p/garli/'>GARLI<\/a>","<a href='https://github.com/broadinstitute/gatk'>GATK<\/a>","<a href='http://molevol.cmima.csic.es/castresana/Gblocks_server.html'>gblocks<\/a>","<a href='https://gcc.gnu.org/'>GCC<\/a>","<a href='http://cnsgenomics.com/software/gcta/'>gcta<\/a>","<a href='https://github.com/gnudatalanguage/gdl'>GDL<\/a>","<a href='http://genome.crg.es/software/geneid/'>geneid<\/a>","<a href='http://exon.gatech.edu/GeneMark/'>Genemark<\/a>","<a href='https://github.com/schatzlab/genomescope'>GenomeScope<\/a>","<a href='https://github.com/PacificBiosciences/GenomicConsensus'>GenomicConsensus (Arrow)<\/a>","<a href='https://git-scm.com/about'>Git<\/a>","<a href='http://glew.sourceforge.net/'>glew<\/a>","<a href='https://github.com/glfw/glfw'>glfw<\/a>","<a href='https://sourceware.org/git/?p=glibc.git'>glibc<\/a>","<a href='http://ccb.jhu.edu/software/glimmer/'>glimmer<\/a>","<a href='http://ccb.jhu.edu/software/glimmerhmm/'>glimmerhmm<\/a>","<a href='https://github.com/GlobalArrays/ga'>Global Arrays (GA) toolkit<\/a>","<a href='http://research-pub.gene.com/gmap/'>gmap<\/a>","<a href='http://research-pub.gene.com/gmap/'>gmap_gsnap<\/a>","<a href='https://github.com/epam/GMP'>gmp<\/a>","<a href='https://www.gnu.org/'>GNU<\/a>","<a href='https://www.gnu.org/software/libtool/manual/libtool.html'>GNU libtool<\/a>","<a href='https://www.gnu.org/software/parallel/'>GNU parallel<\/a>","<a href='https://www.gnu.org/software/gsl/'>GNU scientific library<\/a>","<a href='https://sourceforge.net/projects/cbadvanced/'>GNU Tools<\/a>","<a href='https://geronimp.github.io/graftM/'>GraftM<\/a>","<a href='https://github.com/biobakery/graphlan'>GraPhlAn<\/a>","<a href='http://www.gromacs.org/'>gromacs<\/a>","<a href='http://ecogenomics.github.io/GroopM/'>groopm<\/a>","<a href='https://github.com/sanger-pathogens/gubbins'>gubbins<\/a>","guppy","<a href='https://hadoop.apache.org/'>Hadoop<\/a>","<a href='https://github.com/vpc-ccg/haslr'>HASLR<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php?title=Hawkeye'>Hawkeye<\/a>","<a href='https://github.com/SegataLab/hclust2'>Hclust2<\/a>","<a href='https://support.hdfgroup.org/products/hdf4_tools/'>hdf4<\/a>","<a href='https://support.hdfgroup.org/products/hdf5_tools/'>hdf5<\/a>","<a href='https://github.com/higlass'>HiGlass<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772016/'>HiRise<\/a>","<a href='https://daehwankimlab.github.io/hisat2/'>HISAT2<\/a>","<a href='http://hmmer.org/'>HMMER<\/a>","<a href='https://github.com/horovod/horovod'>horovod<\/a>","<a href='https://github.com/htseq/htseq'>HTSeq<\/a>","<a href='http://www.htslib.org/'>HTSlib<\/a>","<a href='https://github.com/loneknightpy/idba'>idba-ud<\/a>","<a href='https://www.l3harrisgeospatial.com/Software-Technology/IDL'>IDL<\/a>","<a href='https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler'>Improved Phased Assembler (pbipa)<\/a>","<a href='http://eddylab.org/infernal/'>Infernal<\/a>","<a href='http://software.broadinstitute.org/software/igv/home'>Integrated Genomics Viewer<\/a>","<a href='https://interproscan-docs.readthedocs.io/en/latest/'>InterProScan<\/a>","<a href='https://github.com/nerscadmin/IPM'>ipm<\/a>","<a href='https://github.com/dereneaton/pyrad'>iPyRAD<\/a>","<a href='http://www.iqtree.org/'>IQ-TREE<\/a>","<a href='https://github.com/PacificBiosciences/IsoSeq'>IsoSeq3<\/a>","<a href='http://mcmc-jags.sourceforge.net/'>jags<\/a>","<a href='https://github.com/genomecuration/JAMg'>JAMg<\/a>","<a href='https://www.oracle.com/java/'>Java<\/a>","<a href='https://github.com/GMOD/jbrowse'>JBrowse<\/a>","<a href='http://www.genome.umd.edu/jellyfish.html'>jellyfish<\/a>","<a href='https://github.com/aidenlab/Juicebox'>juicebox<\/a>","<a href='https://github.com/aidenlab/juicer'>Juicer<\/a>","<a href='https://julialang.org/'>julia<\/a>","<a href='https://pachterlab.github.io/kallisto/about.html'>Kallisto<\/a>","<a href='https://kat.readthedocs.io/en/latest/'>KAT<\/a>","<a href='https://github.com/dib-lab/khmer'>khmer<\/a>","<a href='https://github.com/refresh-bio/KMC'>KMC<\/a>","<a href='https://ccb.jhu.edu/software/kraken/'>kraken<\/a>","<a href='https://github.com/DerrickWood/kraken2/wiki'>kraken2<\/a>","<a href='https://github.com/marbl/Krona/wiki'>krona<\/a>","<a href='https://kubernetes.io/'>Kubernetes<\/a>","<a href='http://www.bx.psu.edu/~rsharris/lastz/'>lastz<\/a>","<a href='https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst'>LFQ-Analyst<\/a>","<a href='https://ofiwg.github.io/libfabric/'>libfabric<\/a>","<a href='https://github.com/agordon/libgtextutils'>libgtextutils<\/a>","<a href='https://www.nongnu.org/libunwind/'>libunwind<\/a>","<a href='https://github.com/bcgsc/LINKS'>LINKS<\/a>","<a href='https://www.lipidmaps.org/'>Lipid Maps<\/a>","<a href='https://llvm.org/'>llvm<\/a>","<a href='https://lmod.readthedocs.io/en/latest/'>lmod<\/a>","<a href='https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger'>Long Ranger<\/a>","<a href='http://csg.sph.umich.edu/abecasis/mach/index.html'>mach<\/a>","<a href='https://github.com/macs3-project/MACS'>macs<\/a>","<a href='https://github.com/macs3-project/MACS'>macs2<\/a>","<a href='https://github.com/ranwez/MACSE_V2_PIPELINES'>MACSE<\/a>","<a href='http://mafft.cbrc.jp/alignment/software/'>mafft<\/a>","<a href='https://github.com/WatsonLab/MAGpy'>MAGpy<\/a>","<a href='http://www.yandell-lab.org/software/maker.html'>MAKER<\/a>","<a href='http://www.yandell-lab.org/software/maker-p.html'>MAKER-P<\/a>","<a href='https://cran.r-project.org/web/packages/MALDIquant/index.html'>MALDIquant<\/a>","<a href='https://github.com/marbl/MashMap'>mashmap<\/a>","<a href='https://github.com/kwongj/maskrc-svg'>maskrc-svg<\/a>","<a href='https://github.com/alekseyzimin/masurca'>MaSuRCA<\/a>","<a href='https://www.mathworks.com/products/matlab.html'>MATLAB<\/a>","<a href='https://github.com/dtreichler/matserver'>matserver<\/a>","<a href='http://darlinglab.org/mauve/download.html'>Mauve<\/a>","<a href='https://maven.apache.org/'>maven<\/a>","<a href='https://sourceforge.net/projects/maxbin2/'>MaxBin2<\/a>","<a href='https://www.maxquant.org/'>MaxQuant<\/a>","<a href='https://www.maxquant.org/'>MaxQuant (biocontainers)<\/a>","<a href='https://micans.org/mcl/'>mcl<\/a>","<a href='https://github.com/nanoporetech/medaka'>Medaka<\/a>","<a href='https://github.com/voutcn/megahit'>megahit<\/a>","<a href='https://github.com/marbl/merqury'>Merqury<\/a>","<a href='https://bitbucket.org/berkeleylab/metabat/src/master/'>metabat<\/a>","<a href='https://sourceforge.net/projects/metacv/'>metacv<\/a>","<a href='https://github.com/DiltheyLab/MetaMaps'>MetaMaps<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/metaMS.html'>metaMS<\/a>","<a href='https://huttenhower.sph.harvard.edu/metaphlan'>MetaPhlAn<\/a>","<a href='http://cab.spbu.ru/software/spades/'>metaspades<\/a>","<a href='https://peterbeerli.com/migrate-html5/index.html'>MIGRATE<\/a>","<a href='https://gatb.inria.fr/software/minia/'>minia<\/a>","<a href='https://genome.sph.umich.edu/wiki/Minimac4'>Minimac4<\/a>","<a href='https://github.com/lh3/minimap2'>minimap2<\/a>","<a href='https://sourceforge.net/p/mira-assembler/wiki/Home/'>mira<\/a>","<a href='https://www.nature.com/articles/nbt1394#Sec11'>mirdeep<\/a>","<a href='https://github.com/rajewsky-lab/mirdeep2'>mirdeep2<\/a>","<a href='https://github.com/yarden/MISO'>miso<\/a>","<a href='http://mixomics.org/'>mixOmics<\/a>","<a href='https://github.com/tseemann/mlst'>MLST<\/a>","<a href='https://github.com/compomics/moFF'>moFF<\/a>","<a href='https://www.mono-project.com/'>Mono<\/a>","<a href='https://cwenger.github.io/Morpheus/'>Morpheus<\/a>","<a href='https://www.mothur.org/'>Mothur<\/a>","<a href='https://www.mpfr.org/'>mpfr<\/a>","mpiBLAST","<a href='https://github.com/LLNL/mpiP'>mpiP (mpi profiling library)<\/a>","<a href='https://github.com/NBISweden/MrBayes'>MrBayes<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSGFplus.html'>MS-GF+<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSnbase.html'>MSnbase readMSData<\/a>","<a href='https://bitbucket.org/joseph07/multisnv/wiki/Home'>multisnv<\/a>","<a href='http://mummer.sourceforge.net/'>MUMmer<\/a>","<a href='http://www.drive5.com/muscle/'>MUSCLE<\/a>","<a href='https://mvapich.cse.ohio-state.edu/'>MVAPICH<\/a>","<a href='https://software.opensuse.org/package/mxml'>mxml<\/a>","<a href='https://www.nag.com/'>nag<\/a>","<a href='https://github.com/jts/nanopolish'>Nanopolish<\/a>","<a href='https://www.nasm.us/'>NASM<\/a>","<a href='https://github.com/NVIDIA/nccl'>NCCL<\/a>","<a href='https://gitlab.com/remikz/nccmp'>nccmp<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml'>NCL<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml'>NCL<\/a>","<a href='http://nco.sourceforge.net/'>NCO<\/a>","<a href='http://meteora.ucsd.edu/~pierce/ncview_home_page.html'>Ncview<\/a>","<a href='https://github.com/xiaochuanle/NECAT'>NECAT<\/a>","<a href='https://www.unidata.ucar.edu/software/netcdf/'>NetCDF<\/a>","<a href='https://github.com/neurodebian/neurodebian'>neurodebian<\/a>","<a href='https://github.com/Nextomics/NextDenovo'>NextDenovo<\/a>","<a href='https://www.nextflow.io/'>nextflow<\/a>","<a href='http://cibiv.github.io/NextGenMap/'>nextgenmap<\/a>","<a href='https://github.com/Nextomics/NextPolish'>NextPolish<\/a>","<a href='https://github.com/ncbi/ngs'>NGS<\/a>","<a href='https://github.com/shenlab-sinai/ngsplot'>ngsplot<\/a>","<a href='https://nucleoatac.readthedocs.io/en/latest/'>nucleoatac<\/a>","<a href='https://github.com/NVlabs/nvbio'>nvbio<\/a>","<a href='https://developer.nvidia.com/hpc-sdk'>NVIDIA HPC SDK<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/oases/'>Oases<\/a>","<a href='https://www.vi-hps.org/tools/opari2.html'>OPARI2<\/a>","<a href='https://www.open-mpi.org/'>Open MPI<\/a>","<a href='https://www.open-mpi.org/'>Open MPI (ib)<\/a>","<a href='https://www.open-mpi.org/'>Open MPI 2 (ib)<\/a>","<a href='https://www.openms.de/'>OpenMS<\/a>","<a href='https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper'>OPLS-DA_Contrasts (w4mcorcov)<\/a>","<a href='https://github.com/kishwarshafin/pepper'>P.E.P.P.E.R. (+ deep variant)<\/a>","<a href='https://github.com/TomSkelly/PacBioEDA'>PacBioEDA<\/a>","padb","<a href='http://icl.utk.edu/papi/software/'>PAPI (Performance Application Programming Interface)<\/a>","<a href='https://www.vi-hps.org/tools/paraver.html'>PARAVER<\/a>","<a href='https://www.paraview.org/'>ParaView<\/a>","<a href='https://github.com/marbl/parsnp'>Parsnp<\/a>","<a href='https://github.com/PASApipeline/PASApipeline/wiki'>PASA<\/a>","<a href='https://github.com/NixOS/patchelf'>PatchELF<\/a>","<a href='https://downloadcenter.intel.com/download/19792'>PDT (Processor Diagnostic Tool)<\/a>","<a href='http://compomics.github.io/projects/peptide-shaker.html'>Peptide Shaker<\/a>","<a href='http://percolator.ms/'>Percolator<\/a>","<a href='https://www.php.net/'>PHP<\/a>","<a href='https://broadinstitute.github.io/picard/'>PICARD<\/a>","<a href='http://picrust.github.io/picrust/'>PICRUSt<\/a>","<a href='https://github.com/picrust/picrust2/wiki'>PICRUSt 2<\/a>","<a href='https://www.drive5.com/piler/'>PILER-DF<\/a>","<a href='https://github.com/broadinstitute/pilon/wiki'>Pilon<\/a>","<a href='http://zzz.bwh.harvard.edu/plink/'>plink<\/a>","<a href='http://atgu.mgh.harvard.edu/plinkseq/index.shtml'>plinkseq<\/a>","<a href='https://parallel-netcdf.github.io/'>PnetCDF<\/a>","<a href='https://github.com/matsen/pplacer'>pplacer<\/a>","<a href='https://github.com/wtsi-hpag/PretextView'>PretextView / Pretext Suite<\/a>","<a href='http://prinseq.sourceforge.net/'>PRINSEQ<\/a>","<a href='http://www.genabel.org/packages/ProbABEL'>probabel<\/a>","<a href='https://github.com/rsilvabioinfo/ProbMetab'>ProbMetab Tool<\/a>","<a href='https://github.com/hyattpd/Prodigal'>Prodigal<\/a>","<a href='https://github.com/tseemann/prokka'>Prokka<\/a>","<a href='https://github.com/dfguan/purge_dups'>Purge_Dups<\/a>","<a href='https://bitbucket.org/mroachawri/purge_haplotigs/src/master/'>PurgeHaplotigs<\/a>","<a href='https://pylib.readthedocs.io/en/1.4.3/index.html'>pylibs<\/a>","<a href='https://ipyrad.readthedocs.io/en/latest/'>PyRAD<\/a>","<a href='https://github.com/pysam-developers/pysam'>pysam<\/a>","<a href='https://www.python.org/'>python<\/a>","<a href='https://www.python.org/'>python2<\/a>","<a href='https://www.python.org/'>python2-as-python<\/a>","<a href='https://www.python.org/'>python3<\/a>","<a href='https://www.python.org/'>python3-as-python<\/a>","<a href='https://pytorch.org/'>PyTorch<\/a>","<a href='http://qiime.org/'>QIIME<\/a>","<a href='http://qiime.org/'>QIIME<\/a>","<a href='https://qiime2.org/'>QIIME2.0<\/a>","<a href='http://bioinf.spbau.ru/quast'>QUAST<\/a>","<a href='https://github.com/mahulchak/quickmerge'>Quickmerge<\/a>","<a href='https://www.r-project.org/'>R<\/a>","<a href='https://www.rabbitmq.com/'>RabbitMQ server<\/a>","<a href='https://github.com/isovic/racon'>Racon<\/a>","<a href='https://github.com/bcgsc/RAILS'>RAILS/Cobbler<\/a>","<a href='http://ratt.sourceforge.net/'>RATT<\/a>","<a href='https://github.com/comprna/RATTLE'>RATTLE<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/galaxyp/rawtools/51e4aa55ea04'>Raw Tools<\/a>","<a href='https://sco.h-its.org/exelixis/web/software/raxml/index.html'>RAxML<\/a>","<a href='https://github.com/dparks1134/RefineM'>RefineM<\/a>","<a href='https://github.com/simoncchu/REPdenovo'>REPdenovo<\/a>","<a href='http://www.repeatmasker.org/'>RepeatMasker<\/a>","<a href='http://www.repeatmasker.org/RepeatModeler/'>RepeatModeler<\/a>","<a href='https://github.com/Dfam-consortium/RepeatModeler'>RepeatModeler2<\/a>","<a href='https://www.yandell-lab.org/software/repeatrunner.html'>RepeatRunner<\/a>","<a href='https://urgi.versailles.inra.fr/Tools/REPET'>REPET<\/a>","<a href='https://cran.r-project.org/web/packages/rjags/index.html'>rjags<\/a>","<a href='https://deweylab.github.io/RSEM/'>RSEM<\/a>","<a href='http://rseqc.sourceforge.net/'>rseqc<\/a>","<a href='https://rstudio.com/'>Rstudio<\/a>","<a href='https://www.ruby-lang.org/en/downloads/'>Ruby<\/a>","<a href='https://combine-lab.github.io/salmon/'>Salmon<\/a>","<a href='https://github.com/GregoryFaust/samblaster'>samblaster<\/a>","<a href='http://www.htslib.org/'>SAMTools<\/a>","<a href='https://www.sas.com/en_us/software/foundation.html'>SAS<\/a>","<a href='http://compbio.cs.toronto.edu/hapsembler/scarpa.html'>Scarpa<\/a>","<a href='https://www.scipy.org/'>SciPy<\/a>","<a href='https://scons.org/'>SCons<\/a>","<a href='https://pypi.python.org/pypi/screed'>screed<\/a>","<a href='http://compomics.github.io/projects/searchgui.html'>Search GUI<\/a>","<a href='https://github.com/shenwei356/seqkit'>SeqKit<\/a>","<a href='https://github.com/drostlab/seqstats'>seqstats<\/a>","<a href='https://github.com/trinityrnaseq/seqtk-trinity'>seqtk-trinity<\/a>","<a href='https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'>shapeit<\/a>","<a href='https://github.com/najoshi/sickle'>sickle<\/a>","<a href='https://www.bryntum.com/products/siesta/'>Siesta<\/a>","<a href='https://singularity.lbl.gov/'>singularity<\/a>","<a href='https://github.com/relipmoc/skewer'>skewer<\/a>","<a href='https://www.pacb.com/support/software-downloads/'>SMRT Link / Analysis<\/a>","<a href='https://snakemake.readthedocs.io/en/stable/'>Snakemake<\/a>","<a href='https://github.com/KorfLab/SNAP'>SNAP<\/a>","<a href='https://github.com/PMMAraujo/snappy'>SNAPPy<\/a>","<a href='http://lowelab.ucsc.edu/snoscan/'>snoscan<\/a>","<a href='http://snpeff.sourceforge.net/'>snpeff<\/a>","soap","<a href='http://i.cs.hku.hk/2bwt-tools/soap3/index.html'>soap3-gpu<\/a>","<a href='https://github.com/aquaskyline/SOAPdenovo2'>SOAPdenovo2<\/a>","<a href='https://sourceforge.net/projects/soapsnp/'>SOAPsnp<\/a>","<a href='https://github.com/biocore/sortmerna'>sortmerna<\/a>","<a href='https://github.com/ablab/spades'>SPAdes<\/a>","<a href='https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi'>Splign<\/a>","<a href='https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc'>sra-tools<\/a>","<a href='http://catchenlab.life.illinois.edu/stacks/'>Stacks<\/a>","<a href='https://github.com/alexdobin/STAR'>star<\/a>","<a href='http://web.stanford.edu/group/pritchardlab/structure.html'>structure<\/a>","<a href='https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome'>Supernova<\/a>","<a href='https://github.com/hall-lab/svtyper'>svtyper<\/a>","<a href='https://github.com/TimoLassmann/tagdust'>tagdust<\/a>","<a href='https://bitbucket.org/tasseladmin/tassel-5-standalone'>tassel<\/a>","<a href='http://www.cs.uoregon.edu/research/tau/home.php'>TAU<\/a>","<a href='https://www.tensorflow.org/resources/tools'>TensorFlow<\/a>","<a href='https://github.com/compomics/ThermoRawFileParser'>ThermoRawFileParser<\/a>","<a href='https://github.com/bcgsc/tigmint'>Tigmint<\/a>","<a href='https://ccb.jhu.edu/software/tophat/index.shtml'>tophat<\/a>","<a href='https://totalview.io/'>TotalView<\/a>","<a href='https://github.com/TransDecoder/TransDecoder/wiki'>TransDecoder<\/a>","<a href='https://tandem.bu.edu/trf/trf.html'>trf<\/a>","<a href='http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'>Trim Galore<\/a>","<a href='http://www.usadellab.org/cms/?page=trimmomatic'>Trimmomatic<\/a>","<a href='https://github.com/trinityrnaseq/trinityrnaseq/wiki'>Trinity / trinityrnaseq<\/a>","<a href='https://github.com/Trinotate/Trinotate.github.io/wiki'>Trinotate<\/a>","<a href='http://eddylab.org/software.html'>trnascan-se<\/a>","<a href='http://hgdownload.soe.ucsc.edu'>UCSC tools<\/a>","<a href='https://www.openucx.org/'>UCX (Unified Communication X)<\/a>","<a href='https://www.unidata.ucar.edu/software/udunits/'>UDUNITS<\/a>","<a href='https://github.com/rrwick/Unicycler'>Unicycler<\/a>","<a href='http://www.unixodbc.org/utilities.html'>unixODBC<\/a>","<a href='https://github.com/marbl/VALET'>Valet<\/a>","<a href='https://github.com/cc2qe/vawk'>vawk<\/a>","<a href='https://vcftools.github.io/index.html'>VCFTools<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/velvet/'>Velvet<\/a>","<a href='https://www.tbi.univie.ac.at/RNA/documentation.html'>ViennaRNA<\/a>","<a href='https://wci.llnl.gov/simulation/computer-codes/visit/'>VisIt<\/a>","<a href='https://github.com/torognes/vsearch'>vsearch<\/a>","<a href='https://github.com/adigenova/wengan'>Wengan<\/a>","<a href='http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html'>WindowMasker<\/a>","<a href='https://www.winehq.org/'>Wine<\/a>","<a href='http://www.ebi.ac.uk/~birney/wise2/'>wise<\/a>","<a href='https://workflow4metabolomics.org/'>Workflow 4 Metabolomics<\/a>","<a href='https://github.com/ruanjue/wtdbg2'>WTDBG2<\/a>","<a href='https://www.wxwidgets.org/'>wxWidgets<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/xcms.html'>xcms<\/a>","<a href='https://xerces.apache.org/xerces-c/'>Xerces-C++ XML parser<\/a>","<a href='https://github.com/AustralianBioCommons/xml4ena'>xml4ena<\/a>","<a href='https://github.com/freedesktop/xorg-macros'>xorg-macros<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem/29e2df3edf11'>xtandem<\/a>","<a href='https://zlib.net/'>zlib<\/a>","zstd"],["3D-DNA","abricate","abyss","allmaps","allpathslg","amos","anaconda","angsd","ant","apollo","arima mapping","arm-forge","arm-reports","artemis","aspera","augustus","autoconf","automake","bamkit","bamm","bamtools","batchcorrection","bazel","bbcp","bbmap","bcftools","beagle","beagle-lib","beast","beast2","bedtools","binutils","bioconductor","biolinux","Bionano Access","bioperl","biopython","bismark","blast","blast+","blast2","blat","BlobToolKit","blobtools","boost","bowtie","bowtie2","braker1","braker2","breakdancer","bricks","bsmap","bs_seeker","bsseeker2","busco","bwa","bx-python","bzip2","cactus","camera","canu","cap3","cardinal","cat_bins","cdhit","celseq2","celera","centos","centos6","checkm","checkm-genome","chunkchromosome","clustalw","clusteringsuite","cmake","cnvkit","cpc","cpc2","coreutils","cpat","cppunit","cube","cuda","cudnn","cufflinks","curl","cutadapt","cytoscape","darshan","deeptools","deepvariant","dendropy","DIA-Umpire","diamond","discovar_de_novo","dms_tools2","dmtcp","dssp","easybuild","edena","embedded-nimrod","emboss","ensembl","genebuild","erlang","eval","EvidenceModeler","exabayes","exonerate","export2graphlan","extrae","falcon","pb-assembly","fastml","fastqc","faststructure","fastx_toolkit","fastx","fgenesh","fimpute","flashlfq","fltk","flye","fpart","fpm","fraggenescan","freebayes","freesurfer","funannotate","garli","gatk","gblock","gcc","gcta","gdl","geneid","genemark","genomescope","GenomicConsensus","git","glew","glfw","glibc","glimmer","glimmerhmm","ga","gmap","gmap_gsnap","gmp","gnu","libtool","parallel","gsl","gnutools","graftm","graphlan","gromacs","groopm","gubbins","guppy","hadoop","haslr","hawkeye","hclust2","hdf4","hdf5","higlass","hirise","hisat2","hmmer","horovod","htseq","htslib","idba-ud","idl","pbipa","infernal","igv","interproscan","ipm","ipyrad","iq-tree","isoseq3","jags","jamg","java","jbrowse","jellyfish","juicebox","juicer","julia","kallisto","kat","khmer","kmc","kraken","kraken2","krona","k8","lastz","LFQ-Analyst","libfabric","libgtextutils","libunwind","links","lipid_maps","llvm","lmod","longranger","mach","macs","macs2","macse","mafft","magpy","maker","maker-p","maldi_quant","mashmap","maskrc-svg","masurca","matlab","matserver","mauve","maven","maxbin","maxquant","biocontainers-maxquant","mcl","medaka","megahit","merqury","metabat","metacv","metamaps","metams","metaphlan","metaspades","MIGRATE","minia","minimac4","minimap2","mira","mirdeep","mirdeep2","miso","mixomics","mlst","moff","mono","morpheus","Mothur","mpfr","mpiblast","mpip","mrbayes","msgfplus","msnbase","multisnv","mummer","muscle","mvapich2_ib","mxml","nag","nanopolish","nasm","nccl","nccmp","ncl","ncl_ncarg","nco","ncview","necat","netcdf","neurodebian","nextdenovo","nextflow","nextgenmap","nextpolish","ngs","ngsplot","nucleoatac","nvbio","nvidia-hpc-sdk","oases","opari2","openmpi","openmpi_ib","openmpi2_ib","openms","w4mcorcov","pepper+deepvariant","pacbioeda","padb","papi","paraver","paraview","parsnp","PASA","patchelf","pdt","peptideshaker","percolator","php","picard","picrust","picrust2","piler-df","pilon","plink","plinkseq","pnetcdf","pplacer","pretextview","prinseq","probabel","probmetab","prodigal","prokka","purge_dups","purgehaplotigs","pylibs","pyrad","pysam","python","python2","python2-as-python","python3","python3-as-python","pytorch","qiime","qiime-modules","qiime2","quast","quickmerge","r","rabbitmq-server","Racon","rails_cobbler","ratt","rattle","rawtools","raxml","refinem","repdenovo","repeatmasker","repeatmodeler","repeatmodeler2","repeatrunner","repet","rjags","rsem","rseqc","rstudio","ruby","salmon","samblaster","samtools","sas","scarpa","scipy","scons","screed","searchgui","biocontainers-seqkit","seqstats","seqtk-trinity","shapeit","sickle","siesta","singularity","skewer","smrtlink","snakemake","snap","snappy","snoscan","snpeff","soap","soap3-gpu","soapdenovo2","soapsnp","sortmerna","spades","splign","sra-tools","stacks","star","structure","supernova","svtyper","tagdust","tassel","tau","tensorflow","ThermoRawFileParser","tigmint","tophat","totalview","transdecoder","trf","trim_galore","trimmomatic","trinity","trinotate","trnascan-se","ucsc","ucx","udunits","unicycler","unixodbc","valet","vawk","vcftools","velvet","viennarna","visit","vsearch","wengan","windowmasker","wine","wise","workflow4metabolomics","wtdbg2","wxwidgets","xcms","xerces-c","xml4ena","xorg-macros","xtandem","zlib","zstd"],["Sequence assembly","Sequence analysis","Sequence assembly","Sequence assembly","Sequence assembly","Sequence assembly","Gene structure","Population genomics","","Genomics / Genome annotation","Chromosome conformation capture","","","Genomics / Genome annotation","","Genomics / Genome annotation","","","Sequence analysis","Phylogenomics","Sequence analysis","Metabolomics","","","Sequence analysis","Sequence analysis","Population genetics","Population genomics","Phylogenetics","Phylogenetics","Sequence analysis","","","","Sequence analysis","Programming language / platform","Programming language / platform","Epigenomics","Sequence alignment","Sequence alignment","Sequence alignment","Sequence alignment","Quality assurance","Quality assurance","Library","Sequence alignment","Sequence alignment","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","","Epigenomics","Epigenomics","Epigenomics","Quality assurance","Sequence alignment","Sequence analysis","","Sequence alignment","Metabolomics","Sequence assembly","Sequence assembly","Bioimaging","Metagenomics","Sequence analysis","Transcriptomics","Sequence assembly","","","Quality assurance","Quality assurance","Sequence analysis","Sequence alignment","Statistics and probability","","DNA structural variation","Genomics / Genome annotation","Genomics / Genome annotation","Library","Genomics / Genome annotation","","","","Library","Sequence assembly","","Sequence analysis","Multiomics","","Sequence analysis","Variant Calling","Phylogenomics","Proteomics / Metaproteomics","Sequence alignment","Sequence assembly","Genetic variation","","Protein folding, stability and design","","Sequence assembly","","Molecular biology","Data visualisation","Genomics / Genome annotation","Programming language / platform","Quality assurance","Genomics / Genome annotation","Phylogenomics","Sequence analysis","Phylogenomics","","","Sequence assembly","Phylogenomics","Quality assurance","Phylogenomics","Sequence analysis","Sequence analysis","Genomics / Genome annotation","Population genomics","Proteomics / Metaproteomics","","Sequence assembly","","","Sequence analysis","Phylogenomics","Bioimaging","Genomics / Genome annotation","Phylogenetics","Sequence analysis","Phylogenetics","Library","Phylogenomics","Library","Genomics / Genome annotation","Genomics / Genome annotation","Sequence analysis","Sequence assembly","","Library","Library","Library","Genomics / Genome annotation","Genomics / Genome annotation","","Sequence alignment","Sequence alignment","","Programming language / platform","Library","","Library","","Meta-Omics","Phylogenomics","Molecular dynamics","Metagenomics","Phylogenomics","Sequencing","Library","Sequence assembly","Quality assurance","Data visualisation","","","Data visualisation","Sequence assembly","Sequence alignment","Sequence analysis","","Sequence analysis","Library","Sequence assembly","Programming language / platform","Sequence assembly","Genomics / Genome annotation","Data visualisation","Protein sequence features","","Sequence analysis","Phylogenomics","Transcriptomics","Phylogenomics","Genomics / Genome annotation","Programming language / platform","Data visualisation","Sequence analysis","Sequence assembly","Sequence assembly","Programming language / platform","Transcriptomics","Sequence analysis","Sequence analysis","Sequence analysis","Taxonomy","Taxonomy","Data visualisation","Programming language / platform","Sequence alignment","Proteomics / Metaproteomics","Library","Library","","Sequence assembly","Metabolomics","","","Sequence alignment","Phylogenomics","ChIP-seq","ChIP-seq","Sequence alignment","Sequence alignment","Metagenomics","Genomics / Genome annotation","Genomics / Genome annotation","Proteomics","Sequence alignment","","Sequence assembly","","","Sequence alignment","","Metagenomics","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Statistics and probability","Sequence assembly","Metagenomics","Quality assurance","Metagenomics","Metagenomics","Metagenomics","Metabolomics","Metagenomics","Metagenomics","Population genomics","Sequence assembly","Phylogenomics","Sequence alignment","Sequence assembly","Transcriptomics","Transcriptomics","Transcriptomics","Multiomics","Sequence analysis","Proteomics / Metaproteomics","Programming language / platform","Proteomics / Metaproteomics","Metagenomics","","Sequence alignment","Library","Phylogenomics","Proteomics / Metaproteomics","Metabolomics","Variant Calling","Sequence alignment","Genomics / Genome annotation","","Library","Library","Sequence analysis","","","","Data visualisation","Data visualisation","","","Sequence assembly","Library","Neuroscience","Sequence assembly","Workflows","Sequence alignment","Sequence assembly","Knowledge discovery in databases","Data visualisation","Genomics / Genome annotation","Library","Library","Sequence assembly","","","","","Proteomics / Metaproteomics","Metabolomics","Sequence assembly","Sequence analysis","","","Data visualisation","Data visualisation","Phylogenomics","Genomics / Genome annotation","","","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Programming language / platform","Sequence analysis","Phylogenomics","Metagenomics","Genomics / Genome annotation","Sequence assembly","Phylogenomics","Variant Calling","Library","Phylogenomics","Data visualisation","Sequence analysis","Phylogenomics","Metabolomics","Genomics / Genome annotation","Genomics / Genome annotation","Sequence assembly","Sequence assembly","Library","Sequence analysis","Sequence analysis","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","","Metagenomics","Metagenomics","Metagenomics","Quality assurance","Sequence assembly","Programming language / platform","","Sequence assembly","Sequence assembly","Genomics / Genome annotation","Sequence assembly","Proteomics / Metaproteomics","Phylogenomics","Population genomics","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Phylogenomics","Transcriptomics","Quality assurance","Programming language / platform","Programming language / platform","Transcriptomics","Phylogenomics","Sequence analysis","Statistics and probability","Sequence assembly","Programming language / platform","","Sequence analysis","Proteomics / Metaproteomics","Sequence analysis","Sequence analysis","Sequence analysis","Phylogenomics","Sequence analysis","","Programming language / platform","Sequence analysis","Sequence analysis","Workflows","Genomics / Genome annotation","Phylogenomics","Genomics / Genome annotation","Variant Calling","Quality assurance","Sequence alignment","Sequence assembly","Sequence assembly","Metagenomics","Sequence assembly","Genomics / Genome annotation","Knowledge discovery in databases","Sequence assembly","Sequence alignment","Population genomics","Sequence assembly","Phylogenomics","Sequence analysis","Phylogenomics","","","Proteomics / Metaproteomics","Sequence assembly","Sequence alignment","","Transcriptomics","Genomics / Genome annotation","Sequence analysis","Sequence analysis","Sequence assembly","Transcriptomics","Transcriptomics","Sequence analysis","","","Sequence assembly","","Metagenomics","Quality assurance","Genetic variation","Sequence assembly","Transcriptomics","","Sequence analysis","Sequence assembly","Genomics / Genome annotation","","Sequence alignment","Metabolomics","Sequence assembly","","Metabolomics","","Data submission, annotation and curation","","Proteomics / Metaproteomics","",""],["","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abricate'>abricate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abyss'>abyss<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=apollo'>apollo<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=augustus'>augustus<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=batchcorrection'>batchcorrection<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bcftools'>bcftools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beagle'>beagle<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beast'>beast<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bedtools'>bedtools<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bismark'>bismark<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blat'>blat<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=boost'>boost<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie_wrappers'>bowtie_wrappers<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie2'>bowtie2<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=breakdancer_max'>breakdancer_max<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bsmap'>bsmap<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bs_seeker2'>bs_seeker2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=busco'>busco<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bwa'>bwa<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bx_python'>bx_python<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bz2file'>bz2file<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=camera'>camera<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=canu'>canu<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cap3'>cap3<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cardinal'>cardinal<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cat_bins'>cat_bins<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cdhit'>cdhit<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=clustalw'>clustalw<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cmake'>cmake<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=coreutils'>coreutils<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cufflinks'>cufflinks<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=curl'>curl<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cutadapt'>cutadapt<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cytoscape'>cytoscape<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=deeptools'>deeptools<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dendropy'>dendropy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dia_umpire'>dia_umpire<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=diamond'>diamond<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=edena'>edena<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=emboss'>emboss<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ensembl'>ensembl<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=exonerate'>exonerate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=export2graphlan'>export2graphlan<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastqc'>fastqc<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx'>fastx<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx'>fastx<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flashlfq'>flashlfq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flye'>flye<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fraggenescan'>fraggenescan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=freebayes'>freebayes<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=garli'>garli<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gatk'>gatk<\/a>","","","","","","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer'>glimmer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer_hmm'>glimmer_hmm<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=libtool'>libtool<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gsl'>gsl<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=graphlan'>graphlan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gmx'>gmx<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gubbins'>gubbins<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=guppy'>guppy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hadoop'>hadoop<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hisat2'>hisat2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hmmer'>hmmer<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=htseq'>htseq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=idba'>idba<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=infernal'>infernal<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=interproscan'>interproscan<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=iqtree'>iqtree<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jbrowse'>jbrowse<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jellyfish'>jellyfish<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kallisto'>kallisto<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kat'>kat<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=khmer'>khmer<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken'>kraken<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken2'>kraken2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kubernetes'>kubernetes<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lastz'>lastz<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=libgtextutils'>libgtextutils<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lipidmaps'>lipidmaps<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs'>macs<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs2'>macs2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mafft'>mafft<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maker'>maker<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maldi_quant'>maldi_quant<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mauve'>mauve<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxbin2'>maxbin2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant'>maxquant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant'>maxquant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mcl'>mcl<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=medaka'>medaka<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=megahit'>megahit<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metams'>metams<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaphlan'>metaphlan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaspades'>metaspades<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minia'>minia<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minimap2'>minimap2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mira'>mira<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mirdeep2'>mirdeep2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mixomics'>mixomics<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mlst'>mlst<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=proteomics_moff'>proteomics_moff<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=morpheus'>morpheus<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mothur'>mothur<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mrbayes'>mrbayes<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msgfplus'>msgfplus<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msnbase'>msnbase<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mummer'>mummer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=muscle'>muscle<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=nanopolish'>nanopolish<\/a>","","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=netcdf'>netcdf<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ngsplot'>ngsplot<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=oases'>oases<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=openms'>openms<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4mcorcov'>w4mcorcov<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=parsnp'>parsnp<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=peptideshaker'>peptideshaker<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=percolator'>percolator<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picard'>picard<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picrust'>picrust<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pilon'>pilon<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=plink'>plink<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prinseq'>prinseq<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prodigal'>prodigal<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prokka'>prokka<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pysam'>pysam<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime'>qiime<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime'>qiime<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime2'>qiime2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=quast'>quast<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=racon'>racon<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rawtools'>rawtools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=raxml'>raxml<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repeat_masker'>repeat_masker<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repet'>repet<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rsem'>rsem<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rseqc'>rseqc<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ruby'>ruby<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=salmon'>salmon<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samblaster'>samblaster<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samtools'>samtools<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=scipy'>scipy<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=screed'>screed<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=searchgui'>searchgui<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=seqkit'>seqkit<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sickle'>sickle<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snap'>snap<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snpeff'>snpeff<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sortmerna'>sortmerna<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=spades'>spades<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=stacks'>stacks<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=star'>star<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=structure'>structure<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=svtyper'>svtyper<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tagdust'>tagdust<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tassel'>tassel<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=thermo_raw_file_converter'>thermo_raw_file_converter<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tophat'>tophat<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=transdecoder'>transdecoder<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trim_galore'>trim_galore<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trimmomatic'>trimmomatic<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinity'>trinity<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinotate'>trinotate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trnascan-se'>trnascan-se<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ucsc'>ucsc<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=unicycler'>unicycler<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=valet'>valet<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vcftools'>vcftools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=velvet'>velvet<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vienna'>vienna<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vsearch'>vsearch<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=windowmasker'>windowmasker<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4m'>w4m<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xcms'>xcms<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xtandem'>xtandem<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=zlib'>zlib<\/a>",""],["","<a href='https://bio.tools/ABRicate'>&#9679;<\/a>","<a href='https://bio.tools/ABySS'>&#9679;<\/a>","","","<a href='https://bio.tools/amos'>&#9679;<\/a>","<a href='https://bio.tools/anaconda'>&#9679;<\/a>","<a href='https://bio.tools/angsd'>&#9679;<\/a>","","<a href='https://bio.tools/apollo'>&#9679;<\/a>","","","","<a href='https://bio.tools/artemis'>&#9679;<\/a>","","<a href='https://bio.tools/augustus'>&#9679;<\/a>","","","","<a href='https://bio.tools/bamm'>&#9679;<\/a>","<a href='https://bio.tools/bamtools'>&#9679;<\/a>","","","","<a href='https://bio.tools/bbmap'>&#9679;<\/a>","<a href='https://bio.tools/bcftools'>&#9679;<\/a>","<a href='https://bio.tools/beagle'>&#9679;<\/a>","","<a href='https://bio.tools/beast'>&#9679;<\/a>","<a href='https://bio.tools/beast2'>&#9679;<\/a>","<a href='https://bio.tools/bedtools'>&#9679;<\/a>","","<a href='https://bio.tools/bioconductor'>&#9679;<\/a>","","","<a href='https://bio.tools/bioperl'>&#9679;<\/a>","<a href='https://bio.tools/Biopython'>&#9679;<\/a>","<a href='https://bio.tools/bismark'>&#9679;<\/a>","<a href='https://bio.tools/blast'>&#9679;<\/a>","","","<a href='https://bio.tools/blat'>&#9679;<\/a>","<a href='https://bio.tools/BlobToolKit'>&#9679;<\/a>","<a href='https://bio.tools/BlobToolKit'>&#9679;<\/a>","","<a href='https://bio.tools/bowtie'>&#9679;<\/a>","<a href='https://bio.tools/bowtie2'>&#9679;<\/a>","<a href='https://bio.tools/braker1'>&#9679;<\/a>","","<a href='https://bio.tools/breakdancer'>&#9679;<\/a>","","<a href='https://bio.tools/bsmap'>&#9679;<\/a>","","","<a href='https://bio.tools/busco'>&#9679;<\/a>","<a href='https://bio.tools/bwa'>&#9679;<\/a>","<a href='https://bio.tools/bx-python'>&#9679;<\/a>","","","<a href='https://bio.tools/camera'>&#9679;<\/a>","<a href='https://bio.tools/canu'>&#9679;<\/a>","<a href='https://bio.tools/cap3'>&#9679;<\/a>","<a href='https://bio.tools/cardinal'>&#9679;<\/a>","","<a href='https://bio.tools/cd-hit'>&#9679;<\/a>","","<a href='https://bio.tools/cabog'>&#9679;<\/a>","","","<a href='https://bio.tools/checkm'>&#9679;<\/a>","<a href='https://bio.tools/checkm'>&#9679;<\/a>","","<a href='https://bio.tools/clustalw'>&#9679;<\/a>","","","<a href='https://bio.tools/cnvkit'>&#9679;<\/a>","<a href='https://bio.tools/cpc'>&#9679;<\/a>","<a href='https://bio.tools/cpc2'>&#9679;<\/a>","","","","","","","<a href='https://bio.tools/cufflinks'>&#9679;<\/a>","","<a href='https://bio.tools/cutadapt'>&#9679;<\/a>","<a href='https://bio.tools/cytoscape'>&#9679;<\/a>","","<a href='https://bio.tools/deeptools'>&#9679;<\/a>","","","<a href='https://bio.tools/DIA-Umpire'>&#9679;<\/a>","<a href='https://bio.tools/diamond'>&#9679;<\/a>","<a href='https://bio.tools/discovar_de_novo'>&#9679;<\/a>","","","<a href='https://bio.tools/dssp'>&#9679;<\/a>","","<a href='https://bio.tools/edena'>&#9679;<\/a>","","<a href='https://bio.tools/emboss'>&#9679;<\/a>","<a href='https://bio.tools/ensembl'>&#9679;<\/a>","","","","<a href='https://bio.tools/EvidenceModeler'>&#9679;<\/a>","","<a href='https://bio.tools/exonerate'>&#9679;<\/a>","","","","","<a href='https://bio.tools/fastml'>&#9679;<\/a>","<a href='https://bio.tools/fastqc'>&#9679;<\/a>","<a href='https://bio.tools/faststructure'>&#9679;<\/a>","<a href='https://bio.tools/fastx-toolkit'>&#9679;<\/a>","<a href='https://bio.tools/fastx-toolkit'>&#9679;<\/a>","<a href='https://bio.tools/fgenesh'>&#9679;<\/a>","<a href='https://bio.tools/fimpute'>&#9679;<\/a>","","","<a href='https://bio.tools/Flye'>&#9679;<\/a>","","","<a href='https://bio.tools/fraggenescan'>&#9679;<\/a>","<a href='https://bio.tools/freebayes'>&#9679;<\/a>","<a href='https://bio.tools/freesurfer'>&#9679;<\/a>","","","<a href='https://bio.tools/gatk'>&#9679;<\/a>","<a href='https://bio.tools/gblocks'>&#9679;<\/a>","","<a href='https://bio.tools/gcta'>&#9679;<\/a>","","<a href='https://bio.tools/geneid'>&#9679;<\/a>","<a href='https://bio.tools/genemark'>&#9679;<\/a>","<a href='https://bio.tools/GenomeScope_2.0'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/glimmer'>&#9679;<\/a>","<a href='https://bio.tools/glimmer-hmm'>&#9679;<\/a>","","<a href='https://bio.tools/gmap'>&#9679;<\/a>","<a href='https://bio.tools/gmap'>&#9679;<\/a>","","","","","","","<a href='https://bio.tools/graftm'>&#9679;<\/a>","","<a href='https://bio.tools/gromacs'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/hawkeye'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/hisat2'>&#9679;<\/a>","<a href='https://bio.tools/hmmer3'>&#9679;<\/a>","","<a href='https://bio.tools/htseq'>&#9679;<\/a>","<a href='https://bio.tools/htslib'>&#9679;<\/a>","<a href='https://bio.tools/idba'>&#9679;<\/a>","","","","","<a href='https://bio.tools/interproscan_4'>&#9679;<\/a>","","","<a href='https://bio.tools/iq-tree'>&#9679;<\/a>","","","","","<a href='https://bio.tools/jbrowse'>&#9679;<\/a>","<a href='https://bio.tools/jellyfish'>&#9679;<\/a>","","","","<a href='https://bio.tools/kallisto'>&#9679;<\/a>","<a href='https://bio.tools/kat'>&#9679;<\/a>","<a href='https://bio.tools/khmer'>&#9679;<\/a>","<a href='https://bio.tools/kmc'>&#9679;<\/a>","<a href='https://bio.tools/kraken'>&#9679;<\/a>","<a href='https://bio.tools/kraken2'>&#9679;<\/a>","<a href='https://bio.tools/krona'>&#9679;<\/a>","","<a href='https://bio.tools/lastz'>&#9679;<\/a>","<a href='https://bio.tools/LFQ-Analyst'>&#9679;<\/a>","","","","","<a href='https://bio.tools/lipid_maps'>&#9679;<\/a>","","","","<a href='https://bio.tools/mach'>&#9679;<\/a>","<a href='https://bio.tools/macs'>&#9679;<\/a>","<a href='https://bio.tools/macs'>&#9679;<\/a>","","<a href='https://bio.tools/MAFFT'>&#9679;<\/a>","","<a href='https://bio.tools/maker'>&#9679;<\/a>","<a href='https://bio.tools/maker-p'>&#9679;<\/a>","","","","<a href='https://bio.tools/masurca'>&#9679;<\/a>","<a href='https://bio.tools/matlab'>&#9679;<\/a>","","<a href='https://bio.tools/mauve'>&#9679;<\/a>","","<a href='https://bio.tools/maxbin'>&#9679;<\/a>","<a href='https://bio.tools/MaxQuant'>&#9679;<\/a>","<a href='https://bio.tools/MaxQuant'>&#9679;<\/a>","<a href='https://bio.tools/mcl'>&#9679;<\/a>","","<a href='https://bio.tools/megahit'>&#9679;<\/a>","","<a href='https://bio.tools/MetaBAT_2'>&#9679;<\/a>","","","<a href='https://bio.tools/metams'>&#9679;<\/a>","<a href='https://bio.tools/metaphlan'>&#9679;<\/a>","<a href='https://bio.tools/metaspades'>&#9679;<\/a>","<a href='https://bio.tools/MIGRATE'>&#9679;<\/a>","<a href='https://bio.tools/minia'>&#9679;<\/a>","","<a href='https://bio.tools/minimap2'>&#9679;<\/a>","<a href='https://bio.tools/mira'>&#9679;<\/a>","<a href='https://bio.tools/mirdeep'>&#9679;<\/a>","","<a href='https://bio.tools/miso'>&#9679;<\/a>","<a href='https://bio.tools/mixomics'>&#9679;<\/a>","<a href='https://bio.tools/mlst'>&#9679;<\/a>","<a href='https://bio.tools/moff'>&#9679;<\/a>","","<a href='https://bio.tools/morpheus'>&#9679;<\/a>","<a href='https://bio.tools/Mothur'>&#9679;<\/a>","","","","<a href='https://bio.tools/mrbayes'>&#9679;<\/a>","<a href='https://bio.tools/msgfplus'>&#9679;<\/a>","<a href='https://bio.tools/msnbase'>&#9679;<\/a>","","<a href='https://bio.tools/mummer'>&#9679;<\/a>","<a href='https://bio.tools/muscle'>&#9679;<\/a>","","","","<a href='https://bio.tools/nanopolish'>&#9679;<\/a>","","","","","","","","","","","","","","<a href='https://bio.tools/NextPolish'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/oases'>&#9679;<\/a>","","","","","<a href='https://bio.tools/openms'>&#9679;<\/a>","","","","","","","","","<a href='https://bio.tools/PASA'>&#9679;<\/a>","","","<a href='https://bio.tools/peptideshaker'>&#9679;<\/a>","<a href='https://bio.tools/percolator'>&#9679;<\/a>","","<a href='https://bio.tools/picard_tools'>&#9679;<\/a>","","","","<a href='https://bio.tools/pilon'>&#9679;<\/a>","<a href='https://bio.tools/plink'>&#9679;<\/a>","<a href='https://bio.tools/plink-seq'>&#9679;<\/a>","","<a href='https://bio.tools/clstutils'>&#9679;<\/a>","","<a href='https://bio.tools/prinseq'>&#9679;<\/a>","","","<a href='https://bio.tools/gmv'>&#9679;<\/a>","<a href='https://bio.tools/prokka'>&#9679;<\/a>","<a href='https://bio.tools/purge_dups'>&#9679;<\/a>","","","<a href='https://bio.tools/pyrad'>&#9679;<\/a>","<a href='https://bio.tools/pysam'>&#9679;<\/a>","","","","","","","<a href='https://bio.tools/qiime'>&#9679;<\/a>","<a href='https://bio.tools/qiime'>&#9679;<\/a>","","<a href='https://bio.tools/quast'>&#9679;<\/a>","","<a href='https://bio.tools/r'>&#9679;<\/a>","","<a href='https://bio.tools/Racon'>&#9679;<\/a>","","<a href='https://bio.tools/ratt'>&#9679;<\/a>","","<a href='https://bio.tools/rawtools'>&#9679;<\/a>","<a href='https://bio.tools/raxml'>&#9679;<\/a>","","<a href='https://bio.tools/repdenovo'>&#9679;<\/a>","<a href='https://bio.tools/repeatmasker'>&#9679;<\/a>","<a href='https://bio.tools/repeatmodeler'>&#9679;<\/a>","<a href='https://bio.tools/RepeatModeler2'>&#9679;<\/a>","<a href='https://bio.tools/repeatrunner'>&#9679;<\/a>","<a href='https://bio.tools/repet'>&#9679;<\/a>","","<a href='https://bio.tools/rsem'>&#9679;<\/a>","<a href='https://bio.tools/rseqc'>&#9679;<\/a>","<a href='https://bio.tools/rstudio'>&#9679;<\/a>","","<a href='https://bio.tools/salmon'>&#9679;<\/a>","<a href='https://bio.tools/samblaster'>&#9679;<\/a>","<a href='https://bio.tools/samtools'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/searchgui'>&#9679;<\/a>","","","","<a href='https://bio.tools/shapeit'>&#9679;<\/a>","<a href='https://bio.tools/sickle'>&#9679;<\/a>","","","<a href='https://bio.tools/skewer'>&#9679;<\/a>","<a href='https://bio.tools/smrt_analysis'>&#9679;<\/a>","<a href='https://bio.tools/snakemake'>&#9679;<\/a>","<a href='https://bio.tools/snap'>&#9679;<\/a>","<a href='https://bio.tools/SNAPPy'>&#9679;<\/a>","<a href='https://bio.tools/snoscan'>&#9679;<\/a>","<a href='https://bio.tools/snpeff'>&#9679;<\/a>","<a href='https://bio.tools/soap'>&#9679;<\/a>","","","<a href='https://bio.tools/soapsnp'>&#9679;<\/a>","<a href='https://bio.tools/sortmerna'>&#9679;<\/a>","<a href='https://bio.tools/spades'>&#9679;<\/a>","","<a href='https://bio.tools/sra-tools'>&#9679;<\/a>","<a href='https://bio.tools/stacks'>&#9679;<\/a>","<a href='https://bio.tools/star'>&#9679;<\/a>","<a href='https://bio.tools/structure'>&#9679;<\/a>","","","<a href='https://bio.tools/tagdust'>&#9679;<\/a>","<a href='https://bio.tools/tassel'>&#9679;<\/a>","","<a href='https://bio.tools/TensorFlow'>&#9679;<\/a>","<a href='https://bio.tools/ThermoRawFileParser'>&#9679;<\/a>","","<a href='https://bio.tools/tophat'>&#9679;<\/a>","","<a href='https://bio.tools/TransDecoder'>&#9679;<\/a>","<a href='https://bio.tools/trf'>&#9679;<\/a>","<a href='https://bio.tools/trim_galore'>&#9679;<\/a>","<a href='https://bio.tools/trimmomatic'>&#9679;<\/a>","<a href='https://bio.tools/trinity'>&#9679;<\/a>","","<a href='https://bio.tools/trnascan-se'>&#9679;<\/a>","<a href='https://bio.tools/UCSC_Genome_Browser_Utilities'>&#9679;<\/a>","","","<a href='https://bio.tools/unicycler'>&#9679;<\/a>","","","","<a href='https://bio.tools/vcftools'>&#9679;<\/a>","<a href='https://bio.tools/velvet'>&#9679;<\/a>","","","<a href='https://bio.tools/vsearch'>&#9679;<\/a>","<a href='https://bio.tools/WENGAN'>&#9679;<\/a>","","","<a href='https://bio.tools/wise'>&#9679;<\/a>","<a href='https://bio.tools/workflow4metabolomics'>&#9679;<\/a>","","","<a href='https://bio.tools/xcms'>&#9679;<\/a>","","","","<a href='https://bio.tools/xtandem'>&#9679;<\/a>","",""],["","<a href='https://biocontainers.pro/#/tools/abricate'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/abyss'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/amos'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/angsd'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/ant'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/apollo'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/artemis'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/augustus'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/perl-config-autoconf'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/bamtools'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/batchcorrection'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/bcftools'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beagle'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beagle-lib'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beast'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beast2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bedtools'>&#9679;<\/a>","","","","","","","<a href='https://biocontainers.pro/#/tools/bismark'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/blast'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/ncbi-blast-plus'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/blat'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/blobtools'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bowtie'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bowtie2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/braker'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/braker2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/breakdancer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bsmap'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bs-seeker2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/busco'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bwa'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bx-python'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bz2file'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cactus'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-camera'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/canu'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cap3'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-cardinal'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/cd-hit'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/wgs-assembler'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/checkm-genome'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/checkm-genome'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/clustalw'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/cnvkit'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/coreutils'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cpat'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/cufflinks'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/curl'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cutadapt'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cytoscape'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/deeptools'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/deepvariant'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/dendropy'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/dia_umpire'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/diamond'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/discovardenovo'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/dssp'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/edena'>&#9679;<\/a>","","","","","","","<a href='https://biocontainers.pro/#/tools/evidencemodeler'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/exonerate'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/export2graphlan'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/pb-assembly'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pb-assembly'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastml'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastqc'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/faststructure'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastx_toolkit'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastx_toolkit'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/flashlfq'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/flye'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/fraggenescan'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/freebayes'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/funannotate'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/garli'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gatk4'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gblocks'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/gcta'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/tools/genomescope2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/genomicconsensus'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/glimmer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/glimmerhmm'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/gmap'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gmap'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/parallel'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/graftm'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/graphlan'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gromacs'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/groopm'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gubbins'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/haslr'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/hclust2'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/hisat2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/hmmer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/htseq'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/htslib'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/idba'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/pbipa'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/igv'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/interproscan'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/ipyrad'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/iqtree'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/isoseq3'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/jbrowse'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/jellyfish'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/julia'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kallisto'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kat'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/khmer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kmc'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kraken'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kraken2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/krona'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/k8'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/lastz'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/libgtextutils'>&#9679;<\/a>","","<a href='https://biocontainers.pro/tools/links'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/longranger'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/macs2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/macse'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mafft'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/maker'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/r-maldiquant'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mashmap'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/maskrc-svg'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/masurca'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/mauvealigner'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/maxbin2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/maxquant'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/maxquant'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mcl'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/medaka'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/megahit'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/metabat2'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/metamaps'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-metams'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/metaphlan2'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/migrate-n'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/minia'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/minimac4'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/minimap2'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/mira'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/mirdeep2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/misopy'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/r-mixomics'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mlst'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/moff'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mono'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/morpheus'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mothur'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/mrbayes'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/msgf_plus'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-msnbase'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/mummer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/muscle'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/tools/nanopolish'>&#9679;<\/a>","","","","","","","","","","","","<a href='https://biocontainers.pro/#/tools/nextflow'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/nextgenmap'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/nucleoatac'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/oases'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/openms'>&#9679;<\/a>","","","","","","","<a href='https://biocontainers.pro/#/tools/paraview'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/parsnp'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pasa'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/peptideshaker'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/percolator'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/picard'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/picrust'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/picrust2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/piler'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pilon'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/plink'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/pplacer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/tools/prinseq'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/probabel'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/r-probmetab'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/prodigal'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/prokka'>&#9679;<\/a>","","<a href='https://biocontainers.pro/tools/purge_haplotigs'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/pyrad'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pysam'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/python'>&#9679;<\/a>","","","","","","<a href='https://biocontainers.pro/#/tools/qiime'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/qiime'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/quast'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/quickmerge'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/racon'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/ratt'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/rawtools'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/raxml'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/refinem'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repdenovo'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repeatmasker'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repeatmodeler'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repeatmodeler'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/rsem'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/rseqc'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/ruby'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/salmon'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/samblaster'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/samtools'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/scipy'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/screed'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/searchgui'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/seqkit'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/seqstats'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/shapeit'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/sickle'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/singularity'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/skewer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/snakemake'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/snap'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/snoscan'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/snpeff'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/soapdenovo2'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/soapsnp'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/sortmerna'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/spades'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/sra-tools'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/stacks'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/star'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/structure'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/svtyper'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/tassel'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/thermorawfileparser'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/tigmint'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/tophat'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/transdecoder'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trf'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trim-galore'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trimmomatic'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trinity'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trinotate'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trnascan-se'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/tools/unicycler'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/valet'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/vawk'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/vcftools'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/velvet'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/viennarna'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/vsearch'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/wise'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/isa2w4m'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/wtdbg2'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bioconductor-xcms'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/xtandem'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/zstd'>&#9679;<\/a>"]],"container":"<table class=\"cell-border stripe\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Tool / workflow name<\/th>\n      <th>Tool identifier (module name / bio.tools ID / placeholder)<\/th>\n      <th>Primary purpose (EDAM, if available)<\/th>\n      <th>Available in Galaxy toolshed<\/th>\n      <th>bio.tools<\/th>\n      <th>BioContainers<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
<p></font></p>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-02-23</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>The <em>Tool identifier</em> column contains an identifier for the tool / workflow: typically either the module name (used for matching to HPC lists), a bio.tools ID, or a placeholder value.</li>
<li>The <em>Primary purpose</em> column categorises the tools by purpose, using an <a href="https://github.com/edamontology/edamontology">EDAM</a> <em>topic</em> where possible.</li>
<li>More information about a tool can be found by following the link in the <a href="https://bio.tools/">bio.tools</a> column.</li>
<li>When a tool has been containerised to allow for easier installation on any compute infrastructure, a link to the containerised software that can be downloaded is shown in the <a href="https://biocontainers.pro/#/">BioContainers</a> column.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<hr />
<hr />
</div>
</div>
<div id="how-were-the-tools-and-workflows-selected" class="section level1">
<h1>How were the tools and workflows selected?</h1>
<p>The tool list here includes those that have been identified through <a href="https://www.biocommons.org.au/get-involved">BioCommons Community consultations</a> and other engagements, as well as through matching to the tool sets installed across the various BioCommons partner compute infrastructures. The list is not intended to be exhaustive: i.e. it does not contain <em>ALL</em> bioinformatics tools.</p>
<hr />
<hr />
</div>
<div id="how-was-the-list-generated" class="section level1">
<h1>How was the list generated?</h1>
<p>RStudio (see session info at the bottom of this page) was used for processing.</p>
<p>The broad steps are described here:</p>
<p><font size="-1.5"></p>
<ol style="list-style-type: decimal">
<li>The manually curated tool list and separate tool lists provided by the BioCommons partner infrastructures were parsed (Galaxy Australia, NCI, Pawsey and QRISCloud - RCC).</li>
<li>The complete set of tool lists were joined using the <code>toolID</code> as a key, and</li>
<li>Links were embedded, where available, to the original documentation URLs (homepage or GitHub repository, for example) as well as the registry entries on bio.tools and BioContainers.</li>
<li>Galaxy wrapping often entails that tool suites are separated into their individual tool components. As such, if a Galaxy tool was matched to the manually curated tool list, a tool shed search link was embedded in the <code>Available in Galaxy toolshed</code> column. This allows a user to search for the tool using a relevant search term (e.g. vcftools) to identify all the component tools available in the toolshed.</li>
</ol>
<p></font></p>
<hr />
<hr />
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>## R version 4.0.3 (2020-10-10)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## Running under: Windows 10 x64 (build 19041)
## 
## Matrix products: default
## 
## locale:
## [1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
## [3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
## [5] LC_TIME=English_Australia.1252    
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] DT_0.17           lubridate_1.7.9.2 kableExtra_1.3.1  yaml_2.2.1       
##  [5] forcats_0.5.1     stringr_1.4.0     dplyr_1.0.3       purrr_0.3.4      
##  [9] readr_1.4.0       tidyr_1.1.2       tibble_3.0.5      ggplot2_3.3.3    
## [13] tidyverse_1.3.0   knitr_1.31       
## 
## loaded via a namespace (and not attached):
##  [1] tidyselect_1.1.0  xfun_0.20         haven_2.3.1       colorspace_2.0-0 
##  [5] vctrs_0.3.6       generics_0.1.0    htmltools_0.5.1.1 viridisLite_0.3.0
##  [9] rlang_0.4.10      pillar_1.4.7      glue_1.4.2        withr_2.4.1      
## [13] DBI_1.1.1         dbplyr_2.0.0      modelr_0.1.8      readxl_1.3.1     
## [17] lifecycle_0.2.0   munsell_0.5.0     gtable_0.3.0      cellranger_1.1.0 
## [21] rvest_0.3.6       htmlwidgets_1.5.3 evaluate_0.14     crosstalk_1.1.1  
## [25] fansi_0.4.2       broom_0.7.3       Rcpp_1.0.6        scales_1.1.1     
## [29] backports_1.2.0   webshot_0.5.2     jsonlite_1.7.2    fs_1.5.0         
## [33] hms_1.0.0         digest_0.6.27     stringi_1.5.3     grid_4.0.3       
## [37] cli_2.2.0         tools_4.0.3       magrittr_2.0.1    crayon_1.3.4     
## [41] pkgconfig_2.0.3   ellipsis_0.3.1    xml2_1.3.2        reprex_1.0.0     
## [45] assertthat_0.2.1  rmarkdown_2.6     httr_1.4.2        rstudioapi_0.13  
## [49] R6_2.5.0          compiler_4.0.3</code></pre>
<hr />
<hr />
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
