<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Bioinformatics tools &amp; workflows on BioCommons partner infrastructures</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="1_compute_systems.html">Compute systems</a>
</li>
<li>
  <a href="2_tools.html">Tools &amp; workflows</a>
</li>
<li>
  <a href="3_biocommons_activities.html">Featured tools &amp; workflows</a>
</li>
<li>
  <a href="4_help.html">Help!</a>
</li>
<li>
  <a href="5_attributions.html">Attributions</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Bioinformatics tools &amp; workflows on BioCommons partner infrastructures</h1>

</div>


<p>This page details tool request mechanisms and a list of tools and workflows installed across several of the <a href="./1_compute_systems.html">BioCommons infrastructure partner systems</a>. Tool status on other partner infrastructures will be added over time.</p>
<hr />
<hr />
<div id="requesting-tool-installations" class="section level1">
<h1>Requesting tool installations</h1>
<p>Tool installs can be requested for the infrastructures listed in the table:</p>
<font size="-1.5">
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Infrastructure
</th>
<th style="text-align:left;">
Process
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="https://usegalaxy.org.au/">Galaxy Australia</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://request.usegalaxy.org.au/">tool install request</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://nci.org.au/">NCI</a>
</td>
<td style="text-align:left;">
<a href="https://opus.nci.org.au/display/Help/5.+Software+Applications">Contact NCI</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://pawsey.org.au/">Pawsey</a>
</td>
<td style="text-align:left;">
Visit the <a href="https://support.pawsey.org.au/portal/servicedesk/customer/portal">Helpdesk Portal</a> or email: <a href="help@pawsey.org.au">help@pawsey.org.au</a> with your request
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://www.qriscloud.org.au/">QRIScloud</a> / <a href="https://rcc.uq.edu.au/">UQ-RCC</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://docs.google.com/forms/d/e/1FAIpQLSefKCBkZbTQ-dUyU-pd4ZypkA-TA5GiMgpsJ2slWmD-B6elEg/viewform">tool install request</a>
</td>
</tr>
</tbody>
</table>
<p></font></p>
<hr />
<hr />
</div>
<div id="tool-information" class="section level1 tabset tabset-pills">
<h1>Tool information</h1>
<div id="tool-install-status-and-version-availability" class="section level2">
<h2>Tool install status and version availability</h2>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-02-12</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>For tools which are not currently installed on <a href="https://usegalaxy.org.au/">Galaxy Australia</a>, but which are available <a href="https://toolshed.g2.bx.psu.edu/">in the Galaxy app store (aka toolshed)</a>, the <em>Available in Galaxy toolshed</em> column launches a toolshed search using the link label as the search term.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<p><font size="-1.5"></p>
<table>
<colgroup>
<col width="13%" />
<col width="2%" />
<col width="17%" />
<col width="23%" />
<col width="2%" />
<col width="2%" />
<col width="38%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Tool / workflow name</th>
<th align="center">Galaxy Australia</th>
<th align="center">Available in Galaxy toolshed</th>
<th align="center">NCI (Gadi)</th>
<th align="center">Pawsey (Zeus)</th>
<th align="center">Pawsey (Magnus)</th>
<th align="center">QRIScloud / UQ-RCC (Flashlite, Awoonga, Tinaroo)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><a href="https://github.com/aidenlab/3d-dna">3D-DNA</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/tseemann/abricate">ABRicate</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abricate&amp;sort=name">abricate</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/abyss">ABySS</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abyss&amp;sort=name">abyss</a></td>
<td align="center">2.2.3</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/tanghaibao/jcvi/wiki/ALLMAPS">ALLMAPS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://software.broadinstitute.org/allpaths-lg/blog/">allpathslg</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://amos.sourceforge.net/wiki/index.php/AMOS">amos</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://bioinformatics.ua.pt/software/anaconda/">anaconda</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2019.07<br />2020.02<br />4.2.0<br />4.3.1<br />5.2.0</td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.popgen.dk/angsd/index.php/ANGSD">ANGSD</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.931<br />0.933</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://ant.apache.org/">ant</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.10.7</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/GMOD/Apollo">Apollo</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=apollo&amp;sort=name">apollo</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://arimagenomics.com/assembly">arima mapping</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.arm.com/products/development-tools/server-and-hpc/forge">arm-forge</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20.0.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center">arm-reports</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20.0.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/sanger-pathogens/Artemis">Artemis</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.aspera.com/en/">Aspera</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.9.6</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Gaius-Augustus/Augustus">Augustus</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=augustus&amp;sort=name">augustus</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.3.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/autoconf/">Autoconf</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.69-GCC-4.9.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/software/automake/">Automake</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.15-gcc-4.8.2<br />1.15-GCC-4.9.2<br />1.5-gcc-4.8.2</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/hall-lab/bamkit">bamkit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.1</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bamm-project.org/">bamm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201710<br />201811</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/pezmaster31/bamtools">bamtools</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.4.1</td>
<td align="center"></td>
<td align="center">2.4.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/workflow4metabolomics/batch_correction">batchcorrection</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=batchcorrection&amp;sort=name">batchcorrection</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bazel.build/">bazel</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.26.0<br />0.27.1<br />3.1.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/slaclab/bbcp">bbcp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">15.02.03.00.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://jgi.doe.gov/data-and-tools/bbtools/">bbmap</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201710</td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.htslib.org/">BCFtools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bcftools&amp;sort=name">bcftools</a></td>
<td align="center">1.9</td>
<td align="center">1.6</td>
<td align="center"></td>
<td align="center">1.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://faculty.washington.edu/browning/beagle/beagle.html">Beagle</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beagle&amp;sort=name">beagle</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1<br />4.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/beagle-dev/beagle-lib">beagle-lib</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1.2</td>
<td align="center">2.1.2</td>
<td align="center">2.1.2</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/beast-dev/beast-mcmc">BEAST</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beast&amp;sort=name">beast</a></td>
<td align="center">1.10.4</td>
<td align="center">1.8.4<br />2.4.7</td>
<td align="center"></td>
<td align="center">1.8.0<br />1.8.1<br />1.8.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/CompEvol/beast2">BEAST2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/arq5x/bedtools2">BEDTools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bedtools&amp;sort=name">bedtools</a></td>
<td align="center">2.28.0</td>
<td align="center">2.26.0</td>
<td align="center"></td>
<td align="center">2.25.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/software/binutils/">binutils</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.25<br />2.25-GCC-4.9.2</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.bioconductor.org/">bioconductor</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.6-rc2<br />3.8-rc2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Bio-Linux">biolinux</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">8</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bionanogenomics.com/support-page/bionano-access-software/">Bionano Access</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioperl.org/">bioperl</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.7.1</td>
<td align="center"></td>
<td align="center">1.007002<br />1.007002-sw</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://biopython.org/">biopython</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.70</td>
<td align="center">1.70</td>
<td align="center">1.66</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/FelixKrueger/Bismark">bismark</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bismark&amp;sort=name">bismark</a></td>
<td align="center"></td>
<td align="center">0.19.0</td>
<td align="center"></td>
<td align="center">0.15.0<br />0.22.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast&amp;sort=name">blast</a></td>
<td align="center">2.10.1</td>
<td align="center">2.2.26</td>
<td align="center"></td>
<td align="center">2.3.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST+</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast&amp;sort=name">blast</a></td>
<td align="center"></td>
<td align="center">2.10.1<br />2.7.1</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST2</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast&amp;sort=name">blast</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.10.1<br />2.9.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://genome.ucsc.edu/cgi-bin/hgBlat">blat</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blat&amp;sort=name">blat</a></td>
<td align="center"></td>
<td align="center">3.6</td>
<td align="center"></td>
<td align="center">35</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/blobtoolkit/blobtools2">BlobToolKit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/DRL/blobtools">BlobTools1</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.boost.org/">Boost</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=boost&amp;sort=name">boost</a></td>
<td align="center">1.71.0<br />1.72.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.65.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://bowtie-bio.sourceforge.net/index.shtml">Bowtie</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie_wrappers&amp;sort=name">bowtie_wrappers</a></td>
<td align="center"></td>
<td align="center">1.2.1.1</td>
<td align="center"></td>
<td align="center">1.1.2</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie2&amp;sort=name">bowtie2</a></td>
<td align="center">2.3.5.1</td>
<td align="center">2.3.3.1<br />2.3.4.1</td>
<td align="center"></td>
<td align="center">2.2.7<br />2.3.4.3</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Gaius-Augustus/BRAKER">BRAKER</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Gaius-Augustus/BRAKER">BRAKER2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.5</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/genome/breakdancer">breakdancer</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=breakdancer_max&amp;sort=name">breakdancer_max</a></td>
<td align="center"></td>
<td align="center">1.1.2_2013_03_08</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/pace/bricks">BRICKS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20180416</td>
</tr>
<tr class="even">
<td align="center"><a href="https://code.google.com/archive/p/bsmap/">bsmap</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bsmap&amp;sort=name">bsmap</a></td>
<td align="center"></td>
<td align="center">2.90</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker/BS_Seeker.html">bsseeker</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/BSSeeker/BSseeker2#bs-seeker2">bsseeker2</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bs_seeker2&amp;sort=name">bs_seeker2</a></td>
<td align="center"></td>
<td align="center">2.1.3</td>
<td align="center">2.1.3</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://busco.ezlab.org/">BUSCO</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=busco&amp;sort=name">busco</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.22<br />3.0.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/lh3/bwa">bwa</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bwa&amp;sort=name">bwa</a></td>
<td align="center">0.7.17</td>
<td align="center">0.7.17</td>
<td align="center">0.7.17</td>
<td align="center">0.7.13</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bxlab/bx-python">bx-python</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bx_python&amp;sort=name">bx_python</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.8.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.bzip.org/">bzip2</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bz2file&amp;sort=name">bz2file</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.6-intel-2015.3.187</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/ComparativeGenomicsToolkit/cactus">Cactus</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2019.03.01</td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/CAMERA.html">CAMERA</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=camera&amp;sort=name">camera</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/marbl/canu">Canu</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=canu&amp;sort=name">canu</a></td>
<td align="center">1.9<br />2<br />2.0t</td>
<td align="center">1.6</td>
<td align="center"></td>
<td align="center">1.3<br />1.8<br />1.8<br />1.9<br />2.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://seq.cs.iastate.edu/cap3.html">cap3</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cap3&amp;sort=name">cap3</a></td>
<td align="center"></td>
<td align="center">20130608</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cardinalmsi.org/">Cardinal</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cardinal&amp;sort=name">cardinal</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dutilh/CAT">CAT</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cat_bins&amp;sort=name">cat_bins</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/weizhongli/cdhit">cd-hit</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cdhit&amp;sort=name">cdhit</a></td>
<td align="center"></td>
<td align="center">4.6.8</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/yanailab/celseq2">CEL-Seq 2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.5.3.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page">Celera / wgs-assembler</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">8.3rc2</td>
<td align="center"></td>
<td align="center">8.3rc2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://hub.docker.com/r/centos/tools/">centos</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">6.9-tinaroo2018<br />rocks6_roll_modules<br />rocks7_roll_modules</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://hub.docker.com/r/centos/tools/">centos6</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">9</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Ecogenomics/CheckM/wiki">checkm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.11</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Ecogenomics/CheckM/wiki">checkm-genome</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.13</td>
</tr>
<tr class="even">
<td align="center"><a href="https://genome.sph.umich.edu/wiki/ChunkChromosome">chunkchromosome</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2014-05-27</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.clustal.org/clustal2/">clustalw</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=clustalw&amp;sort=name">clustalw</a></td>
<td align="center"></td>
<td align="center">2.1</td>
<td align="center">2.1</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://tools.bsc.es/doc/clusteringsuite_manual_0.pdf">clusteringsuite</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.6.8</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cmake.org/">cmake</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cmake&amp;sort=name">cmake</a></td>
<td align="center">3.16.2<br />3.18.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1.3-GCC-4.9.2<br />3.15.0-gnu-7.2.0<br />3.2.3-intel-2015.3.187<br />3.9.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/etal/cnvkit">CNVkit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.9.4a0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/biocoder/cpc">Coding Potential Calculator</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/biocoder/CPC2">Coding Potential Calculator 2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/coreutils/">coreutils</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=coreutils&amp;sort=name">coreutils</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">8.22</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/rna-cpat/files/">CPAT: Coding Potential Assessment Tool</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sourceforge.net/projects/cppunit/">cppunit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.14.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cube-score-p-scalasca">cube</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.4<br />4.5</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://developer.nvidia.com/cuda-toolkit">cuda</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">10.1<br />11.0.3</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://developer.nvidia.com/cudnn">cudnn</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">7.6.5-cuda10.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://cole-trapnell-lab.github.io/cufflinks/">Cufflinks</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cufflinks&amp;sort=name">cufflinks</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://curl.se/">curl</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=curl&amp;sort=name">curl</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">7.43.0-intel-2015.3.187</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/marcelm/cutadapt">cutadapt</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cutadapt&amp;sort=name">cutadapt</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.18</td>
</tr>
<tr class="even">
<td align="center"><a href="https://cytoscape.org/">cytoscape</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cytoscape&amp;sort=name">cytoscape</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.7.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.mcs.anl.gov/research/projects/darshan/">darshan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.2.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://deeptools.readthedocs.io/en/develop/">DeepTools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=deeptools&amp;sort=name">deeptools</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.3.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/google/deepvariant">deepvariant</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://dendropy.org/">dendropy</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dendropy&amp;sort=name">dendropy</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.0.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://diaumpire.nesvilab.org/">DIA-Umpire</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dia_umpire&amp;sort=name">dia_umpire</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/bbuchfink/diamond">diamond</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=diamond&amp;sort=name">diamond</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.7.12<br />0.9.22<br />0.9.23<br />0.9.30</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/broadinstitute/discovar_de_novo">DISCOVAR de novo</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://jbloomlab.github.io/dms_tools2/">dms_tools2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.5.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/dmtcp/dmtcp">dmtcp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.5.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/cmbi/dssp">dssp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/easybuilders/easybuild">easybuild</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.3.0<br />2.3.1<br />2.6.0<br />3.6.1<br />3.8.0<br />3.9.4</td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.genomic.ch/edena.php">Edena</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=edena&amp;sort=name">edena</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.131028</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/UQ-RCC/nimrod-embedded">embedded-nimrod</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.8.6<br />1.1.0<br />1.1.4<br />1.10.3<br />1.10.5<br />1.11.0<br />1.3.0<br />1.4.0<br />1.8.0<br />1.8.1<br />1.9.0</td>
</tr>
<tr class="even">
<td align="center"><a href="http://emboss.sourceforge.net/">EMBOSS (European Molecular Biology Open Software Suite)</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=emboss&amp;sort=name">emboss</a></td>
<td align="center"></td>
<td align="center">6.6.0</td>
<td align="center"></td>
<td align="center">6.5.7</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://asia.ensembl.org/info/website/index.html">Ensembl browser</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ensembl&amp;sort=name">ensembl</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html">Ensembl Genebuild</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://erlang.org/faq/introduction.html">Erlang</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">22.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://mblab.wustl.edu/software.html">eval</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.8</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/EVidenceModeler/EVidenceModeler/releases">EVidenceModeler (EVM)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/aberer/exabayes">ExaBayes</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.4.1</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate">Exonerate</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=exonerate&amp;sort=name">exonerate</a></td>
<td align="center"></td>
<td align="center">2.2.0</td>
<td align="center"></td>
<td align="center">2.4.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/SegataLab/export2graphlan/">export2graphlan</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=export2graphlan&amp;sort=name">export2graphlan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.22</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://tools.bsc.es/extrae">Extrae</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.7.1-impi_2018.2.046<br />3.7.1-openmpi_2.0.2<br />3.7.1-openmpi_ib_1.8.4-intel</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/PacificBiosciences/pb-assembly">Falcon</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201710</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PacificBiosciences/pb-assembly">Falcon (pb-assembly)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.6<br />0.0.8</td>
</tr>
<tr class="even">
<td align="center"><a href="http://fastml.tau.ac.il/">fastml</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FASTQC</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastqc&amp;sort=name">fastqc</a></td>
<td align="center"></td>
<td align="center">0.11.4</td>
<td align="center"></td>
<td align="center">0.11.4</td>
</tr>
<tr class="even">
<td align="center"><a href="https://rajanil.github.io/fastStructure/">faststructure</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://hannonlab.cshl.edu/fastx_toolkit/">FASTX-Toolkit</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx&amp;sort=name">fastx</a></td>
<td align="center"></td>
<td align="center">0.0.14</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://hannonlab.cshl.edu/fastx_toolkit/">FASTX-Toolkit</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx&amp;sort=name">fastx</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.14</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.softberry.com/berry.phtml?topic=fgenesh_plus_plus&amp;group=help&amp;subgroup=pipelines">FgenesH++</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://animalbiosciences.uoguelph.ca/~msargol/fimpute/">FImpute</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/smith-chem-wisc/FlashLFQ">flashLFQ</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flashlfq&amp;sort=name">flashlfq</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.fltk.org/index.php">fltk</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.3.5</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/fenderglass/Flye">Flye</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flye&amp;sort=name">flye</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.4.2<br />2.6</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/fpart/">fpart</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://fpm.readthedocs.io/en/latest/">fpm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.3.3-Ruby-2.1.6</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/fraggenescan/">FragGeneScan</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fraggenescan&amp;sort=name">fraggenescan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/ekg/freebayes">freebayes</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=freebayes&amp;sort=name">freebayes</a></td>
<td align="center"></td>
<td align="center">0.9.21</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://surfer.nmr.mgh.harvard.edu/">freesurfer</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">6.0<br />6.1dev</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://funannotate.readthedocs.io/en/latest/install.html">funannotate</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://code.google.com/archive/p/garli/">GARLI</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=garli&amp;sort=name">garli</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/broadinstitute/gatk">GATK</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gatk&amp;sort=name">gatk</a></td>
<td align="center">4.1.4.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.5<br />4.0.4.0</td>
</tr>
<tr class="even">
<td align="center"><a href="http://molevol.cmima.csic.es/castresana/Gblocks_server.html">gblocks</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.91b</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://gcc.gnu.org/">GCC</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">system</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.9.2</td>
</tr>
<tr class="even">
<td align="center"><a href="http://cnsgenomics.com/software/gcta/">gcta</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.26.0</td>
<td align="center"></td>
<td align="center">1.26.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/gnudatalanguage/gdl">GDL</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.9.9</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://genome.crg.es/software/geneid/">geneid</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.4.4</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://exon.gatech.edu/GeneMark/">Genemark</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/schatzlab/genomescope">GenomeScope</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PacificBiosciences/GenomicConsensus">GenomicConsensus (Arrow)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://git-scm.com/about">Git</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.24.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://glew.sourceforge.net/">glew</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/glfw/glfw">glfw</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.3.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sourceware.org/git/?p=glibc.git">glibc</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.17<br />2.19<br />2.23</td>
</tr>
<tr class="even">
<td align="center"><a href="http://ccb.jhu.edu/software/glimmer/">glimmer</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer&amp;sort=name">glimmer</a></td>
<td align="center"></td>
<td align="center">3.02b</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://ccb.jhu.edu/software/glimmerhmm/">glimmerhmm</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer_hmm&amp;sort=name">glimmer_hmm</a></td>
<td align="center"></td>
<td align="center">3.0.4</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/GlobalArrays/ga">Global Arrays (GA) toolkit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.7<br />5.7.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://research-pub.gene.com/gmap/">gmap</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2017-11-15</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://research-pub.gene.com/gmap/">gmap_gsnap</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20151231</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/epam/GMP">gmp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">6.0.0a</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/">GNU</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">7.2.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/libtool/manual/libtool.html">GNU libtool</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=libtool&amp;sort=name">libtool</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.4.2-GCC-4.9.2<br />2.4.5-GCC-4.9.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/software/parallel/">GNU parallel</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20191022</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20170122<br />20180322</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/gsl/">GNU scientific library</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gsl&amp;sort=name">gsl</a></td>
<td align="center">2.6</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.16<br />2.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/cbadvanced/">GNU Tools</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.69</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://geronimp.github.io/graftM/">GraftM</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.9.4</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/biobakery/graphlan">GraPhlAn</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=graphlan&amp;sort=name">graphlan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.1.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.gromacs.org/">gromacs</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gmx&amp;sort=name">gmx</a></td>
<td align="center">2018.3<br />2018.3-plumed<br />2019.3<br />2019.3-gpuvolta<br />2019.3-plumed<br />2020.1<br />2020.1-gpuvolta<br />2020.1-plumed<br />2020.3<br />2020.3-gpuvolta</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2018<br />5.1.4-intel<br />5.1.4-intelMPI</td>
</tr>
<tr class="even">
<td align="center"><a href="http://ecogenomics.github.io/GroopM/">groopm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.3.5<br />201811</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/sanger-pathogens/gubbins">gubbins</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gubbins&amp;sort=name">gubbins</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.4.1</td>
</tr>
<tr class="even">
<td align="center">guppy</td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=guppy&amp;sort=name">guppy</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.3.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://hadoop.apache.org/">Hadoop</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hadoop&amp;sort=name">hadoop</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.5.0-cdh5.3.1-native</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/vpc-ccg/haslr">HASLR</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://amos.sourceforge.net/wiki/index.php?title=Hawkeye">Hawkeye</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/SegataLab/hclust2">Hclust2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://support.hdfgroup.org/products/hdf4_tools/">hdf4</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.2.14<br />4.2.15</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.10</td>
</tr>
<tr class="even">
<td align="center"><a href="https://support.hdfgroup.org/products/hdf5_tools/">hdf5</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.10.5<br />1.10.5p<br />1.8.21<br />1.8.21p</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.8.14</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/higlass">HiGlass</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772016/">HiRise</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://daehwankimlab.github.io/hisat2/">HISAT2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hisat2&amp;sort=name">hisat2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://hmmer.org/">HMMER</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hmmer&amp;sort=name">hmmer</a></td>
<td align="center"></td>
<td align="center">3.0<br />3.1b2</td>
<td align="center"></td>
<td align="center">3.1b2<br />3.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/horovod/horovod">horovod</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.19.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/htseq/htseq">HTSeq</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=htseq&amp;sort=name">htseq</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.6.1p1<br />htseq-0.6.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.htslib.org/">HTSlib</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.9</td>
<td align="center">1.2.1<br />1.6</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/loneknightpy/idba">idba-ud</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=idba&amp;sort=name">idba</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.1.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.l3harrisgeospatial.com/Software-Technology/IDL">IDL</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">8.6<br />8.7.2<br />8.8</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler">Improved Phased Assembler (pbipa)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://eddylab.org/infernal/">Infernal</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=infernal&amp;sort=name">infernal</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://software.broadinstitute.org/software/igv/home">Integrated Genomics Viewer</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.4.14</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://interproscan-docs.readthedocs.io/en/latest/">InterProScan</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=interproscan&amp;sort=name">interproscan</a></td>
<td align="center"></td>
<td align="center">5.27-66.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/nerscadmin/IPM">ipm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.6</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/dereneaton/pyrad">iPyRAD</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.iqtree.org/">IQ-TREE</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=iqtree&amp;sort=name">iqtree</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PacificBiosciences/IsoSeq">IsoSeq3</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.2.2</td>
</tr>
<tr class="even">
<td align="center"><a href="http://mcmc-jags.sourceforge.net/">jags</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.2.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/genomecuration/JAMg">JAMg</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.oracle.com/java/">Java</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">jdk-13.33<br />jdk-8.40</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.7.0_79<br />1.8.0_45</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/GMOD/jbrowse">JBrowse</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jbrowse&amp;sort=name">jbrowse</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.genome.umd.edu/jellyfish.html">jellyfish</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jellyfish&amp;sort=name">jellyfish</a></td>
<td align="center">2.3.0</td>
<td align="center">2.2.6</td>
<td align="center"></td>
<td align="center">2.2.10</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/aidenlab/Juicebox">juicebox</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/aidenlab/juicer">Juicer</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://julialang.org/">julia</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.5.3</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.5.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://pachterlab.github.io/kallisto/about.html">Kallisto</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kallisto&amp;sort=name">kallisto</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://kat.readthedocs.io/en/latest/">KAT</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kat&amp;sort=name">kat</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dib-lab/khmer">khmer</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=khmer&amp;sort=name">khmer</a></td>
<td align="center"></td>
<td align="center">2.1.2</td>
<td align="center">2.1.2</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/refresh-bio/KMC">KMC</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://ccb.jhu.edu/software/kraken/">kraken</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken&amp;sort=name">kraken</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.1.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/DerrickWood/kraken2/wiki">kraken2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken2&amp;sort=name">kraken2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.8</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/Krona/wiki">krona</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.7.1<br />201712</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://kubernetes.io/">Kubernetes</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kubernetes&amp;sort=name">kubernetes</a></td>
<td align="center">0.2.5</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.bx.psu.edu/~rsharris/lastz/">lastz</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lastz&amp;sort=name">lastz</a></td>
<td align="center"></td>
<td align="center">1.04.00</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst">LFQ-Analyst</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://ofiwg.github.io/libfabric/">libfabric</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.8.0a1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/agordon/libgtextutils">libgtextutils</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=libgtextutils&amp;sort=name">libgtextutils</a></td>
<td align="center"></td>
<td align="center">0.7</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nongnu.org/libunwind/">libunwind</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.2.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/LINKS">LINKS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.lipidmaps.org/">Lipid Maps</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lipidmaps&amp;sort=name">lipidmaps</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://llvm.org/">llvm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">10.0.0<br />11.0.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.0.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://lmod.readthedocs.io/en/latest/">lmod</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">7.4.17</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger">Long Ranger</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.2</td>
</tr>
<tr class="even">
<td align="center"><a href="http://csg.sph.umich.edu/abecasis/mach/index.html">mach</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.18</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/macs3-project/MACS">macs</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs&amp;sort=name">macs</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2-2.1.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/macs3-project/MACS">macs2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs2&amp;sort=name">macs2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.7.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/ranwez/MACSE_V2_PIPELINES">MACSE</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://mafft.cbrc.jp/alignment/software/">mafft</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mafft&amp;sort=name">mafft</a></td>
<td align="center"></td>
<td align="center">7.313</td>
<td align="center"></td>
<td align="center">v7.407</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/WatsonLab/MAGpy">MAGpy</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.yandell-lab.org/software/maker.html">MAKER</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maker&amp;sort=name">maker</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.31.10<br />3.01.03</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.yandell-lab.org/software/maker-p.html">MAKER-P</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://cran.r-project.org/web/packages/MALDIquant/index.html">MALDIquant</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maldi_quant&amp;sort=name">maldi_quant</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/marbl/MashMap">mashmap</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/kwongj/maskrc-svg">maskrc-svg</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.5</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/alekseyzimin/masurca">MaSuRCA</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.2.4</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.mathworks.com/products/matlab.html">MATLAB</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">R2019b</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2015a<br />2017a<br />2018a<br />2018a<br />2019b<br />qcif-R2018a<br />qcif-R2018a</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/dtreichler/matserver">matserver</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2018<br />2020</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://darlinglab.org/mauve/download.html">Mauve</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mauve&amp;sort=name">mauve</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20200714</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://maven.apache.org/">maven</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.3.3</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/maxbin2/">MaxBin2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxbin2&amp;sort=name">maxbin2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.maxquant.org/">MaxQuant</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant&amp;sort=name">maxquant</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.6.10.43<br />1.6.17.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.maxquant.org/">MaxQuant (biocontainers)</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant&amp;sort=name">maxquant</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.6.10.43</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://micans.org/mcl/">mcl</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mcl&amp;sort=name">mcl</a></td>
<td align="center"></td>
<td align="center">14-137</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/nanoporetech/medaka">Medaka</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=medaka&amp;sort=name">medaka</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/voutcn/megahit">megahit</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=megahit&amp;sort=name">megahit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201710</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/merqury">Merqury</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/berkeleylab/metabat/src/master/">metabat</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201710</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/metacv/">metacv</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.3.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/DiltheyLab/MetaMaps">MetaMaps</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/metaMS.html">metaMS</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metams&amp;sort=name">metams</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://huttenhower.sph.harvard.edu/metaphlan">MetaPhlAn</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaphlan&amp;sort=name">metaphlan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.0.2</td>
</tr>
<tr class="even">
<td align="center"><a href="http://cab.spbu.ru/software/spades/">metaspades</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaspades&amp;sort=name">metaspades</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://peterbeerli.com/migrate-html5/index.html">MIGRATE</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://gatb.inria.fr/software/minia/">minia</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minia&amp;sort=name">minia</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://genome.sph.umich.edu/wiki/Minimac4">Minimac4</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/lh3/minimap2">minimap2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minimap2&amp;sort=name">minimap2</a></td>
<td align="center">2.17</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sourceforge.net/p/mira-assembler/wiki/Home/">mira</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mira&amp;sort=name">mira</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.9.6</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nature.com/articles/nbt1394#Sec11">mirdeep</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/rajewsky-lab/mirdeep2">mirdeep2</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mirdeep2&amp;sort=name">mirdeep2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.7</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/yarden/MISO">miso</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.5.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://mixomics.org/">mixOmics</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mixomics&amp;sort=name">mixomics</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/tseemann/mlst">MLST</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mlst&amp;sort=name">mlst</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.19.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/compomics/moFF">moFF</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=proteomics_moff&amp;sort=name">proteomics_moff</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.mono-project.com/">Mono</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">6.10.0.104-0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cwenger.github.io/Morpheus/">Morpheus</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=morpheus&amp;sort=name">morpheus</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.mothur.org/">Mothur</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mothur&amp;sort=name">mothur</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.mpfr.org/">mpfr</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1.2</td>
</tr>
<tr class="even">
<td align="center">mpiBLAST</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/LLNL/mpiP">mpiP (mpi profiling library)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.4.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/NBISweden/MrBayes">MrBayes</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mrbayes&amp;sort=name">mrbayes</a></td>
<td align="center"></td>
<td align="center">3.2.6</td>
<td align="center">3.2.6</td>
<td align="center">3.2.3<br />3.2.6</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/MSGFplus.html">MS-GF+</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msgfplus&amp;sort=name">msgfplus</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/MSnbase.html">MSnbase readMSData</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msnbase&amp;sort=name">msnbase</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/joseph07/multisnv/wiki/Home">multisnv</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201801</td>
</tr>
<tr class="even">
<td align="center"><a href="http://mummer.sourceforge.net/">MUMmer</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mummer&amp;sort=name">mummer</a></td>
<td align="center"></td>
<td align="center">3.23</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.drive5.com/muscle/">MUSCLE</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=muscle&amp;sort=name">muscle</a></td>
<td align="center"></td>
<td align="center">3.8.1551</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://mvapich.cse.ohio-state.edu/">MVAPICH</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://software.opensuse.org/package/mxml">mxml</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.9</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nag.com/">nag</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">fll6i26dcl<br />fll6i26dcl-mkl<br />nll6i27dbl<br />nll6i27dbl-i8<br />nll6i27dbl-mkl</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/jts/nanopolish">Nanopolish</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=nanopolish&amp;sort=name">nanopolish</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nasm.us/">NASM</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.11.08-intel-2015.3.187</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/NVIDIA/nccl">NCCL</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.5.6-1+cuda10.1<br />2.6.4-1+cuda10.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://gitlab.com/remikz/nccmp">nccmp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.8.5.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ncl.ucar.edu/index.shtml">NCL</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">6.6.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.ncl.ucar.edu/index.shtml">NCL</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">6.3.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://nco.sourceforge.net/">NCO</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.7.7<br />4.9.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://meteora.ucsd.edu/~pierce/ncview_home_page.html">Ncview</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.7</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.7</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/xiaochuanle/NECAT">NECAT</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.unidata.ucar.edu/software/netcdf/">NetCDF</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=netcdf&amp;sort=name">netcdf</a></td>
<td align="center">4.6.3<br />4.6.3-i8r8<br />4.6.3p<br />4.7.1<br />4.7.1p<br />4.7.3<br />4.7.3p<br />4.7.4<br />4.7.4p</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.6.2<br />4.3.2<br />4.4.1-gnu<br />4.4.1-intel</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/neurodebian/neurodebian">neurodebian</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.37.2<br />0.37.6<br />0.39.0<br />0.39.0+eddy</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Nextomics/NextDenovo">NextDenovo</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">v2.2-beta.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.nextflow.io/">nextflow</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">19.07</td>
</tr>
<tr class="even">
<td align="center"><a href="http://cibiv.github.io/NextGenMap/">nextgenmap</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Nextomics/NextPolish">NextPolish</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.1.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/ncbi/ngs">NGS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.3.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/shenlab-sinai/ngsplot">ngsplot</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ngsplot&amp;sort=name">ngsplot</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.61</td>
</tr>
<tr class="even">
<td align="center"><a href="https://nucleoatac.readthedocs.io/en/latest/">nucleoatac</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.3.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/NVlabs/nvbio">nvbio</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://developer.nvidia.com/hpc-sdk">NVIDIA HPC SDK</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20.7</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ebi.ac.uk/~zerbino/oases/">Oases</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=oases&amp;sort=name">oases</a></td>
<td align="center"></td>
<td align="center">0.2.08</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.vi-hps.org/tools/opari2.html">OPARI2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.3<br />2.0.5</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.open-mpi.org/">Open MPI</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.6<br />2.1.6-debug<br />2.1.6-mt<br />2.1.6-mt-debug<br />3.0.4<br />3.0.4-debug<br />3.1.4<br />3.1.4-debug<br />4.0.1<br />4.0.1-debug<br />4.0.2<br />4.0.2-debug<br />4.0.3<br />4.0.3-debug</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.8.4<br />2.0.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.open-mpi.org/">Open MPI (ib)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.8.4</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.open-mpi.org/">Open MPI 2 (ib)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.2<br />2.1.6</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.openms.de/">OpenMS</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=openms&amp;sort=name">openms</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper">OPLS-DA_Contrasts (w4mcorcov)</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4mcorcov&amp;sort=name">w4mcorcov</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/kishwarshafin/pepper">P.E.P.P.E.R. (+ deep variant)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.1.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/TomSkelly/PacBioEDA">PacBioEDA</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">20140812</td>
</tr>
<tr class="even">
<td align="center">padb</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://icl.utk.edu/papi/software/">PAPI (Performance Application Programming Interface)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.7.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.4.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.vi-hps.org/tools/paraver.html">PARAVER</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.8.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.paraview.org/">ParaView</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.8.0<br />5.8.0-gpu<br />5.8.0-mesa</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.4.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/parsnp">Parsnp</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=parsnp&amp;sort=name">parsnp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.2<br />1.5.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PASApipeline/PASApipeline/wiki">PASA</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/NixOS/patchelf">PatchELF</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://downloadcenter.intel.com/download/19792">PDT (Processor Diagnostic Tool)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.20</td>
</tr>
<tr class="even">
<td align="center"><a href="http://compomics.github.io/projects/peptide-shaker.html">Peptide Shaker</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=peptideshaker&amp;sort=name">peptideshaker</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://percolator.ms/">Percolator</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=percolator&amp;sort=name">percolator</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.php.net/">PHP</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.6.40</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://broadinstitute.github.io/picard/">PICARD</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picard&amp;sort=name">picard</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.18.2<br />2.3.0</td>
</tr>
<tr class="even">
<td align="center"><a href="http://picrust.github.io/picrust/">PICRUSt</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picrust&amp;sort=name">picrust</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.1.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/picrust/picrust2/wiki">PICRUSt 2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.0_b</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.drive5.com/piler/">PILER-DF</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/broadinstitute/pilon/wiki">Pilon</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pilon&amp;sort=name">pilon</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://zzz.bwh.harvard.edu/plink/">plink</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=plink&amp;sort=name">plink</a></td>
<td align="center"></td>
<td align="center">1.07</td>
<td align="center"></td>
<td align="center">1.9</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://atgu.mgh.harvard.edu/plinkseq/index.shtml">plinkseq</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.10</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://parallel-netcdf.github.io/">PnetCDF</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.11.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/matsen/pplacer">pplacer</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">v1.1.alpha17</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/wtsi-hpag/PretextView">PretextView / Pretext Suite</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://prinseq.sourceforge.net/">PRINSEQ</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prinseq&amp;sort=name">prinseq</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.genabel.org/packages/ProbABEL">probabel</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.5.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/rsilvabioinfo/ProbMetab">ProbMetab Tool</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/hyattpd/Prodigal">Prodigal</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prodigal&amp;sort=name">prodigal</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">201804</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/tseemann/prokka">Prokka</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prokka&amp;sort=name">prokka</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.12<br />1.13<br />1.14.5</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dfguan/purge_dups">Purge_Dups</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/mroachawri/purge_haplotigs/src/master/">PurgeHaplotigs</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://pylib.readthedocs.io/en/1.4.3/index.html">pylibs</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://ipyrad.readthedocs.io/en/latest/">PyRAD</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.0.66</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/pysam-developers/pysam">pysam</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pysam&amp;sort=name">pysam</a></td>
<td align="center"></td>
<td align="center">0.13</td>
<td align="center">0.13</td>
<td align="center">0.8.4</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.python.org/">python</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.7.14</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.python.org/">python2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.7.16<br />2.7.17</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.python.org/">python2-as-python</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.python.org/">python3</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.7.4<br />3.8.5</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.4.9-centos<br />3.6.2-sdsc</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.python.org/">python3-as-python</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://pytorch.org/">PyTorch</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.4.0a0<br />1.5.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://qiime.org/">QIIME</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime&amp;sort=name">qiime</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://qiime.org/">QIIME</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime&amp;sort=name">qiime</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.9.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://qiime2.org/">QIIME2.0</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime2&amp;sort=name">qiime2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2018.6.0<br />2019.4</td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioinf.spbau.ru/quast">QUAST</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=quast&amp;sort=name">quast</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.0.2</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/mahulchak/quickmerge">Quickmerge</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.r-project.org/">R</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.6.1<br />4.0.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.4.3<br />3.5.0<br />3.5.0-gnu<br />3.5.0-intel<br />3.6.0-container<br />3.6.0+tidyverse+rstan+rtdists<br />3.6.1-container<br />3.6.1+CrocSim<br />3.6.1+exSTRa<br />3.6.1+Monocle3<br />3.6.1+Monocle3+Giotto<br />3.6.1+Seurat<br />3.6.3+Shawan<br />4.0.0<br />4.0.0+Spatial<br />4.0.2+Spatial<br />4.0.2+tradeSeq+Seurat</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.rabbitmq.com/">RabbitMQ server</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.8.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/isovic/racon">Racon</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=racon&amp;sort=name">racon</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/RAILS">RAILS/Cobbler</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://ratt.sourceforge.net/">RATT</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/comprna/RATTLE">RATTLE</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/rawtools/51e4aa55ea04">Raw Tools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rawtools&amp;sort=name">rawtools</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sco.h-its.org/exelixis/web/software/raxml/index.html">RAxML</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=raxml&amp;sort=name">raxml</a></td>
<td align="center"></td>
<td align="center">8.2.11</td>
<td align="center"></td>
<td align="center">8.2.12</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dparks1134/RefineM">RefineM</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.24</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/simoncchu/REPdenovo">REPdenovo</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.repeatmasker.org/">RepeatMasker</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repeat_masker&amp;sort=name">repeat_masker</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.0.9_p2</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.repeatmasker.org/RepeatModeler/">RepeatModeler</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.1</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Dfam-consortium/RepeatModeler">RepeatModeler2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.yandell-lab.org/software/repeatrunner.html">RepeatRunner</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://urgi.versailles.inra.fr/Tools/REPET">REPET</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repet&amp;sort=name">repet</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cran.r-project.org/web/packages/rjags/index.html">rjags</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.5.1<br />4.6.2–R-3.4.4</td>
</tr>
<tr class="even">
<td align="center"><a href="https://deweylab.github.io/RSEM/">RSEM</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rsem&amp;sort=name">rsem</a></td>
<td align="center"></td>
<td align="center">1.3.0<br />1.3.3</td>
<td align="center"></td>
<td align="center">1.3.3</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://rseqc.sourceforge.net/">rseqc</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rseqc&amp;sort=name">rseqc</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.6.4</td>
</tr>
<tr class="even">
<td align="center"><a href="https://rstudio.com/">Rstudio</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0.143<br />1.1.456-3.4.4<br />1.1.456-3.5.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ruby-lang.org/en/downloads/">Ruby</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ruby&amp;sort=name">ruby</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.6</td>
</tr>
<tr class="even">
<td align="center"><a href="https://combine-lab.github.io/salmon/">Salmon</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=salmon&amp;sort=name">salmon</a></td>
<td align="center">1.1.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/GregoryFaust/samblaster">samblaster</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samblaster&amp;sort=name">samblaster</a></td>
<td align="center"></td>
<td align="center">0.1.24</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.htslib.org/">SAMTools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samtools&amp;sort=name">samtools</a></td>
<td align="center">1.1<br />1.9</td>
<td align="center">0.1.19<br />1.6</td>
<td align="center">0.1.19<br />1.6</td>
<td align="center">0.1.17<br />1.10<br />1.2<br />1.3<br />1.6<br />1.9</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.sas.com/en_us/software/foundation.html">SAS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">9.4</td>
</tr>
<tr class="even">
<td align="center"><a href="http://compbio.cs.toronto.edu/hapsembler/scarpa.html">Scarpa</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.241</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.scipy.org/">SciPy</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=scipy&amp;sort=name">scipy</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.7</td>
</tr>
<tr class="even">
<td align="center"><a href="https://scons.org/">SCons</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://pypi.python.org/pypi/screed">screed</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=screed&amp;sort=name">screed</a></td>
<td align="center"></td>
<td align="center">1.0</td>
<td align="center">1.0</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://compomics.github.io/projects/searchgui.html">Search GUI</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=searchgui&amp;sort=name">searchgui</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/shenwei356/seqkit">SeqKit</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=seqkit&amp;sort=name">seqkit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.13.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/drostlab/seqstats">seqstats</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/trinityrnaseq/seqtk-trinity">seqtk-trinity</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.2</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">shapeit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">r904</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/najoshi/sickle">sickle</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sickle&amp;sort=name">sickle</a></td>
<td align="center"></td>
<td align="center">1.33</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.bryntum.com/products/siesta/">Siesta</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.1-b4<br />4.1.5</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://singularity.lbl.gov/">singularity</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.1<br />2.3.1<br />2.4.0<br />2.4.2<br />2.5.1<br />2.5.2<br />3.5.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/relipmoc/skewer">skewer</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.1.127</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.pacb.com/support/software-downloads/">SMRT Link / Analysis</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">smrtlink_5.0.1<br />smrtlink_5.1<br />smrtlink_6.0<br />smrtlink_7.0<br />smrtlink_9.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://snakemake.readthedocs.io/en/stable/">Snakemake</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.6</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/KorfLab/SNAP">SNAP</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snap&amp;sort=name">snap</a></td>
<td align="center"></td>
<td align="center">0.15</td>
<td align="center"></td>
<td align="center">2006-07-28</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/PMMAraujo/snappy">SNAPPy</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.1.2-GCC-4.9.2</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://lowelab.ucsc.edu/snoscan/">snoscan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://snpeff.sourceforge.net/">snpeff</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snpeff&amp;sort=name">snpeff</a></td>
<td align="center"></td>
<td align="center">4.3t</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center">soap</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.21</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://i.cs.hku.hk/2bwt-tools/soap3/index.html">soap3-gpu</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">r146</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/aquaskyline/SOAPdenovo2">SOAPdenovo2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.41</td>
<td align="center">r240</td>
<td align="center"></td>
<td align="center">240</td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/soapsnp/">SOAPsnp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.03</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/biocore/sortmerna">sortmerna</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sortmerna&amp;sort=name">sortmerna</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.0.3<br />4.0.0</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/ablab/spades">SPAdes</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=spades&amp;sort=name">spades</a></td>
<td align="center"></td>
<td align="center">3.11.1</td>
<td align="center"></td>
<td align="center">3.11.1<br />3.9.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi">Splign</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc">sra-tools</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.8.2-5</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://catchenlab.life.illinois.edu/stacks/">Stacks</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=stacks&amp;sort=name">stacks</a></td>
<td align="center"></td>
<td align="center">1.48<br />2.1</td>
<td align="center"></td>
<td align="center">1.37</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/alexdobin/STAR">star</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=star&amp;sort=name">star</a></td>
<td align="center"></td>
<td align="center">2.5.3a</td>
<td align="center"></td>
<td align="center">2.5.2b<br />2.5.2b-static</td>
</tr>
<tr class="odd">
<td align="center"><a href="http://web.stanford.edu/group/pritchardlab/structure.html">structure</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=structure&amp;sort=name">structure</a></td>
<td align="center"></td>
<td align="center">2.3.4</td>
<td align="center"></td>
<td align="center">2.3.4</td>
</tr>
<tr class="even">
<td align="center"><a href="https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome">Supernova</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/hall-lab/svtyper">svtyper</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=svtyper&amp;sort=name">svtyper</a></td>
<td align="center"></td>
<td align="center">0.1.4</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/TimoLassmann/tagdust">tagdust</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tagdust&amp;sort=name">tagdust</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.33</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/tasseladmin/tassel-5-standalone">tassel</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tassel&amp;sort=name">tassel</a></td>
<td align="center"></td>
<td align="center">4.3</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.cs.uoregon.edu/research/tau/home.php">TAU</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.23</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.tensorflow.org/resources/tools">TensorFlow</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.0.0<br />2.1.0<br />2.3.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/compomics/ThermoRawFileParser">ThermoRawFileParser</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=thermo_raw_file_converter&amp;sort=name">thermo_raw_file_converter</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/tigmint">Tigmint</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://ccb.jhu.edu/software/tophat/index.shtml">tophat</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tophat&amp;sort=name">tophat</a></td>
<td align="center">2.1.1</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.1.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://totalview.io/">TotalView</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2019.3.14<br />2020.1.13</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/TransDecoder/TransDecoder/wiki">TransDecoder</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=transdecoder&amp;sort=name">transdecoder</a></td>
<td align="center"></td>
<td align="center">5.0.2</td>
<td align="center"></td>
<td align="center">5.5.0</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://tandem.bu.edu/trf/trf.html">trf</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">4.09</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">Trim Galore</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trim_galore&amp;sort=name">trim_galore</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.usadellab.org/cms/?page=trimmomatic">Trimmomatic</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trimmomatic&amp;sort=name">trimmomatic</a></td>
<td align="center"></td>
<td align="center">0.36</td>
<td align="center"></td>
<td align="center">0.35<br />0.39</td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki">Trinity / trinityrnaseq</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinity&amp;sort=name">trinity</a></td>
<td align="center">2.9.1</td>
<td align="center">2.5.1</td>
<td align="center"></td>
<td align="center">2.1.1<br />2.9.1</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Trinotate/Trinotate.github.io/wiki">Trinotate</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinotate&amp;sort=name">trinotate</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.2.1</td>
</tr>
<tr class="even">
<td align="center"><a href="http://eddylab.org/software.html">trnascan-se</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trnascan-se&amp;sort=name">trnascan-se</a></td>
<td align="center"></td>
<td align="center">2.0.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://hgdownload.soe.ucsc.edu">UCSC tools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ucsc&amp;sort=name">ucsc</a></td>
<td align="center"></td>
<td align="center">357</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.openucx.org/">UCX (Unified Communication X)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.6.1<br />1.8.0<br />1.8.1<br />1.9.0</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.unidata.ucar.edu/software/udunits/">UDUNITS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.12.11<br />2.2.26</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/rrwick/Unicycler">Unicycler</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=unicycler&amp;sort=name">unicycler</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.unixodbc.org/utilities.html">unixODBC</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.3.7</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/VALET">Valet</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=valet&amp;sort=name">valet</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/cc2qe/vawk">vawk</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">0.0.1</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://vcftools.github.io/index.html">VCFTools</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vcftools&amp;sort=name">vcftools</a></td>
<td align="center"></td>
<td align="center">0.1.15</td>
<td align="center"></td>
<td align="center">0.1.14</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ebi.ac.uk/~zerbino/velvet/">Velvet</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=velvet&amp;sort=name">velvet</a></td>
<td align="center">1.2.10</td>
<td align="center">1.2.10</td>
<td align="center"></td>
<td align="center">1.2.10</td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.tbi.univie.ac.at/RNA/documentation.html">ViennaRNA</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vienna&amp;sort=name">vienna</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.2.4</td>
</tr>
<tr class="odd">
<td align="center"><a href="https://wci.llnl.gov/simulation/computer-codes/visit/">VisIt</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.1.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/torognes/vsearch">vsearch</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vsearch&amp;sort=name">vsearch</a></td>
<td align="center"></td>
<td align="center">2.6.0</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/adigenova/wengan">Wengan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html">WindowMasker</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=windowmasker&amp;sort=name">windowmasker</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.winehq.org/">Wine</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">5.0.2</td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.ebi.ac.uk/~birney/wise2/">wise</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">2.4.1</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://workflow4metabolomics.org/">Workflow 4 Metabolomics</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4m&amp;sort=name">w4m</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/ruanjue/wtdbg2">WTDBG2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.wxwidgets.org/">wxWidgets</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.0.4</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xcms&amp;sort=name">xcms</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://xerces.apache.org/xerces-c/">Xerces-C++ XML parser</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.2.2</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/AustralianBioCommons/xml4ena">xml4ena</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/freedesktop/xorg-macros">xorg-macros</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.17-GCC-4.7.2</td>
</tr>
<tr class="even">
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem/29e2df3edf11">xtandem</a></td>
<td align="center">●</td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xtandem&amp;sort=name">xtandem</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://zlib.net/">zlib</a></td>
<td align="center"></td>
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=zlib&amp;sort=name">zlib</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.2.11-gnu-7.1.0</td>
</tr>
<tr class="even">
<td align="center">zstd</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.4.3</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
</tbody>
</table>
<p></font></p>
<hr />
<hr />
</div>
<div id="tool-information-1" class="section level2">
<h2>Tool information</h2>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-02-12</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>The <em>Primary purpose</em> column categorises the tools by purpose, using an <a href="https://github.com/edamontology/edamontology">EDAM</a> <em>topic</em> where possible.</li>
<li>More information about a tool can be found by following the link in the <a href="https://bio.tools/">bio.tools</a> column.</li>
<li>When a tool has been containerised to allow for easier installation on any compute infrastructure, a link to the containerised software that can be downloaded is shown in the <a href="https://biocontainers.pro/#/">BioContainers</a> column.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<p><font size="-1.5"></p>
<table>
<colgroup>
<col width="40%" />
<col width="14%" />
<col width="20%" />
<col width="23%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Tool / workflow name</th>
<th align="center">Primary purpose (EDAM, if available)</th>
<th align="center">bio.tools</th>
<th align="center">BioContainers</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center"><a href="https://github.com/aidenlab/3d-dna">3D-DNA</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/tseemann/abricate">ABRicate</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/ABRicate">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/abricate">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/abyss">ABySS</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/ABySS">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/abyss">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/tanghaibao/jcvi/wiki/ALLMAPS">ALLMAPS</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://software.broadinstitute.org/allpaths-lg/blog/">allpathslg</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://amos.sourceforge.net/wiki/index.php/AMOS">amos</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/amos">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/amos">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://bioinformatics.ua.pt/software/anaconda/">anaconda</a></td>
<td align="center">Gene structure</td>
<td align="center"><a href="https://bio.tools/anaconda">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.popgen.dk/angsd/index.php/ANGSD">ANGSD</a></td>
<td align="center">Population genomics</td>
<td align="center"><a href="https://bio.tools/angsd">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/angsd">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://ant.apache.org/">ant</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/ant">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/GMOD/Apollo">Apollo</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/apollo">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/apollo">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://arimagenomics.com/assembly">arima mapping</a></td>
<td align="center">Chromosome conformation capture</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.arm.com/products/development-tools/server-and-hpc/forge">arm-forge</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center">arm-reports</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/sanger-pathogens/Artemis">Artemis</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/artemis">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/artemis">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.aspera.com/en/">Aspera</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Gaius-Augustus/Augustus">Augustus</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/augustus">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/augustus">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/autoconf/">Autoconf</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/perl-config-autoconf">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/software/automake/">Automake</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/hall-lab/bamkit">bamkit</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bamm-project.org/">bamm</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/bamm">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/pezmaster31/bamtools">bamtools</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/bamtools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bamtools">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/workflow4metabolomics/batch_correction">batchcorrection</a></td>
<td align="center">Metabolomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/batchcorrection">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bazel.build/">bazel</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/slaclab/bbcp">bbcp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://jgi.doe.gov/data-and-tools/bbtools/">bbmap</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/bbmap">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.htslib.org/">BCFtools</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/bcftools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bcftools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://faculty.washington.edu/browning/beagle/beagle.html">Beagle</a></td>
<td align="center">Population genetics</td>
<td align="center"><a href="https://bio.tools/beagle">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/beagle">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/beagle-dev/beagle-lib">beagle-lib</a></td>
<td align="center">Population genomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/beagle-lib">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/beast-dev/beast-mcmc">BEAST</a></td>
<td align="center">Phylogenetics</td>
<td align="center"><a href="https://bio.tools/beast">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/beast">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/CompEvol/beast2">BEAST2</a></td>
<td align="center">Phylogenetics</td>
<td align="center"><a href="https://bio.tools/beast2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/beast2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/arq5x/bedtools2">BEDTools</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/bedtools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bedtools">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/software/binutils/">binutils</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.bioconductor.org/">bioconductor</a></td>
<td align="center"></td>
<td align="center"><a href="https://bio.tools/bioconductor">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Bio-Linux">biolinux</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bionanogenomics.com/support-page/bionano-access-software/">Bionano Access</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioperl.org/">bioperl</a></td>
<td align="center">Programming language / platform</td>
<td align="center"><a href="https://bio.tools/bioperl">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://biopython.org/">biopython</a></td>
<td align="center">Programming language / platform</td>
<td align="center"><a href="https://bio.tools/Biopython">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/FelixKrueger/Bismark">bismark</a></td>
<td align="center">Epigenomics</td>
<td align="center"><a href="https://bio.tools/bismark">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bismark">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/blast">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/blast">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST+</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/ncbi-blast-plus">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST2</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://genome.ucsc.edu/cgi-bin/hgBlat">blat</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/blat">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/blat">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/blobtoolkit/blobtools2">BlobToolKit</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/BlobToolKit">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/DRL/blobtools">BlobTools1</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/BlobToolKit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/blobtools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.boost.org/">Boost</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bowtie-bio.sourceforge.net/index.shtml">Bowtie</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/bowtie">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bowtie">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie2</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/bowtie2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bowtie2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Gaius-Augustus/BRAKER">BRAKER</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/braker1">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/braker">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Gaius-Augustus/BRAKER">BRAKER2</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/braker2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/genome/breakdancer">breakdancer</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/breakdancer">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/breakdancer">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/pace/bricks">BRICKS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://code.google.com/archive/p/bsmap/">bsmap</a></td>
<td align="center">Epigenomics</td>
<td align="center"><a href="https://bio.tools/bsmap">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bsmap">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker/BS_Seeker.html">bsseeker</a></td>
<td align="center">Epigenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/BSSeeker/BSseeker2#bs-seeker2">bsseeker2</a></td>
<td align="center">Epigenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bs-seeker2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://busco.ezlab.org/">BUSCO</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/busco">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/busco">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/lh3/bwa">bwa</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/bwa">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bwa">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bxlab/bx-python">bx-python</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/bx-python">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bx-python">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.bzip.org/">bzip2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bz2file">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/ComparativeGenomicsToolkit/cactus">Cactus</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cactus">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/CAMERA.html">CAMERA</a></td>
<td align="center">Metabolomics</td>
<td align="center"><a href="https://bio.tools/camera">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bioconductor-camera">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/marbl/canu">Canu</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/canu">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/canu">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://seq.cs.iastate.edu/cap3.html">cap3</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/cap3">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cap3">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cardinalmsi.org/">Cardinal</a></td>
<td align="center">Bioimaging</td>
<td align="center"><a href="https://bio.tools/cardinal">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bioconductor-cardinal">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dutilh/CAT">CAT</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/weizhongli/cdhit">cd-hit</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/cd-hit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cd-hit">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/yanailab/celseq2">CEL-Seq 2</a></td>
<td align="center">Transcriptomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page">Celera / wgs-assembler</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/cabog">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/wgs-assembler">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://hub.docker.com/r/centos/tools/">centos</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://hub.docker.com/r/centos/tools/">centos6</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Ecogenomics/CheckM/wiki">checkm</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/checkm">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/checkm-genome">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Ecogenomics/CheckM/wiki">checkm-genome</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/checkm">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/checkm-genome">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://genome.sph.umich.edu/wiki/ChunkChromosome">chunkchromosome</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.clustal.org/clustal2/">clustalw</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/clustalw">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/clustalw">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://tools.bsc.es/doc/clusteringsuite_manual_0.pdf">clusteringsuite</a></td>
<td align="center">Statistics and probability</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cmake.org/">cmake</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/etal/cnvkit">CNVkit</a></td>
<td align="center">DNA structural variation</td>
<td align="center"><a href="https://bio.tools/cnvkit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cnvkit">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/biocoder/cpc">Coding Potential Calculator</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/cpc">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/biocoder/CPC2">Coding Potential Calculator 2</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/cpc2">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/coreutils/">coreutils</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/coreutils">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/rna-cpat/files/">CPAT: Coding Potential Assessment Tool</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cpat">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sourceforge.net/projects/cppunit/">cppunit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cube-score-p-scalasca">cube</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://developer.nvidia.com/cuda-toolkit">cuda</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://developer.nvidia.com/cudnn">cudnn</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://cole-trapnell-lab.github.io/cufflinks/">Cufflinks</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/cufflinks">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cufflinks">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://curl.se/">curl</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/curl">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/marcelm/cutadapt">cutadapt</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/cutadapt">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cutadapt">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://cytoscape.org/">cytoscape</a></td>
<td align="center">Multiomics</td>
<td align="center"><a href="https://bio.tools/cytoscape">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/cytoscape">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.mcs.anl.gov/research/projects/darshan/">darshan</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://deeptools.readthedocs.io/en/develop/">DeepTools</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/deeptools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/deeptools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/google/deepvariant">deepvariant</a></td>
<td align="center">Variant Calling</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/deepvariant">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://dendropy.org/">dendropy</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/dendropy">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://diaumpire.nesvilab.org/">DIA-Umpire</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/DIA-Umpire">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/dia_umpire">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/bbuchfink/diamond">diamond</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/diamond">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/diamond">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/broadinstitute/discovar_de_novo">DISCOVAR de novo</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/discovar_de_novo">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/discovardenovo">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://jbloomlab.github.io/dms_tools2/">dms_tools2</a></td>
<td align="center">Genetic variation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/dmtcp/dmtcp">dmtcp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/cmbi/dssp">dssp</a></td>
<td align="center">Protein folding, stability and design</td>
<td align="center"><a href="https://bio.tools/dssp">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/dssp">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/easybuilders/easybuild">easybuild</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.genomic.ch/edena.php">Edena</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/edena">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/edena">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/UQ-RCC/nimrod-embedded">embedded-nimrod</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://emboss.sourceforge.net/">EMBOSS (European Molecular Biology Open Software Suite)</a></td>
<td align="center">Molecular biology</td>
<td align="center"><a href="https://bio.tools/emboss">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://asia.ensembl.org/info/website/index.html">Ensembl browser</a></td>
<td align="center">Data visualisation</td>
<td align="center"><a href="https://bio.tools/ensembl">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html">Ensembl Genebuild</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://erlang.org/faq/introduction.html">Erlang</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://mblab.wustl.edu/software.html">eval</a></td>
<td align="center">Quality assurance</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/EVidenceModeler/EVidenceModeler/releases">EVidenceModeler (EVM)</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/EvidenceModeler">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/evidencemodeler">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/aberer/exabayes">ExaBayes</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate">Exonerate</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/exonerate">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/exonerate">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/SegataLab/export2graphlan/">export2graphlan</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/export2graphlan">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://tools.bsc.es/extrae">Extrae</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/PacificBiosciences/pb-assembly">Falcon</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pb-assembly">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PacificBiosciences/pb-assembly">Falcon (pb-assembly)</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pb-assembly">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://fastml.tau.ac.il/">fastml</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/fastml">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/fastml">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FASTQC</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/fastqc">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/fastqc">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://rajanil.github.io/fastStructure/">faststructure</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/faststructure">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/faststructure">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://hannonlab.cshl.edu/fastx_toolkit/">FASTX-Toolkit</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/fastx-toolkit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/fastx_toolkit">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://hannonlab.cshl.edu/fastx_toolkit/">FASTX-Toolkit</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/fastx-toolkit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/fastx_toolkit">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.softberry.com/berry.phtml?topic=fgenesh_plus_plus&amp;group=help&amp;subgroup=pipelines">FgenesH++</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/fgenesh">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://animalbiosciences.uoguelph.ca/~msargol/fimpute/">FImpute</a></td>
<td align="center">Population genomics</td>
<td align="center"><a href="https://bio.tools/fimpute">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/smith-chem-wisc/FlashLFQ">flashLFQ</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/flashlfq">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.fltk.org/index.php">fltk</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/fenderglass/Flye">Flye</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/Flye">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/flye">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/fpart/">fpart</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://fpm.readthedocs.io/en/latest/">fpm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/fraggenescan/">FragGeneScan</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/fraggenescan">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/fraggenescan">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/ekg/freebayes">freebayes</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/freebayes">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/freebayes">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://surfer.nmr.mgh.harvard.edu/">freesurfer</a></td>
<td align="center">Bioimaging</td>
<td align="center"><a href="https://bio.tools/freesurfer">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://funannotate.readthedocs.io/en/latest/install.html">funannotate</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/funannotate">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://code.google.com/archive/p/garli/">GARLI</a></td>
<td align="center">Phylogenetics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/garli">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/broadinstitute/gatk">GATK</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/gatk">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gatk4">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://molevol.cmima.csic.es/castresana/Gblocks_server.html">gblocks</a></td>
<td align="center">Phylogenetics</td>
<td align="center"><a href="https://bio.tools/gblocks">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gblocks">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://gcc.gnu.org/">GCC</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://cnsgenomics.com/software/gcta/">gcta</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/gcta">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gcta">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/gnudatalanguage/gdl">GDL</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://genome.crg.es/software/geneid/">geneid</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/geneid">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://exon.gatech.edu/GeneMark/">Genemark</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/genemark">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/schatzlab/genomescope">GenomeScope</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/GenomeScope_2.0">●</a></td>
<td align="center"><a href="https://biocontainers.pro/tools/genomescope2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PacificBiosciences/GenomicConsensus">GenomicConsensus (Arrow)</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/genomicconsensus">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://git-scm.com/about">Git</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://glew.sourceforge.net/">glew</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/glfw/glfw">glfw</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sourceware.org/git/?p=glibc.git">glibc</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://ccb.jhu.edu/software/glimmer/">glimmer</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/glimmer">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/glimmer">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://ccb.jhu.edu/software/glimmerhmm/">glimmerhmm</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/glimmer-hmm">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/glimmerhmm">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/GlobalArrays/ga">Global Arrays (GA) toolkit</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://research-pub.gene.com/gmap/">gmap</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/gmap">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gmap">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://research-pub.gene.com/gmap/">gmap_gsnap</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/gmap">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gmap">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/epam/GMP">gmp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/">GNU</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/libtool/manual/libtool.html">GNU libtool</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.gnu.org/software/parallel/">GNU parallel</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/parallel">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.gnu.org/software/gsl/">GNU scientific library</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/cbadvanced/">GNU Tools</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://geronimp.github.io/graftM/">GraftM</a></td>
<td align="center">Meta-Omics</td>
<td align="center"><a href="https://bio.tools/graftm">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/graftm">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/biobakery/graphlan">GraPhlAn</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/graphlan">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.gromacs.org/">gromacs</a></td>
<td align="center">Molecular dynamics</td>
<td align="center"><a href="https://bio.tools/gromacs">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gromacs">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://ecogenomics.github.io/GroopM/">groopm</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/groopm">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/sanger-pathogens/gubbins">gubbins</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/gubbins">●</a></td>
</tr>
<tr class="even">
<td align="center">guppy</td>
<td align="center">Sequencing</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://hadoop.apache.org/">Hadoop</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/vpc-ccg/haslr">HASLR</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/haslr">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://amos.sourceforge.net/wiki/index.php?title=Hawkeye">Hawkeye</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/hawkeye">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/SegataLab/hclust2">Hclust2</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/hclust2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://support.hdfgroup.org/products/hdf4_tools/">hdf4</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://support.hdfgroup.org/products/hdf5_tools/">hdf5</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/higlass">HiGlass</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772016/">HiRise</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://daehwankimlab.github.io/hisat2/">HISAT2</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/hisat2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/hisat2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://hmmer.org/">HMMER</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/hmmer3">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/hmmer">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/horovod/horovod">horovod</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/htseq/htseq">HTSeq</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/htseq">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/htseq">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.htslib.org/">HTSlib</a></td>
<td align="center">Library</td>
<td align="center"><a href="https://bio.tools/htslib">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/htslib">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/loneknightpy/idba">idba-ud</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/idba">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/idba">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.l3harrisgeospatial.com/Software-Technology/IDL">IDL</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler">Improved Phased Assembler (pbipa)</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pbipa">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://eddylab.org/infernal/">Infernal</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://software.broadinstitute.org/software/igv/home">Integrated Genomics Viewer</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/igv">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://interproscan-docs.readthedocs.io/en/latest/">InterProScan</a></td>
<td align="center">Protein sequence features</td>
<td align="center"><a href="https://bio.tools/interproscan_4">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/interproscan">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/nerscadmin/IPM">ipm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/dereneaton/pyrad">iPyRAD</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/ipyrad">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.iqtree.org/">IQ-TREE</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/iq-tree">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/iqtree">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PacificBiosciences/IsoSeq">IsoSeq3</a></td>
<td align="center">Transcriptomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/isoseq3">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://mcmc-jags.sourceforge.net/">jags</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/genomecuration/JAMg">JAMg</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.oracle.com/java/">Java</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/GMOD/jbrowse">JBrowse</a></td>
<td align="center">Data visualisation</td>
<td align="center"><a href="https://bio.tools/jbrowse">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/jbrowse">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.genome.umd.edu/jellyfish.html">jellyfish</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/jellyfish">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/jellyfish">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/aidenlab/Juicebox">juicebox</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/aidenlab/juicer">Juicer</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://julialang.org/">julia</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/julia">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://pachterlab.github.io/kallisto/about.html">Kallisto</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/kallisto">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/kallisto">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://kat.readthedocs.io/en/latest/">KAT</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/kat">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/kat">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dib-lab/khmer">khmer</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/khmer">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/khmer">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/refresh-bio/KMC">KMC</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/kmc">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/kmc">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://ccb.jhu.edu/software/kraken/">kraken</a></td>
<td align="center">Taxonomy</td>
<td align="center"><a href="https://bio.tools/kraken">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/kraken">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/DerrickWood/kraken2/wiki">kraken2</a></td>
<td align="center">Taxonomy</td>
<td align="center"><a href="https://bio.tools/kraken2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/kraken2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/Krona/wiki">krona</a></td>
<td align="center">Data visualisation</td>
<td align="center"><a href="https://bio.tools/krona">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/krona">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://kubernetes.io/">Kubernetes</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/k8">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.bx.psu.edu/~rsharris/lastz/">lastz</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/lastz">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/lastz">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst">LFQ-Analyst</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/LFQ-Analyst">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://ofiwg.github.io/libfabric/">libfabric</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/agordon/libgtextutils">libgtextutils</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/libgtextutils">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nongnu.org/libunwind/">libunwind</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/LINKS">LINKS</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/tools/links">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.lipidmaps.org/">Lipid Maps</a></td>
<td align="center">Metabolomics</td>
<td align="center"><a href="https://bio.tools/lipid_maps">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://llvm.org/">llvm</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://lmod.readthedocs.io/en/latest/">lmod</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger">Long Ranger</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/longranger">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://csg.sph.umich.edu/abecasis/mach/index.html">mach</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/mach">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/macs3-project/MACS">macs</a></td>
<td align="center">ChIP-seq</td>
<td align="center"><a href="https://bio.tools/macs">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/macs3-project/MACS">macs2</a></td>
<td align="center">ChIP-seq</td>
<td align="center"><a href="https://bio.tools/macs">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/macs2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/ranwez/MACSE_V2_PIPELINES">MACSE</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/macse">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://mafft.cbrc.jp/alignment/software/">mafft</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/MAFFT">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mafft">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/WatsonLab/MAGpy">MAGpy</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.yandell-lab.org/software/maker.html">MAKER</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/maker">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/maker">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.yandell-lab.org/software/maker-p.html">MAKER-P</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/maker-p">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://cran.r-project.org/web/packages/MALDIquant/index.html">MALDIquant</a></td>
<td align="center">Proteomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/r-maldiquant">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/marbl/MashMap">mashmap</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mashmap">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/kwongj/maskrc-svg">maskrc-svg</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/maskrc-svg">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/alekseyzimin/masurca">MaSuRCA</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/masurca">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/masurca">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.mathworks.com/products/matlab.html">MATLAB</a></td>
<td align="center"></td>
<td align="center"><a href="https://bio.tools/matlab">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/dtreichler/matserver">matserver</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://darlinglab.org/mauve/download.html">Mauve</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/mauve">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mauvealigner">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://maven.apache.org/">maven</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/maxbin2/">MaxBin2</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/maxbin">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/maxbin2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.maxquant.org/">MaxQuant</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/MaxQuant">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/maxquant">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.maxquant.org/">MaxQuant (biocontainers)</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/MaxQuant">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/maxquant">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://micans.org/mcl/">mcl</a></td>
<td align="center">Statistics and probability</td>
<td align="center"><a href="https://bio.tools/mcl">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mcl">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/nanoporetech/medaka">Medaka</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/tools/medaka">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/voutcn/megahit">megahit</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/megahit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/megahit">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/merqury">Merqury</a></td>
<td align="center">Quality assurance</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/berkeleylab/metabat/src/master/">metabat</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/MetaBAT_2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/metabat2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/metacv/">metacv</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/DiltheyLab/MetaMaps">MetaMaps</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/metamaps">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/metaMS.html">metaMS</a></td>
<td align="center">Metabolomics</td>
<td align="center"><a href="https://bio.tools/metams">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bioconductor-metams">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://huttenhower.sph.harvard.edu/metaphlan">MetaPhlAn</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/metaphlan">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/metaphlan2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://cab.spbu.ru/software/spades/">metaspades</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/metaspades">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://peterbeerli.com/migrate-html5/index.html">MIGRATE</a></td>
<td align="center">Population genomics</td>
<td align="center"><a href="https://bio.tools/MIGRATE">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/migrate-n">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://gatb.inria.fr/software/minia/">minia</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/minia">●</a></td>
<td align="center"><a href="https://biocontainers.pro/tools/minia">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://genome.sph.umich.edu/wiki/Minimac4">Minimac4</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/minimac4">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/lh3/minimap2">minimap2</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/minimap2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/minimap2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sourceforge.net/p/mira-assembler/wiki/Home/">mira</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/mira">●</a></td>
<td align="center"><a href="https://biocontainers.pro/tools/mira">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nature.com/articles/nbt1394#Sec11">mirdeep</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/mirdeep">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/rajewsky-lab/mirdeep2">mirdeep2</a></td>
<td align="center">Transcriptomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mirdeep2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/yarden/MISO">miso</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/miso">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/misopy">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://mixomics.org/">mixOmics</a></td>
<td align="center">Multiomics</td>
<td align="center"><a href="https://bio.tools/mixomics">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/r-mixomics">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/tseemann/mlst">MLST</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/mlst">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mlst">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/compomics/moFF">moFF</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/moff">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/moff">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.mono-project.com/">Mono</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mono">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cwenger.github.io/Morpheus/">Morpheus</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/morpheus">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/morpheus">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.mothur.org/">Mothur</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/Mothur">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mothur">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.mpfr.org/">mpfr</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">mpiBLAST</td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/LLNL/mpiP">mpiP (mpi profiling library)</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/NBISweden/MrBayes">MrBayes</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/mrbayes">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mrbayes">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/MSGFplus.html">MS-GF+</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/msgfplus">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/msgf_plus">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/MSnbase.html">MSnbase readMSData</a></td>
<td align="center">Metabolomics</td>
<td align="center"><a href="https://bio.tools/msnbase">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bioconductor-msnbase">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/joseph07/multisnv/wiki/Home">multisnv</a></td>
<td align="center">Variant Calling</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://mummer.sourceforge.net/">MUMmer</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/mummer">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/mummer">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.drive5.com/muscle/">MUSCLE</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/muscle">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/muscle">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://mvapich.cse.ohio-state.edu/">MVAPICH</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://software.opensuse.org/package/mxml">mxml</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nag.com/">nag</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/jts/nanopolish">Nanopolish</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/nanopolish">●</a></td>
<td align="center"><a href="https://biocontainers.pro/tools/nanopolish">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.nasm.us/">NASM</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/NVIDIA/nccl">NCCL</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://gitlab.com/remikz/nccmp">nccmp</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ncl.ucar.edu/index.shtml">NCL</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.ncl.ucar.edu/index.shtml">NCL</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://nco.sourceforge.net/">NCO</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://meteora.ucsd.edu/~pierce/ncview_home_page.html">Ncview</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/xiaochuanle/NECAT">NECAT</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.unidata.ucar.edu/software/netcdf/">NetCDF</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/neurodebian/neurodebian">neurodebian</a></td>
<td align="center">Neuroscience</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Nextomics/NextDenovo">NextDenovo</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.nextflow.io/">nextflow</a></td>
<td align="center">Workflows</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/nextflow">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://cibiv.github.io/NextGenMap/">nextgenmap</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/nextgenmap">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Nextomics/NextPolish">NextPolish</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/NextPolish">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/ncbi/ngs">NGS</a></td>
<td align="center">Knowledge discovery in databases</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/shenlab-sinai/ngsplot">ngsplot</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://nucleoatac.readthedocs.io/en/latest/">nucleoatac</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/nucleoatac">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/NVlabs/nvbio">nvbio</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://developer.nvidia.com/hpc-sdk">NVIDIA HPC SDK</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ebi.ac.uk/~zerbino/oases/">Oases</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/oases">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/oases">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.vi-hps.org/tools/opari2.html">OPARI2</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.open-mpi.org/">Open MPI</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.open-mpi.org/">Open MPI (ib)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.open-mpi.org/">Open MPI 2 (ib)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.openms.de/">OpenMS</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/openms">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/openms">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper">OPLS-DA_Contrasts (w4mcorcov)</a></td>
<td align="center">Metabolomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/kishwarshafin/pepper">P.E.P.P.E.R. (+ deep variant)</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/TomSkelly/PacBioEDA">PacBioEDA</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">padb</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://icl.utk.edu/papi/software/">PAPI (Performance Application Programming Interface)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.vi-hps.org/tools/paraver.html">PARAVER</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.paraview.org/">ParaView</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/paraview">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/parsnp">Parsnp</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/parsnp">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/PASApipeline/PASApipeline/wiki">PASA</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/PASA">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pasa">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/NixOS/patchelf">PatchELF</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://downloadcenter.intel.com/download/19792">PDT (Processor Diagnostic Tool)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://compomics.github.io/projects/peptide-shaker.html">Peptide Shaker</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/peptideshaker">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/peptideshaker">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://percolator.ms/">Percolator</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/percolator">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/percolator">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.php.net/">PHP</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://broadinstitute.github.io/picard/">PICARD</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/picard_tools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/picard">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://picrust.github.io/picrust/">PICRUSt</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/picrust">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/picrust/picrust2/wiki">PICRUSt 2</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/picrust2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.drive5.com/piler/">PILER-DF</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/piler">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/broadinstitute/pilon/wiki">Pilon</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/pilon">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pilon">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://zzz.bwh.harvard.edu/plink/">plink</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/plink">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/plink">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://atgu.mgh.harvard.edu/plinkseq/index.shtml">plinkseq</a></td>
<td align="center">Variant Calling</td>
<td align="center"><a href="https://bio.tools/plink-seq">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://parallel-netcdf.github.io/">PnetCDF</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/matsen/pplacer">pplacer</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/clstutils">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pplacer">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/wtsi-hpag/PretextView">PretextView / Pretext Suite</a></td>
<td align="center">Data visualisation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://prinseq.sourceforge.net/">PRINSEQ</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/prinseq">●</a></td>
<td align="center"><a href="https://biocontainers.pro/tools/prinseq">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.genabel.org/packages/ProbABEL">probabel</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/probabel">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/rsilvabioinfo/ProbMetab">ProbMetab Tool</a></td>
<td align="center">Metabolomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/r-probmetab">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/hyattpd/Prodigal">Prodigal</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/gmv">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/prodigal">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/tseemann/prokka">Prokka</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/prokka">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/prokka">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dfguan/purge_dups">Purge_Dups</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/purge_dups">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/mroachawri/purge_haplotigs/src/master/">PurgeHaplotigs</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/tools/purge_haplotigs">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://pylib.readthedocs.io/en/1.4.3/index.html">pylibs</a></td>
<td align="center">Library</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://ipyrad.readthedocs.io/en/latest/">PyRAD</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/pyrad">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pyrad">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/pysam-developers/pysam">pysam</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/pysam">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/pysam">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.python.org/">python</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/python">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.python.org/">python2</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.python.org/">python2-as-python</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.python.org/">python3</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.python.org/">python3-as-python</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://pytorch.org/">PyTorch</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://qiime.org/">QIIME</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/qiime">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/qiime">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://qiime.org/">QIIME</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/qiime">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/qiime">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://qiime2.org/">QIIME2.0</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioinf.spbau.ru/quast">QUAST</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/quast">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/quast">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/mahulchak/quickmerge">Quickmerge</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/tools/quickmerge">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.r-project.org/">R</a></td>
<td align="center">Programming language / platform</td>
<td align="center"><a href="https://bio.tools/r">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.rabbitmq.com/">RabbitMQ server</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/isovic/racon">Racon</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/Racon">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/racon">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/RAILS">RAILS/Cobbler</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://ratt.sourceforge.net/">RATT</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/ratt">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/ratt">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/comprna/RATTLE">RATTLE</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/view/galaxyp/rawtools/51e4aa55ea04">Raw Tools</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/rawtools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/rawtools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://sco.h-its.org/exelixis/web/software/raxml/index.html">RAxML</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/raxml">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/raxml">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/dparks1134/RefineM">RefineM</a></td>
<td align="center">Population genomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/refinem">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/simoncchu/REPdenovo">REPdenovo</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/repdenovo">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/repdenovo">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.repeatmasker.org/">RepeatMasker</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/repeatmasker">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/repeatmasker">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.repeatmasker.org/RepeatModeler/">RepeatModeler</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/repeatmodeler">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/repeatmodeler">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/Dfam-consortium/RepeatModeler">RepeatModeler2</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/RepeatModeler2">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/repeatmodeler">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.yandell-lab.org/software/repeatrunner.html">RepeatRunner</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/repeatrunner">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://urgi.versailles.inra.fr/Tools/REPET">REPET</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/repet">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://cran.r-project.org/web/packages/rjags/index.html">rjags</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://deweylab.github.io/RSEM/">RSEM</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/rsem">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/rsem">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://rseqc.sourceforge.net/">rseqc</a></td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/rseqc">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/rseqc">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://rstudio.com/">Rstudio</a></td>
<td align="center">Programming language / platform</td>
<td align="center"><a href="https://bio.tools/rstudio">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ruby-lang.org/en/downloads/">Ruby</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/ruby">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://combine-lab.github.io/salmon/">Salmon</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/salmon">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/salmon">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/GregoryFaust/samblaster">samblaster</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/samblaster">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/samblaster">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.htslib.org/">SAMTools</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/samtools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/samtools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.sas.com/en_us/software/foundation.html">SAS</a></td>
<td align="center">Statistics and probability</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://compbio.cs.toronto.edu/hapsembler/scarpa.html">Scarpa</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.scipy.org/">SciPy</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/scipy">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://scons.org/">SCons</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://pypi.python.org/pypi/screed">screed</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/screed">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://compomics.github.io/projects/searchgui.html">Search GUI</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/searchgui">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/searchgui">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/shenwei356/seqkit">SeqKit</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/seqkit">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/drostlab/seqstats">seqstats</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/tools/seqstats">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/trinityrnaseq/seqtk-trinity">seqtk-trinity</a></td>
<td align="center">Sequence analysis</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html">shapeit</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/shapeit">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/shapeit">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/najoshi/sickle">sickle</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/sickle">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/sickle">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.bryntum.com/products/siesta/">Siesta</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://singularity.lbl.gov/">singularity</a></td>
<td align="center">Programming language / platform</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/singularity">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/relipmoc/skewer">skewer</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/skewer">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/skewer">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.pacb.com/support/software-downloads/">SMRT Link / Analysis</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/smrt_analysis">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://snakemake.readthedocs.io/en/stable/">Snakemake</a></td>
<td align="center">Workflows</td>
<td align="center"><a href="https://bio.tools/snakemake">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/snakemake">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/KorfLab/SNAP">SNAP</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/snap">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/snap">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/PMMAraujo/snappy">SNAPPy</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/SNAPPy">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://lowelab.ucsc.edu/snoscan/">snoscan</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/snoscan">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/snoscan">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://snpeff.sourceforge.net/">snpeff</a></td>
<td align="center">Variant Calling</td>
<td align="center"><a href="https://bio.tools/snpeff">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/snpeff">●</a></td>
</tr>
<tr class="odd">
<td align="center">soap</td>
<td align="center">Quality assurance</td>
<td align="center"><a href="https://bio.tools/soap">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://i.cs.hku.hk/2bwt-tools/soap3/index.html">soap3-gpu</a></td>
<td align="center">Sequence alignment</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/aquaskyline/SOAPdenovo2">SOAPdenovo2</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/soapdenovo2">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://sourceforge.net/projects/soapsnp/">SOAPsnp</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/soapsnp">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/soapsnp">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/biocore/sortmerna">sortmerna</a></td>
<td align="center">Metagenomics</td>
<td align="center"><a href="https://bio.tools/sortmerna">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/sortmerna">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/ablab/spades">SPAdes</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/spades">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/spades">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi">Splign</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc">sra-tools</a></td>
<td align="center">Knowledge discovery in databases</td>
<td align="center"><a href="https://bio.tools/sra-tools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/sra-tools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://catchenlab.life.illinois.edu/stacks/">Stacks</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/stacks">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/stacks">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/alexdobin/STAR">star</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/star">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/star">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://web.stanford.edu/group/pritchardlab/structure.html">structure</a></td>
<td align="center">Population genomics</td>
<td align="center"><a href="https://bio.tools/structure">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/structure">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome">Supernova</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/hall-lab/svtyper">svtyper</a></td>
<td align="center">Phylogenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/svtyper">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/TimoLassmann/tagdust">tagdust</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/tagdust">●</a></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://bitbucket.org/tasseladmin/tassel-5-standalone">tassel</a></td>
<td align="center">Phylogenomics</td>
<td align="center"><a href="https://bio.tools/tassel">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/tassel">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.cs.uoregon.edu/research/tau/home.php">TAU</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.tensorflow.org/resources/tools">TensorFlow</a></td>
<td align="center"></td>
<td align="center"><a href="https://bio.tools/TensorFlow">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/compomics/ThermoRawFileParser">ThermoRawFileParser</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/ThermoRawFileParser">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/thermorawfileparser">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/bcgsc/tigmint">Tigmint</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/tools/tigmint">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://ccb.jhu.edu/software/tophat/index.shtml">tophat</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/tophat">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/tophat">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://totalview.io/">TotalView</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/TransDecoder/TransDecoder/wiki">TransDecoder</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/TransDecoder">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/transdecoder">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://tandem.bu.edu/trf/trf.html">trf</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"><a href="https://bio.tools/trf">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/trf">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">Trim Galore</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/trim_galore">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/trim-galore">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.usadellab.org/cms/?page=trimmomatic">Trimmomatic</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/trimmomatic">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/trimmomatic">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki">Trinity / trinityrnaseq</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/trinity">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/trinity">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/Trinotate/Trinotate.github.io/wiki">Trinotate</a></td>
<td align="center">Transcriptomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/trinotate">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="http://eddylab.org/software.html">trnascan-se</a></td>
<td align="center">Transcriptomics</td>
<td align="center"><a href="https://bio.tools/trnascan-se">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/trnascan-se">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://hgdownload.soe.ucsc.edu">UCSC tools</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/UCSC_Genome_Browser_Utilities">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.openucx.org/">UCX (Unified Communication X)</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.unidata.ucar.edu/software/udunits/">UDUNITS</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/rrwick/Unicycler">Unicycler</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/unicycler">●</a></td>
<td align="center"><a href="https://biocontainers.pro/tools/unicycler">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="http://www.unixodbc.org/utilities.html">unixODBC</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/marbl/VALET">Valet</a></td>
<td align="center">Metagenomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/valet">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/cc2qe/vawk">vawk</a></td>
<td align="center">Quality assurance</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/vawk">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://vcftools.github.io/index.html">VCFTools</a></td>
<td align="center">Genetic variation</td>
<td align="center"><a href="https://bio.tools/vcftools">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/vcftools">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.ebi.ac.uk/~zerbino/velvet/">Velvet</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/velvet">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/velvet">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://www.tbi.univie.ac.at/RNA/documentation.html">ViennaRNA</a></td>
<td align="center">Transcriptomics</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/viennarna">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://wci.llnl.gov/simulation/computer-codes/visit/">VisIt</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/torognes/vsearch">vsearch</a></td>
<td align="center">Sequence analysis</td>
<td align="center"><a href="https://bio.tools/vsearch">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/vsearch">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/adigenova/wengan">Wengan</a></td>
<td align="center">Sequence assembly</td>
<td align="center"><a href="https://bio.tools/WENGAN">●</a></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html">WindowMasker</a></td>
<td align="center">Genomics / Genome annotation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.winehq.org/">Wine</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://www.ebi.ac.uk/~birney/wise2/">wise</a></td>
<td align="center">Sequence alignment</td>
<td align="center"><a href="https://bio.tools/wise">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/wise">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://workflow4metabolomics.org/">Workflow 4 Metabolomics</a></td>
<td align="center">Metabolomics</td>
<td align="center"><a href="https://bio.tools/workflow4metabolomics">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/isa2w4m">●</a></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/ruanjue/wtdbg2">WTDBG2</a></td>
<td align="center">Sequence assembly</td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/wtdbg2">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://www.wxwidgets.org/">wxWidgets</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="http://bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a></td>
<td align="center">Metabolomics</td>
<td align="center"><a href="https://bio.tools/xcms">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/bioconductor-xcms">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://xerces.apache.org/xerces-c/">Xerces-C++ XML parser</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://github.com/AustralianBioCommons/xml4ena">xml4ena</a></td>
<td align="center">Data submission, annotation and curation</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://github.com/freedesktop/xorg-macros">xorg-macros</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center"><a href="https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem/29e2df3edf11">xtandem</a></td>
<td align="center">Proteomics / Metaproteomics</td>
<td align="center"><a href="https://bio.tools/xtandem">●</a></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/xtandem">●</a></td>
</tr>
<tr class="odd">
<td align="center"><a href="https://zlib.net/">zlib</a></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">zstd</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"><a href="https://biocontainers.pro/#/tools/zstd">●</a></td>
</tr>
</tbody>
</table>
<p></font></p>
<hr />
<hr />
</div>
</div>
<div id="how-were-the-tools-and-workflows-selected" class="section level1">
<h1>How were the tools and workflows selected?</h1>
<p>The tool list here includes those that have been identified through <a href="https://www.biocommons.org.au/get-involved">BioCommons Community consultations</a> and other engagements, as well as through matching to the tool sets installed across the various BioCommons partner compute infrastructures. The list is not intended to be exhaustive: i.e. it does not contain <em>ALL</em> bioinformatics tools.</p>
<hr />
<hr />
</div>
<div id="how-was-the-list-generated" class="section level1">
<h1>How was the list generated?</h1>
<p>RStudio (see session info at the bottom of this page) was used for processing.</p>
<p>The broad steps are described here:</p>
<p><font size="-1.5"></p>
<ol style="list-style-type: decimal">
<li>The manually curated tool list and separate tool lists provided by the BioCommons partner infrastructures were parsed (Galaxy Australia, NCI, Pawsey and QRISCloud - RCC).</li>
<li>The complete set of tool lists were joined using the <code>tool ID</code> as a key, and</li>
<li>Links were embedded, where available, to the original documentation URLs (homepage or GitHub repository, for example) as well as the registry entries on bio.tools and BioContainers.</li>
<li>Galaxy wrapping often entails that tool suites are separated into their individual tool components. As such, if a Galaxy Australia tool was matched to the manually curated tool list, a tool shed search link was embedded in the Galaxy Australia column. This allows a user to search for the tool using a relevant search term (e.g. vcftools) to identify all the component tools available in the toolshed.</li>
</ol>
<p></font></p>
<hr />
<hr />
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>## R version 4.0.3 (2020-10-10)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## Running under: Windows 10 x64 (build 19041)
## 
## Matrix products: default
## 
## locale:
## [1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
## [3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
## [5] LC_TIME=English_Australia.1252    
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] lubridate_1.7.9.2 kableExtra_1.3.1  yaml_2.2.1        forcats_0.5.1    
##  [5] stringr_1.4.0     dplyr_1.0.3       purrr_0.3.4       readr_1.4.0      
##  [9] tidyr_1.1.2       tibble_3.0.5      ggplot2_3.3.3     tidyverse_1.3.0  
## [13] knitr_1.31       
## 
## loaded via a namespace (and not attached):
##  [1] tidyselect_1.1.0  xfun_0.20         haven_2.3.1       colorspace_2.0-0 
##  [5] vctrs_0.3.6       generics_0.1.0    htmltools_0.5.1.1 viridisLite_0.3.0
##  [9] rlang_0.4.10      pillar_1.4.7      glue_1.4.2        withr_2.4.1      
## [13] DBI_1.1.1         dbplyr_2.0.0      modelr_0.1.8      readxl_1.3.1     
## [17] lifecycle_0.2.0   munsell_0.5.0     gtable_0.3.0      cellranger_1.1.0 
## [21] rvest_0.3.6       evaluate_0.14     fansi_0.4.2       highr_0.8        
## [25] broom_0.7.3       Rcpp_1.0.6        scales_1.1.1      backports_1.2.0  
## [29] webshot_0.5.2     jsonlite_1.7.2    fs_1.5.0          hms_1.0.0        
## [33] digest_0.6.27     stringi_1.5.3     grid_4.0.3        cli_2.2.0        
## [37] tools_4.0.3       magrittr_2.0.1    crayon_1.3.4      pkgconfig_2.0.3  
## [41] ellipsis_0.3.1    xml2_1.3.2        reprex_1.0.0      assertthat_0.2.1 
## [45] rmarkdown_2.6     httr_1.4.2        rstudioapi_0.13   R6_2.5.0         
## [49] compiler_4.0.3</code></pre>
<hr />
<hr />
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
