<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Bioinformatics tools &amp; workflows on BioCommons partner infrastructures</title>

<script src="site_libs/jquery-3.5.1/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.17/datatables.js"></script>
<link href="site_libs/dt-core-bootstrap-1.10.20/css/dataTables.bootstrap.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-bootstrap-1.10.20/css/dataTables.bootstrap.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-bootstrap-1.10.20/js/jquery.dataTables.min.js"></script>
<script src="site_libs/dt-core-bootstrap-1.10.20/js/dataTables.bootstrap.min.js"></script>
<link href="site_libs/crosstalk-1.1.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.1/js/crosstalk.min.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>

<!DOCTYPE html>
<link rel="shortcut icon" href="./images/Australian-Biocommons-Favicon.png" />

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-T8GWK081T9"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-T8GWK081T9');
</script>
<!--/html_preserve-->

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="1_compute_systems.html">Compute systems</a>
</li>
<li>
  <a href="2_tools.html">Tools &amp; workflows</a>
</li>
<li>
  <a href="3_biocommons_activities.html">Featured tools &amp; workflows</a>
</li>
<li>
  <a href="4_help.html">Help!</a>
</li>
<li>
  <a href="5_attributions.html">Attributions</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Bioinformatics tools &amp; workflows on BioCommons partner infrastructures</h1>

</div>


<style type="text/css">
.main-container {
  max-width: 1500px;
  margin-left: auto;
  margin-right: auto;
}
</style>
<p>This page details tool request mechanisms and a list of tools and workflows installed across several of the <a href="./1_compute_systems.html">BioCommons infrastructure partner systems</a>. Tool status on other partner infrastructures will be added over time.</p>
<hr />
<hr />
<p><a name="tool_request"></a></p>
<div id="requesting-tool-installations" class="section level1">
<h1>Requesting tool installations</h1>
<p>Tool installs can be requested for the infrastructures listed in the table:</p>
<font size="-1.5">
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
Infrastructure
</th>
<th style="text-align:left;">
Process
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<a href="https://usegalaxy.org.au/">Galaxy Australia</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://request.usegalaxy.org.au/">tool install request</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://nci.org.au/">NCI</a>
</td>
<td style="text-align:left;">
<a href="https://opus.nci.org.au/display/Help/5.+Software+Applications">Contact NCI</a>
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://pawsey.org.au/">Pawsey</a>
</td>
<td style="text-align:left;">
Visit the <a href="https://support.pawsey.org.au/portal/servicedesk/customer/portal">Helpdesk Portal</a> or email: <a href="help@pawsey.org.au">help@pawsey.org.au</a> with your request
</td>
</tr>
<tr>
<td style="text-align:left;">
<a href="https://www.qriscloud.org.au/">QRIScloud</a> / <a href="https://rcc.uq.edu.au/">UQ-RCC</a>
</td>
<td style="text-align:left;">
Complete a <a href="https://docs.google.com/forms/d/e/1FAIpQLSefKCBkZbTQ-dUyU-pd4ZypkA-TA5GiMgpsJ2slWmD-B6elEg/viewform">tool install request</a>
</td>
</tr>
</tbody>
</table>
<p></font></p>
<div id="faq-how-can-i-get-a-hpc-allocation" class="section level3">
<h3>FAQ: How can I get a HPC allocation?</h3>
<p>Allocation of computational resources to individual researchers on partner HPC and cloud infrastructures is through the usual channels offered via each partner (see instructions for each by following these links: <a href="https://nci.org.au/users/how-access-nci">NCI</a>, <a href="https://support.pawsey.org.au/documentation/display/US/Getting+Access">Pawsey</a>, <a href="https://www.qcif.edu.au/services/computing/">QCIF</a>, <a href="http://cloud.nectar.org.au/start-now/">ARDC’s Nectar Research Cloud</a>). The <a href="https://www.biocommons.org.au/biocloud">BioCommons BioCloud</a> activity is working with these partners to examine ways to make it easier for life science researchers to receive HPC and HTC allocations.</p>
<p>Note that access to computational resources via the <a href="https://usegalaxy.org.au/">Galaxy Australia</a> hosted web platform is unlimited for Australian researchers thanks to our computational partners (ARDC’s Nectar Research Cloud, University of Queensland, QCIF, University of Melbourne and Pawsey) who underpin the Galaxy Australia service.</p>
<hr />
<hr />
<p><a name="tool_information"></a></p>
</div>
</div>
<div id="tool-information" class="section level1 tabset tabset-pills">
<h1>Tool information</h1>
<div id="tool-install-status-and-version-availability" class="section level2">
<h2>Tool install status and version availability</h2>
<p><font size="-1.5"></p>
<div id="htmlwidget-096f991ab36bb327bbff" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-096f991ab36bb327bbff">{"x":{"style":"bootstrap","filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432"],["<a href='https://github.com/aidenlab/3d-dna' target='_blank'  rel='noopener noreferrer'>3D-DNA<\/a>","<a href='https://github.com/tseemann/abricate' target='_blank'  rel='noopener noreferrer'>ABRicate<\/a>","<a href='https://github.com/bcgsc/abyss' target='_blank'  rel='noopener noreferrer'>ABySS<\/a>","<a href='https://github.com/tanghaibao/jcvi/wiki/ALLMAPS' target='_blank'  rel='noopener noreferrer'>ALLMAPS<\/a>","<a href='http://software.broadinstitute.org/allpaths-lg/blog/' target='_blank'  rel='noopener noreferrer'>allpathslg<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php/AMOS' target='_blank'  rel='noopener noreferrer'>amos<\/a>","<a href='http://bioinformatics.ua.pt/software/anaconda/' target='_blank'  rel='noopener noreferrer'>anaconda<\/a>","<a href='http://www.popgen.dk/angsd/index.php/ANGSD' target='_blank'  rel='noopener noreferrer'>ANGSD<\/a>","<a href='https://ant.apache.org/' target='_blank'  rel='noopener noreferrer'>ant<\/a>","<a href='https://github.com/GMOD/Apollo' target='_blank'  rel='noopener noreferrer'>Apollo<\/a>","<a href='https://arimagenomics.com/assembly' target='_blank'  rel='noopener noreferrer'>arima mapping<\/a>","<a href='https://www.arm.com/products/development-tools/server-and-hpc/forge' target='_blank'  rel='noopener noreferrer'>arm-forge<\/a>","arm-reports","<a href='https://github.com/sanger-pathogens/Artemis' target='_blank'  rel='noopener noreferrer'>Artemis<\/a>","<a href='https://www.aspera.com/en/' target='_blank'  rel='noopener noreferrer'>Aspera<\/a>","<a href='https://github.com/Gaius-Augustus/Augustus' target='_blank'  rel='noopener noreferrer'>Augustus<\/a>","<a href='https://www.gnu.org/software/autoconf/' target='_blank'  rel='noopener noreferrer'>Autoconf<\/a>","<a href='https://www.gnu.org/software/automake/' target='_blank'  rel='noopener noreferrer'>Automake<\/a>","<a href='https://github.com/hall-lab/bamkit' target='_blank'  rel='noopener noreferrer'>bamkit<\/a>","<a href='http://bamm-project.org/' target='_blank'  rel='noopener noreferrer'>bamm<\/a>","<a href='https://github.com/pezmaster31/bamtools' target='_blank'  rel='noopener noreferrer'>bamtools<\/a>","<a href='https://github.com/workflow4metabolomics/batch_correction' target='_blank'  rel='noopener noreferrer'>batchcorrection<\/a>","<a href='https://bazel.build/' target='_blank'  rel='noopener noreferrer'>bazel<\/a>","<a href='https://github.com/slaclab/bbcp' target='_blank'  rel='noopener noreferrer'>bbcp<\/a>","<a href='https://jgi.doe.gov/data-and-tools/bbtools/' target='_blank'  rel='noopener noreferrer'>bbmap<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>BCFtools<\/a>","<a href='https://faculty.washington.edu/browning/beagle/beagle.html' target='_blank'  rel='noopener noreferrer'>Beagle<\/a>","<a href='https://github.com/beagle-dev/beagle-lib' target='_blank'  rel='noopener noreferrer'>beagle-lib<\/a>","<a href='https://github.com/beast-dev/beast-mcmc' target='_blank'  rel='noopener noreferrer'>BEAST<\/a>","<a href='https://github.com/CompEvol/beast2' target='_blank'  rel='noopener noreferrer'>BEAST2<\/a>","<a href='https://github.com/arq5x/bedtools2' target='_blank'  rel='noopener noreferrer'>BEDTools<\/a>","<a href='https://www.gnu.org/software/binutils/' target='_blank'  rel='noopener noreferrer'>binutils<\/a>","<a href='https://www.bioconductor.org/' target='_blank'  rel='noopener noreferrer'>bioconductor<\/a>","<a href='https://github.com/Bio-Linux' target='_blank'  rel='noopener noreferrer'>biolinux<\/a>","<a href='https://bionanogenomics.com/support-page/bionano-access-software/' target='_blank'  rel='noopener noreferrer'>Bionano Access<\/a>","<a href='http://bioperl.org/' target='_blank'  rel='noopener noreferrer'>bioperl<\/a>","<a href='http://biopython.org/' target='_blank'  rel='noopener noreferrer'>biopython<\/a>","<a href='https://github.com/FelixKrueger/Bismark' target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST+<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST2<\/a>","<a href='https://genome.ucsc.edu/cgi-bin/hgBlat' target='_blank'  rel='noopener noreferrer'>blat<\/a>","<a href='https://github.com/blobtoolkit/blobtools2' target='_blank'  rel='noopener noreferrer'>BlobToolKit<\/a>","<a href='https://github.com/DRL/blobtools' target='_blank'  rel='noopener noreferrer'>BlobTools1<\/a>","<a href='https://www.boost.org/' target='_blank'  rel='noopener noreferrer'>Boost<\/a>","<a href='http://bowtie-bio.sourceforge.net/index.shtml' target='_blank'  rel='noopener noreferrer'>Bowtie<\/a>","<a href='http://bowtie-bio.sourceforge.net/bowtie2/index.shtml' target='_blank'  rel='noopener noreferrer'>Bowtie2<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER' target='_blank'  rel='noopener noreferrer'>BRAKER<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER' target='_blank'  rel='noopener noreferrer'>BRAKER2<\/a>","<a href='https://github.com/genome/breakdancer' target='_blank'  rel='noopener noreferrer'>breakdancer<\/a>","<a href='https://github.com/pace/bricks' target='_blank'  rel='noopener noreferrer'>BRICKS<\/a>","<a href='https://code.google.com/archive/p/bsmap/' target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","<a href='http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker/BS_Seeker.html' target='_blank'  rel='noopener noreferrer'>bsseeker<\/a>","<a href='https://github.com/BSSeeker/BSseeker2#bs-seeker2' target='_blank'  rel='noopener noreferrer'>bsseeker2<\/a>","<a href='https://busco.ezlab.org/' target='_blank'  rel='noopener noreferrer'>BUSCO<\/a>","<a href='https://github.com/lh3/bwa' target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://github.com/bxlab/bx-python' target='_blank'  rel='noopener noreferrer'>bx-python<\/a>","<a href='http://www.bzip.org/' target='_blank'  rel='noopener noreferrer'>bzip2<\/a>","<a href='https://github.com/ComparativeGenomicsToolkit/cactus' target='_blank'  rel='noopener noreferrer'>Cactus<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/CAMERA.html' target='_blank'  rel='noopener noreferrer'>CAMERA<\/a>","<a href='https://github.com/marbl/canu' target='_blank'  rel='noopener noreferrer'>Canu<\/a>","<a href='http://seq.cs.iastate.edu/cap3.html' target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='https://cardinalmsi.org/' target='_blank'  rel='noopener noreferrer'>Cardinal<\/a>","<a href='https://github.com/dutilh/CAT' target='_blank'  rel='noopener noreferrer'>CAT<\/a>","<a href='https://github.com/weizhongli/cdhit' target='_blank'  rel='noopener noreferrer'>cd-hit<\/a>","<a href='https://github.com/yanailab/celseq2' target='_blank'  rel='noopener noreferrer'>CEL-Seq 2<\/a>","<a href='http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page' target='_blank'  rel='noopener noreferrer'>Celera / wgs-assembler<\/a>","<a href='https://hub.docker.com/r/centos/tools/' target='_blank'  rel='noopener noreferrer'>centos<\/a>","<a href='https://hub.docker.com/r/centos/tools/' target='_blank'  rel='noopener noreferrer'>centos6<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki' target='_blank'  rel='noopener noreferrer'>checkm-genome<\/a>","<a href='https://genome.sph.umich.edu/wiki/ChunkChromosome' target='_blank'  rel='noopener noreferrer'>chunkchromosome<\/a>","<a href='http://www.clustal.org/clustal2/' target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","<a href='https://tools.bsc.es/doc/clusteringsuite_manual_0.pdf' target='_blank'  rel='noopener noreferrer'>clusteringsuite<\/a>","<a href='https://cmake.org/' target='_blank'  rel='noopener noreferrer'>cmake<\/a>","<a href='https://github.com/etal/cnvkit' target='_blank'  rel='noopener noreferrer'>CNVkit<\/a>","<a href='https://github.com/biocoder/cpc' target='_blank'  rel='noopener noreferrer'>Coding Potential Calculator<\/a>","<a href='https://github.com/biocoder/CPC2' target='_blank'  rel='noopener noreferrer'>Coding Potential Calculator 2<\/a>","<a href='https://www.gnu.org/software/coreutils/' target='_blank'  rel='noopener noreferrer'>coreutils<\/a>","<a href='https://sourceforge.net/projects/rna-cpat/files/' target='_blank'  rel='noopener noreferrer'>CPAT: Coding Potential Assessment Tool<\/a>","<a href='https://sourceforge.net/projects/cppunit/' target='_blank'  rel='noopener noreferrer'>cppunit<\/a>","<a href='https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cube-score-p-scalasca' target='_blank'  rel='noopener noreferrer'>cube<\/a>","<a href='https://developer.nvidia.com/cuda-toolkit' target='_blank'  rel='noopener noreferrer'>cuda<\/a>","<a href='https://developer.nvidia.com/cudnn' target='_blank'  rel='noopener noreferrer'>cudnn<\/a>","<a href='http://cole-trapnell-lab.github.io/cufflinks/' target='_blank'  rel='noopener noreferrer'>Cufflinks<\/a>","<a href='https://curl.se/' target='_blank'  rel='noopener noreferrer'>curl<\/a>","<a href='https://github.com/marcelm/cutadapt' target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='https://cytoscape.org/' target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","<a href='https://www.mcs.anl.gov/research/projects/darshan/' target='_blank'  rel='noopener noreferrer'>darshan<\/a>","<a href='https://deeptools.readthedocs.io/en/develop/' target='_blank'  rel='noopener noreferrer'>DeepTools<\/a>","<a href='https://github.com/google/deepvariant' target='_blank'  rel='noopener noreferrer'>deepvariant<\/a>","<a href='https://dendropy.org/' target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://diaumpire.nesvilab.org/' target='_blank'  rel='noopener noreferrer'>DIA-Umpire<\/a>","<a href='https://github.com/bbuchfink/diamond' target='_blank'  rel='noopener noreferrer'>diamond<\/a>","<a href='https://github.com/broadinstitute/discovar_de_novo' target='_blank'  rel='noopener noreferrer'>DISCOVAR de novo<\/a>","<a href='https://jbloomlab.github.io/dms_tools2/' target='_blank'  rel='noopener noreferrer'>dms_tools2<\/a>","<a href='https://github.com/dmtcp/dmtcp' target='_blank'  rel='noopener noreferrer'>dmtcp<\/a>","<a href='https://github.com/cmbi/dssp' target='_blank'  rel='noopener noreferrer'>dssp<\/a>","<a href='https://github.com/easybuilders/easybuild' target='_blank'  rel='noopener noreferrer'>easybuild<\/a>","<a href='http://www.genomic.ch/edena.php' target='_blank'  rel='noopener noreferrer'>Edena<\/a>","<a href='https://github.com/UQ-RCC/nimrod-embedded' target='_blank'  rel='noopener noreferrer'>embedded-nimrod<\/a>","<a href='http://emboss.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>EMBOSS (European Molecular Biology Open Software Suite)<\/a>","<a href='https://asia.ensembl.org/info/website/index.html' target='_blank'  rel='noopener noreferrer'>Ensembl browser<\/a>","<a href='http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html' target='_blank'  rel='noopener noreferrer'>Ensembl Genebuild<\/a>","<a href='http://erlang.org/faq/introduction.html' target='_blank'  rel='noopener noreferrer'>Erlang<\/a>","<a href='http://mblab.wustl.edu/software.html' target='_blank'  rel='noopener noreferrer'>eval<\/a>","<a href='https://github.com/EVidenceModeler/EVidenceModeler/releases' target='_blank'  rel='noopener noreferrer'>EVidenceModeler (EVM)<\/a>","<a href='https://github.com/aberer/exabayes' target='_blank'  rel='noopener noreferrer'>ExaBayes<\/a>","<a href='http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate' target='_blank'  rel='noopener noreferrer'>Exonerate<\/a>","<a href='https://github.com/SegataLab/export2graphlan/' target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","<a href='https://tools.bsc.es/extrae' target='_blank'  rel='noopener noreferrer'>Extrae<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly' target='_blank'  rel='noopener noreferrer'>Falcon<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly' target='_blank'  rel='noopener noreferrer'>Falcon (pb-assembly)<\/a>","<a href='http://fastml.tau.ac.il/' target='_blank'  rel='noopener noreferrer'>fastml<\/a>","<a href='https://www.bioinformatics.babraham.ac.uk/projects/fastqc/' target='_blank'  rel='noopener noreferrer'>FASTQC<\/a>","<a href='https://rajanil.github.io/fastStructure/' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/' target='_blank'  rel='noopener noreferrer'>FASTX-Toolkit<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/' target='_blank'  rel='noopener noreferrer'>FASTX-Toolkit<\/a>","<a href='http://www.softberry.com/berry.phtml?topic=fgenesh_plus_plus&group=help&subgroup=pipelines' target='_blank'  rel='noopener noreferrer'>FgenesH++<\/a>","<a href='http://animalbiosciences.uoguelph.ca/~msargol/fimpute/' target='_blank'  rel='noopener noreferrer'>FImpute<\/a>","<a href='https://github.com/smith-chem-wisc/FlashLFQ' target='_blank'  rel='noopener noreferrer'>flashLFQ<\/a>","<a href='https://www.fltk.org/index.php' target='_blank'  rel='noopener noreferrer'>fltk<\/a>","<a href='https://github.com/fenderglass/Flye' target='_blank'  rel='noopener noreferrer'>Flye<\/a>","<a href='https://sourceforge.net/projects/fpart/' target='_blank'  rel='noopener noreferrer'>fpart<\/a>","<a href='https://fpm.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>fpm<\/a>","<a href='https://sourceforge.net/projects/fraggenescan/' target='_blank'  rel='noopener noreferrer'>FragGeneScan<\/a>","<a href='https://github.com/ekg/freebayes' target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","<a href='https://surfer.nmr.mgh.harvard.edu/' target='_blank'  rel='noopener noreferrer'>freesurfer<\/a>","<a href='https://funannotate.readthedocs.io/en/latest/install.html' target='_blank'  rel='noopener noreferrer'>funannotate<\/a>","<a href='https://code.google.com/archive/p/garli/' target='_blank'  rel='noopener noreferrer'>GARLI<\/a>","<a href='https://github.com/broadinstitute/gatk' target='_blank'  rel='noopener noreferrer'>GATK<\/a>","<a href='http://molevol.cmima.csic.es/castresana/Gblocks_server.html' target='_blank'  rel='noopener noreferrer'>gblocks<\/a>","<a href='https://gcc.gnu.org/' target='_blank'  rel='noopener noreferrer'>GCC<\/a>","<a href='http://cnsgenomics.com/software/gcta/' target='_blank'  rel='noopener noreferrer'>gcta<\/a>","<a href='https://github.com/gnudatalanguage/gdl' target='_blank'  rel='noopener noreferrer'>GDL<\/a>","<a href='http://genome.crg.es/software/geneid/' target='_blank'  rel='noopener noreferrer'>geneid<\/a>","<a href='http://exon.gatech.edu/GeneMark/' target='_blank'  rel='noopener noreferrer'>Genemark<\/a>","<a href='https://github.com/schatzlab/genomescope' target='_blank'  rel='noopener noreferrer'>GenomeScope<\/a>","<a href='https://github.com/PacificBiosciences/GenomicConsensus' target='_blank'  rel='noopener noreferrer'>GenomicConsensus (Arrow)<\/a>","<a href='https://git-scm.com/about' target='_blank'  rel='noopener noreferrer'>Git<\/a>","<a href='http://glew.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>glew<\/a>","<a href='https://github.com/glfw/glfw' target='_blank'  rel='noopener noreferrer'>glfw<\/a>","<a href='https://sourceware.org/git/?p=glibc.git' target='_blank'  rel='noopener noreferrer'>glibc<\/a>","<a href='http://ccb.jhu.edu/software/glimmer/' target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='http://ccb.jhu.edu/software/glimmerhmm/' target='_blank'  rel='noopener noreferrer'>glimmerhmm<\/a>","<a href='https://github.com/GlobalArrays/ga' target='_blank'  rel='noopener noreferrer'>Global Arrays (GA) toolkit<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap_gsnap<\/a>","<a href='https://github.com/epam/GMP' target='_blank'  rel='noopener noreferrer'>gmp<\/a>","<a href='https://www.gnu.org/' target='_blank'  rel='noopener noreferrer'>GNU<\/a>","<a href='https://www.gnu.org/software/libtool/manual/libtool.html' target='_blank'  rel='noopener noreferrer'>GNU libtool<\/a>","<a href='https://www.gnu.org/software/parallel/' target='_blank'  rel='noopener noreferrer'>GNU parallel<\/a>","<a href='https://www.gnu.org/software/gsl/' target='_blank'  rel='noopener noreferrer'>GNU scientific library<\/a>","<a href='https://sourceforge.net/projects/cbadvanced/' target='_blank'  rel='noopener noreferrer'>GNU Tools<\/a>","<a href='https://geronimp.github.io/graftM/' target='_blank'  rel='noopener noreferrer'>GraftM<\/a>","<a href='https://github.com/biobakery/graphlan' target='_blank'  rel='noopener noreferrer'>GraPhlAn<\/a>","<a href='http://www.gromacs.org/' target='_blank'  rel='noopener noreferrer'>gromacs<\/a>","<a href='http://ecogenomics.github.io/GroopM/' target='_blank'  rel='noopener noreferrer'>groopm<\/a>","<a href='https://github.com/sanger-pathogens/gubbins' target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","guppy","<a href='https://hadoop.apache.org/' target='_blank'  rel='noopener noreferrer'>Hadoop<\/a>","<a href='https://github.com/vpc-ccg/haslr' target='_blank'  rel='noopener noreferrer'>HASLR<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php?title=Hawkeye' target='_blank'  rel='noopener noreferrer'>Hawkeye<\/a>","<a href='https://github.com/SegataLab/hclust2' target='_blank'  rel='noopener noreferrer'>Hclust2<\/a>","<a href='https://support.hdfgroup.org/products/hdf4_tools/' target='_blank'  rel='noopener noreferrer'>hdf4<\/a>","<a href='https://support.hdfgroup.org/products/hdf5_tools/' target='_blank'  rel='noopener noreferrer'>hdf5<\/a>","<a href='https://github.com/higlass' target='_blank'  rel='noopener noreferrer'>HiGlass<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772016/' target='_blank'  rel='noopener noreferrer'>HiRise<\/a>","<a href='https://daehwankimlab.github.io/hisat2/' target='_blank'  rel='noopener noreferrer'>HISAT2<\/a>","<a href='http://hmmer.org/' target='_blank'  rel='noopener noreferrer'>HMMER<\/a>","<a href='https://github.com/horovod/horovod' target='_blank'  rel='noopener noreferrer'>horovod<\/a>","<a href='https://github.com/htseq/htseq' target='_blank'  rel='noopener noreferrer'>HTSeq<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>HTSlib<\/a>","<a href='https://github.com/loneknightpy/idba' target='_blank'  rel='noopener noreferrer'>idba-ud<\/a>","<a href='https://www.l3harrisgeospatial.com/Software-Technology/IDL' target='_blank'  rel='noopener noreferrer'>IDL<\/a>","<a href='https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler' target='_blank'  rel='noopener noreferrer'>Improved Phased Assembler (pbipa)<\/a>","<a href='http://eddylab.org/infernal/' target='_blank'  rel='noopener noreferrer'>Infernal<\/a>","<a href='http://software.broadinstitute.org/software/igv/home' target='_blank'  rel='noopener noreferrer'>Integrated Genomics Viewer<\/a>","<a href='https://interproscan-docs.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>InterProScan<\/a>","<a href='https://github.com/nerscadmin/IPM' target='_blank'  rel='noopener noreferrer'>ipm<\/a>","<a href='https://github.com/dereneaton/pyrad' target='_blank'  rel='noopener noreferrer'>iPyRAD<\/a>","<a href='http://www.iqtree.org/' target='_blank'  rel='noopener noreferrer'>IQ-TREE<\/a>","<a href='https://github.com/PacificBiosciences/IsoSeq' target='_blank'  rel='noopener noreferrer'>IsoSeq3<\/a>","<a href='http://mcmc-jags.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>jags<\/a>","<a href='https://github.com/genomecuration/JAMg' target='_blank'  rel='noopener noreferrer'>JAMg<\/a>","<a href='https://www.oracle.com/java/' target='_blank'  rel='noopener noreferrer'>Java<\/a>","<a href='https://github.com/GMOD/jbrowse' target='_blank'  rel='noopener noreferrer'>JBrowse<\/a>","<a href='http://www.genome.umd.edu/jellyfish.html' target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","<a href='https://github.com/aidenlab/Juicebox' target='_blank'  rel='noopener noreferrer'>juicebox<\/a>","<a href='https://github.com/aidenlab/juicer' target='_blank'  rel='noopener noreferrer'>Juicer<\/a>","<a href='https://julialang.org/' target='_blank'  rel='noopener noreferrer'>julia<\/a>","<a href='https://pachterlab.github.io/kallisto/about.html' target='_blank'  rel='noopener noreferrer'>Kallisto<\/a>","<a href='https://kat.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>KAT<\/a>","<a href='https://github.com/dib-lab/khmer' target='_blank'  rel='noopener noreferrer'>khmer<\/a>","<a href='https://github.com/refresh-bio/KMC' target='_blank'  rel='noopener noreferrer'>KMC<\/a>","<a href='https://ccb.jhu.edu/software/kraken/' target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://github.com/DerrickWood/kraken2/wiki' target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","<a href='https://github.com/marbl/Krona/wiki' target='_blank'  rel='noopener noreferrer'>krona<\/a>","<a href='https://kubernetes.io/' target='_blank'  rel='noopener noreferrer'>Kubernetes<\/a>","<a href='http://www.bx.psu.edu/~rsharris/lastz/' target='_blank'  rel='noopener noreferrer'>lastz<\/a>","<a href='https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst' target='_blank'  rel='noopener noreferrer'>LFQ-Analyst<\/a>","<a href='https://ofiwg.github.io/libfabric/' target='_blank'  rel='noopener noreferrer'>libfabric<\/a>","<a href='https://github.com/agordon/libgtextutils' target='_blank'  rel='noopener noreferrer'>libgtextutils<\/a>","<a href='https://www.nongnu.org/libunwind/' target='_blank'  rel='noopener noreferrer'>libunwind<\/a>","<a href='https://github.com/bcgsc/LINKS' target='_blank'  rel='noopener noreferrer'>LINKS<\/a>","<a href='https://www.lipidmaps.org/' target='_blank'  rel='noopener noreferrer'>Lipid Maps<\/a>","<a href='https://llvm.org/' target='_blank'  rel='noopener noreferrer'>llvm<\/a>","<a href='https://lmod.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>lmod<\/a>","<a href='https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger' target='_blank'  rel='noopener noreferrer'>Long Ranger<\/a>","<a href='http://csg.sph.umich.edu/abecasis/mach/index.html' target='_blank'  rel='noopener noreferrer'>mach<\/a>","<a href='https://github.com/macs3-project/MACS' target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='https://github.com/macs3-project/MACS' target='_blank'  rel='noopener noreferrer'>macs2<\/a>","<a href='https://github.com/ranwez/MACSE_V2_PIPELINES' target='_blank'  rel='noopener noreferrer'>MACSE<\/a>","<a href='http://mafft.cbrc.jp/alignment/software/' target='_blank'  rel='noopener noreferrer'>mafft<\/a>","<a href='https://github.com/WatsonLab/MAGpy' target='_blank'  rel='noopener noreferrer'>MAGpy<\/a>","<a href='http://www.yandell-lab.org/software/maker.html' target='_blank'  rel='noopener noreferrer'>MAKER<\/a>","<a href='http://www.yandell-lab.org/software/maker-p.html' target='_blank'  rel='noopener noreferrer'>MAKER-P<\/a>","<a href='https://cran.r-project.org/web/packages/MALDIquant/index.html' target='_blank'  rel='noopener noreferrer'>MALDIquant<\/a>","<a href='https://github.com/marbl/MashMap' target='_blank'  rel='noopener noreferrer'>mashmap<\/a>","<a href='https://github.com/kwongj/maskrc-svg' target='_blank'  rel='noopener noreferrer'>maskrc-svg<\/a>","<a href='https://github.com/alekseyzimin/masurca' target='_blank'  rel='noopener noreferrer'>MaSuRCA<\/a>","<a href='https://www.mathworks.com/products/matlab.html' target='_blank'  rel='noopener noreferrer'>MATLAB<\/a>","<a href='https://github.com/dtreichler/matserver' target='_blank'  rel='noopener noreferrer'>matserver<\/a>","<a href='http://darlinglab.org/mauve/download.html' target='_blank'  rel='noopener noreferrer'>Mauve<\/a>","<a href='https://maven.apache.org/' target='_blank'  rel='noopener noreferrer'>maven<\/a>","<a href='https://sourceforge.net/projects/maxbin2/' target='_blank'  rel='noopener noreferrer'>MaxBin2<\/a>","<a href='https://www.maxquant.org/' target='_blank'  rel='noopener noreferrer'>MaxQuant<\/a>","<a href='https://www.maxquant.org/' target='_blank'  rel='noopener noreferrer'>MaxQuant (biocontainers)<\/a>","<a href='https://micans.org/mcl/' target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://github.com/nanoporetech/medaka' target='_blank'  rel='noopener noreferrer'>Medaka<\/a>","<a href='https://github.com/voutcn/megahit' target='_blank'  rel='noopener noreferrer'>megahit<\/a>","<a href='https://github.com/marbl/merqury' target='_blank'  rel='noopener noreferrer'>Merqury<\/a>","<a href='https://bitbucket.org/berkeleylab/metabat/src/master/' target='_blank'  rel='noopener noreferrer'>metabat<\/a>","<a href='https://sourceforge.net/projects/metacv/' target='_blank'  rel='noopener noreferrer'>metacv<\/a>","<a href='https://github.com/DiltheyLab/MetaMaps' target='_blank'  rel='noopener noreferrer'>MetaMaps<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/metaMS.html' target='_blank'  rel='noopener noreferrer'>metaMS<\/a>","<a href='https://huttenhower.sph.harvard.edu/metaphlan' target='_blank'  rel='noopener noreferrer'>MetaPhlAn<\/a>","<a href='http://cab.spbu.ru/software/spades/' target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","<a href='https://peterbeerli.com/migrate-html5/index.html' target='_blank'  rel='noopener noreferrer'>MIGRATE<\/a>","<a href='https://gatb.inria.fr/software/minia/' target='_blank'  rel='noopener noreferrer'>minia<\/a>","<a href='https://genome.sph.umich.edu/wiki/Minimac4' target='_blank'  rel='noopener noreferrer'>Minimac4<\/a>","<a href='https://github.com/lh3/minimap2' target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","<a href='https://sourceforge.net/p/mira-assembler/wiki/Home/' target='_blank'  rel='noopener noreferrer'>mira<\/a>","<a href='https://www.nature.com/articles/nbt1394#Sec11' target='_blank'  rel='noopener noreferrer'>mirdeep<\/a>","<a href='https://github.com/rajewsky-lab/mirdeep2' target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","<a href='https://github.com/yarden/MISO' target='_blank'  rel='noopener noreferrer'>miso<\/a>","<a href='http://mixomics.org/' target='_blank'  rel='noopener noreferrer'>mixOmics<\/a>","<a href='https://github.com/tseemann/mlst' target='_blank'  rel='noopener noreferrer'>MLST<\/a>","<a href='https://github.com/compomics/moFF' target='_blank'  rel='noopener noreferrer'>moFF<\/a>","<a href='https://www.mono-project.com/' target='_blank'  rel='noopener noreferrer'>Mono<\/a>","<a href='https://cwenger.github.io/Morpheus/' target='_blank'  rel='noopener noreferrer'>Morpheus<\/a>","<a href='https://www.mothur.org/' target='_blank'  rel='noopener noreferrer'>Mothur<\/a>","<a href='https://www.mpfr.org/' target='_blank'  rel='noopener noreferrer'>mpfr<\/a>","mpiBLAST","<a href='https://github.com/LLNL/mpiP' target='_blank'  rel='noopener noreferrer'>mpiP (mpi profiling library)<\/a>","<a href='https://github.com/NBISweden/MrBayes' target='_blank'  rel='noopener noreferrer'>MrBayes<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSGFplus.html' target='_blank'  rel='noopener noreferrer'>MS-GF+<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSnbase.html' target='_blank'  rel='noopener noreferrer'>MSnbase readMSData<\/a>","<a href='https://bitbucket.org/joseph07/multisnv/wiki/Home' target='_blank'  rel='noopener noreferrer'>multisnv<\/a>","<a href='http://mummer.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>MUMmer<\/a>","<a href='http://www.drive5.com/muscle/' target='_blank'  rel='noopener noreferrer'>MUSCLE<\/a>","<a href='https://mvapich.cse.ohio-state.edu/' target='_blank'  rel='noopener noreferrer'>MVAPICH<\/a>","<a href='https://software.opensuse.org/package/mxml' target='_blank'  rel='noopener noreferrer'>mxml<\/a>","<a href='https://www.nag.com/' target='_blank'  rel='noopener noreferrer'>nag<\/a>","<a href='https://github.com/jts/nanopolish' target='_blank'  rel='noopener noreferrer'>Nanopolish<\/a>","<a href='https://www.nasm.us/' target='_blank'  rel='noopener noreferrer'>NASM<\/a>","<a href='https://github.com/NVIDIA/nccl' target='_blank'  rel='noopener noreferrer'>NCCL<\/a>","<a href='https://gitlab.com/remikz/nccmp' target='_blank'  rel='noopener noreferrer'>nccmp<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml' target='_blank'  rel='noopener noreferrer'>NCL<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml' target='_blank'  rel='noopener noreferrer'>NCL<\/a>","<a href='http://nco.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>NCO<\/a>","<a href='http://meteora.ucsd.edu/~pierce/ncview_home_page.html' target='_blank'  rel='noopener noreferrer'>Ncview<\/a>","<a href='https://github.com/xiaochuanle/NECAT' target='_blank'  rel='noopener noreferrer'>NECAT<\/a>","<a href='https://www.unidata.ucar.edu/software/netcdf/' target='_blank'  rel='noopener noreferrer'>NetCDF<\/a>","<a href='https://github.com/neurodebian/neurodebian' target='_blank'  rel='noopener noreferrer'>neurodebian<\/a>","<a href='https://github.com/Nextomics/NextDenovo' target='_blank'  rel='noopener noreferrer'>NextDenovo<\/a>","<a href='https://www.nextflow.io/' target='_blank'  rel='noopener noreferrer'>nextflow<\/a>","<a href='http://cibiv.github.io/NextGenMap/' target='_blank'  rel='noopener noreferrer'>nextgenmap<\/a>","<a href='https://github.com/Nextomics/NextPolish' target='_blank'  rel='noopener noreferrer'>NextPolish<\/a>","<a href='https://github.com/ncbi/ngs' target='_blank'  rel='noopener noreferrer'>NGS<\/a>","<a href='https://github.com/shenlab-sinai/ngsplot' target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","<a href='https://nucleoatac.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>nucleoatac<\/a>","<a href='https://github.com/NVlabs/nvbio' target='_blank'  rel='noopener noreferrer'>nvbio<\/a>","<a href='https://developer.nvidia.com/hpc-sdk' target='_blank'  rel='noopener noreferrer'>NVIDIA HPC SDK<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/oases/' target='_blank'  rel='noopener noreferrer'>Oases<\/a>","<a href='https://www.vi-hps.org/tools/opari2.html' target='_blank'  rel='noopener noreferrer'>OPARI2<\/a>","<a href='https://www.open-mpi.org/' target='_blank'  rel='noopener noreferrer'>Open MPI<\/a>","<a href='https://www.open-mpi.org/' target='_blank'  rel='noopener noreferrer'>Open MPI (ib)<\/a>","<a href='https://www.open-mpi.org/' target='_blank'  rel='noopener noreferrer'>Open MPI 2 (ib)<\/a>","<a href='https://www.openms.de/' target='_blank'  rel='noopener noreferrer'>OpenMS<\/a>","<a href='https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper' target='_blank'  rel='noopener noreferrer'>OPLS-DA_Contrasts (w4mcorcov)<\/a>","<a href='https://github.com/kishwarshafin/pepper' target='_blank'  rel='noopener noreferrer'>P.E.P.P.E.R. (+ deep variant)<\/a>","<a href='https://github.com/TomSkelly/PacBioEDA' target='_blank'  rel='noopener noreferrer'>PacBioEDA<\/a>","padb","<a href='http://icl.utk.edu/papi/software/' target='_blank'  rel='noopener noreferrer'>PAPI (Performance Application Programming Interface)<\/a>","<a href='https://www.vi-hps.org/tools/paraver.html' target='_blank'  rel='noopener noreferrer'>PARAVER<\/a>","<a href='https://www.paraview.org/' target='_blank'  rel='noopener noreferrer'>ParaView<\/a>","<a href='https://github.com/marbl/parsnp' target='_blank'  rel='noopener noreferrer'>Parsnp<\/a>","<a href='https://github.com/PASApipeline/PASApipeline/wiki' target='_blank'  rel='noopener noreferrer'>PASA<\/a>","<a href='https://github.com/NixOS/patchelf' target='_blank'  rel='noopener noreferrer'>PatchELF<\/a>","<a href='https://downloadcenter.intel.com/download/19792' target='_blank'  rel='noopener noreferrer'>PDT (Processor Diagnostic Tool)<\/a>","<a href='http://compomics.github.io/projects/peptide-shaker.html' target='_blank'  rel='noopener noreferrer'>Peptide Shaker<\/a>","<a href='http://percolator.ms/' target='_blank'  rel='noopener noreferrer'>Percolator<\/a>","<a href='https://www.php.net/' target='_blank'  rel='noopener noreferrer'>PHP<\/a>","<a href='https://broadinstitute.github.io/picard/' target='_blank'  rel='noopener noreferrer'>PICARD<\/a>","<a href='http://picrust.github.io/picrust/' target='_blank'  rel='noopener noreferrer'>PICRUSt<\/a>","<a href='https://github.com/picrust/picrust2/wiki' target='_blank'  rel='noopener noreferrer'>PICRUSt 2<\/a>","<a href='https://www.drive5.com/piler/' target='_blank'  rel='noopener noreferrer'>PILER-DF<\/a>","<a href='https://github.com/broadinstitute/pilon/wiki' target='_blank'  rel='noopener noreferrer'>Pilon<\/a>","<a href='http://zzz.bwh.harvard.edu/plink/' target='_blank'  rel='noopener noreferrer'>plink<\/a>","<a href='http://atgu.mgh.harvard.edu/plinkseq/index.shtml' target='_blank'  rel='noopener noreferrer'>plinkseq<\/a>","<a href='https://parallel-netcdf.github.io/' target='_blank'  rel='noopener noreferrer'>PnetCDF<\/a>","<a href='https://github.com/matsen/pplacer' target='_blank'  rel='noopener noreferrer'>pplacer<\/a>","<a href='https://github.com/wtsi-hpag/PretextView' target='_blank'  rel='noopener noreferrer'>PretextView / Pretext Suite<\/a>","<a href='http://prinseq.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>PRINSEQ<\/a>","<a href='http://www.genabel.org/packages/ProbABEL' target='_blank'  rel='noopener noreferrer'>probabel<\/a>","<a href='https://github.com/rsilvabioinfo/ProbMetab' target='_blank'  rel='noopener noreferrer'>ProbMetab Tool<\/a>","<a href='https://github.com/hyattpd/Prodigal' target='_blank'  rel='noopener noreferrer'>Prodigal<\/a>","<a href='https://github.com/tseemann/prokka' target='_blank'  rel='noopener noreferrer'>Prokka<\/a>","<a href='https://github.com/dfguan/purge_dups' target='_blank'  rel='noopener noreferrer'>Purge_Dups<\/a>","<a href='https://bitbucket.org/mroachawri/purge_haplotigs/src/master/' target='_blank'  rel='noopener noreferrer'>PurgeHaplotigs<\/a>","<a href='https://pylib.readthedocs.io/en/1.4.3/index.html' target='_blank'  rel='noopener noreferrer'>pylibs<\/a>","<a href='https://ipyrad.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>PyRAD<\/a>","<a href='https://github.com/pysam-developers/pysam' target='_blank'  rel='noopener noreferrer'>pysam<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python2<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python2-as-python<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python3<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python3-as-python<\/a>","<a href='https://pytorch.org/' target='_blank'  rel='noopener noreferrer'>PyTorch<\/a>","<a href='http://qiime.org/' target='_blank'  rel='noopener noreferrer'>QIIME<\/a>","<a href='http://qiime.org/' target='_blank'  rel='noopener noreferrer'>QIIME<\/a>","<a href='https://qiime2.org/' target='_blank'  rel='noopener noreferrer'>QIIME2.0<\/a>","<a href='http://bioinf.spbau.ru/quast' target='_blank'  rel='noopener noreferrer'>QUAST<\/a>","<a href='https://github.com/mahulchak/quickmerge' target='_blank'  rel='noopener noreferrer'>Quickmerge<\/a>","<a href='https://www.r-project.org/' target='_blank'  rel='noopener noreferrer'>R<\/a>","<a href='https://www.rabbitmq.com/' target='_blank'  rel='noopener noreferrer'>RabbitMQ server<\/a>","<a href='https://github.com/isovic/racon' target='_blank'  rel='noopener noreferrer'>Racon<\/a>","<a href='https://github.com/bcgsc/RAILS' target='_blank'  rel='noopener noreferrer'>RAILS/Cobbler<\/a>","<a href='http://ratt.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>RATT<\/a>","<a href='https://github.com/comprna/RATTLE' target='_blank'  rel='noopener noreferrer'>RATTLE<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/galaxyp/rawtools/51e4aa55ea04' target='_blank'  rel='noopener noreferrer'>Raw Tools<\/a>","<a href='https://sco.h-its.org/exelixis/web/software/raxml/index.html' target='_blank'  rel='noopener noreferrer'>RAxML<\/a>","<a href='https://github.com/dparks1134/RefineM' target='_blank'  rel='noopener noreferrer'>RefineM<\/a>","<a href='https://github.com/simoncchu/REPdenovo' target='_blank'  rel='noopener noreferrer'>REPdenovo<\/a>","<a href='http://www.repeatmasker.org/' target='_blank'  rel='noopener noreferrer'>RepeatMasker<\/a>","<a href='http://www.repeatmasker.org/RepeatModeler/' target='_blank'  rel='noopener noreferrer'>RepeatModeler<\/a>","<a href='https://github.com/Dfam-consortium/RepeatModeler' target='_blank'  rel='noopener noreferrer'>RepeatModeler2<\/a>","<a href='https://www.yandell-lab.org/software/repeatrunner.html' target='_blank'  rel='noopener noreferrer'>RepeatRunner<\/a>","<a href='https://urgi.versailles.inra.fr/Tools/REPET' target='_blank'  rel='noopener noreferrer'>REPET<\/a>","<a href='https://cran.r-project.org/web/packages/rjags/index.html' target='_blank'  rel='noopener noreferrer'>rjags<\/a>","<a href='https://deweylab.github.io/RSEM/' target='_blank'  rel='noopener noreferrer'>RSEM<\/a>","<a href='http://rseqc.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","<a href='https://rstudio.com/' target='_blank'  rel='noopener noreferrer'>Rstudio<\/a>","<a href='https://www.ruby-lang.org/en/downloads/' target='_blank'  rel='noopener noreferrer'>Ruby<\/a>","<a href='https://combine-lab.github.io/salmon/' target='_blank'  rel='noopener noreferrer'>Salmon<\/a>","<a href='https://github.com/GregoryFaust/samblaster' target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>SAMTools<\/a>","<a href='https://www.sas.com/en_us/software/foundation.html' target='_blank'  rel='noopener noreferrer'>SAS<\/a>","<a href='http://compbio.cs.toronto.edu/hapsembler/scarpa.html' target='_blank'  rel='noopener noreferrer'>Scarpa<\/a>","<a href='https://www.scipy.org/' target='_blank'  rel='noopener noreferrer'>SciPy<\/a>","<a href='https://scons.org/' target='_blank'  rel='noopener noreferrer'>SCons<\/a>","<a href='https://pypi.python.org/pypi/screed' target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='http://compomics.github.io/projects/searchgui.html' target='_blank'  rel='noopener noreferrer'>Search GUI<\/a>","<a href='https://github.com/shenwei356/seqkit' target='_blank'  rel='noopener noreferrer'>SeqKit<\/a>","<a href='https://github.com/drostlab/seqstats' target='_blank'  rel='noopener noreferrer'>seqstats<\/a>","<a href='https://github.com/trinityrnaseq/seqtk-trinity' target='_blank'  rel='noopener noreferrer'>seqtk-trinity<\/a>","<a href='https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html' target='_blank'  rel='noopener noreferrer'>shapeit<\/a>","<a href='https://github.com/najoshi/sickle' target='_blank'  rel='noopener noreferrer'>sickle<\/a>","<a href='https://www.bryntum.com/products/siesta/' target='_blank'  rel='noopener noreferrer'>Siesta<\/a>","<a href='https://singularity.lbl.gov/' target='_blank'  rel='noopener noreferrer'>singularity<\/a>","<a href='https://github.com/relipmoc/skewer' target='_blank'  rel='noopener noreferrer'>skewer<\/a>","<a href='https://www.pacb.com/support/software-downloads/' target='_blank'  rel='noopener noreferrer'>SMRT Link / Analysis<\/a>","<a href='https://snakemake.readthedocs.io/en/stable/' target='_blank'  rel='noopener noreferrer'>Snakemake<\/a>","<a href='https://github.com/KorfLab/SNAP' target='_blank'  rel='noopener noreferrer'>SNAP<\/a>","<a href='https://github.com/PMMAraujo/snappy' target='_blank'  rel='noopener noreferrer'>SNAPPy<\/a>","<a href='http://lowelab.ucsc.edu/snoscan/' target='_blank'  rel='noopener noreferrer'>snoscan<\/a>","<a href='http://snpeff.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","soap","<a href='http://i.cs.hku.hk/2bwt-tools/soap3/index.html' target='_blank'  rel='noopener noreferrer'>soap3-gpu<\/a>","<a href='https://github.com/aquaskyline/SOAPdenovo2' target='_blank'  rel='noopener noreferrer'>SOAPdenovo2<\/a>","<a href='https://sourceforge.net/projects/soapsnp/' target='_blank'  rel='noopener noreferrer'>SOAPsnp<\/a>","<a href='https://github.com/biocore/sortmerna' target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='https://github.com/ablab/spades' target='_blank'  rel='noopener noreferrer'>SPAdes<\/a>","<a href='https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi' target='_blank'  rel='noopener noreferrer'>Splign<\/a>","<a href='https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc' target='_blank'  rel='noopener noreferrer'>sra-tools<\/a>","<a href='http://catchenlab.life.illinois.edu/stacks/' target='_blank'  rel='noopener noreferrer'>Stacks<\/a>","<a href='https://github.com/alexdobin/STAR' target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='http://web.stanford.edu/group/pritchardlab/structure.html' target='_blank'  rel='noopener noreferrer'>structure<\/a>","<a href='https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome' target='_blank'  rel='noopener noreferrer'>Supernova<\/a>","<a href='https://github.com/hall-lab/svtyper' target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","<a href='https://github.com/TimoLassmann/tagdust' target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://bitbucket.org/tasseladmin/tassel-5-standalone' target='_blank'  rel='noopener noreferrer'>tassel<\/a>","<a href='http://www.cs.uoregon.edu/research/tau/home.php' target='_blank'  rel='noopener noreferrer'>TAU<\/a>","<a href='https://www.tensorflow.org/resources/tools' target='_blank'  rel='noopener noreferrer'>TensorFlow<\/a>","<a href='https://github.com/compomics/ThermoRawFileParser' target='_blank'  rel='noopener noreferrer'>ThermoRawFileParser<\/a>","<a href='https://github.com/bcgsc/tigmint' target='_blank'  rel='noopener noreferrer'>Tigmint<\/a>","<a href='https://ccb.jhu.edu/software/tophat/index.shtml' target='_blank'  rel='noopener noreferrer'>tophat<\/a>","<a href='https://totalview.io/' target='_blank'  rel='noopener noreferrer'>TotalView<\/a>","<a href='https://github.com/TransDecoder/TransDecoder/wiki' target='_blank'  rel='noopener noreferrer'>TransDecoder<\/a>","<a href='https://tandem.bu.edu/trf/trf.html' target='_blank'  rel='noopener noreferrer'>trf<\/a>","<a href='http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/' target='_blank'  rel='noopener noreferrer'>Trim Galore<\/a>","<a href='http://www.usadellab.org/cms/?page=trimmomatic' target='_blank'  rel='noopener noreferrer'>Trimmomatic<\/a>","<a href='https://github.com/trinityrnaseq/trinityrnaseq/wiki' target='_blank'  rel='noopener noreferrer'>Trinity / trinityrnaseq<\/a>","<a href='https://github.com/Trinotate/Trinotate.github.io/wiki' target='_blank'  rel='noopener noreferrer'>Trinotate<\/a>","<a href='http://eddylab.org/software.html' target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","<a href='http://hgdownload.soe.ucsc.edu' target='_blank'  rel='noopener noreferrer'>UCSC tools<\/a>","<a href='https://www.openucx.org/' target='_blank'  rel='noopener noreferrer'>UCX (Unified Communication X)<\/a>","<a href='https://www.unidata.ucar.edu/software/udunits/' target='_blank'  rel='noopener noreferrer'>UDUNITS<\/a>","<a href='https://github.com/rrwick/Unicycler' target='_blank'  rel='noopener noreferrer'>Unicycler<\/a>","<a href='http://www.unixodbc.org/utilities.html' target='_blank'  rel='noopener noreferrer'>unixODBC<\/a>","<a href='https://github.com/marbl/VALET' target='_blank'  rel='noopener noreferrer'>Valet<\/a>","<a href='https://github.com/cc2qe/vawk' target='_blank'  rel='noopener noreferrer'>vawk<\/a>","<a href='https://vcftools.github.io/index.html' target='_blank'  rel='noopener noreferrer'>VCFTools<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/velvet/' target='_blank'  rel='noopener noreferrer'>Velvet<\/a>","<a href='https://www.tbi.univie.ac.at/RNA/documentation.html' target='_blank'  rel='noopener noreferrer'>ViennaRNA<\/a>","<a href='https://wci.llnl.gov/simulation/computer-codes/visit/' target='_blank'  rel='noopener noreferrer'>VisIt<\/a>","<a href='https://github.com/torognes/vsearch' target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","<a href='https://github.com/adigenova/wengan' target='_blank'  rel='noopener noreferrer'>Wengan<\/a>","<a href='http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html' target='_blank'  rel='noopener noreferrer'>WindowMasker<\/a>","<a href='https://www.winehq.org/' target='_blank'  rel='noopener noreferrer'>Wine<\/a>","<a href='http://www.ebi.ac.uk/~birney/wise2/' target='_blank'  rel='noopener noreferrer'>wise<\/a>","<a href='https://workflow4metabolomics.org/' target='_blank'  rel='noopener noreferrer'>Workflow 4 Metabolomics<\/a>","<a href='https://github.com/ruanjue/wtdbg2' target='_blank'  rel='noopener noreferrer'>WTDBG2<\/a>","<a href='https://www.wxwidgets.org/' target='_blank'  rel='noopener noreferrer'>wxWidgets<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/xcms.html' target='_blank'  rel='noopener noreferrer'>xcms<\/a>","<a href='https://xerces.apache.org/xerces-c/' target='_blank'  rel='noopener noreferrer'>Xerces-C++ XML parser<\/a>","<a href='https://github.com/AustralianBioCommons/xml4ena' target='_blank'  rel='noopener noreferrer'>xml4ena<\/a>","<a href='https://github.com/freedesktop/xorg-macros' target='_blank'  rel='noopener noreferrer'>xorg-macros<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem/29e2df3edf11' target='_blank'  rel='noopener noreferrer'>xtandem<\/a>","<a href='https://zlib.net/' target='_blank'  rel='noopener noreferrer'>zlib<\/a>","zstd"],["3D-DNA","abricate","abyss","allmaps","allpathslg","amos","anaconda","angsd","ant","apollo","arima mapping","arm-forge","arm-reports","artemis","aspera","augustus","autoconf","automake","bamkit","bamm","bamtools","batchcorrection","bazel","bbcp","bbmap","bcftools","beagle","beagle-lib","beast","beast2","bedtools","binutils","bioconductor","biolinux","Bionano Access","bioperl","biopython","bismark","blast","blast+","blast2","blat","BlobToolKit","blobtools","boost","bowtie","bowtie2","braker1","braker2","breakdancer","bricks","bsmap","bs_seeker","bsseeker2","busco","bwa","bx-python","bzip2","cactus","camera","canu","cap3","cardinal","cat_bins","cdhit","celseq2","celera","centos","centos6","checkm","checkm-genome","chunkchromosome","clustalw","clusteringsuite","cmake","cnvkit","cpc","cpc2","coreutils","cpat","cppunit","cube","cuda","cudnn","cufflinks","curl","cutadapt","cytoscape","darshan","deeptools","deepvariant","dendropy","DIA-Umpire","diamond","discovar_de_novo","dms_tools2","dmtcp","dssp","easybuild","edena","embedded-nimrod","emboss","ensembl","genebuild","erlang","eval","EvidenceModeler","exabayes","exonerate","export2graphlan","extrae","falcon","pb-assembly","fastml","fastqc","faststructure","fastx_toolkit","fastx","fgenesh","fimpute","flashlfq","fltk","flye","fpart","fpm","fraggenescan","freebayes","freesurfer","funannotate","garli","gatk","gblock","gcc","gcta","gdl","geneid","genemark","genomescope","GenomicConsensus","git","glew","glfw","glibc","glimmer","glimmerhmm","ga","gmap","gmap_gsnap","gmp","gnu","libtool","parallel","gsl","gnutools","graftm","graphlan","gromacs","groopm","gubbins","guppy","hadoop","haslr","hawkeye","hclust2","hdf4","hdf5","higlass","hirise","hisat2","hmmer","horovod","htseq","htslib","idba-ud","idl","pbipa","infernal","igv","interproscan","ipm","ipyrad","iq-tree","isoseq3","jags","jamg","java","jbrowse","jellyfish","juicebox","juicer","julia","kallisto","kat","khmer","kmc","kraken","kraken2","krona","k8","lastz","LFQ-Analyst","libfabric","libgtextutils","libunwind","links","lipid_maps","llvm","lmod","longranger","mach","macs","macs2","macse","mafft","magpy","maker","maker-p","maldi_quant","mashmap","maskrc-svg","masurca","matlab","matserver","mauve","maven","maxbin","maxquant","biocontainers-maxquant","mcl","medaka","megahit","merqury","metabat","metacv","metamaps","metams","metaphlan","metaspades","MIGRATE","minia","minimac4","minimap2","mira","mirdeep","mirdeep2","miso","mixomics","mlst","moff","mono","morpheus","Mothur","mpfr","mpiblast","mpip","mrbayes","msgfplus","msnbase","multisnv","mummer","muscle","mvapich2_ib","mxml","nag","nanopolish","nasm","nccl","nccmp","ncl","ncl_ncarg","nco","ncview","necat","netcdf","neurodebian","nextdenovo","nextflow","nextgenmap","nextpolish","ngs","ngsplot","nucleoatac","nvbio","nvidia-hpc-sdk","oases","opari2","openmpi","openmpi_ib","openmpi2_ib","openms","w4mcorcov","pepper+deepvariant","pacbioeda","padb","papi","paraver","paraview","parsnp","PASA","patchelf","pdt","peptideshaker","percolator","php","picard","picrust","picrust2","piler-df","pilon","plink","plinkseq","pnetcdf","pplacer","pretextview","prinseq","probabel","probmetab","prodigal","prokka","purge_dups","purgehaplotigs","pylibs","pyrad","pysam","python","python2","python2-as-python","python3","python3-as-python","pytorch","qiime","qiime-modules","qiime2","quast","quickmerge","r","rabbitmq-server","Racon","rails_cobbler","ratt","rattle","rawtools","raxml","refinem","repdenovo","repeatmasker","repeatmodeler","repeatmodeler2","repeatrunner","repet","rjags","rsem","rseqc","rstudio","ruby","salmon","samblaster","samtools","sas","scarpa","scipy","scons","screed","searchgui","biocontainers-seqkit","seqstats","seqtk-trinity","shapeit","sickle","siesta","singularity","skewer","smrtlink","snakemake","snap","snappy","snoscan","snpeff","soap","soap3-gpu","soapdenovo2","soapsnp","sortmerna","spades","splign","sra-tools","stacks","star","structure","supernova","svtyper","tagdust","tassel","tau","tensorflow","ThermoRawFileParser","tigmint","tophat","totalview","transdecoder","trf","trim_galore","trimmomatic","trinity","trinotate","trnascan-se","ucsc","ucx","udunits","unicycler","unixodbc","valet","vawk","vcftools","velvet","viennarna","visit","vsearch","wengan","windowmasker","wine","wise","workflow4metabolomics","wtdbg2","wxwidgets","xcms","xerces-c","xml4ena","xorg-macros","xtandem","zlib","zstd"],["Sequence assembly","Sequence analysis","Sequence assembly","Sequence assembly","Sequence assembly","Sequence assembly","Gene structure","Population genomics","","Genomics / Genome annotation","Chromosome conformation capture","","","Genomics / Genome annotation","","Genomics / Genome annotation","","","Sequence analysis","Phylogenomics","Sequence analysis","Metabolomics","","","Sequence analysis","Sequence analysis","Population genetics","Population genomics","Phylogenetics","Phylogenetics","Sequence analysis","","","","Sequence analysis","Programming language / platform","Programming language / platform","Epigenomics","Sequence alignment","Sequence alignment","Sequence alignment","Sequence alignment","Quality assurance","Quality assurance","Library","Sequence alignment","Sequence alignment","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","","Epigenomics","Epigenomics","Epigenomics","Quality assurance","Sequence alignment","Sequence analysis","","Sequence alignment","Metabolomics","Sequence assembly","Sequence assembly","Bioimaging","Metagenomics","Sequence analysis","Transcriptomics","Sequence assembly","","","Quality assurance","Quality assurance","Sequence analysis","Sequence alignment","Statistics and probability","","DNA structural variation","Genomics / Genome annotation","Genomics / Genome annotation","Library","Genomics / Genome annotation","","","","Library","Sequence assembly","","Sequence analysis","Multiomics","","Sequence analysis","Variant Calling","Phylogenomics","Proteomics / Metaproteomics","Sequence alignment","Sequence assembly","Genetic variation","","Protein folding, stability and design","","Sequence assembly","","Molecular biology","Data visualisation","Genomics / Genome annotation","Programming language / platform","Quality assurance","Genomics / Genome annotation","Phylogenomics","Sequence analysis","Phylogenomics","","","Sequence assembly","Phylogenomics","Quality assurance","Phylogenomics","Sequence analysis","Sequence analysis","Genomics / Genome annotation","Population genomics","Proteomics / Metaproteomics","","Sequence assembly","","","Sequence analysis","Phylogenomics","Bioimaging","Genomics / Genome annotation","Phylogenetics","Sequence analysis","Phylogenetics","Library","Phylogenomics","Library","Genomics / Genome annotation","Genomics / Genome annotation","Sequence analysis","Sequence assembly","","Library","Library","Library","Genomics / Genome annotation","Genomics / Genome annotation","","Sequence alignment","Sequence alignment","","Programming language / platform","Library","","Library","","Meta-Omics","Phylogenomics","Molecular dynamics","Metagenomics","Phylogenomics","Sequencing","Library","Sequence assembly","Quality assurance","Data visualisation","","","Data visualisation","Sequence assembly","Sequence alignment","Sequence analysis","","Sequence analysis","Library","Sequence assembly","Programming language / platform","Sequence assembly","Genomics / Genome annotation","Data visualisation","Protein sequence features","","Sequence analysis","Phylogenomics","Transcriptomics","Phylogenomics","Genomics / Genome annotation","Programming language / platform","Data visualisation","Sequence analysis","Sequence assembly","Sequence assembly","Programming language / platform","Transcriptomics","Sequence analysis","Sequence analysis","Sequence analysis","Taxonomy","Taxonomy","Data visualisation","Programming language / platform","Sequence alignment","Proteomics / Metaproteomics","Library","Library","","Sequence assembly","Metabolomics","","","Sequence alignment","Phylogenomics","ChIP-seq","ChIP-seq","Sequence alignment","Sequence alignment","Metagenomics","Genomics / Genome annotation","Genomics / Genome annotation","Proteomics","Sequence alignment","","Sequence assembly","","","Sequence alignment","","Metagenomics","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Statistics and probability","Sequence assembly","Metagenomics","Quality assurance","Metagenomics","Metagenomics","Metagenomics","Metabolomics","Metagenomics","Metagenomics","Population genomics","Sequence assembly","Phylogenomics","Sequence alignment","Sequence assembly","Transcriptomics","Transcriptomics","Transcriptomics","Multiomics","Sequence analysis","Proteomics / Metaproteomics","Programming language / platform","Proteomics / Metaproteomics","Metagenomics","","Sequence alignment","Library","Phylogenomics","Proteomics / Metaproteomics","Metabolomics","Variant Calling","Sequence alignment","Genomics / Genome annotation","","Library","Library","Sequence analysis","","","","Data visualisation","Data visualisation","","","Sequence assembly","Library","Neuroscience","Sequence assembly","Workflows","Sequence alignment","Sequence assembly","Knowledge discovery in databases","Data visualisation","Genomics / Genome annotation","Library","Library","Sequence assembly","","","","","Proteomics / Metaproteomics","Metabolomics","Sequence assembly","Sequence analysis","","","Data visualisation","Data visualisation","Phylogenomics","Genomics / Genome annotation","","","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Programming language / platform","Sequence analysis","Phylogenomics","Metagenomics","Genomics / Genome annotation","Sequence assembly","Phylogenomics","Variant Calling","Library","Phylogenomics","Data visualisation","Sequence analysis","Phylogenomics","Metabolomics","Genomics / Genome annotation","Genomics / Genome annotation","Sequence assembly","Sequence assembly","Library","Sequence analysis","Sequence analysis","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","","Metagenomics","Metagenomics","Metagenomics","Quality assurance","Sequence assembly","Programming language / platform","","Sequence assembly","Sequence assembly","Genomics / Genome annotation","Sequence assembly","Proteomics / Metaproteomics","Phylogenomics","Population genomics","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Phylogenomics","Transcriptomics","Quality assurance","Programming language / platform","Programming language / platform","Transcriptomics","Phylogenomics","Sequence analysis","Statistics and probability","Sequence assembly","Programming language / platform","","Sequence analysis","Proteomics / Metaproteomics","Sequence analysis","Sequence analysis","Sequence analysis","Phylogenomics","Sequence analysis","","Programming language / platform","Sequence analysis","Sequence analysis","Workflows","Genomics / Genome annotation","Phylogenomics","Genomics / Genome annotation","Variant Calling","Quality assurance","Sequence alignment","Sequence assembly","Sequence assembly","Metagenomics","Sequence assembly","Genomics / Genome annotation","Knowledge discovery in databases","Sequence assembly","Sequence alignment","Population genomics","Sequence assembly","Phylogenomics","Sequence analysis","Phylogenomics","","","Proteomics / Metaproteomics","Sequence assembly","Sequence alignment","","Transcriptomics","Genomics / Genome annotation","Sequence analysis","Sequence analysis","Sequence assembly","Transcriptomics","Transcriptomics","Sequence analysis","","","Sequence assembly","","Metagenomics","Quality assurance","Genetic variation","Sequence assembly","Transcriptomics","","Sequence analysis","Sequence assembly","Genomics / Genome annotation","","Sequence alignment","Metabolomics","Sequence assembly","","Metabolomics","","Data submission, annotation and curation","","Proteomics / Metaproteomics","",""],["","Yes","","","","","","","","","","","","","","Yes","","","","","","Yes","","","","Yes","","","","","Yes","","","","","","","","","Yes","","","","","","","Yes","","","","","","","","Yes","Yes","","","","Yes","Yes","","Yes","Yes","Yes","","","","","","","","Yes","","","","","","","","","","","","Yes","","Yes","","","Yes","","","Yes","Yes","","","","","","","","Yes","","","","","","","","","","","","","Yes","","Yes","Yes","","","Yes","","Yes","","","Yes","Yes","","","","","","","","","","","","","","","","","","","","","","","","","","","","","Yes","Yes","","Yes","","","","","","","","","","Yes","Yes","","Yes","","","","","Yes","","","","","Yes","","","","","Yes","","","","","Yes","","","","Yes","Yes","","","Yes","","","","","","Yes","","","","","Yes","Yes","","Yes","","","","Yes","","","","","","","","Yes","Yes","Yes","","","Yes","","","","","","","Yes","","","","Yes","","","Yes","","Yes","Yes","Yes","","Yes","Yes","","","","","Yes","Yes","","","","","","","Yes","","","","","","","","","","","","","","","","","","","","","","","","","Yes","Yes","","","","","","","","","","","Yes","Yes","","Yes","Yes","","","Yes","","","","","","","","","","Yes","","","","","","","","","","","","","","","Yes","","","","Yes","","","","Yes","Yes","","","","","","","","","","Yes","","","Yes","","Yes","","","","","","Yes","","","","","","","","","","","Yes","","","Yes","","","","","Yes","Yes","","","Yes","Yes","","","","","","","","Yes","","Yes","","Yes","","Yes","Yes","Yes","Yes","Yes","Yes","","","Yes","","","","Yes","Yes","","","Yes","","","","","Yes","","","Yes","","","","Yes","",""],["","","2.2.3","","","","","","1.10.7","","","20.0.2","20.0.2","","3.9.6","","","","","","","","0.26.0<br />0.27.1<br />3.1.0","15.02.03.00.1","","1.9","","3.1.2","1.10.4","","2.28.0","","","","","","","","2.10.1","","","","","","1.71.0<br />1.72.0","","2.3.5.1","","","","","","","","","0.7.17","","","","","1.9<br />2<br />2.0t","","","","","","","","","","","","","","3.16.2<br />3.18.2","","","","","","1.14.0","4.4<br />4.5","10.1<br />11.0.3","7.6.5-cuda10.1","","","","","3.2.1","","","","","","","","","","","","","","","","22.1","","","","","","","","","","","","","","","","","1.3.5","","","","","","","","","4.1.4.0","","system","","0.9.9","","","","","2.24.1","2.1.0","3.3.2","","","","5.7<br />5.7.2","","","","","","20191022","2.6","","","","2018.3<br />2018.3-plumed<br />2019.3<br />2019.3-gpuvolta<br />2019.3-plumed<br />2020.1<br />2020.1-gpuvolta<br />2020.1-plumed<br />2020.3<br />2020.3-gpuvolta","","","","","","","","4.2.14<br />4.2.15","1.10.5<br />1.10.5p<br />1.8.21<br />1.8.21p","","","","","0.19.0","","1.9","","8.6<br />8.7.2<br />8.8","","","","","2.0.6","","","","","","jdk-13.33<br />jdk-8.40","","2.3.0","","","1.5.3","","","","","","","","0.2.5","","","1.8.0a1","","1.2.1","","","10.0.0<br />11.0.0","","","","","","","","","","","","","","","R2019b","2018<br />2020","","","","","","","","","","","","","","","","","","","2.17","","","","","","","","","","","","","3.4.1","","","","","","","","","fll6i26dcl<br />fll6i26dcl-mkl<br />nll6i27dbl<br />nll6i27dbl-i8<br />nll6i27dbl-mkl","","","2.5.6-1+cuda10.1<br />2.6.4-1+cuda10.1","1.8.5.0","6.6.2","","4.7.7<br />4.9.2","2.1.7","","4.6.3<br />4.6.3-i8r8<br />4.6.3p<br />4.7.1<br />4.7.1p<br />4.7.3<br />4.7.3p<br />4.7.4<br />4.7.4p","","","","","","","","","","20.7","","2.0.3<br />2.0.5","2.1.6<br />2.1.6-debug<br />2.1.6-mt<br />2.1.6-mt-debug<br />3.0.4<br />3.0.4-debug<br />3.1.4<br />3.1.4-debug<br />4.0.1<br />4.0.1-debug<br />4.0.2<br />4.0.2-debug<br />4.0.3<br />4.0.3-debug","","","","","","","3.2","5.7.0","","5.8.0<br />5.8.0-gpu<br />5.8.0-mesa","","","0.1","","","","5.6.40","","","","","","","","1.11.2","","","","","","","","","","","","","","2.7.16<br />2.7.17","","3.7.4<br />3.8.5","","1.4.0a0<br />1.5.1","","","","","","3.6.1<br />4.0.0","3.8.0","","","","","","","","","","","","","","","","","","","1.1.0","","1.1<br />1.9","","","","3.1.1","","","","","","","","4.1-b4<br />4.1.5","","","","","","","","","","","2.41","","","","","","","","","","","","","","2.0.0<br />2.1.0<br />2.3.0","","","2.1.1","2019.3.14<br />2020.1.13","","","","","2.9.1","","","","1.6.1<br />1.8.0<br />1.8.1<br />1.9.0","1.12.11<br />2.2.26","","2.3.7","","","","1.2.10","","3.1.2","","","","","","","","3.0.4","","3.2.2","","","","","1.4.3"],["","","","","","3.1.0","","","","","","","","","","","","","0.1","","2.4.1","","","","","1.6","","2.1.2","1.8.4<br />2.4.7","","2.26.0","","","","","1.7.1","1.70","0.19.0","2.2.26","2.10.1<br />2.7.1","","3.6","","","","1.2.1.1","2.3.3.1<br />2.3.4.1","","","1.1.2_2013_03_08","","2.90","","2.1.3","","0.7.17","","","","","1.6","20130608","","","4.6.8","","8.3rc2","","","","","2014-05-27","2.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","6.6.0","","","","2.2.8","","","2.2.0","","","","","3.1","0.11.4","1.0","0.0.14","","","","","","","","","","0.9.21","","","","","0.91b","","1.26.0","","1.4.4","","","","","","","","3.02b","3.0.4","","2017-11-15","","","","","","","","","","","","","","","","","","","","","","","3.0<br />3.1b2","","","1.2.1<br />1.6","","","","","2.4.14","5.27-66.0","","","","","","","","","2.2.6","","","","","","2.1.2","","","","","","1.04.00","","","0.7","","","","","","","1.0.18","","","","7.313","","","","","","","","","","","","","","","14-137","","","","","","","","","","","","","","","","","","","","","","","","","","","3.2.6","","","","3.23","3.8.1551","","","","","","","","","","","","","","","","","","","1.3.0","","","","","0.2.08","","","","","","","","","","","","","","","","","","","","","","","","","1.07","0.10","","","","","0.5.0","","","","","","","3.0.66","0.13","","","","","","","","","","","","","","","","","","","8.2.11","","","","","","","","","1.3.0<br />1.3.3","","","","","0.1.24","0.1.19<br />1.6","","0.241","","","1.0","","","","0.0.2","r904","1.33","","","0.1.127","","","0.15","","","4.3t","2.21","r146","r240","","","3.11.1","","2.8.2-5","1.48<br />2.1","2.5.3a","2.3.4","","0.1.4","","4.3","","","","","","","5.0.2","4.09","","0.36","2.5.1","","2.0.0","357","","","","","","0.0.1","0.1.15","1.2.10","","","2.6.0","","","","2.4.1","","","","","","","","","",""],["","","","","","","","","","","","","","","","","","","","","","","","","","","","2.1.2","","","","","","","","","1.70","","","","","","","","","","","","","","","","","2.1.3","","0.7.17","","","","","","","","","","","","","","","","","2.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","1.4.1","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","2.1.2","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","3.2.6","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","0.13","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","0.1.19<br />1.6","","","","","1.0","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""],["","1.0.1","2.0.2","","","","2019.07<br />2020.02<br />4.2.0<br />4.3.1<br />5.2.0","0.931<br />0.933","","","","","","","","3.3.3","2.69-GCC-4.9.2","1.15-gcc-4.8.2<br />1.15-GCC-4.9.2<br />1.5-gcc-4.8.2","","201710<br />201811","2.4.0","","","","201710","1.3","2.1<br />4.1","","1.8.0<br />1.8.1<br />1.8.2","","2.25.0","2.25<br />2.25-GCC-4.9.2","3.6-rc2<br />3.8-rc2","8","","1.007002<br />1.007002-sw","1.66","0.15.0<br />0.22.3","2.3.0","","2.10.1<br />2.9.0","35","","","1.65.1","1.1.2","2.2.7<br />2.3.4.3","","2.1.5","","20180416","","","","1.22<br />3.0.2","0.7.13","0.8.1","1.0.6-intel-2015.3.187","2019.03.01","","1.3<br />1.8<br />1.8<br />1.9<br />2.1","","","","","0.5.3.3","8.3rc2","6.9-tinaroo2018<br />rocks6_roll_modules<br />rocks7_roll_modules","9","1.0.11","1.0.13","","","2.6.8","3.1.3-GCC-4.9.2<br />3.15.0-gnu-7.2.0<br />3.2.3-intel-2015.3.187<br />3.9.1","0.9.4a0","","","8.22","","","","","","2.2.1","7.43.0-intel-2015.3.187","1.18","3.7.1","","3.3.1","1.0.0","4.0.3","","0.7.12<br />0.9.22<br />0.9.23<br />0.9.30","","2.5.0","2.5.0","2.2.1","2.3.0<br />2.3.1<br />2.6.0<br />3.6.1<br />3.8.0<br />3.9.4","3.131028","0.8.6<br />1.1.0<br />1.1.4<br />1.10.3<br />1.10.5<br />1.11.0<br />1.3.0<br />1.4.0<br />1.8.0<br />1.8.1<br />1.9.0","6.5.7","","","","","","","2.4.0","0.22","3.7.1-impi_2018.2.046<br />3.7.1-openmpi_2.0.2<br />3.7.1-openmpi_ib_1.8.4-intel","201710","0.0.6<br />0.0.8","","0.11.4","","","0.0.14","","","","","2.4.2<br />2.6","1.0.0","1.3.3-Ruby-2.1.6","","","6.0<br />6.1dev","","","3.5<br />4.0.4.0","","4.9.2","1.26.0","","","","","","","","","2.17<br />2.19<br />2.23","","","","","20151231","6.0.0a","7.2.0","2.4.2-GCC-4.9.2<br />2.4.5-GCC-4.9.2","20170122<br />20180322","1.16<br />2.1","2.69","0.9.4","1.1.3","2018<br />5.1.4-intel<br />5.1.4-intelMPI","0.3.5<br />201811","2.4.1","3.3.0","2.5.0-cdh5.3.1-native","","","1.0.0","2.10","1.8.14","","","2.1","3.1b2<br />3.3","","0.6.1p1<br />htseq-0.6.1","","1.1.1","","","","","","2.0.3","","","3.2.2","4.2.0","","1.7.0_79<br />1.8.0_45","","2.2.10","","","1.5.0","","","","","1.1.1","2.0.8","2.7.1<br />201712","","","","","","","","","4.0.1","7.4.17","2.2.2","","2-2.1.1","2.2.7.1","","v7.407","","2.31.10<br />3.01.03","","","","0.5","3.2.4","2015a<br />2017a<br />2018a<br />2018a<br />2019b<br />qcif-R2018a<br />qcif-R2018a","","20200714","3.3.3","","1.6.10.43<br />1.6.17.0","1.6.10.43","","","201710","","201710","2.3.0","0.1","","3.0.2","","","","1.0.2","","4.9.6","","0.0.7","0.5.3","","2.19.0","","6.10.0.104-0","","","3.1.2","","","3.2.3<br />3.2.6","","","201801","","","2.1","2.9","","","2.11.08-intel-2015.3.187","","","","6.3.0","","2.1.7","","3.6.2<br />4.3.2<br />4.4.1-gnu<br />4.4.1-intel","0.37.2<br />0.37.6<br />0.39.0<br />0.39.0+eddy","v2.2-beta.0","19.07","","1.1.0","","2.61","0.3.1","","","","","1.8.4<br />2.0.2","1.8.4","2.0.2<br />2.1.6","","","0.1.0","20140812","","5.4.1","4.8.1","5.4.1","1.2<br />1.5.3","","","3.20","","","","2.18.2<br />2.3.0","1.1.3","2.2.0_b","","","1.9","","","v1.1.alpha17","","","","","201804","1.12<br />1.13<br />1.14.5","","","1.0","","0.8.4","2.7.14","","","3.4.9-centos<br />3.6.2-sdsc","","","","1.9.1","2018.6.0<br />2019.4","5.0.2","","3.4.3<br />3.5.0<br />3.5.0-gnu<br />3.5.0-intel<br />3.6.0-container<br />3.6.0+tidyverse+rstan+rtdists<br />3.6.1-container<br />3.6.1+CrocSim<br />3.6.1+exSTRa<br />3.6.1+Monocle3<br />3.6.1+Monocle3+Giotto<br />3.6.1+Seurat<br />3.6.3+Shawan<br />4.0.0<br />4.0.0+Spatial<br />4.0.2+Spatial<br />4.0.2+tradeSeq+Seurat","","","","","","","8.2.12","0.0.24","0.0.1","4.0.9_p2","2.0.1","","","","4.5.1<br />4.6.2--R-3.4.4","1.3.3","2.6.4","1.0.143<br />1.1.456-3.4.4<br />1.1.456-3.5.1","2.1.6","","","0.1.17<br />1.10<br />1.2<br />1.3<br />1.6<br />1.9","9.4","","2.7","","","","0.13.2","","","","","","2.2.1<br />2.3.1<br />2.4.0<br />2.4.2<br />2.5.1<br />2.5.2<br />3.5.0","","smrtlink_5.0.1<br />smrtlink_5.1<br />smrtlink_6.0<br />smrtlink_7.0<br />smrtlink_9.0","5.6","2006-07-28","1.1.2-GCC-4.9.2","","","","","240","1.03","3.0.3<br />4.0.0","3.11.1<br />3.9.0","","","1.37","2.5.2b<br />2.5.2b-static","2.3.4","2.1.1","","2.33","","2.23","","","","2.1.1","","5.5.0","","","0.35<br />0.39","2.1.1<br />2.9.1","3.2.1","","","","","","","","","0.1.14","1.2.10","2.2.4","","","","","5.0.2","","","","","","","","1.17-GCC-4.7.2","","1.2.11-gnu-7.1.0",""]],"container":"<table class=\"table table-bordered table-striped\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Tool / workflow name<\/th>\n      <th>Tool identifier (module name / bio.tools ID / placeholder)<\/th>\n      <th>Primary purpose (EDAM, if available)<\/th>\n      <th>Galaxy Australia<\/th>\n      <th>NCI (Gadi)<\/th>\n      <th>Pawsey (Zeus)<\/th>\n      <th>Pawsey (Magnus)<\/th>\n      <th>QRIScloud / UQ-RCC (Flashlite, Awoonga, Tinaroo)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":20,"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}],"lengthMenu":[10,20,25,50,100]}},"evals":[],"jsHooks":[]}</script>
<p></font></p>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-03-03</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>The <em>Tool identifier</em> column contains an identifier for the tool / workflow: typically either the module name (used for matching to HPC lists), a bio.tools ID, or a placeholder value.</li>
<li>For tools which are not currently installed on <a href="https://usegalaxy.org.au/">Galaxy Australia</a>, but which are available <a href="https://toolshed.g2.bx.psu.edu/">in the Galaxy app store (aka toolshed)</a>, the <em>Available in Galaxy toolshed</em> column launches a toolshed search using the link label as the search term.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<hr />
<hr />
</div>
<div id="tool-information-1" class="section level2">
<h2>Tool information</h2>
<p><font size="-1.5"></p>
<div id="htmlwidget-9f37d713ba357f0138c5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9f37d713ba357f0138c5">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432"],["<a href='https://github.com/aidenlab/3d-dna' target='_blank'  rel='noopener noreferrer'>3D-DNA<\/a>","<a href='https://github.com/tseemann/abricate' target='_blank'  rel='noopener noreferrer'>ABRicate<\/a>","<a href='https://github.com/bcgsc/abyss' target='_blank'  rel='noopener noreferrer'>ABySS<\/a>","<a href='https://github.com/tanghaibao/jcvi/wiki/ALLMAPS' target='_blank'  rel='noopener noreferrer'>ALLMAPS<\/a>","<a href='http://software.broadinstitute.org/allpaths-lg/blog/' target='_blank'  rel='noopener noreferrer'>allpathslg<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php/AMOS' target='_blank'  rel='noopener noreferrer'>amos<\/a>","<a href='http://bioinformatics.ua.pt/software/anaconda/' target='_blank'  rel='noopener noreferrer'>anaconda<\/a>","<a href='http://www.popgen.dk/angsd/index.php/ANGSD' target='_blank'  rel='noopener noreferrer'>ANGSD<\/a>","<a href='https://ant.apache.org/' target='_blank'  rel='noopener noreferrer'>ant<\/a>","<a href='https://github.com/GMOD/Apollo' target='_blank'  rel='noopener noreferrer'>Apollo<\/a>","<a href='https://arimagenomics.com/assembly' target='_blank'  rel='noopener noreferrer'>arima mapping<\/a>","<a href='https://www.arm.com/products/development-tools/server-and-hpc/forge' target='_blank'  rel='noopener noreferrer'>arm-forge<\/a>","arm-reports","<a href='https://github.com/sanger-pathogens/Artemis' target='_blank'  rel='noopener noreferrer'>Artemis<\/a>","<a href='https://www.aspera.com/en/' target='_blank'  rel='noopener noreferrer'>Aspera<\/a>","<a href='https://github.com/Gaius-Augustus/Augustus' target='_blank'  rel='noopener noreferrer'>Augustus<\/a>","<a href='https://www.gnu.org/software/autoconf/' target='_blank'  rel='noopener noreferrer'>Autoconf<\/a>","<a href='https://www.gnu.org/software/automake/' target='_blank'  rel='noopener noreferrer'>Automake<\/a>","<a href='https://github.com/hall-lab/bamkit' target='_blank'  rel='noopener noreferrer'>bamkit<\/a>","<a href='http://bamm-project.org/' target='_blank'  rel='noopener noreferrer'>bamm<\/a>","<a href='https://github.com/pezmaster31/bamtools' target='_blank'  rel='noopener noreferrer'>bamtools<\/a>","<a href='https://github.com/workflow4metabolomics/batch_correction' target='_blank'  rel='noopener noreferrer'>batchcorrection<\/a>","<a href='https://bazel.build/' target='_blank'  rel='noopener noreferrer'>bazel<\/a>","<a href='https://github.com/slaclab/bbcp' target='_blank'  rel='noopener noreferrer'>bbcp<\/a>","<a href='https://jgi.doe.gov/data-and-tools/bbtools/' target='_blank'  rel='noopener noreferrer'>bbmap<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>BCFtools<\/a>","<a href='https://faculty.washington.edu/browning/beagle/beagle.html' target='_blank'  rel='noopener noreferrer'>Beagle<\/a>","<a href='https://github.com/beagle-dev/beagle-lib' target='_blank'  rel='noopener noreferrer'>beagle-lib<\/a>","<a href='https://github.com/beast-dev/beast-mcmc' target='_blank'  rel='noopener noreferrer'>BEAST<\/a>","<a href='https://github.com/CompEvol/beast2' target='_blank'  rel='noopener noreferrer'>BEAST2<\/a>","<a href='https://github.com/arq5x/bedtools2' target='_blank'  rel='noopener noreferrer'>BEDTools<\/a>","<a href='https://www.gnu.org/software/binutils/' target='_blank'  rel='noopener noreferrer'>binutils<\/a>","<a href='https://www.bioconductor.org/' target='_blank'  rel='noopener noreferrer'>bioconductor<\/a>","<a href='https://github.com/Bio-Linux' target='_blank'  rel='noopener noreferrer'>biolinux<\/a>","<a href='https://bionanogenomics.com/support-page/bionano-access-software/' target='_blank'  rel='noopener noreferrer'>Bionano Access<\/a>","<a href='http://bioperl.org/' target='_blank'  rel='noopener noreferrer'>bioperl<\/a>","<a href='http://biopython.org/' target='_blank'  rel='noopener noreferrer'>biopython<\/a>","<a href='https://github.com/FelixKrueger/Bismark' target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST+<\/a>","<a href='https://blast.ncbi.nlm.nih.gov/Blast.cgi' target='_blank'  rel='noopener noreferrer'>BLAST2<\/a>","<a href='https://genome.ucsc.edu/cgi-bin/hgBlat' target='_blank'  rel='noopener noreferrer'>blat<\/a>","<a href='https://github.com/blobtoolkit/blobtools2' target='_blank'  rel='noopener noreferrer'>BlobToolKit<\/a>","<a href='https://github.com/DRL/blobtools' target='_blank'  rel='noopener noreferrer'>BlobTools1<\/a>","<a href='https://www.boost.org/' target='_blank'  rel='noopener noreferrer'>Boost<\/a>","<a href='http://bowtie-bio.sourceforge.net/index.shtml' target='_blank'  rel='noopener noreferrer'>Bowtie<\/a>","<a href='http://bowtie-bio.sourceforge.net/bowtie2/index.shtml' target='_blank'  rel='noopener noreferrer'>Bowtie2<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER' target='_blank'  rel='noopener noreferrer'>BRAKER<\/a>","<a href='https://github.com/Gaius-Augustus/BRAKER' target='_blank'  rel='noopener noreferrer'>BRAKER2<\/a>","<a href='https://github.com/genome/breakdancer' target='_blank'  rel='noopener noreferrer'>breakdancer<\/a>","<a href='https://github.com/pace/bricks' target='_blank'  rel='noopener noreferrer'>BRICKS<\/a>","<a href='https://code.google.com/archive/p/bsmap/' target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","<a href='http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker/BS_Seeker.html' target='_blank'  rel='noopener noreferrer'>bsseeker<\/a>","<a href='https://github.com/BSSeeker/BSseeker2#bs-seeker2' target='_blank'  rel='noopener noreferrer'>bsseeker2<\/a>","<a href='https://busco.ezlab.org/' target='_blank'  rel='noopener noreferrer'>BUSCO<\/a>","<a href='https://github.com/lh3/bwa' target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://github.com/bxlab/bx-python' target='_blank'  rel='noopener noreferrer'>bx-python<\/a>","<a href='http://www.bzip.org/' target='_blank'  rel='noopener noreferrer'>bzip2<\/a>","<a href='https://github.com/ComparativeGenomicsToolkit/cactus' target='_blank'  rel='noopener noreferrer'>Cactus<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/CAMERA.html' target='_blank'  rel='noopener noreferrer'>CAMERA<\/a>","<a href='https://github.com/marbl/canu' target='_blank'  rel='noopener noreferrer'>Canu<\/a>","<a href='http://seq.cs.iastate.edu/cap3.html' target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='https://cardinalmsi.org/' target='_blank'  rel='noopener noreferrer'>Cardinal<\/a>","<a href='https://github.com/dutilh/CAT' target='_blank'  rel='noopener noreferrer'>CAT<\/a>","<a href='https://github.com/weizhongli/cdhit' target='_blank'  rel='noopener noreferrer'>cd-hit<\/a>","<a href='https://github.com/yanailab/celseq2' target='_blank'  rel='noopener noreferrer'>CEL-Seq 2<\/a>","<a href='http://wgs-assembler.sourceforge.net/wiki/index.php?title=Main_Page' target='_blank'  rel='noopener noreferrer'>Celera / wgs-assembler<\/a>","<a href='https://hub.docker.com/r/centos/tools/' target='_blank'  rel='noopener noreferrer'>centos<\/a>","<a href='https://hub.docker.com/r/centos/tools/' target='_blank'  rel='noopener noreferrer'>centos6<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki' target='_blank'  rel='noopener noreferrer'>checkm<\/a>","<a href='https://github.com/Ecogenomics/CheckM/wiki' target='_blank'  rel='noopener noreferrer'>checkm-genome<\/a>","<a href='https://genome.sph.umich.edu/wiki/ChunkChromosome' target='_blank'  rel='noopener noreferrer'>chunkchromosome<\/a>","<a href='http://www.clustal.org/clustal2/' target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","<a href='https://tools.bsc.es/doc/clusteringsuite_manual_0.pdf' target='_blank'  rel='noopener noreferrer'>clusteringsuite<\/a>","<a href='https://cmake.org/' target='_blank'  rel='noopener noreferrer'>cmake<\/a>","<a href='https://github.com/etal/cnvkit' target='_blank'  rel='noopener noreferrer'>CNVkit<\/a>","<a href='https://github.com/biocoder/cpc' target='_blank'  rel='noopener noreferrer'>Coding Potential Calculator<\/a>","<a href='https://github.com/biocoder/CPC2' target='_blank'  rel='noopener noreferrer'>Coding Potential Calculator 2<\/a>","<a href='https://www.gnu.org/software/coreutils/' target='_blank'  rel='noopener noreferrer'>coreutils<\/a>","<a href='https://sourceforge.net/projects/rna-cpat/files/' target='_blank'  rel='noopener noreferrer'>CPAT: Coding Potential Assessment Tool<\/a>","<a href='https://sourceforge.net/projects/cppunit/' target='_blank'  rel='noopener noreferrer'>cppunit<\/a>","<a href='https://hbp-hpc-platform.fz-juelich.de/?hbp_software=cube-score-p-scalasca' target='_blank'  rel='noopener noreferrer'>cube<\/a>","<a href='https://developer.nvidia.com/cuda-toolkit' target='_blank'  rel='noopener noreferrer'>cuda<\/a>","<a href='https://developer.nvidia.com/cudnn' target='_blank'  rel='noopener noreferrer'>cudnn<\/a>","<a href='http://cole-trapnell-lab.github.io/cufflinks/' target='_blank'  rel='noopener noreferrer'>Cufflinks<\/a>","<a href='https://curl.se/' target='_blank'  rel='noopener noreferrer'>curl<\/a>","<a href='https://github.com/marcelm/cutadapt' target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='https://cytoscape.org/' target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","<a href='https://www.mcs.anl.gov/research/projects/darshan/' target='_blank'  rel='noopener noreferrer'>darshan<\/a>","<a href='https://deeptools.readthedocs.io/en/develop/' target='_blank'  rel='noopener noreferrer'>DeepTools<\/a>","<a href='https://github.com/google/deepvariant' target='_blank'  rel='noopener noreferrer'>deepvariant<\/a>","<a href='https://dendropy.org/' target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://diaumpire.nesvilab.org/' target='_blank'  rel='noopener noreferrer'>DIA-Umpire<\/a>","<a href='https://github.com/bbuchfink/diamond' target='_blank'  rel='noopener noreferrer'>diamond<\/a>","<a href='https://github.com/broadinstitute/discovar_de_novo' target='_blank'  rel='noopener noreferrer'>DISCOVAR de novo<\/a>","<a href='https://jbloomlab.github.io/dms_tools2/' target='_blank'  rel='noopener noreferrer'>dms_tools2<\/a>","<a href='https://github.com/dmtcp/dmtcp' target='_blank'  rel='noopener noreferrer'>dmtcp<\/a>","<a href='https://github.com/cmbi/dssp' target='_blank'  rel='noopener noreferrer'>dssp<\/a>","<a href='https://github.com/easybuilders/easybuild' target='_blank'  rel='noopener noreferrer'>easybuild<\/a>","<a href='http://www.genomic.ch/edena.php' target='_blank'  rel='noopener noreferrer'>Edena<\/a>","<a href='https://github.com/UQ-RCC/nimrod-embedded' target='_blank'  rel='noopener noreferrer'>embedded-nimrod<\/a>","<a href='http://emboss.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>EMBOSS (European Molecular Biology Open Software Suite)<\/a>","<a href='https://asia.ensembl.org/info/website/index.html' target='_blank'  rel='noopener noreferrer'>Ensembl browser<\/a>","<a href='http://may2012.archive.ensembl.org/info/docs/genebuild/genome_annotation.html' target='_blank'  rel='noopener noreferrer'>Ensembl Genebuild<\/a>","<a href='http://erlang.org/faq/introduction.html' target='_blank'  rel='noopener noreferrer'>Erlang<\/a>","<a href='http://mblab.wustl.edu/software.html' target='_blank'  rel='noopener noreferrer'>eval<\/a>","<a href='https://github.com/EVidenceModeler/EVidenceModeler/releases' target='_blank'  rel='noopener noreferrer'>EVidenceModeler (EVM)<\/a>","<a href='https://github.com/aberer/exabayes' target='_blank'  rel='noopener noreferrer'>ExaBayes<\/a>","<a href='http://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate' target='_blank'  rel='noopener noreferrer'>Exonerate<\/a>","<a href='https://github.com/SegataLab/export2graphlan/' target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","<a href='https://tools.bsc.es/extrae' target='_blank'  rel='noopener noreferrer'>Extrae<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly' target='_blank'  rel='noopener noreferrer'>Falcon<\/a>","<a href='https://github.com/PacificBiosciences/pb-assembly' target='_blank'  rel='noopener noreferrer'>Falcon (pb-assembly)<\/a>","<a href='http://fastml.tau.ac.il/' target='_blank'  rel='noopener noreferrer'>fastml<\/a>","<a href='https://www.bioinformatics.babraham.ac.uk/projects/fastqc/' target='_blank'  rel='noopener noreferrer'>FASTQC<\/a>","<a href='https://rajanil.github.io/fastStructure/' target='_blank'  rel='noopener noreferrer'>faststructure<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/' target='_blank'  rel='noopener noreferrer'>FASTX-Toolkit<\/a>","<a href='http://hannonlab.cshl.edu/fastx_toolkit/' target='_blank'  rel='noopener noreferrer'>FASTX-Toolkit<\/a>","<a href='http://www.softberry.com/berry.phtml?topic=fgenesh_plus_plus&group=help&subgroup=pipelines' target='_blank'  rel='noopener noreferrer'>FgenesH++<\/a>","<a href='http://animalbiosciences.uoguelph.ca/~msargol/fimpute/' target='_blank'  rel='noopener noreferrer'>FImpute<\/a>","<a href='https://github.com/smith-chem-wisc/FlashLFQ' target='_blank'  rel='noopener noreferrer'>flashLFQ<\/a>","<a href='https://www.fltk.org/index.php' target='_blank'  rel='noopener noreferrer'>fltk<\/a>","<a href='https://github.com/fenderglass/Flye' target='_blank'  rel='noopener noreferrer'>Flye<\/a>","<a href='https://sourceforge.net/projects/fpart/' target='_blank'  rel='noopener noreferrer'>fpart<\/a>","<a href='https://fpm.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>fpm<\/a>","<a href='https://sourceforge.net/projects/fraggenescan/' target='_blank'  rel='noopener noreferrer'>FragGeneScan<\/a>","<a href='https://github.com/ekg/freebayes' target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","<a href='https://surfer.nmr.mgh.harvard.edu/' target='_blank'  rel='noopener noreferrer'>freesurfer<\/a>","<a href='https://funannotate.readthedocs.io/en/latest/install.html' target='_blank'  rel='noopener noreferrer'>funannotate<\/a>","<a href='https://code.google.com/archive/p/garli/' target='_blank'  rel='noopener noreferrer'>GARLI<\/a>","<a href='https://github.com/broadinstitute/gatk' target='_blank'  rel='noopener noreferrer'>GATK<\/a>","<a href='http://molevol.cmima.csic.es/castresana/Gblocks_server.html' target='_blank'  rel='noopener noreferrer'>gblocks<\/a>","<a href='https://gcc.gnu.org/' target='_blank'  rel='noopener noreferrer'>GCC<\/a>","<a href='http://cnsgenomics.com/software/gcta/' target='_blank'  rel='noopener noreferrer'>gcta<\/a>","<a href='https://github.com/gnudatalanguage/gdl' target='_blank'  rel='noopener noreferrer'>GDL<\/a>","<a href='http://genome.crg.es/software/geneid/' target='_blank'  rel='noopener noreferrer'>geneid<\/a>","<a href='http://exon.gatech.edu/GeneMark/' target='_blank'  rel='noopener noreferrer'>Genemark<\/a>","<a href='https://github.com/schatzlab/genomescope' target='_blank'  rel='noopener noreferrer'>GenomeScope<\/a>","<a href='https://github.com/PacificBiosciences/GenomicConsensus' target='_blank'  rel='noopener noreferrer'>GenomicConsensus (Arrow)<\/a>","<a href='https://git-scm.com/about' target='_blank'  rel='noopener noreferrer'>Git<\/a>","<a href='http://glew.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>glew<\/a>","<a href='https://github.com/glfw/glfw' target='_blank'  rel='noopener noreferrer'>glfw<\/a>","<a href='https://sourceware.org/git/?p=glibc.git' target='_blank'  rel='noopener noreferrer'>glibc<\/a>","<a href='http://ccb.jhu.edu/software/glimmer/' target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='http://ccb.jhu.edu/software/glimmerhmm/' target='_blank'  rel='noopener noreferrer'>glimmerhmm<\/a>","<a href='https://github.com/GlobalArrays/ga' target='_blank'  rel='noopener noreferrer'>Global Arrays (GA) toolkit<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap<\/a>","<a href='http://research-pub.gene.com/gmap/' target='_blank'  rel='noopener noreferrer'>gmap_gsnap<\/a>","<a href='https://github.com/epam/GMP' target='_blank'  rel='noopener noreferrer'>gmp<\/a>","<a href='https://www.gnu.org/' target='_blank'  rel='noopener noreferrer'>GNU<\/a>","<a href='https://www.gnu.org/software/libtool/manual/libtool.html' target='_blank'  rel='noopener noreferrer'>GNU libtool<\/a>","<a href='https://www.gnu.org/software/parallel/' target='_blank'  rel='noopener noreferrer'>GNU parallel<\/a>","<a href='https://www.gnu.org/software/gsl/' target='_blank'  rel='noopener noreferrer'>GNU scientific library<\/a>","<a href='https://sourceforge.net/projects/cbadvanced/' target='_blank'  rel='noopener noreferrer'>GNU Tools<\/a>","<a href='https://geronimp.github.io/graftM/' target='_blank'  rel='noopener noreferrer'>GraftM<\/a>","<a href='https://github.com/biobakery/graphlan' target='_blank'  rel='noopener noreferrer'>GraPhlAn<\/a>","<a href='http://www.gromacs.org/' target='_blank'  rel='noopener noreferrer'>gromacs<\/a>","<a href='http://ecogenomics.github.io/GroopM/' target='_blank'  rel='noopener noreferrer'>groopm<\/a>","<a href='https://github.com/sanger-pathogens/gubbins' target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","guppy","<a href='https://hadoop.apache.org/' target='_blank'  rel='noopener noreferrer'>Hadoop<\/a>","<a href='https://github.com/vpc-ccg/haslr' target='_blank'  rel='noopener noreferrer'>HASLR<\/a>","<a href='http://amos.sourceforge.net/wiki/index.php?title=Hawkeye' target='_blank'  rel='noopener noreferrer'>Hawkeye<\/a>","<a href='https://github.com/SegataLab/hclust2' target='_blank'  rel='noopener noreferrer'>Hclust2<\/a>","<a href='https://support.hdfgroup.org/products/hdf4_tools/' target='_blank'  rel='noopener noreferrer'>hdf4<\/a>","<a href='https://support.hdfgroup.org/products/hdf5_tools/' target='_blank'  rel='noopener noreferrer'>hdf5<\/a>","<a href='https://github.com/higlass' target='_blank'  rel='noopener noreferrer'>HiGlass<\/a>","<a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4772016/' target='_blank'  rel='noopener noreferrer'>HiRise<\/a>","<a href='https://daehwankimlab.github.io/hisat2/' target='_blank'  rel='noopener noreferrer'>HISAT2<\/a>","<a href='http://hmmer.org/' target='_blank'  rel='noopener noreferrer'>HMMER<\/a>","<a href='https://github.com/horovod/horovod' target='_blank'  rel='noopener noreferrer'>horovod<\/a>","<a href='https://github.com/htseq/htseq' target='_blank'  rel='noopener noreferrer'>HTSeq<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>HTSlib<\/a>","<a href='https://github.com/loneknightpy/idba' target='_blank'  rel='noopener noreferrer'>idba-ud<\/a>","<a href='https://www.l3harrisgeospatial.com/Software-Technology/IDL' target='_blank'  rel='noopener noreferrer'>IDL<\/a>","<a href='https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler' target='_blank'  rel='noopener noreferrer'>Improved Phased Assembler (pbipa)<\/a>","<a href='http://eddylab.org/infernal/' target='_blank'  rel='noopener noreferrer'>Infernal<\/a>","<a href='http://software.broadinstitute.org/software/igv/home' target='_blank'  rel='noopener noreferrer'>Integrated Genomics Viewer<\/a>","<a href='https://interproscan-docs.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>InterProScan<\/a>","<a href='https://github.com/nerscadmin/IPM' target='_blank'  rel='noopener noreferrer'>ipm<\/a>","<a href='https://github.com/dereneaton/pyrad' target='_blank'  rel='noopener noreferrer'>iPyRAD<\/a>","<a href='http://www.iqtree.org/' target='_blank'  rel='noopener noreferrer'>IQ-TREE<\/a>","<a href='https://github.com/PacificBiosciences/IsoSeq' target='_blank'  rel='noopener noreferrer'>IsoSeq3<\/a>","<a href='http://mcmc-jags.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>jags<\/a>","<a href='https://github.com/genomecuration/JAMg' target='_blank'  rel='noopener noreferrer'>JAMg<\/a>","<a href='https://www.oracle.com/java/' target='_blank'  rel='noopener noreferrer'>Java<\/a>","<a href='https://github.com/GMOD/jbrowse' target='_blank'  rel='noopener noreferrer'>JBrowse<\/a>","<a href='http://www.genome.umd.edu/jellyfish.html' target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","<a href='https://github.com/aidenlab/Juicebox' target='_blank'  rel='noopener noreferrer'>juicebox<\/a>","<a href='https://github.com/aidenlab/juicer' target='_blank'  rel='noopener noreferrer'>Juicer<\/a>","<a href='https://julialang.org/' target='_blank'  rel='noopener noreferrer'>julia<\/a>","<a href='https://pachterlab.github.io/kallisto/about.html' target='_blank'  rel='noopener noreferrer'>Kallisto<\/a>","<a href='https://kat.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>KAT<\/a>","<a href='https://github.com/dib-lab/khmer' target='_blank'  rel='noopener noreferrer'>khmer<\/a>","<a href='https://github.com/refresh-bio/KMC' target='_blank'  rel='noopener noreferrer'>KMC<\/a>","<a href='https://ccb.jhu.edu/software/kraken/' target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://github.com/DerrickWood/kraken2/wiki' target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","<a href='https://github.com/marbl/Krona/wiki' target='_blank'  rel='noopener noreferrer'>krona<\/a>","<a href='https://kubernetes.io/' target='_blank'  rel='noopener noreferrer'>Kubernetes<\/a>","<a href='http://www.bx.psu.edu/~rsharris/lastz/' target='_blank'  rel='noopener noreferrer'>lastz<\/a>","<a href='https://github.com/MonashBioinformaticsPlatform/LFQ-Analyst' target='_blank'  rel='noopener noreferrer'>LFQ-Analyst<\/a>","<a href='https://ofiwg.github.io/libfabric/' target='_blank'  rel='noopener noreferrer'>libfabric<\/a>","<a href='https://github.com/agordon/libgtextutils' target='_blank'  rel='noopener noreferrer'>libgtextutils<\/a>","<a href='https://www.nongnu.org/libunwind/' target='_blank'  rel='noopener noreferrer'>libunwind<\/a>","<a href='https://github.com/bcgsc/LINKS' target='_blank'  rel='noopener noreferrer'>LINKS<\/a>","<a href='https://www.lipidmaps.org/' target='_blank'  rel='noopener noreferrer'>Lipid Maps<\/a>","<a href='https://llvm.org/' target='_blank'  rel='noopener noreferrer'>llvm<\/a>","<a href='https://lmod.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>lmod<\/a>","<a href='https://support.10xgenomics.com/genome-exome/software/pipelines/latest/what-is-long-ranger' target='_blank'  rel='noopener noreferrer'>Long Ranger<\/a>","<a href='http://csg.sph.umich.edu/abecasis/mach/index.html' target='_blank'  rel='noopener noreferrer'>mach<\/a>","<a href='https://github.com/macs3-project/MACS' target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='https://github.com/macs3-project/MACS' target='_blank'  rel='noopener noreferrer'>macs2<\/a>","<a href='https://github.com/ranwez/MACSE_V2_PIPELINES' target='_blank'  rel='noopener noreferrer'>MACSE<\/a>","<a href='http://mafft.cbrc.jp/alignment/software/' target='_blank'  rel='noopener noreferrer'>mafft<\/a>","<a href='https://github.com/WatsonLab/MAGpy' target='_blank'  rel='noopener noreferrer'>MAGpy<\/a>","<a href='http://www.yandell-lab.org/software/maker.html' target='_blank'  rel='noopener noreferrer'>MAKER<\/a>","<a href='http://www.yandell-lab.org/software/maker-p.html' target='_blank'  rel='noopener noreferrer'>MAKER-P<\/a>","<a href='https://cran.r-project.org/web/packages/MALDIquant/index.html' target='_blank'  rel='noopener noreferrer'>MALDIquant<\/a>","<a href='https://github.com/marbl/MashMap' target='_blank'  rel='noopener noreferrer'>mashmap<\/a>","<a href='https://github.com/kwongj/maskrc-svg' target='_blank'  rel='noopener noreferrer'>maskrc-svg<\/a>","<a href='https://github.com/alekseyzimin/masurca' target='_blank'  rel='noopener noreferrer'>MaSuRCA<\/a>","<a href='https://www.mathworks.com/products/matlab.html' target='_blank'  rel='noopener noreferrer'>MATLAB<\/a>","<a href='https://github.com/dtreichler/matserver' target='_blank'  rel='noopener noreferrer'>matserver<\/a>","<a href='http://darlinglab.org/mauve/download.html' target='_blank'  rel='noopener noreferrer'>Mauve<\/a>","<a href='https://maven.apache.org/' target='_blank'  rel='noopener noreferrer'>maven<\/a>","<a href='https://sourceforge.net/projects/maxbin2/' target='_blank'  rel='noopener noreferrer'>MaxBin2<\/a>","<a href='https://www.maxquant.org/' target='_blank'  rel='noopener noreferrer'>MaxQuant<\/a>","<a href='https://www.maxquant.org/' target='_blank'  rel='noopener noreferrer'>MaxQuant (biocontainers)<\/a>","<a href='https://micans.org/mcl/' target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://github.com/nanoporetech/medaka' target='_blank'  rel='noopener noreferrer'>Medaka<\/a>","<a href='https://github.com/voutcn/megahit' target='_blank'  rel='noopener noreferrer'>megahit<\/a>","<a href='https://github.com/marbl/merqury' target='_blank'  rel='noopener noreferrer'>Merqury<\/a>","<a href='https://bitbucket.org/berkeleylab/metabat/src/master/' target='_blank'  rel='noopener noreferrer'>metabat<\/a>","<a href='https://sourceforge.net/projects/metacv/' target='_blank'  rel='noopener noreferrer'>metacv<\/a>","<a href='https://github.com/DiltheyLab/MetaMaps' target='_blank'  rel='noopener noreferrer'>MetaMaps<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/metaMS.html' target='_blank'  rel='noopener noreferrer'>metaMS<\/a>","<a href='https://huttenhower.sph.harvard.edu/metaphlan' target='_blank'  rel='noopener noreferrer'>MetaPhlAn<\/a>","<a href='http://cab.spbu.ru/software/spades/' target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","<a href='https://peterbeerli.com/migrate-html5/index.html' target='_blank'  rel='noopener noreferrer'>MIGRATE<\/a>","<a href='https://gatb.inria.fr/software/minia/' target='_blank'  rel='noopener noreferrer'>minia<\/a>","<a href='https://genome.sph.umich.edu/wiki/Minimac4' target='_blank'  rel='noopener noreferrer'>Minimac4<\/a>","<a href='https://github.com/lh3/minimap2' target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","<a href='https://sourceforge.net/p/mira-assembler/wiki/Home/' target='_blank'  rel='noopener noreferrer'>mira<\/a>","<a href='https://www.nature.com/articles/nbt1394#Sec11' target='_blank'  rel='noopener noreferrer'>mirdeep<\/a>","<a href='https://github.com/rajewsky-lab/mirdeep2' target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","<a href='https://github.com/yarden/MISO' target='_blank'  rel='noopener noreferrer'>miso<\/a>","<a href='http://mixomics.org/' target='_blank'  rel='noopener noreferrer'>mixOmics<\/a>","<a href='https://github.com/tseemann/mlst' target='_blank'  rel='noopener noreferrer'>MLST<\/a>","<a href='https://github.com/compomics/moFF' target='_blank'  rel='noopener noreferrer'>moFF<\/a>","<a href='https://www.mono-project.com/' target='_blank'  rel='noopener noreferrer'>Mono<\/a>","<a href='https://cwenger.github.io/Morpheus/' target='_blank'  rel='noopener noreferrer'>Morpheus<\/a>","<a href='https://www.mothur.org/' target='_blank'  rel='noopener noreferrer'>Mothur<\/a>","<a href='https://www.mpfr.org/' target='_blank'  rel='noopener noreferrer'>mpfr<\/a>","mpiBLAST","<a href='https://github.com/LLNL/mpiP' target='_blank'  rel='noopener noreferrer'>mpiP (mpi profiling library)<\/a>","<a href='https://github.com/NBISweden/MrBayes' target='_blank'  rel='noopener noreferrer'>MrBayes<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSGFplus.html' target='_blank'  rel='noopener noreferrer'>MS-GF+<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/MSnbase.html' target='_blank'  rel='noopener noreferrer'>MSnbase readMSData<\/a>","<a href='https://bitbucket.org/joseph07/multisnv/wiki/Home' target='_blank'  rel='noopener noreferrer'>multisnv<\/a>","<a href='http://mummer.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>MUMmer<\/a>","<a href='http://www.drive5.com/muscle/' target='_blank'  rel='noopener noreferrer'>MUSCLE<\/a>","<a href='https://mvapich.cse.ohio-state.edu/' target='_blank'  rel='noopener noreferrer'>MVAPICH<\/a>","<a href='https://software.opensuse.org/package/mxml' target='_blank'  rel='noopener noreferrer'>mxml<\/a>","<a href='https://www.nag.com/' target='_blank'  rel='noopener noreferrer'>nag<\/a>","<a href='https://github.com/jts/nanopolish' target='_blank'  rel='noopener noreferrer'>Nanopolish<\/a>","<a href='https://www.nasm.us/' target='_blank'  rel='noopener noreferrer'>NASM<\/a>","<a href='https://github.com/NVIDIA/nccl' target='_blank'  rel='noopener noreferrer'>NCCL<\/a>","<a href='https://gitlab.com/remikz/nccmp' target='_blank'  rel='noopener noreferrer'>nccmp<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml' target='_blank'  rel='noopener noreferrer'>NCL<\/a>","<a href='https://www.ncl.ucar.edu/index.shtml' target='_blank'  rel='noopener noreferrer'>NCL<\/a>","<a href='http://nco.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>NCO<\/a>","<a href='http://meteora.ucsd.edu/~pierce/ncview_home_page.html' target='_blank'  rel='noopener noreferrer'>Ncview<\/a>","<a href='https://github.com/xiaochuanle/NECAT' target='_blank'  rel='noopener noreferrer'>NECAT<\/a>","<a href='https://www.unidata.ucar.edu/software/netcdf/' target='_blank'  rel='noopener noreferrer'>NetCDF<\/a>","<a href='https://github.com/neurodebian/neurodebian' target='_blank'  rel='noopener noreferrer'>neurodebian<\/a>","<a href='https://github.com/Nextomics/NextDenovo' target='_blank'  rel='noopener noreferrer'>NextDenovo<\/a>","<a href='https://www.nextflow.io/' target='_blank'  rel='noopener noreferrer'>nextflow<\/a>","<a href='http://cibiv.github.io/NextGenMap/' target='_blank'  rel='noopener noreferrer'>nextgenmap<\/a>","<a href='https://github.com/Nextomics/NextPolish' target='_blank'  rel='noopener noreferrer'>NextPolish<\/a>","<a href='https://github.com/ncbi/ngs' target='_blank'  rel='noopener noreferrer'>NGS<\/a>","<a href='https://github.com/shenlab-sinai/ngsplot' target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","<a href='https://nucleoatac.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>nucleoatac<\/a>","<a href='https://github.com/NVlabs/nvbio' target='_blank'  rel='noopener noreferrer'>nvbio<\/a>","<a href='https://developer.nvidia.com/hpc-sdk' target='_blank'  rel='noopener noreferrer'>NVIDIA HPC SDK<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/oases/' target='_blank'  rel='noopener noreferrer'>Oases<\/a>","<a href='https://www.vi-hps.org/tools/opari2.html' target='_blank'  rel='noopener noreferrer'>OPARI2<\/a>","<a href='https://www.open-mpi.org/' target='_blank'  rel='noopener noreferrer'>Open MPI<\/a>","<a href='https://www.open-mpi.org/' target='_blank'  rel='noopener noreferrer'>Open MPI (ib)<\/a>","<a href='https://www.open-mpi.org/' target='_blank'  rel='noopener noreferrer'>Open MPI 2 (ib)<\/a>","<a href='https://www.openms.de/' target='_blank'  rel='noopener noreferrer'>OpenMS<\/a>","<a href='https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper' target='_blank'  rel='noopener noreferrer'>OPLS-DA_Contrasts (w4mcorcov)<\/a>","<a href='https://github.com/kishwarshafin/pepper' target='_blank'  rel='noopener noreferrer'>P.E.P.P.E.R. (+ deep variant)<\/a>","<a href='https://github.com/TomSkelly/PacBioEDA' target='_blank'  rel='noopener noreferrer'>PacBioEDA<\/a>","padb","<a href='http://icl.utk.edu/papi/software/' target='_blank'  rel='noopener noreferrer'>PAPI (Performance Application Programming Interface)<\/a>","<a href='https://www.vi-hps.org/tools/paraver.html' target='_blank'  rel='noopener noreferrer'>PARAVER<\/a>","<a href='https://www.paraview.org/' target='_blank'  rel='noopener noreferrer'>ParaView<\/a>","<a href='https://github.com/marbl/parsnp' target='_blank'  rel='noopener noreferrer'>Parsnp<\/a>","<a href='https://github.com/PASApipeline/PASApipeline/wiki' target='_blank'  rel='noopener noreferrer'>PASA<\/a>","<a href='https://github.com/NixOS/patchelf' target='_blank'  rel='noopener noreferrer'>PatchELF<\/a>","<a href='https://downloadcenter.intel.com/download/19792' target='_blank'  rel='noopener noreferrer'>PDT (Processor Diagnostic Tool)<\/a>","<a href='http://compomics.github.io/projects/peptide-shaker.html' target='_blank'  rel='noopener noreferrer'>Peptide Shaker<\/a>","<a href='http://percolator.ms/' target='_blank'  rel='noopener noreferrer'>Percolator<\/a>","<a href='https://www.php.net/' target='_blank'  rel='noopener noreferrer'>PHP<\/a>","<a href='https://broadinstitute.github.io/picard/' target='_blank'  rel='noopener noreferrer'>PICARD<\/a>","<a href='http://picrust.github.io/picrust/' target='_blank'  rel='noopener noreferrer'>PICRUSt<\/a>","<a href='https://github.com/picrust/picrust2/wiki' target='_blank'  rel='noopener noreferrer'>PICRUSt 2<\/a>","<a href='https://www.drive5.com/piler/' target='_blank'  rel='noopener noreferrer'>PILER-DF<\/a>","<a href='https://github.com/broadinstitute/pilon/wiki' target='_blank'  rel='noopener noreferrer'>Pilon<\/a>","<a href='http://zzz.bwh.harvard.edu/plink/' target='_blank'  rel='noopener noreferrer'>plink<\/a>","<a href='http://atgu.mgh.harvard.edu/plinkseq/index.shtml' target='_blank'  rel='noopener noreferrer'>plinkseq<\/a>","<a href='https://parallel-netcdf.github.io/' target='_blank'  rel='noopener noreferrer'>PnetCDF<\/a>","<a href='https://github.com/matsen/pplacer' target='_blank'  rel='noopener noreferrer'>pplacer<\/a>","<a href='https://github.com/wtsi-hpag/PretextView' target='_blank'  rel='noopener noreferrer'>PretextView / Pretext Suite<\/a>","<a href='http://prinseq.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>PRINSEQ<\/a>","<a href='http://www.genabel.org/packages/ProbABEL' target='_blank'  rel='noopener noreferrer'>probabel<\/a>","<a href='https://github.com/rsilvabioinfo/ProbMetab' target='_blank'  rel='noopener noreferrer'>ProbMetab Tool<\/a>","<a href='https://github.com/hyattpd/Prodigal' target='_blank'  rel='noopener noreferrer'>Prodigal<\/a>","<a href='https://github.com/tseemann/prokka' target='_blank'  rel='noopener noreferrer'>Prokka<\/a>","<a href='https://github.com/dfguan/purge_dups' target='_blank'  rel='noopener noreferrer'>Purge_Dups<\/a>","<a href='https://bitbucket.org/mroachawri/purge_haplotigs/src/master/' target='_blank'  rel='noopener noreferrer'>PurgeHaplotigs<\/a>","<a href='https://pylib.readthedocs.io/en/1.4.3/index.html' target='_blank'  rel='noopener noreferrer'>pylibs<\/a>","<a href='https://ipyrad.readthedocs.io/en/latest/' target='_blank'  rel='noopener noreferrer'>PyRAD<\/a>","<a href='https://github.com/pysam-developers/pysam' target='_blank'  rel='noopener noreferrer'>pysam<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python2<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python2-as-python<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python3<\/a>","<a href='https://www.python.org/' target='_blank'  rel='noopener noreferrer'>python3-as-python<\/a>","<a href='https://pytorch.org/' target='_blank'  rel='noopener noreferrer'>PyTorch<\/a>","<a href='http://qiime.org/' target='_blank'  rel='noopener noreferrer'>QIIME<\/a>","<a href='http://qiime.org/' target='_blank'  rel='noopener noreferrer'>QIIME<\/a>","<a href='https://qiime2.org/' target='_blank'  rel='noopener noreferrer'>QIIME2.0<\/a>","<a href='http://bioinf.spbau.ru/quast' target='_blank'  rel='noopener noreferrer'>QUAST<\/a>","<a href='https://github.com/mahulchak/quickmerge' target='_blank'  rel='noopener noreferrer'>Quickmerge<\/a>","<a href='https://www.r-project.org/' target='_blank'  rel='noopener noreferrer'>R<\/a>","<a href='https://www.rabbitmq.com/' target='_blank'  rel='noopener noreferrer'>RabbitMQ server<\/a>","<a href='https://github.com/isovic/racon' target='_blank'  rel='noopener noreferrer'>Racon<\/a>","<a href='https://github.com/bcgsc/RAILS' target='_blank'  rel='noopener noreferrer'>RAILS/Cobbler<\/a>","<a href='http://ratt.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>RATT<\/a>","<a href='https://github.com/comprna/RATTLE' target='_blank'  rel='noopener noreferrer'>RATTLE<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/galaxyp/rawtools/51e4aa55ea04' target='_blank'  rel='noopener noreferrer'>Raw Tools<\/a>","<a href='https://sco.h-its.org/exelixis/web/software/raxml/index.html' target='_blank'  rel='noopener noreferrer'>RAxML<\/a>","<a href='https://github.com/dparks1134/RefineM' target='_blank'  rel='noopener noreferrer'>RefineM<\/a>","<a href='https://github.com/simoncchu/REPdenovo' target='_blank'  rel='noopener noreferrer'>REPdenovo<\/a>","<a href='http://www.repeatmasker.org/' target='_blank'  rel='noopener noreferrer'>RepeatMasker<\/a>","<a href='http://www.repeatmasker.org/RepeatModeler/' target='_blank'  rel='noopener noreferrer'>RepeatModeler<\/a>","<a href='https://github.com/Dfam-consortium/RepeatModeler' target='_blank'  rel='noopener noreferrer'>RepeatModeler2<\/a>","<a href='https://www.yandell-lab.org/software/repeatrunner.html' target='_blank'  rel='noopener noreferrer'>RepeatRunner<\/a>","<a href='https://urgi.versailles.inra.fr/Tools/REPET' target='_blank'  rel='noopener noreferrer'>REPET<\/a>","<a href='https://cran.r-project.org/web/packages/rjags/index.html' target='_blank'  rel='noopener noreferrer'>rjags<\/a>","<a href='https://deweylab.github.io/RSEM/' target='_blank'  rel='noopener noreferrer'>RSEM<\/a>","<a href='http://rseqc.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","<a href='https://rstudio.com/' target='_blank'  rel='noopener noreferrer'>Rstudio<\/a>","<a href='https://www.ruby-lang.org/en/downloads/' target='_blank'  rel='noopener noreferrer'>Ruby<\/a>","<a href='https://combine-lab.github.io/salmon/' target='_blank'  rel='noopener noreferrer'>Salmon<\/a>","<a href='https://github.com/GregoryFaust/samblaster' target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='http://www.htslib.org/' target='_blank'  rel='noopener noreferrer'>SAMTools<\/a>","<a href='https://www.sas.com/en_us/software/foundation.html' target='_blank'  rel='noopener noreferrer'>SAS<\/a>","<a href='http://compbio.cs.toronto.edu/hapsembler/scarpa.html' target='_blank'  rel='noopener noreferrer'>Scarpa<\/a>","<a href='https://www.scipy.org/' target='_blank'  rel='noopener noreferrer'>SciPy<\/a>","<a href='https://scons.org/' target='_blank'  rel='noopener noreferrer'>SCons<\/a>","<a href='https://pypi.python.org/pypi/screed' target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='http://compomics.github.io/projects/searchgui.html' target='_blank'  rel='noopener noreferrer'>Search GUI<\/a>","<a href='https://github.com/shenwei356/seqkit' target='_blank'  rel='noopener noreferrer'>SeqKit<\/a>","<a href='https://github.com/drostlab/seqstats' target='_blank'  rel='noopener noreferrer'>seqstats<\/a>","<a href='https://github.com/trinityrnaseq/seqtk-trinity' target='_blank'  rel='noopener noreferrer'>seqtk-trinity<\/a>","<a href='https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html' target='_blank'  rel='noopener noreferrer'>shapeit<\/a>","<a href='https://github.com/najoshi/sickle' target='_blank'  rel='noopener noreferrer'>sickle<\/a>","<a href='https://www.bryntum.com/products/siesta/' target='_blank'  rel='noopener noreferrer'>Siesta<\/a>","<a href='https://singularity.lbl.gov/' target='_blank'  rel='noopener noreferrer'>singularity<\/a>","<a href='https://github.com/relipmoc/skewer' target='_blank'  rel='noopener noreferrer'>skewer<\/a>","<a href='https://www.pacb.com/support/software-downloads/' target='_blank'  rel='noopener noreferrer'>SMRT Link / Analysis<\/a>","<a href='https://snakemake.readthedocs.io/en/stable/' target='_blank'  rel='noopener noreferrer'>Snakemake<\/a>","<a href='https://github.com/KorfLab/SNAP' target='_blank'  rel='noopener noreferrer'>SNAP<\/a>","<a href='https://github.com/PMMAraujo/snappy' target='_blank'  rel='noopener noreferrer'>SNAPPy<\/a>","<a href='http://lowelab.ucsc.edu/snoscan/' target='_blank'  rel='noopener noreferrer'>snoscan<\/a>","<a href='http://snpeff.sourceforge.net/' target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","soap","<a href='http://i.cs.hku.hk/2bwt-tools/soap3/index.html' target='_blank'  rel='noopener noreferrer'>soap3-gpu<\/a>","<a href='https://github.com/aquaskyline/SOAPdenovo2' target='_blank'  rel='noopener noreferrer'>SOAPdenovo2<\/a>","<a href='https://sourceforge.net/projects/soapsnp/' target='_blank'  rel='noopener noreferrer'>SOAPsnp<\/a>","<a href='https://github.com/biocore/sortmerna' target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='https://github.com/ablab/spades' target='_blank'  rel='noopener noreferrer'>SPAdes<\/a>","<a href='https://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi' target='_blank'  rel='noopener noreferrer'>Splign<\/a>","<a href='https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=toolkit_doc' target='_blank'  rel='noopener noreferrer'>sra-tools<\/a>","<a href='http://catchenlab.life.illinois.edu/stacks/' target='_blank'  rel='noopener noreferrer'>Stacks<\/a>","<a href='https://github.com/alexdobin/STAR' target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='http://web.stanford.edu/group/pritchardlab/structure.html' target='_blank'  rel='noopener noreferrer'>structure<\/a>","<a href='https://support.10xgenomics.com/de-novo-assembly/software/overview/latest/welcome' target='_blank'  rel='noopener noreferrer'>Supernova<\/a>","<a href='https://github.com/hall-lab/svtyper' target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","<a href='https://github.com/TimoLassmann/tagdust' target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://bitbucket.org/tasseladmin/tassel-5-standalone' target='_blank'  rel='noopener noreferrer'>tassel<\/a>","<a href='http://www.cs.uoregon.edu/research/tau/home.php' target='_blank'  rel='noopener noreferrer'>TAU<\/a>","<a href='https://www.tensorflow.org/resources/tools' target='_blank'  rel='noopener noreferrer'>TensorFlow<\/a>","<a href='https://github.com/compomics/ThermoRawFileParser' target='_blank'  rel='noopener noreferrer'>ThermoRawFileParser<\/a>","<a href='https://github.com/bcgsc/tigmint' target='_blank'  rel='noopener noreferrer'>Tigmint<\/a>","<a href='https://ccb.jhu.edu/software/tophat/index.shtml' target='_blank'  rel='noopener noreferrer'>tophat<\/a>","<a href='https://totalview.io/' target='_blank'  rel='noopener noreferrer'>TotalView<\/a>","<a href='https://github.com/TransDecoder/TransDecoder/wiki' target='_blank'  rel='noopener noreferrer'>TransDecoder<\/a>","<a href='https://tandem.bu.edu/trf/trf.html' target='_blank'  rel='noopener noreferrer'>trf<\/a>","<a href='http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/' target='_blank'  rel='noopener noreferrer'>Trim Galore<\/a>","<a href='http://www.usadellab.org/cms/?page=trimmomatic' target='_blank'  rel='noopener noreferrer'>Trimmomatic<\/a>","<a href='https://github.com/trinityrnaseq/trinityrnaseq/wiki' target='_blank'  rel='noopener noreferrer'>Trinity / trinityrnaseq<\/a>","<a href='https://github.com/Trinotate/Trinotate.github.io/wiki' target='_blank'  rel='noopener noreferrer'>Trinotate<\/a>","<a href='http://eddylab.org/software.html' target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","<a href='http://hgdownload.soe.ucsc.edu' target='_blank'  rel='noopener noreferrer'>UCSC tools<\/a>","<a href='https://www.openucx.org/' target='_blank'  rel='noopener noreferrer'>UCX (Unified Communication X)<\/a>","<a href='https://www.unidata.ucar.edu/software/udunits/' target='_blank'  rel='noopener noreferrer'>UDUNITS<\/a>","<a href='https://github.com/rrwick/Unicycler' target='_blank'  rel='noopener noreferrer'>Unicycler<\/a>","<a href='http://www.unixodbc.org/utilities.html' target='_blank'  rel='noopener noreferrer'>unixODBC<\/a>","<a href='https://github.com/marbl/VALET' target='_blank'  rel='noopener noreferrer'>Valet<\/a>","<a href='https://github.com/cc2qe/vawk' target='_blank'  rel='noopener noreferrer'>vawk<\/a>","<a href='https://vcftools.github.io/index.html' target='_blank'  rel='noopener noreferrer'>VCFTools<\/a>","<a href='https://www.ebi.ac.uk/~zerbino/velvet/' target='_blank'  rel='noopener noreferrer'>Velvet<\/a>","<a href='https://www.tbi.univie.ac.at/RNA/documentation.html' target='_blank'  rel='noopener noreferrer'>ViennaRNA<\/a>","<a href='https://wci.llnl.gov/simulation/computer-codes/visit/' target='_blank'  rel='noopener noreferrer'>VisIt<\/a>","<a href='https://github.com/torognes/vsearch' target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","<a href='https://github.com/adigenova/wengan' target='_blank'  rel='noopener noreferrer'>Wengan<\/a>","<a href='http://nebc.nox.ac.uk/bioinformatics/docs/windowmasker.html' target='_blank'  rel='noopener noreferrer'>WindowMasker<\/a>","<a href='https://www.winehq.org/' target='_blank'  rel='noopener noreferrer'>Wine<\/a>","<a href='http://www.ebi.ac.uk/~birney/wise2/' target='_blank'  rel='noopener noreferrer'>wise<\/a>","<a href='https://workflow4metabolomics.org/' target='_blank'  rel='noopener noreferrer'>Workflow 4 Metabolomics<\/a>","<a href='https://github.com/ruanjue/wtdbg2' target='_blank'  rel='noopener noreferrer'>WTDBG2<\/a>","<a href='https://www.wxwidgets.org/' target='_blank'  rel='noopener noreferrer'>wxWidgets<\/a>","<a href='http://bioconductor.org/packages/release/bioc/html/xcms.html' target='_blank'  rel='noopener noreferrer'>xcms<\/a>","<a href='https://xerces.apache.org/xerces-c/' target='_blank'  rel='noopener noreferrer'>Xerces-C++ XML parser<\/a>","<a href='https://github.com/AustralianBioCommons/xml4ena' target='_blank'  rel='noopener noreferrer'>xml4ena<\/a>","<a href='https://github.com/freedesktop/xorg-macros' target='_blank'  rel='noopener noreferrer'>xorg-macros<\/a>","<a href='https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem/29e2df3edf11' target='_blank'  rel='noopener noreferrer'>xtandem<\/a>","<a href='https://zlib.net/' target='_blank'  rel='noopener noreferrer'>zlib<\/a>","zstd"],["3D-DNA","abricate","abyss","allmaps","allpathslg","amos","anaconda","angsd","ant","apollo","arima mapping","arm-forge","arm-reports","artemis","aspera","augustus","autoconf","automake","bamkit","bamm","bamtools","batchcorrection","bazel","bbcp","bbmap","bcftools","beagle","beagle-lib","beast","beast2","bedtools","binutils","bioconductor","biolinux","Bionano Access","bioperl","biopython","bismark","blast","blast+","blast2","blat","BlobToolKit","blobtools","boost","bowtie","bowtie2","braker1","braker2","breakdancer","bricks","bsmap","bs_seeker","bsseeker2","busco","bwa","bx-python","bzip2","cactus","camera","canu","cap3","cardinal","cat_bins","cdhit","celseq2","celera","centos","centos6","checkm","checkm-genome","chunkchromosome","clustalw","clusteringsuite","cmake","cnvkit","cpc","cpc2","coreutils","cpat","cppunit","cube","cuda","cudnn","cufflinks","curl","cutadapt","cytoscape","darshan","deeptools","deepvariant","dendropy","DIA-Umpire","diamond","discovar_de_novo","dms_tools2","dmtcp","dssp","easybuild","edena","embedded-nimrod","emboss","ensembl","genebuild","erlang","eval","EvidenceModeler","exabayes","exonerate","export2graphlan","extrae","falcon","pb-assembly","fastml","fastqc","faststructure","fastx_toolkit","fastx","fgenesh","fimpute","flashlfq","fltk","flye","fpart","fpm","fraggenescan","freebayes","freesurfer","funannotate","garli","gatk","gblock","gcc","gcta","gdl","geneid","genemark","genomescope","GenomicConsensus","git","glew","glfw","glibc","glimmer","glimmerhmm","ga","gmap","gmap_gsnap","gmp","gnu","libtool","parallel","gsl","gnutools","graftm","graphlan","gromacs","groopm","gubbins","guppy","hadoop","haslr","hawkeye","hclust2","hdf4","hdf5","higlass","hirise","hisat2","hmmer","horovod","htseq","htslib","idba-ud","idl","pbipa","infernal","igv","interproscan","ipm","ipyrad","iq-tree","isoseq3","jags","jamg","java","jbrowse","jellyfish","juicebox","juicer","julia","kallisto","kat","khmer","kmc","kraken","kraken2","krona","k8","lastz","LFQ-Analyst","libfabric","libgtextutils","libunwind","links","lipid_maps","llvm","lmod","longranger","mach","macs","macs2","macse","mafft","magpy","maker","maker-p","maldi_quant","mashmap","maskrc-svg","masurca","matlab","matserver","mauve","maven","maxbin","maxquant","biocontainers-maxquant","mcl","medaka","megahit","merqury","metabat","metacv","metamaps","metams","metaphlan","metaspades","MIGRATE","minia","minimac4","minimap2","mira","mirdeep","mirdeep2","miso","mixomics","mlst","moff","mono","morpheus","Mothur","mpfr","mpiblast","mpip","mrbayes","msgfplus","msnbase","multisnv","mummer","muscle","mvapich2_ib","mxml","nag","nanopolish","nasm","nccl","nccmp","ncl","ncl_ncarg","nco","ncview","necat","netcdf","neurodebian","nextdenovo","nextflow","nextgenmap","nextpolish","ngs","ngsplot","nucleoatac","nvbio","nvidia-hpc-sdk","oases","opari2","openmpi","openmpi_ib","openmpi2_ib","openms","w4mcorcov","pepper+deepvariant","pacbioeda","padb","papi","paraver","paraview","parsnp","PASA","patchelf","pdt","peptideshaker","percolator","php","picard","picrust","picrust2","piler-df","pilon","plink","plinkseq","pnetcdf","pplacer","pretextview","prinseq","probabel","probmetab","prodigal","prokka","purge_dups","purgehaplotigs","pylibs","pyrad","pysam","python","python2","python2-as-python","python3","python3-as-python","pytorch","qiime","qiime-modules","qiime2","quast","quickmerge","r","rabbitmq-server","Racon","rails_cobbler","ratt","rattle","rawtools","raxml","refinem","repdenovo","repeatmasker","repeatmodeler","repeatmodeler2","repeatrunner","repet","rjags","rsem","rseqc","rstudio","ruby","salmon","samblaster","samtools","sas","scarpa","scipy","scons","screed","searchgui","biocontainers-seqkit","seqstats","seqtk-trinity","shapeit","sickle","siesta","singularity","skewer","smrtlink","snakemake","snap","snappy","snoscan","snpeff","soap","soap3-gpu","soapdenovo2","soapsnp","sortmerna","spades","splign","sra-tools","stacks","star","structure","supernova","svtyper","tagdust","tassel","tau","tensorflow","ThermoRawFileParser","tigmint","tophat","totalview","transdecoder","trf","trim_galore","trimmomatic","trinity","trinotate","trnascan-se","ucsc","ucx","udunits","unicycler","unixodbc","valet","vawk","vcftools","velvet","viennarna","visit","vsearch","wengan","windowmasker","wine","wise","workflow4metabolomics","wtdbg2","wxwidgets","xcms","xerces-c","xml4ena","xorg-macros","xtandem","zlib","zstd"],["Sequence assembly","Sequence analysis","Sequence assembly","Sequence assembly","Sequence assembly","Sequence assembly","Gene structure","Population genomics","","Genomics / Genome annotation","Chromosome conformation capture","","","Genomics / Genome annotation","","Genomics / Genome annotation","","","Sequence analysis","Phylogenomics","Sequence analysis","Metabolomics","","","Sequence analysis","Sequence analysis","Population genetics","Population genomics","Phylogenetics","Phylogenetics","Sequence analysis","","","","Sequence analysis","Programming language / platform","Programming language / platform","Epigenomics","Sequence alignment","Sequence alignment","Sequence alignment","Sequence alignment","Quality assurance","Quality assurance","Library","Sequence alignment","Sequence alignment","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","","Epigenomics","Epigenomics","Epigenomics","Quality assurance","Sequence alignment","Sequence analysis","","Sequence alignment","Metabolomics","Sequence assembly","Sequence assembly","Bioimaging","Metagenomics","Sequence analysis","Transcriptomics","Sequence assembly","","","Quality assurance","Quality assurance","Sequence analysis","Sequence alignment","Statistics and probability","","DNA structural variation","Genomics / Genome annotation","Genomics / Genome annotation","Library","Genomics / Genome annotation","","","","Library","Sequence assembly","","Sequence analysis","Multiomics","","Sequence analysis","Variant Calling","Phylogenomics","Proteomics / Metaproteomics","Sequence alignment","Sequence assembly","Genetic variation","","Protein folding, stability and design","","Sequence assembly","","Molecular biology","Data visualisation","Genomics / Genome annotation","Programming language / platform","Quality assurance","Genomics / Genome annotation","Phylogenomics","Sequence analysis","Phylogenomics","","","Sequence assembly","Phylogenomics","Quality assurance","Phylogenomics","Sequence analysis","Sequence analysis","Genomics / Genome annotation","Population genomics","Proteomics / Metaproteomics","","Sequence assembly","","","Sequence analysis","Phylogenomics","Bioimaging","Genomics / Genome annotation","Phylogenetics","Sequence analysis","Phylogenetics","Library","Phylogenomics","Library","Genomics / Genome annotation","Genomics / Genome annotation","Sequence analysis","Sequence assembly","","Library","Library","Library","Genomics / Genome annotation","Genomics / Genome annotation","","Sequence alignment","Sequence alignment","","Programming language / platform","Library","","Library","","Meta-Omics","Phylogenomics","Molecular dynamics","Metagenomics","Phylogenomics","Sequencing","Library","Sequence assembly","Quality assurance","Data visualisation","","","Data visualisation","Sequence assembly","Sequence alignment","Sequence analysis","","Sequence analysis","Library","Sequence assembly","Programming language / platform","Sequence assembly","Genomics / Genome annotation","Data visualisation","Protein sequence features","","Sequence analysis","Phylogenomics","Transcriptomics","Phylogenomics","Genomics / Genome annotation","Programming language / platform","Data visualisation","Sequence analysis","Sequence assembly","Sequence assembly","Programming language / platform","Transcriptomics","Sequence analysis","Sequence analysis","Sequence analysis","Taxonomy","Taxonomy","Data visualisation","Programming language / platform","Sequence alignment","Proteomics / Metaproteomics","Library","Library","","Sequence assembly","Metabolomics","","","Sequence alignment","Phylogenomics","ChIP-seq","ChIP-seq","Sequence alignment","Sequence alignment","Metagenomics","Genomics / Genome annotation","Genomics / Genome annotation","Proteomics","Sequence alignment","","Sequence assembly","","","Sequence alignment","","Metagenomics","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Statistics and probability","Sequence assembly","Metagenomics","Quality assurance","Metagenomics","Metagenomics","Metagenomics","Metabolomics","Metagenomics","Metagenomics","Population genomics","Sequence assembly","Phylogenomics","Sequence alignment","Sequence assembly","Transcriptomics","Transcriptomics","Transcriptomics","Multiomics","Sequence analysis","Proteomics / Metaproteomics","Programming language / platform","Proteomics / Metaproteomics","Metagenomics","","Sequence alignment","Library","Phylogenomics","Proteomics / Metaproteomics","Metabolomics","Variant Calling","Sequence alignment","Genomics / Genome annotation","","Library","Library","Sequence analysis","","","","Data visualisation","Data visualisation","","","Sequence assembly","Library","Neuroscience","Sequence assembly","Workflows","Sequence alignment","Sequence assembly","Knowledge discovery in databases","Data visualisation","Genomics / Genome annotation","Library","Library","Sequence assembly","","","","","Proteomics / Metaproteomics","Metabolomics","Sequence assembly","Sequence analysis","","","Data visualisation","Data visualisation","Phylogenomics","Genomics / Genome annotation","","","Proteomics / Metaproteomics","Proteomics / Metaproteomics","Programming language / platform","Sequence analysis","Phylogenomics","Metagenomics","Genomics / Genome annotation","Sequence assembly","Phylogenomics","Variant Calling","Library","Phylogenomics","Data visualisation","Sequence analysis","Phylogenomics","Metabolomics","Genomics / Genome annotation","Genomics / Genome annotation","Sequence assembly","Sequence assembly","Library","Sequence analysis","Sequence analysis","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","Programming language / platform","","Metagenomics","Metagenomics","Metagenomics","Quality assurance","Sequence assembly","Programming language / platform","","Sequence assembly","Sequence assembly","Genomics / Genome annotation","Sequence assembly","Proteomics / Metaproteomics","Phylogenomics","Population genomics","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Genomics / Genome annotation","Phylogenomics","Transcriptomics","Quality assurance","Programming language / platform","Programming language / platform","Transcriptomics","Phylogenomics","Sequence analysis","Statistics and probability","Sequence assembly","Programming language / platform","","Sequence analysis","Proteomics / Metaproteomics","Sequence analysis","Sequence analysis","Sequence analysis","Phylogenomics","Sequence analysis","","Programming language / platform","Sequence analysis","Sequence analysis","Workflows","Genomics / Genome annotation","Phylogenomics","Genomics / Genome annotation","Variant Calling","Quality assurance","Sequence alignment","Sequence assembly","Sequence assembly","Metagenomics","Sequence assembly","Genomics / Genome annotation","Knowledge discovery in databases","Sequence assembly","Sequence alignment","Population genomics","Sequence assembly","Phylogenomics","Sequence analysis","Phylogenomics","","","Proteomics / Metaproteomics","Sequence assembly","Sequence alignment","","Transcriptomics","Genomics / Genome annotation","Sequence analysis","Sequence analysis","Sequence assembly","Transcriptomics","Transcriptomics","Sequence analysis","","","Sequence assembly","","Metagenomics","Quality assurance","Genetic variation","Sequence assembly","Transcriptomics","","Sequence analysis","Sequence assembly","Genomics / Genome annotation","","Sequence alignment","Metabolomics","Sequence assembly","","Metabolomics","","Data submission, annotation and curation","","Proteomics / Metaproteomics","",""],["","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abricate'  target='_blank'  rel='noopener noreferrer'>abricate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=abyss'  target='_blank'  rel='noopener noreferrer'>abyss<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=apollo'  target='_blank'  rel='noopener noreferrer'>apollo<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=augustus'  target='_blank'  rel='noopener noreferrer'>augustus<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=batchcorrection'  target='_blank'  rel='noopener noreferrer'>batchcorrection<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bcftools'  target='_blank'  rel='noopener noreferrer'>bcftools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beagle'  target='_blank'  rel='noopener noreferrer'>beagle<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=beast'  target='_blank'  rel='noopener noreferrer'>beast<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bedtools'  target='_blank'  rel='noopener noreferrer'>bedtools<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bismark'  target='_blank'  rel='noopener noreferrer'>bismark<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'  target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'  target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blast'  target='_blank'  rel='noopener noreferrer'>blast<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=blat'  target='_blank'  rel='noopener noreferrer'>blat<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=boost'  target='_blank'  rel='noopener noreferrer'>boost<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie_wrappers'  target='_blank'  rel='noopener noreferrer'>bowtie_wrappers<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bowtie2'  target='_blank'  rel='noopener noreferrer'>bowtie2<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=breakdancer_max'  target='_blank'  rel='noopener noreferrer'>breakdancer_max<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bsmap'  target='_blank'  rel='noopener noreferrer'>bsmap<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bs_seeker2'  target='_blank'  rel='noopener noreferrer'>bs_seeker2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=busco'  target='_blank'  rel='noopener noreferrer'>busco<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bwa'  target='_blank'  rel='noopener noreferrer'>bwa<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bx_python'  target='_blank'  rel='noopener noreferrer'>bx_python<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=bz2file'  target='_blank'  rel='noopener noreferrer'>bz2file<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=camera'  target='_blank'  rel='noopener noreferrer'>camera<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=canu'  target='_blank'  rel='noopener noreferrer'>canu<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cap3'  target='_blank'  rel='noopener noreferrer'>cap3<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cardinal'  target='_blank'  rel='noopener noreferrer'>cardinal<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cat_bins'  target='_blank'  rel='noopener noreferrer'>cat_bins<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cdhit'  target='_blank'  rel='noopener noreferrer'>cdhit<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=clustalw'  target='_blank'  rel='noopener noreferrer'>clustalw<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cmake'  target='_blank'  rel='noopener noreferrer'>cmake<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=coreutils'  target='_blank'  rel='noopener noreferrer'>coreutils<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cufflinks'  target='_blank'  rel='noopener noreferrer'>cufflinks<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=curl'  target='_blank'  rel='noopener noreferrer'>curl<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cutadapt'  target='_blank'  rel='noopener noreferrer'>cutadapt<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=cytoscape'  target='_blank'  rel='noopener noreferrer'>cytoscape<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=deeptools'  target='_blank'  rel='noopener noreferrer'>deeptools<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dendropy'  target='_blank'  rel='noopener noreferrer'>dendropy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=dia_umpire'  target='_blank'  rel='noopener noreferrer'>dia_umpire<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=diamond'  target='_blank'  rel='noopener noreferrer'>diamond<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=edena'  target='_blank'  rel='noopener noreferrer'>edena<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=emboss'  target='_blank'  rel='noopener noreferrer'>emboss<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ensembl'  target='_blank'  rel='noopener noreferrer'>ensembl<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=exonerate'  target='_blank'  rel='noopener noreferrer'>exonerate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=export2graphlan'  target='_blank'  rel='noopener noreferrer'>export2graphlan<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastqc'  target='_blank'  rel='noopener noreferrer'>fastqc<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx'  target='_blank'  rel='noopener noreferrer'>fastx<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fastx'  target='_blank'  rel='noopener noreferrer'>fastx<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flashlfq'  target='_blank'  rel='noopener noreferrer'>flashlfq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=flye'  target='_blank'  rel='noopener noreferrer'>flye<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=fraggenescan'  target='_blank'  rel='noopener noreferrer'>fraggenescan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=freebayes'  target='_blank'  rel='noopener noreferrer'>freebayes<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=garli'  target='_blank'  rel='noopener noreferrer'>garli<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gatk'  target='_blank'  rel='noopener noreferrer'>gatk<\/a>","","","","","","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer'  target='_blank'  rel='noopener noreferrer'>glimmer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=glimmer_hmm'  target='_blank'  rel='noopener noreferrer'>glimmer_hmm<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=libtool'  target='_blank'  rel='noopener noreferrer'>libtool<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gsl'  target='_blank'  rel='noopener noreferrer'>gsl<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=graphlan'  target='_blank'  rel='noopener noreferrer'>graphlan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gmx'  target='_blank'  rel='noopener noreferrer'>gmx<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=gubbins'  target='_blank'  rel='noopener noreferrer'>gubbins<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=guppy'  target='_blank'  rel='noopener noreferrer'>guppy<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hadoop'  target='_blank'  rel='noopener noreferrer'>hadoop<\/a>","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hisat2'  target='_blank'  rel='noopener noreferrer'>hisat2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=hmmer'  target='_blank'  rel='noopener noreferrer'>hmmer<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=htseq'  target='_blank'  rel='noopener noreferrer'>htseq<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=idba'  target='_blank'  rel='noopener noreferrer'>idba<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=infernal'  target='_blank'  rel='noopener noreferrer'>infernal<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=interproscan'  target='_blank'  rel='noopener noreferrer'>interproscan<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=iqtree'  target='_blank'  rel='noopener noreferrer'>iqtree<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jbrowse'  target='_blank'  rel='noopener noreferrer'>jbrowse<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=jellyfish'  target='_blank'  rel='noopener noreferrer'>jellyfish<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kallisto'  target='_blank'  rel='noopener noreferrer'>kallisto<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kat'  target='_blank'  rel='noopener noreferrer'>kat<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=khmer'  target='_blank'  rel='noopener noreferrer'>khmer<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken'  target='_blank'  rel='noopener noreferrer'>kraken<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kraken2'  target='_blank'  rel='noopener noreferrer'>kraken2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=kubernetes'  target='_blank'  rel='noopener noreferrer'>kubernetes<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lastz'  target='_blank'  rel='noopener noreferrer'>lastz<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=libgtextutils'  target='_blank'  rel='noopener noreferrer'>libgtextutils<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=lipidmaps'  target='_blank'  rel='noopener noreferrer'>lipidmaps<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs'  target='_blank'  rel='noopener noreferrer'>macs<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=macs2'  target='_blank'  rel='noopener noreferrer'>macs2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mafft'  target='_blank'  rel='noopener noreferrer'>mafft<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maker'  target='_blank'  rel='noopener noreferrer'>maker<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maldi_quant'  target='_blank'  rel='noopener noreferrer'>maldi_quant<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mauve'  target='_blank'  rel='noopener noreferrer'>mauve<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxbin2'  target='_blank'  rel='noopener noreferrer'>maxbin2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant'  target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=maxquant'  target='_blank'  rel='noopener noreferrer'>maxquant<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mcl'  target='_blank'  rel='noopener noreferrer'>mcl<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=medaka'  target='_blank'  rel='noopener noreferrer'>medaka<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=megahit'  target='_blank'  rel='noopener noreferrer'>megahit<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metams'  target='_blank'  rel='noopener noreferrer'>metams<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaphlan'  target='_blank'  rel='noopener noreferrer'>metaphlan<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=metaspades'  target='_blank'  rel='noopener noreferrer'>metaspades<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minia'  target='_blank'  rel='noopener noreferrer'>minia<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=minimap2'  target='_blank'  rel='noopener noreferrer'>minimap2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mira'  target='_blank'  rel='noopener noreferrer'>mira<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mirdeep2'  target='_blank'  rel='noopener noreferrer'>mirdeep2<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mixomics'  target='_blank'  rel='noopener noreferrer'>mixomics<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mlst'  target='_blank'  rel='noopener noreferrer'>mlst<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=proteomics_moff'  target='_blank'  rel='noopener noreferrer'>proteomics_moff<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=morpheus'  target='_blank'  rel='noopener noreferrer'>morpheus<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mothur'  target='_blank'  rel='noopener noreferrer'>mothur<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mrbayes'  target='_blank'  rel='noopener noreferrer'>mrbayes<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msgfplus'  target='_blank'  rel='noopener noreferrer'>msgfplus<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=msnbase'  target='_blank'  rel='noopener noreferrer'>msnbase<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=mummer'  target='_blank'  rel='noopener noreferrer'>mummer<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=muscle'  target='_blank'  rel='noopener noreferrer'>muscle<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=nanopolish'  target='_blank'  rel='noopener noreferrer'>nanopolish<\/a>","","","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=netcdf'  target='_blank'  rel='noopener noreferrer'>netcdf<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ngsplot'  target='_blank'  rel='noopener noreferrer'>ngsplot<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=oases'  target='_blank'  rel='noopener noreferrer'>oases<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=openms'  target='_blank'  rel='noopener noreferrer'>openms<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4mcorcov'  target='_blank'  rel='noopener noreferrer'>w4mcorcov<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=parsnp'  target='_blank'  rel='noopener noreferrer'>parsnp<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=peptideshaker'  target='_blank'  rel='noopener noreferrer'>peptideshaker<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=percolator'  target='_blank'  rel='noopener noreferrer'>percolator<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picard'  target='_blank'  rel='noopener noreferrer'>picard<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=picrust'  target='_blank'  rel='noopener noreferrer'>picrust<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pilon'  target='_blank'  rel='noopener noreferrer'>pilon<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=plink'  target='_blank'  rel='noopener noreferrer'>plink<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prinseq'  target='_blank'  rel='noopener noreferrer'>prinseq<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prodigal'  target='_blank'  rel='noopener noreferrer'>prodigal<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=prokka'  target='_blank'  rel='noopener noreferrer'>prokka<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=pysam'  target='_blank'  rel='noopener noreferrer'>pysam<\/a>","","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime'  target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime'  target='_blank'  rel='noopener noreferrer'>qiime<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=qiime2'  target='_blank'  rel='noopener noreferrer'>qiime2<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=quast'  target='_blank'  rel='noopener noreferrer'>quast<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=racon'  target='_blank'  rel='noopener noreferrer'>racon<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rawtools'  target='_blank'  rel='noopener noreferrer'>rawtools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=raxml'  target='_blank'  rel='noopener noreferrer'>raxml<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repeat_masker'  target='_blank'  rel='noopener noreferrer'>repeat_masker<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=repet'  target='_blank'  rel='noopener noreferrer'>repet<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rsem'  target='_blank'  rel='noopener noreferrer'>rsem<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=rseqc'  target='_blank'  rel='noopener noreferrer'>rseqc<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ruby'  target='_blank'  rel='noopener noreferrer'>ruby<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=salmon'  target='_blank'  rel='noopener noreferrer'>salmon<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samblaster'  target='_blank'  rel='noopener noreferrer'>samblaster<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=samtools'  target='_blank'  rel='noopener noreferrer'>samtools<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=scipy'  target='_blank'  rel='noopener noreferrer'>scipy<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=screed'  target='_blank'  rel='noopener noreferrer'>screed<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=searchgui'  target='_blank'  rel='noopener noreferrer'>searchgui<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=seqkit'  target='_blank'  rel='noopener noreferrer'>seqkit<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sickle'  target='_blank'  rel='noopener noreferrer'>sickle<\/a>","","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snap'  target='_blank'  rel='noopener noreferrer'>snap<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=snpeff'  target='_blank'  rel='noopener noreferrer'>snpeff<\/a>","","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=sortmerna'  target='_blank'  rel='noopener noreferrer'>sortmerna<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=spades'  target='_blank'  rel='noopener noreferrer'>spades<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=stacks'  target='_blank'  rel='noopener noreferrer'>stacks<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=star'  target='_blank'  rel='noopener noreferrer'>star<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=structure'  target='_blank'  rel='noopener noreferrer'>structure<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=svtyper'  target='_blank'  rel='noopener noreferrer'>svtyper<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tagdust'  target='_blank'  rel='noopener noreferrer'>tagdust<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tassel'  target='_blank'  rel='noopener noreferrer'>tassel<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=thermo_raw_file_converter'  target='_blank'  rel='noopener noreferrer'>thermo_raw_file_converter<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=tophat'  target='_blank'  rel='noopener noreferrer'>tophat<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=transdecoder'  target='_blank'  rel='noopener noreferrer'>transdecoder<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trim_galore'  target='_blank'  rel='noopener noreferrer'>trim_galore<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trimmomatic'  target='_blank'  rel='noopener noreferrer'>trimmomatic<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinity'  target='_blank'  rel='noopener noreferrer'>trinity<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trinotate'  target='_blank'  rel='noopener noreferrer'>trinotate<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=trnascan-se'  target='_blank'  rel='noopener noreferrer'>trnascan-se<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=ucsc'  target='_blank'  rel='noopener noreferrer'>ucsc<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=unicycler'  target='_blank'  rel='noopener noreferrer'>unicycler<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=valet'  target='_blank'  rel='noopener noreferrer'>valet<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vcftools'  target='_blank'  rel='noopener noreferrer'>vcftools<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=velvet'  target='_blank'  rel='noopener noreferrer'>velvet<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vienna'  target='_blank'  rel='noopener noreferrer'>vienna<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=vsearch'  target='_blank'  rel='noopener noreferrer'>vsearch<\/a>","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=windowmasker'  target='_blank'  rel='noopener noreferrer'>windowmasker<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=w4m'  target='_blank'  rel='noopener noreferrer'>w4m<\/a>","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xcms'  target='_blank'  rel='noopener noreferrer'>xcms<\/a>","","","","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=xtandem'  target='_blank'  rel='noopener noreferrer'>xtandem<\/a>","<a href='https://toolshed.g2.bx.psu.edu/repository/browse_repositories?f-free-text-search=zlib'  target='_blank'  rel='noopener noreferrer'>zlib<\/a>",""],["","<a href='https://bio.tools/ABRicate' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/ABySS' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/amos' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/anaconda' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/angsd' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/apollo' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/artemis' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/augustus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/bamm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bamtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/bbmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bcftools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/beagle' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/beast' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/beast2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bedtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/bioconductor' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/bioperl' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/Biopython' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bismark' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/blast' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/blat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/BlobToolKit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/BlobToolKit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/bowtie' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bowtie2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/braker1' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/breakdancer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/bsmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/busco' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bwa' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/bx-python' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/camera' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/canu' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/cap3' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/cardinal' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/cd-hit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/cabog' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/checkm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/checkm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/clustalw' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/cnvkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/cpc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/cpc2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","<a href='https://bio.tools/cufflinks' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/cutadapt' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/cytoscape' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/deeptools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/DIA-Umpire' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/diamond' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/discovar_de_novo' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/dssp' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/edena' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/emboss' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/ensembl' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/EvidenceModeler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/exonerate' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://bio.tools/fastml' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/fastqc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/faststructure' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/fastx-toolkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/fastx-toolkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/fgenesh' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/fimpute' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/Flye' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/fraggenescan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/freebayes' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/freesurfer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/gatk' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/gblocks' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/gcta' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/geneid' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/genemark' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/GenomeScope_2.0' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/glimmer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/glimmer-hmm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/gmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/gmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","<a href='https://bio.tools/graftm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/gromacs' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/hawkeye' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/hisat2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/hmmer3' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/htseq' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/htslib' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/idba' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://bio.tools/interproscan_4' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/iq-tree' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://bio.tools/jbrowse' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/jellyfish' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/kallisto' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/kat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/khmer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/kmc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/kraken' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/kraken2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/krona' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/lastz' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/LFQ-Analyst' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://bio.tools/lipid_maps' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/mach' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/macs' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/macs' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/MAFFT' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/maker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/maker-p' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/masurca' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/matlab' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/mauve' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/maxbin' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/MaxQuant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/MaxQuant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/mcl' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/megahit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/MetaBAT_2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/metams' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/metaphlan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/metaspades' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/MIGRATE' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/minia' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/minimap2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/mira' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/mirdeep' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/miso' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/mixomics' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/mlst' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/moff' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/morpheus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/Mothur' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/mrbayes' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/msgfplus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/msnbase' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/mummer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/muscle' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/nanopolish' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","","","","","","","","<a href='https://bio.tools/NextPolish' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/oases' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://bio.tools/openms' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","","","<a href='https://bio.tools/PASA' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/peptideshaker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/percolator' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/picard_tools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/pilon' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/plink' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/plink-seq' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/clstutils' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/prinseq' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/gmv' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/prokka' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/purge_dups' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/pyrad' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/pysam' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","<a href='https://bio.tools/qiime' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/qiime' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/quast' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/r' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/Racon' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/ratt' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/rawtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/raxml' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/repdenovo' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/repeatmasker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/repeatmodeler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/RepeatModeler2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/repeatrunner' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/repet' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/rsem' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/rseqc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/rstudio' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/salmon' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/samblaster' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/samtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","<a href='https://bio.tools/searchgui' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/shapeit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/sickle' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/skewer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/smrt_analysis' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/snakemake' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/snap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/SNAPPy' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/snoscan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/snpeff' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/soap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/soapsnp' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/sortmerna' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/spades' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/sra-tools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/stacks' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/star' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/structure' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/tagdust' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/tassel' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/TensorFlow' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/ThermoRawFileParser' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/tophat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/TransDecoder' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/trf' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/trim_galore' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/trimmomatic' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/trinity' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://bio.tools/trnascan-se' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/UCSC_Genome_Browser_Utilities' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/unicycler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/vcftools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/velvet' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/vsearch' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/WENGAN' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/wise' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://bio.tools/workflow4metabolomics' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://bio.tools/xcms' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://bio.tools/xtandem' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","",""],["","<a href='https://biocontainers.pro/#/tools/abricate' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/abyss' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/amos' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/angsd' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/ant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/apollo' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/artemis' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/augustus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/perl-config-autoconf' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/bamtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/batchcorrection' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/bcftools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beagle' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beagle-lib' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beast' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/beast2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bedtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","<a href='https://biocontainers.pro/#/tools/bismark' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/blast' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/ncbi-blast-plus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/blat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/blobtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bowtie' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bowtie2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/braker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/braker2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/breakdancer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bsmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bs-seeker2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/busco' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bwa' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bx-python' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bz2file' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cactus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-camera' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/canu' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cap3' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-cardinal' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/cd-hit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/wgs-assembler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/checkm-genome' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/checkm-genome' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/clustalw' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/cnvkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/coreutils' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cpat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/cufflinks' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/curl' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cutadapt' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/cytoscape' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/deeptools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/deepvariant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/dendropy' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/dia_umpire' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/diamond' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/discovardenovo' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/dssp' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/edena' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","<a href='https://biocontainers.pro/#/tools/evidencemodeler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/exonerate' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/export2graphlan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/pb-assembly' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pb-assembly' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastml' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastqc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/faststructure' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastx_toolkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/fastx_toolkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/flashlfq' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/flye' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/fraggenescan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/freebayes' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/funannotate' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/garli' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gatk4' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gblocks' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/gcta' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/tools/genomescope2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/genomicconsensus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/glimmer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/glimmerhmm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/gmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/parallel' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/graftm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/graphlan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gromacs' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/groopm' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/gubbins' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/haslr' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/hclust2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/hisat2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/hmmer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/htseq' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/htslib' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/idba' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/pbipa' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/igv' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/interproscan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/ipyrad' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/iqtree' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/isoseq3' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/jbrowse' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/jellyfish' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/julia' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kallisto' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/khmer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kmc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kraken' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/kraken2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/krona' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/k8' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/lastz' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/libgtextutils' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/tools/links' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/longranger' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/macs2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/macse' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mafft' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/maker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/r-maldiquant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mashmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/maskrc-svg' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/masurca' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/mauvealigner' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/maxbin2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/maxquant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/maxquant' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mcl' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/medaka' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/megahit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/metabat2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/metamaps' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-metams' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/metaphlan2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/migrate-n' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/minia' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/minimac4' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/minimap2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/mira' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/mirdeep2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/misopy' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/r-mixomics' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mlst' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/moff' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mono' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/morpheus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/mothur' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/mrbayes' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/msgf_plus' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/bioconductor-msnbase' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/mummer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/muscle' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/tools/nanopolish' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","","","","","","<a href='https://biocontainers.pro/#/tools/nextflow' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/nextgenmap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/nucleoatac' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/oases' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","<a href='https://biocontainers.pro/#/tools/openms' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","","<a href='https://biocontainers.pro/#/tools/paraview' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/parsnp' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pasa' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/peptideshaker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/percolator' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/picard' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/picrust' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/picrust2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/piler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pilon' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/plink' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/pplacer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/tools/prinseq' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/probabel' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/r-probmetab' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/prodigal' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/prokka' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/tools/purge_haplotigs' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/pyrad' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/pysam' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/python' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","","","<a href='https://biocontainers.pro/#/tools/qiime' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/qiime' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/quast' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/quickmerge' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/racon' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/ratt' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/rawtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/raxml' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/refinem' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repdenovo' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repeatmasker' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repeatmodeler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/repeatmodeler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/rsem' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/rseqc' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/ruby' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/salmon' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/samblaster' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/samtools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/scipy' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/screed' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/searchgui' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/seqkit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/seqstats' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/shapeit' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/sickle' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/singularity' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/skewer' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/snakemake' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/snap' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/snoscan' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/snpeff' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/soapdenovo2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/soapsnp' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/sortmerna' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/spades' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/sra-tools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/stacks' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/star' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/structure' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/svtyper' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/tassel' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","<a href='https://biocontainers.pro/#/tools/thermorawfileparser' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/tools/tigmint' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/tophat' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/transdecoder' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trf' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trim-galore' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trimmomatic' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trinity' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trinotate' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/trnascan-se' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/tools/unicycler' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/valet' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/vawk' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/vcftools' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/velvet' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/viennarna' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/vsearch' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/wise' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/isa2w4m' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","<a href='https://biocontainers.pro/#/tools/wtdbg2' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/bioconductor-xcms' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","","","<a href='https://biocontainers.pro/#/tools/xtandem' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>","","<a href='https://biocontainers.pro/#/tools/zstd' target='_blank'  rel='noopener noreferrer'>&#9679;<\/a>"]],"container":"<table class=\"cell-border stripe\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Tool / workflow name<\/th>\n      <th>Tool identifier (module name / bio.tools ID / placeholder)<\/th>\n      <th>Primary purpose (EDAM, if available)<\/th>\n      <th>Available in Galaxy toolshed<\/th>\n      <th>bio.tools<\/th>\n      <th>BioContainers<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
<p></font></p>
<p><strong>Note:</strong></p>
<ul>
<li>Table last updated <code>2021-03-03</code>.</li>
<li>Software documentation is linked from the <em>Tool / workflow name</em> in the first column.</li>
<li>The <em>Tool identifier</em> column contains an identifier for the tool / workflow: typically either the module name (used for matching to HPC lists), a bio.tools ID, or a placeholder value.</li>
<li>The <em>Primary purpose</em> column categorises the tools by purpose, using an <a href="https://github.com/edamontology/edamontology">EDAM</a> <em>topic</em> where possible.</li>
<li>More information about a tool can be found by following the link in the <a href="https://bio.tools/">bio.tools</a> column.</li>
<li>When a tool has been containerised to allow for easier installation on any compute infrastructure, a link to the containerised software that can be downloaded is shown in the <a href="https://biocontainers.pro/#/">BioContainers</a> column.</li>
<li>The list currently includes some tools that are not used for bioinformatics.</li>
<li>The source material for the table is currently manually curated, and while we endeavour to keep the information as current as possible, there is a natural limit to the volume of information maintained here. Production of this information will be automated during 2021, and tools that are not relevant for bioinformatics analyses removed.</li>
</ul>
<hr />
<hr />
</div>
</div>
<div id="how-were-the-tools-and-workflows-selected" class="section level1">
<h1>How were the tools and workflows selected?</h1>
<p>The tool list here includes those that have been identified through <a href="https://www.biocommons.org.au/get-involved">BioCommons Community consultations</a> and other engagements, as well as through matching to the tool sets installed across the various BioCommons partner compute infrastructures. The list is not intended to be exhaustive: i.e. it does not contain <em>ALL</em> bioinformatics tools.</p>
<hr />
<hr />
</div>
<div id="how-was-the-list-generated" class="section level1">
<h1>How was the list generated?</h1>
<p>RStudio (see session info at the bottom of this page) was used for processing.</p>
<p>The broad steps are described here:</p>
<p><font size="-1.5"></p>
<ol style="list-style-type: decimal">
<li>The manually curated tool list and separate tool lists provided by the BioCommons partner infrastructures were parsed (Galaxy Australia, NCI, Pawsey and QRISCloud - RCC).</li>
<li>The complete set of tool lists were joined using the <code>toolID</code> as a key, and</li>
<li>Links were embedded, where available, to the original documentation URLs (homepage or GitHub repository, for example) as well as the registry entries on bio.tools and BioContainers.</li>
<li>Galaxy wrapping often entails that tool suites are separated into their individual tool components. As such, if a Galaxy tool was matched to the manually curated tool list, a tool shed search link was embedded in the <code>Available in Galaxy toolshed</code> column. This allows a user to search for the tool using a relevant search term (e.g. vcftools) to identify all the component tools available in the toolshed.</li>
</ol>
<p></font></p>
<hr />
<hr />
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre><code>## R version 4.0.3 (2020-10-10)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## Running under: Windows 10 x64 (build 19041)
## 
## Matrix products: default
## 
## locale:
## [1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
## [3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
## [5] LC_TIME=English_Australia.1252    
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] DT_0.17           lubridate_1.7.9.2 yaml_2.2.1        knitr_1.31       
##  [5] kableExtra_1.3.1  forcats_0.5.1     stringr_1.4.0     dplyr_1.0.3      
##  [9] purrr_0.3.4       readr_1.4.0       tidyr_1.1.2       tibble_3.0.5     
## [13] ggplot2_3.3.3     tidyverse_1.3.0  
## 
## loaded via a namespace (and not attached):
##  [1] tidyselect_1.1.0  xfun_0.20         haven_2.3.1       colorspace_2.0-0 
##  [5] vctrs_0.3.6       generics_0.1.0    htmltools_0.5.1.1 viridisLite_0.3.0
##  [9] rlang_0.4.10      pillar_1.4.7      glue_1.4.2        withr_2.4.1      
## [13] DBI_1.1.1         dbplyr_2.0.0      modelr_0.1.8      readxl_1.3.1     
## [17] lifecycle_0.2.0   munsell_0.5.0     gtable_0.3.0      cellranger_1.1.0 
## [21] rvest_0.3.6       htmlwidgets_1.5.3 evaluate_0.14     crosstalk_1.1.1  
## [25] fansi_0.4.2       broom_0.7.3       Rcpp_1.0.6        scales_1.1.1     
## [29] backports_1.2.0   webshot_0.5.2     jsonlite_1.7.2    fs_1.5.0         
## [33] hms_1.0.0         digest_0.6.27     stringi_1.5.3     grid_4.0.3       
## [37] cli_2.2.0         tools_4.0.3       magrittr_2.0.1    crayon_1.3.4     
## [41] pkgconfig_2.0.3   ellipsis_0.3.1    xml2_1.3.2        reprex_1.0.0     
## [45] assertthat_0.2.1  rmarkdown_2.6     httr_1.4.2        rstudioapi_0.13  
## [49] R6_2.5.0          compiler_4.0.3</code></pre>
<hr />
<hr />
</div>
</div>


<!DOCTYPE html>

<div class="footer">This site uses Cookies. By proceeding, you are agreeing to our <a href="https://www.biocommons.org.au/cookies">Cookies Policy</a>. We use cookies to help the website run effectively.</div>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
